<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Milton Pividori" />
  <meta name="author" content="Sumei Lu" />
  <meta name="author" content="Binglan Li" />
  <meta name="author" content="Chun Su" />
  <meta name="author" content="Matthew E. Johnson" />
  <meta name="author" content="Wei-Qi Wei" />
  <meta name="author" content="Qiping Feng" />
  <meta name="author" content="Bahram Namjou" />
  <meta name="author" content="Krzysztof Kiryluk" />
  <meta name="author" content="Iftikhar Kullo" />
  <meta name="author" content="Yuan Luo" />
  <meta name="author" content="Blair D. Sullivan" />
  <meta name="author" content="Benjamin F. Voight" />
  <meta name="author" content="Carsten Skarke" />
  <meta name="author" content="Marylyn D. Ritchie" />
  <meta name="author" content="Struan F.A. Grant" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="dcterms.date" content="2023-01-01" />
  <meta name="keywords" content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits" />
  <title>Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms" />
  <meta name="citation_title" content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms" />
  <meta property="og:title" content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms" />
  <meta property="twitter:title" content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms" />
  <meta name="dc.date" content="2023-01-01" />
  <meta name="citation_publication_date" content="2023-01-01" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Milton Pividori" />
  <meta name="citation_author_institution" content="Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0002-3035-4403" />
  <meta name="twitter:creator" content="@miltondp" />
  <meta name="citation_author" content="Sumei Lu" />
  <meta name="citation_author_institution" content="Center for Spatial and Functional Genomics, Children&#39;s Hospital of Philadelphia, Philadelphia, PA, 19104, USA" />
  <meta name="citation_author" content="Binglan Li" />
  <meta name="citation_author_institution" content="Department of Biomedical Data Science, Stanford University, Stanford, CA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-0103-6107" />
  <meta name="citation_author" content="Chun Su" />
  <meta name="citation_author_institution" content="Center for Spatial and Functional Genomics, Children&#39;s Hospital of Philadelphia, Philadelphia, PA, 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0001-6388-8666" />
  <meta name="citation_author" content="Matthew E. Johnson" />
  <meta name="citation_author_institution" content="Center for Spatial and Functional Genomics, Children&#39;s Hospital of Philadelphia, Philadelphia, PA, 19104, USA" />
  <meta name="citation_author" content="Wei-Qi Wei" />
  <meta name="citation_author_institution" content="Vanderbilt University Medical Center" />
  <meta name="citation_author" content="Qiping Feng" />
  <meta name="citation_author_institution" content="Vanderbilt University Medical Center" />
  <meta name="citation_author_orcid" content="0000-0002-6213-793X" />
  <meta name="citation_author" content="Bahram Namjou" />
  <meta name="citation_author_institution" content="Cincinnati Children&#39;s Hospital Medical Center" />
  <meta name="citation_author" content="Krzysztof Kiryluk" />
  <meta name="citation_author_institution" content="Department of Medicine, Division of Nephrology, Vagelos College of Physicians &amp; Surgeons, Columbia University, New York, New York" />
  <meta name="citation_author_orcid" content="0000-0002-5047-6715" />
  <meta name="twitter:creator" content="@kirylukk" />
  <meta name="citation_author" content="Iftikhar Kullo" />
  <meta name="citation_author_institution" content="Mayo Clinic" />
  <meta name="citation_author" content="Yuan Luo" />
  <meta name="citation_author_institution" content="Northwestern University" />
  <meta name="citation_author" content="Blair D. Sullivan" />
  <meta name="citation_author_institution" content="School of Computing, University of Utah, Salt Lake City, UT, USA" />
  <meta name="citation_author" content="Benjamin F. Voight" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_institution" content="Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0002-6205-9994" />
  <meta name="twitter:creator" content="@bvoight28" />
  <meta name="citation_author" content="Carsten Skarke" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Department of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0001-5145-3681" />
  <meta name="twitter:creator" content="@CarstenSkarke" />
  <meta name="citation_author" content="Marylyn D. Ritchie" />
  <meta name="citation_author_institution" content="Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0002-1208-1720" />
  <meta name="twitter:creator" content="@MarylynRitchie" />
  <meta name="citation_author" content="Struan F.A. Grant" />
  <meta name="citation_author_institution" content="Center for Spatial and Functional Genomics, Children&#39;s Hospital of Philadelphia, Philadelphia, PA, 19104, USA" />
  <meta name="citation_author_institution" content="Division of Endocrinology and Diabetes, Children&#39;s Hospital of Philadelphia, Philadelphia, PA, 19104, USA" />
  <meta name="citation_author_institution" content="Division of Human Genetics, Children&#39;s Hospital of Philadelphia, Philadelphia, PA, 19104, USA" />
  <meta name="citation_author_institution" content="Department of Pediatrics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, 19104, USA" />
  <meta name="citation_author_institution" content="Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0003-2025-5302" />
  <meta name="twitter:creator" content="@STRUANGRANT" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, CO 80045, USA" />
  <meta name="citation_author_institution" content="Department of Biomedical Informatics, University of Colorado School of Medicine, Aurora, CO 80045, USA" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@GreeneScientist" />
  <link rel="canonical" href="https://greenelab.github.io/phenoplier_manuscript/" />
  <meta property="og:url" content="https://greenelab.github.io/phenoplier_manuscript/" />
  <meta property="twitter:url" content="https://greenelab.github.io/phenoplier_manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/phenoplier_manuscript/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/phenoplier_manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/phenoplier_manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/phenoplier_manuscript/v/e37277b585e05f4f1c06519c374488c6311d1ae5/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/phenoplier_manuscript/v/e37277b585e05f4f1c06519c374488c6311d1ae5/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/phenoplier_manuscript/v/e37277b585e05f4f1c06519c374488c6311d1ae5/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  
  <!-- pandoc-eqnos: equation style -->
  <style>
    .eqnos { display: inline-block; position: relative; width: 100%; }
    .eqnos br { display: none; }
    .eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</h1>
</header>
<p><em>A DOI-citable version of this manuscript is available at <a href="https://doi.org/10.1101/2021.07.05.450786" class="uri">https://doi.org/10.1101/2021.07.05.450786</a></em>.</p>
<!-- 
<small><em>
This manuscript
([permalink](https://greenelab.github.io/phenoplier_manuscript/v/e37277b585e05f4f1c06519c374488c6311d1ae5/))
was automatically generated
from [greenelab/phenoplier_manuscript@e37277b](https://github.com/greenelab/phenoplier_manuscript/tree/e37277b585e05f4f1c06519c374488c6311d1ae5)
on January 1, 2023.
</em></small> -->
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Milton Pividori</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3035-4403">0000-0002-3035-4403</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/miltondp">miltondp</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/miltondp">miltondp</a>
<br>
<small>
Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
· Funded by The Gordon and Betty Moore Foundation GBMF 4552; The National Human Genome Research Institute (R01 HG010067); The National Human Genome Research Institute (K99HG011898)
</small></p></li>
<li><p><strong>Sumei Lu</strong>
<br>
<small>
Center for Spatial and Functional Genomics, Children’s Hospital of Philadelphia, Philadelphia, PA, 19104, USA
</small></p></li>
<li><p><strong>Binglan Li</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0103-6107">0000-0002-0103-6107</a>
<br>
<small>
Department of Biomedical Data Science, Stanford University, Stanford, CA, USA
</small></p></li>
<li><p><strong>Chun Su</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6388-8666">0000-0001-6388-8666</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/sckinta">sckinta</a>
<br>
<small>
Center for Spatial and Functional Genomics, Children’s Hospital of Philadelphia, Philadelphia, PA, 19104, USA
</small></p></li>
<li><p><strong>Matthew E. Johnson</strong>
<br>
<small>
Center for Spatial and Functional Genomics, Children’s Hospital of Philadelphia, Philadelphia, PA, 19104, USA
</small></p></li>
<li><p><strong>Wei-Qi Wei</strong>
<br>
<small>
Vanderbilt University Medical Center
</small></p></li>
<li><p><strong>Qiping Feng</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-6213-793X">0000-0002-6213-793X</a>
<br>
<small>
Vanderbilt University Medical Center
</small></p></li>
<li><p><strong>Bahram Namjou</strong>
<br>
<small>
Cincinnati Children’s Hospital Medical Center
</small></p></li>
<li><p><strong>Krzysztof Kiryluk</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5047-6715">0000-0002-5047-6715</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/kirylukk">kirylukk</a>
<br>
<small>
Department of Medicine, Division of Nephrology, Vagelos College of Physicians &amp; Surgeons, Columbia University, New York, New York
</small></p></li>
<li><p><strong>Iftikhar Kullo</strong>
<br>
<small>
Mayo Clinic
</small></p></li>
<li><p><strong>Yuan Luo</strong>
<br>
<small>
Northwestern University
</small></p></li>
<li><p><strong>Blair D. Sullivan</strong>
<br>
<small>
School of Computing, University of Utah, Salt Lake City, UT, USA
</small></p></li>
<li><p><strong>Benjamin F. Voight</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-6205-9994">0000-0002-6205-9994</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/bvoight">bvoight</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/bvoight28">bvoight28</a>
<br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA; Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA; Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Carsten Skarke</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5145-3681">0000-0001-5145-3681</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/CarstenSkarke">CarstenSkarke</a>
<br>
<small>
Institute for Translational Medicine and Therapeutics, Department of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Marylyn D. Ritchie</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1208-1720">0000-0002-1208-1720</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/MarylynRitchie">MarylynRitchie</a>
<br>
<small>
Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Struan F.A. Grant</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2025-5302">0000-0003-2025-5302</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/STRUANGRANT">STRUANGRANT</a>
<br>
<small>
Center for Spatial and Functional Genomics, Children’s Hospital of Philadelphia, Philadelphia, PA, 19104, USA; Division of Endocrinology and Diabetes, Children’s Hospital of Philadelphia, Philadelphia, PA, 19104, USA; Division of Human Genetics, Children’s Hospital of Philadelphia, Philadelphia, PA, 19104, USA; Department of Pediatrics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, 19104, USA; Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, 19104, USA
</small></p></li>
<li><p><strong>Casey S. Greene</strong>
<sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/GreeneScientist">GreeneScientist</a>
<br>
<small>
Center for Health AI, University of Colorado School of Medicine, Aurora, CO 80045, USA; Department of Biomedical Informatics, University of Colorado School of Medicine, Aurora, CO 80045, USA
· Funded by The Gordon and Betty Moore Foundation (GBMF 4552); The National Human Genome Research Institute (R01 HG010067); The National Cancer Institute (R01 CA237170)
</small></p></li>
</ul>
<div id="correspondence">
<p>✉ — Correspondence possible via <a href="https://github.com/greenelab/phenoplier_manuscript/issues">GitHub Issues</a>
or email to
Casey S. Greene &lt;casey.s.greene@cuanschutz.edu&gt;.</p>
</div>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Genes act in concert with each other in specific contexts to perform their functions.
Determining how these genes influence complex traits requires a mechanistic understanding of expression regulation across different conditions.
It has been shown that this insight is critical for developing new therapies.
In this regard, the role of individual genes in disease-relevant mechanisms can be hypothesized with transcriptome-wide association studies (TWAS), which have represented a significant step forward in testing the mediating role of gene expression in GWAS associations.
However, modern models of the architecture of complex traits predict that gene-gene interactions play a crucial role in disease origin and progression.
Here we introduce PhenoPLIER, a computational approach that maps gene-trait associations and pharmacological perturbation data into a common latent representation for a joint analysis.
This representation is based on modules of genes with similar expression patterns across the same conditions.
We observed that diseases were significantly associated with gene modules expressed in relevant cell types, and our approach was accurate in predicting known drug-disease pairs and inferring mechanisms of action.
Furthermore, using a CRISPR screen to analyze lipid regulation, we found that functionally important players lacked TWAS associations but were prioritized in trait-associated modules by PhenoPLIER.
By incorporating groups of co-expressed genes, PhenoPLIER can contextualize genetic associations and reveal potential targets missed by single-gene strategies.</p>
<h2 id="introduction">Introduction</h2>
<p>Genes work together in context-specific networks to carry out different functions <span class="citation" data-cites="e0tRKjE5 CVF61Un5">[<a href="#ref-e0tRKjE5" role="doc-biblioref">1</a>,<a href="#ref-CVF61Un5" role="doc-biblioref">2</a>]</span>.
Variations in these genes can change their functional role and, at a higher level, affect disease-relevant biological processes <span class="citation" data-cites="Om8ZhS06">[<a href="#ref-Om8ZhS06" role="doc-biblioref">3</a>]</span>.
In this context, determining how genes influence complex traits requires mechanistically understanding expression regulation across different cell types <span class="citation" data-cites="9Pr9idng 1DoyZS7y0 nhaocxmR">[<a href="#ref-9Pr9idng" role="doc-biblioref">4</a>,<a href="#ref-1DoyZS7y0" role="doc-biblioref">5</a>,<a href="#ref-nhaocxmR" role="doc-biblioref">6</a>]</span>, which in turn should lead to improved treatments <span class="citation" data-cites="REXpV7nA PgEwSS4Q">[<a href="#ref-REXpV7nA" role="doc-biblioref">7</a>,<a href="#ref-PgEwSS4Q" role="doc-biblioref">8</a>]</span>.
Previous studies have described different regulatory DNA elements <span class="citation" data-cites="15J98V2qM sLkFMFZj SxuuTQTQ xRGqPsT2 1DoyZS7y0">[<a href="#ref-1DoyZS7y0" role="doc-biblioref">5</a>,<a href="#ref-15J98V2qM" role="doc-biblioref">9</a>,<a href="#ref-sLkFMFZj" role="doc-biblioref">10</a>,<a href="#ref-SxuuTQTQ" role="doc-biblioref">11</a>,<a href="#ref-xRGqPsT2" role="doc-biblioref">12</a>]</span> including genetic effects on gene expression across different tissues <span class="citation" data-cites="9Pr9idng">[<a href="#ref-9Pr9idng" role="doc-biblioref">4</a>]</span>.
Integrating functional genomics data and GWAS data <span class="citation" data-cites="WFslDIWl aIyQY5ZT WFslDIWl z8MQTAnJ">[<a href="#ref-WFslDIWl" role="doc-biblioref">13</a>,<a href="#ref-WFslDIWl" role="doc-biblioref">13</a>,<a href="#ref-aIyQY5ZT" role="doc-biblioref">14</a>,<a href="#ref-z8MQTAnJ" role="doc-biblioref">15</a>]</span> has improved the identification of these transcriptional mechanisms that, when dysregulated, commonly result in tissue- and cell lineage-specific pathology <span class="citation" data-cites="O0e3EhY6 18jYvPauB wNE0EQlN">[<a href="#ref-O0e3EhY6" role="doc-biblioref">16</a>,<a href="#ref-18jYvPauB" role="doc-biblioref">17</a>,<a href="#ref-wNE0EQlN" role="doc-biblioref">18</a>]</span>.</p>
<p>Given the availability of gene expression data across several tissues <span class="citation" data-cites="6SPTvFXq Nz3IMEzd 9Pr9idng 57TjOMEA">[<a href="#ref-9Pr9idng" role="doc-biblioref">4</a>,<a href="#ref-6SPTvFXq" role="doc-biblioref">19</a>,<a href="#ref-Nz3IMEzd" role="doc-biblioref">20</a>,<a href="#ref-57TjOMEA" role="doc-biblioref">21</a>]</span>, an effective approach to identify these biological processes is the transcription-wide association study (TWAS), which integrates expression quantitative trait loci (eQTLs) data to provide a mechanistic interpretation for GWAS findings.
TWAS relies on testing whether perturbations in gene regulatory mechanisms mediate the association between genetic variants and human diseases <span class="citation" data-cites="yEdRP9Xx 1D63fEEPb 1FFzCXo1s Z8bvDdVq">[<a href="#ref-yEdRP9Xx" role="doc-biblioref">22</a>,<a href="#ref-1D63fEEPb" role="doc-biblioref">23</a>,<a href="#ref-1FFzCXo1s" role="doc-biblioref">24</a>,<a href="#ref-Z8bvDdVq" role="doc-biblioref">25</a>]</span>, and these approaches have been highly successful not only in understanding disease etiology at the transcriptome level <span class="citation" data-cites="ktVcsmYD gZAOkumx zwpq2IXD">[<a href="#ref-ktVcsmYD" role="doc-biblioref">26</a>,<a href="#ref-gZAOkumx" role="doc-biblioref">27</a>,<a href="#ref-zwpq2IXD" role="doc-biblioref">28</a>]</span> but also in disease-risk prediction (polygenic scores) <span class="citation" data-cites="hSYqogYZ">[<a href="#ref-hSYqogYZ" role="doc-biblioref">29</a>]</span> and drug repurposing <span class="citation" data-cites="17oeJ0CXy">[<a href="#ref-17oeJ0CXy" role="doc-biblioref">30</a>]</span> tasks.
However, TWAS works at the individual gene level, which does not capture more complex interactions at the network level.</p>
<p>These gene-gene interactions play a crucial role in current theories of the architecture of complex traits, such as the omnigenic model <span class="citation" data-cites="vpIDZCSa">[<a href="#ref-vpIDZCSa" role="doc-biblioref">31</a>]</span>, which suggests that methods need to incorporate this complexity to disentangle disease-relevant mechanisms.
Widespread gene pleiotropy, for instance, reveals the highly interconnected nature of transcriptional networks <span class="citation" data-cites="pZZn28he PDWEwciL">[<a href="#ref-pZZn28he" role="doc-biblioref">32</a>,<a href="#ref-PDWEwciL" role="doc-biblioref">33</a>]</span>, where potentially all genes expressed in disease-relevant cell types have a non-zero effect on the trait <span class="citation" data-cites="vpIDZCSa LXvTZzEA">[<a href="#ref-vpIDZCSa" role="doc-biblioref">31</a>,<a href="#ref-LXvTZzEA" role="doc-biblioref">34</a>]</span>.
One way to learn these gene-gene interactions is using the concept of gene module: a group of genes with similar expression profiles across different conditions <span class="citation" data-cites="jrAMOJCD SjGoBywE CVF61Un5">[<a href="#ref-CVF61Un5" role="doc-biblioref">2</a>,<a href="#ref-jrAMOJCD" role="doc-biblioref">35</a>,<a href="#ref-SjGoBywE" role="doc-biblioref">36</a>]</span>.
In this context, several unsupervised approaches have been proposed to infer these gene-gene connections by extracting gene modules from co-expression patterns <span class="citation" data-cites="S4e4WaP3 f2r8LLIn VzZoy0BD">[<a href="#ref-S4e4WaP3" role="doc-biblioref">37</a>,<a href="#ref-f2r8LLIn" role="doc-biblioref">38</a>,<a href="#ref-VzZoy0BD" role="doc-biblioref">39</a>]</span>.
Matrix factorization techniques like independent or principal component analysis (ICA/PCA) have shown superior performance in this task <span class="citation" data-cites="1BVbSrr6M">[<a href="#ref-1BVbSrr6M" role="doc-biblioref">40</a>]</span> since they capture local expression effects from a subset of samples and can handle modules overlap effectively.
Therefore, integrating genetic studies with gene modules extracted using unsupervised learning could further improve our understanding of disease origin <span class="citation" data-cites="SjGoBywE">[<a href="#ref-SjGoBywE" role="doc-biblioref">36</a>]</span> and progression <span class="citation" data-cites="S56q1qoc">[<a href="#ref-S56q1qoc" role="doc-biblioref">41</a>]</span>.</p>
<p>Here we propose PhenoPLIER, an omnigenic approach that provides a gene module perspective to genetic studies.
The flexibility of our method allows integrating different data modalities into the same representation for a joint analysis.
In this work, we show that this module perspective can infer how groups of functionally-related genes influence complex traits, detect shared and distinct transcriptomic properties among traits, and predict how pharmacological perturbations affect genes’ activity to exert their effects.
PhenoPLIER maps gene-trait associations and drug-induced transcriptional responses into a common latent representation.
For this, we integrated thousands of gene-trait associations (using TWAS from PhenomeXcan <span class="citation" data-cites="lY5ln3dB">[<a href="#ref-lY5ln3dB" role="doc-biblioref">42</a>]</span>) and transcriptional profiles of drugs (from LINCS L1000 <span class="citation" data-cites="F7lIlh2N">[<a href="#ref-F7lIlh2N" role="doc-biblioref">43</a>]</span>) into a low-dimensional space learned from public gene expression data on tens of thousands of RNA-seq samples (recount2 <span class="citation" data-cites="14rnBunuZ 6SPTvFXq">[<a href="#ref-6SPTvFXq" role="doc-biblioref">19</a>,<a href="#ref-14rnBunuZ" role="doc-biblioref">44</a>]</span>).
We used a latent representation defined by a matrix factorization approach <span class="citation" data-cites="Ki2ij7zE 14rnBunuZ">[<a href="#ref-14rnBunuZ" role="doc-biblioref">44</a>,<a href="#ref-Ki2ij7zE" role="doc-biblioref">45</a>]</span> that extracts gene modules with certain sparsity constraints and preferences for those that align with prior knowledge (pathways).
When mapping gene-trait associations to this reduced expression space, we observed that diseases were significantly associated with gene modules expressed in relevant cell types: such as hypothyroidism with T cells, corneal endothelial cells with keratometry measurements, hematological assays on specific blood cell types, plasma lipids with adipose tissue, and neuropsychiatric disorders with different brain cell types.
Moreover, since PhenoPLIER can use models derived from large and heterogeneous RNA-seq datasets, we could also identify modules associated with cell types under specific stimuli or disease states.
We observed that significant module-trait associations in PhenomeXcan (our discovery cohort) replicated in the Electronic Medical Records and Genomics (eMERGE) network phase III <span class="citation" data-cites="wfqjCerX gZAOkumx">[<a href="#ref-gZAOkumx" role="doc-biblioref">27</a>,<a href="#ref-wfqjCerX" role="doc-biblioref">46</a>]</span> (our replication cohort).
Furthermore, we performed a CRISPR screen to analyze lipid regulation in HepG2 cells.
We observed more robust trait associations with modules than with individual genes, even when single genes known to be involved in lipid metabolism did not reach genome-wide significance.
Compared to a single-gene approach, our module-based method also better predicted FDA-approved drug-disease links by capturing tissue-specific pathophysiological mechanisms linked with the mechanism of action of drugs (e.g., niacin with cardiovascular traits via a known immune mechanism).
This improved drug-disease prediction suggested that modules may provide a better means to examine drug-disease relationships than individual genes.
Finally, exploring the phenotype-module space revealed stable trait clusters associated with relevant tissues, including a complex branch involving lipids with cardiovascular, autoimmune, and neuropsychiatric disorders.
In summary, instead of considering single genes associated with different complex traits, PhenoPLIER incorporates groups of genes that act together to carry out different functions in specific cell types.
This approach improves robustness in detecting and interpreting genetic associations, and here we show how it can prioritize alternative and potentially more promising candidate targets even when known single gene associations are not detected.
The approach represents a conceptual shift in the interpretation of genetic studies.
It has the potential to extract mechanistic insight from statistical associations to enhance the understanding of complex diseases and their therapeutic modalities.</p>
<h2 id="results">Results</h2>
<!--

Some papers that might be interesting:

https://www.nature.com/articles/s41591-020-01221-5
Air pollution linked to neurodegeneration markers

-->
<h3 id="phenoplier-an-integration-framework-based-on-gene-co-expression-patterns">PhenoPLIER: an integration framework based on gene co-expression patterns</h3>
<div id="fig:entire_process" class="fignos">
<figure>
<img src="images/entire_process/entire_process.svg" style="width:100.0%" alt="Figure 1: Schematic of the PhenoPLIER framework. a) High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (genetic variants). PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations. b) The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top). This process generates matrices \mathbf{\hat{L}} and \mathbf{\hat{M}}, where LVs now describe each drug/trait. In the middle, we show the three main computational components provided by PhenoPLIER to perform this integration: 1) an LV-based regression model, 2) a clustering framework to learn groups of traits, and 3) an LV-based drug repurposing approach. c) LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603 after projecting gene-trait associations in PhenomeXcan). Genes that are part of LV603 were expressed in relevant cell types (top). PBMC: peripheral blood mononuclear cells; mDCs: myeloid dendritic cells." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Schematic of the PhenoPLIER framework.</strong>
<strong>a)</strong> High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (genetic variants).
PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations.
<strong>b)</strong> The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top).
This process generates matrices <span class="math inline">\(\mathbf{\hat{L}}\)</span> and <span class="math inline">\(\mathbf{\hat{M}}\)</span>, where LVs now describe each drug/trait.
In the middle, we show the three main computational components provided by PhenoPLIER to perform this integration:
1) an LV-based regression model,
2) a clustering framework to learn groups of traits,
and 3) an LV-based drug repurposing approach.
<strong>c)</strong> LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603 after projecting gene-trait associations in PhenomeXcan).
Genes that are part of LV603 were expressed in relevant cell types (top).
PBMC: peripheral blood mononuclear cells;
mDCs: myeloid dendritic cells.</figcaption>
</figure>
</div>
<p>PhenoPLIER is a flexible computational framework that combines gene-trait and gene-drug associations with gene modules expressed in specific contexts (Figure <a href="#fig:entire_process">1</a>a).
The approach uses a latent representation (with latent variables or LVs representing gene modules) derived from a large gene expression compendium (Figure <a href="#fig:entire_process">1</a>b, top) to integrate TWAS with drug-induced transcriptional responses (Figure <a href="#fig:entire_process">1</a>b, bottom) for a joint analysis.
The approach consists in three main components (Figure <a href="#fig:entire_process">1</a>b, middle, see <a href="#sec:methods">Methods</a>):
1) an LV-based regression model to compute an association between an LV and a trait,
2) a clustering framework to learn groups of traits with shared transcriptomic properties,
and 3) an LV-based drug repurposing approach that links diseases to potential treatments.
We performed extensive simulations for our regression model (<a href="#sm:reg:null_sim">Supplementary Note 1</a>) and clustering framework (<a href="#sm:clustering:null_sim">Supplementary Note 2</a>) to ensure proper calibration and expected results under a model of no association.</p>
<p>We used TWAS results from PhenomeXcan <span class="citation" data-cites="lY5ln3dB">[<a href="#ref-lY5ln3dB" role="doc-biblioref">42</a>]</span> and the eMERGE network <span class="citation" data-cites="gZAOkumx">[<a href="#ref-gZAOkumx" role="doc-biblioref">27</a>]</span> as discovery and replication cohorts, respectively (<a href="#sec:methods:twas">Methods</a>).
PhenomeXcan provides gene-trait associations for 4,091 different diseases and traits from the UK Biobank <span class="citation" data-cites="nmJxPpE5">[<a href="#ref-nmJxPpE5" role="doc-biblioref">47</a>]</span> and other studies, whereas the analyses on eMERGE were performed across 309 phecodes.
TWAS results were derived using two statistical methods (see <a href="#sec:methods:predixcan">Methods</a>):
1) Summary-MultiXcan (S-MultiXcan) associations were used for the regression and clustering components,
and 2) Summary-PrediXcan (S-PrediXcan) associations were used for the drug repurposing component.
In addition, we also used colocalization results, which provide a probability of overlap between the GWAS and eQTL signals.
For the drug-repurposing approach, we used transcriptional responses to small molecule perturbations from LINCS L1000 <span class="citation" data-cites="F7lIlh2N">[<a href="#ref-F7lIlh2N" role="doc-biblioref">43</a>]</span> comprising 1,170 compounds.</p>
<p>The latent gene expression representation was obtained from the MultiPLIER models <span class="citation" data-cites="14rnBunuZ">[<a href="#ref-14rnBunuZ" role="doc-biblioref">44</a>]</span>, which were derived by applying a matrix factorization method (the pathway-level information extractor or PLIER <span class="citation" data-cites="Ki2ij7zE">[<a href="#ref-Ki2ij7zE" role="doc-biblioref">45</a>]</span>) to recount2 <span class="citation" data-cites="6SPTvFXq">[<a href="#ref-6SPTvFXq" role="doc-biblioref">19</a>]</span> – a uniformly-curated collection of transcript-level gene expression quantified by RNA-seq in a large, diverse set of samples collected across a range of disease states, cell types differentiation stages, and various stimuli (see <a href="#sec:methods:multiplier">Methods</a>).
The MultiPLIER models extracted 987 LVs by optimizing data reconstruction but also the alignment of LVs with prior knowledge/pathways.</p>
<p>Each LV or gene module represents a group of weighted genes expressed together in the same tissues and cell types as a functional unit.
Since LVs might represent a functional set of genes regulated by the same transcriptional program <span class="citation" data-cites="1FVd2WW6G DN7TyZzb">[<a href="#ref-1FVd2WW6G" role="doc-biblioref">48</a>,<a href="#ref-DN7TyZzb" role="doc-biblioref">49</a>]</span>, we conjecture that the projection of TWAS and pharmacologic perturbations data into this latent space could provide a better mechanistic understanding.
For this projection of different data modalities into the same space, PhenoPLIER converts gene associations to an LV score: all genes’ standardized effect sizes for a trait (from TWAS) or differential expression values for a drug (from pharmacologic perturbation data) are multiplied by the LV genes’ weights and summed, producing a single value.
Instead of looking at individual genes, this process links different traits and drugs to functionally-related groups of genes or LVs.
PhenoPLIER uses LVs annotations about the specific conditions where the group of genes is expressed, such as cell types and tissues, even at specific developmental stages, disease stages or under distinct stimuli.
Although this is not strictly necessary for PhenoPLIER to work, these annotations can dramatically improve the interpretability of results.
MultiPLIER’s models provide this information by linking LVs to samples, which may be annotated for experimental conditions (represented by matrix <span class="math inline">\(\mathbf{B}\)</span> at the top of Figure <a href="#fig:entire_process">1</a>b) in which genes in an LV are expressed.
An example of this is shown in Figure <a href="#fig:entire_process">1</a>c.
In the original MultiPLIER study, the authors reported that one of the latent variables, identified as LV603, was associated with a known neutrophil pathway and highly correlated with neutrophil count estimates from whole blood RNA-seq profiles <span class="citation" data-cites="18TSqd1tG">[<a href="#ref-18TSqd1tG" role="doc-biblioref">50</a>]</span>.
We analyzed LV603 using PhenoPLIER and found that
1) neutrophil counts and other white blood cell traits were ranked among the top 10 traits out of 4,091 (Figure <a href="#fig:entire_process">1</a>c, bottom), and basophils count and percentage were significantly associated with this LV when using our regression method (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv603">5</a>),
and 2) LV603’s genes were expressed in highly relevant cell types (Figure <a href="#fig:entire_process">1</a>c, top).
These initial results suggested that groups of functionally related and co-expressed genes tend to correspond to groups of trait-associated genes, and the approach can link transcriptional mechanisms from large and diverse dataset collections to complex traits.</p>
<p>Therefore, PhenoPLIER allows the user to address specific questions, namely:
do disease-associated genes belong to modules expressed in specific tissues and cell types?
Are these cell type-specific modules associated with <em>different</em> diseases, thus potentially representing a “network pleiotropy” example from an omnigenic point of view <span class="citation" data-cites="vpIDZCSa">[<a href="#ref-vpIDZCSa" role="doc-biblioref">31</a>]</span>?
Is there a subset of module’s genes that is closer to the definition of “core” genes (i.e., directly affecting the trait with no mediated regulation of other genes <span class="citation" data-cites="LXvTZzEA">[<a href="#ref-LXvTZzEA" role="doc-biblioref">34</a>]</span>) and thus represents alternative and potentially better candidate targets?
Are drugs perturbing these transcriptional mechanisms, and can they suggest potential mechanisms of action?</p>
<h3 id="lvs-link-genes-that-alter-lipid-accumulation-with-relevant-traits-and-tissues">LVs link genes that alter lipid accumulation with relevant traits and tissues</h3>
<p>Our first experiment attempted to answer whether genes in a disease-relevant LV could represent potential therapeutic targets.
For this, the first step was to obtain a set of genes strongly associated with a phenotype of interest.
Therefore, we performed a fluorescence-based CRISPR-Cas9 in the HepG2 cell line and identified 462 genes associated with lipid regulation (<a href="#sec:methods:crispr">Methods</a>).
From these, we selected two high-confidence gene sets that either caused a decrease or increase of lipids:
a lipids-decreasing gene-set with eight genes: <em>BLCAP</em>, <em>FBXW7</em>, <em>INSIG2</em>, <em>PCYT2</em>, <em>PTEN</em>, <em>SOX9</em>, <em>TCF7L2</em>, <em>UBE2J2</em>;
and a lipids-increasing gene-set with six genes: <em>ACACA</em>, <em>DGAT2</em>, <em>HILPDA</em>, <em>MBTPS1</em>, <em>SCAP</em>, <em>SRPR</em> (Supplementary File 2).</p>
<div id="fig:lv246" class="fignos">
<figure>
<img src="images/lvs_analysis/lv246/lv246.svg" style="width:100.0%" alt="Figure 2: Tissues and traits associated with a gene module related to lipid metabolism (LV246).  a) Top cell types/tissues in which LV246’s genes are expressed. Values in the y-axis come from matrix \mathbf{B} in the MultiPLIER models (Figure 1b, see Methods). In the x-axis, cell types/tissues are sorted by the maximum sample value.  b) Gene-trait associations (S-MultiXcan; threshold at -log(p)=10) and colocalization probability (fastENLOC) for the top traits in LV246. The top 40 genes in LV246 are shown, sorted by their LV weight (matrix \mathbf{Z}), from largest (the top gene SCD) to smallest (FAR2); DGAT2 and ACACA, in boldface, are two of the six high-confidence genes in the lipids-increasing gene set from the CRISPR screen. Cardiovascular-related traits are in boldface.  SGBS: Simpson Golabi Behmel Syndrome; CH2DB: CH2 groups to double bonds ratio; HDL: high-density lipoprotein; RCP: locus regional colocalization probability. " />
<figcaption aria-hidden="true"><span>Figure 2:</span> <strong>Tissues and traits associated with a gene module related to lipid metabolism (LV246).</strong>
<!--  -->
<strong>a)</strong> Top cell types/tissues in which LV246’s genes are expressed.
Values in the <span class="math inline">\(y\)</span>-axis come from matrix <span class="math inline">\(\mathbf{B}\)</span> in the MultiPLIER models (Figure <a href="#fig:entire_process">1</a>b, see Methods).
In the <span class="math inline">\(x\)</span>-axis, cell types/tissues are sorted by the maximum sample value.
<!--  -->
<strong>b)</strong> Gene-trait associations (S-MultiXcan; threshold at -log(<span class="math inline">\(p\)</span>)=10) and colocalization probability (fastENLOC) for the top traits in LV246.
The top 40 genes in LV246 are shown, sorted by their LV weight (matrix <span class="math inline">\(\mathbf{Z}\)</span>), from largest (the top gene <em>SCD</em>) to smallest (<em>FAR2</em>);
<em>DGAT2</em> and <em>ACACA</em>, in boldface, are two of the six high-confidence genes in the lipids-increasing gene set from the CRISPR screen.
Cardiovascular-related traits are in boldface.
<!--  -->
SGBS: Simpson Golabi Behmel Syndrome;
CH2DB: CH<sub>2</sub> groups to double bonds ratio;
HDL: high-density lipoprotein;
RCP: locus regional colocalization probability.
<!--  --></figcaption>
</figure>
</div>
<p>Next, we analyzed all 987 LVs using Fast Gene Set Enrichment Analysis (FGSEA) <span class="citation" data-cites="Z8WXLD67">[<a href="#ref-Z8WXLD67" role="doc-biblioref">51</a>]</span>, and found 15 LVs nominally enriched (unadjusted <em>P</em> &lt; 0.01) with these lipid-altering gene-sets (Supplementary Tables <a href="#tbl:sup:lipids_crispr:modules_enriched_increase">2</a> and <a href="#tbl:sup:lipids_crispr:modules_enriched_decrease">3</a>).
Among those with reliable sample metadata, LV246, the top LV associated with the lipids-increasing gene-set, contained genes mainly co-expressed in adipose tissue (Figure <a href="#fig:lv246">2</a>a), which plays a key role in coordinating and regulating lipid metabolism.
Using our regression framework across all traits in PhenomeXcan, we found that gene weights for this LV were predictive of gene associations for plasma lipids, high cholesterol, and Alzheimer’s disease (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv246">8</a>, FDR &lt; 1e-23).
These lipids-related associations also replicated across the 309 traits in eMERGE (Supplementary Table <a href="#tbl:sup:emerge_assocs:lv246">9</a>), where LV246 was significantly associated with hypercholesterolemia (phecode: 272.11, FDR &lt; 4e-9), hyperlipidemia (phecode: 272.1, FDR &lt; 4e-7) and disorders of lipoid metabolism (phecode: 272, FDR &lt; 4e-7).</p>
<p>Two high-confidence genes from our CRISPR screening, <em>DGAT2</em> and <em>ACACA</em>, are responsible for encoding enzymes for triglycerides and fatty acid synthesis and were among the highest-weighted genes of LV246 (Figure <a href="#fig:lv246">2</a>b, in boldface).
However, in contrast to other members of LV246, <em>DGAT2</em> and <em>ACACA</em> were not associated nor colocalized with any of the cardiovascular-related traits and thus would not have been prioritized by TWAS alone;
instead, other members of LV246, such as <em>SCD</em>, <em>LPL</em>, <em>FADS2</em>, <em>HMGCR</em>, and <em>LDLR</em>, were significantly associated and colocalized with lipid-related traits.
This lack of association of two high-confidence genes from our CRISPR screen might be explained from an omnigenic point of view <span class="citation" data-cites="LXvTZzEA">[<a href="#ref-LXvTZzEA" role="doc-biblioref">34</a>]</span>.
Assuming that the TWAS models for <em>DGAT2</em> and <em>ACACA</em> capture all common <em>cis</em>-eQTLs (the only genetic component of gene expression that TWAS can capture) and there are no rare <em>cis</em>-eQTLs, these two genes might represent “core” genes (i.e., they directly affect the trait with no mediated regulation of other genes), and many of the rest in the LV are “peripheral” genes that <em>trans</em>-regulate them.</p>
<h3 id="lvs-predict-drug-disease-pairs-better-than-single-genes">LVs predict drug-disease pairs better than single genes</h3>
<p>We next determined how substituting LVs for individual genes predicted known treatment-disease relationships.
For this, we used the transcriptional responses to small molecule perturbations profiled in LINCS L1000 <span class="citation" data-cites="F7lIlh2N">[<a href="#ref-F7lIlh2N" role="doc-biblioref">43</a>]</span>, which were further processed and mapped to DrugBank IDs <span class="citation" data-cites="6PR8LEXK O21tn8vf 1DJZvtwP1">[<a href="#ref-6PR8LEXK" role="doc-biblioref">52</a>,<a href="#ref-O21tn8vf" role="doc-biblioref">53</a>,<a href="#ref-1DJZvtwP1" role="doc-biblioref">54</a>]</span>.
Based on an established drug repurposing strategy that matches reversed transcriptome patterns between genes and drug-induced perturbations <span class="citation" data-cites="1ClBKizD7 mZjkE1xU">[<a href="#ref-1ClBKizD7" role="doc-biblioref">55</a>,<a href="#ref-mZjkE1xU" role="doc-biblioref">56</a>]</span>, we adopted a previously described framework that uses imputed transcriptomes from TWAS to prioritize drug candidates <span class="citation" data-cites="17oeJ0CXy">[<a href="#ref-17oeJ0CXy" role="doc-biblioref">30</a>]</span>.
For this, we computed a drug-disease score by calculating the negative dot product between the <span class="math inline">\(z\)</span>-scores for a disease (from TWAS) and the <span class="math inline">\(z\)</span>-scores for a drug (from LINCS) across sets of genes of different sizes (see <a href="#sec:methods:drug">Methods</a>).
Therefore, a large score for a drug-disease pair indicated that higher (lower) predicted expression values of disease-associated genes are down (up)-regulated by the drug, thus predicting a potential treatment.
Similarly, for the LV-based approach, we estimated how pharmacological perturbations affected the gene module activity by projecting expression profiles of drugs into our latent representation (Figure <a href="#fig:entire_process">1</a>b).
We used a manually-curated gold standard set of drug-disease medical indications <span class="citation" data-cites="O21tn8vf 10KA5jTBQ">[<a href="#ref-O21tn8vf" role="doc-biblioref">53</a>,<a href="#ref-10KA5jTBQ" role="doc-biblioref">57</a>]</span> for 322 drugs across 53 diseases to evaluate the prediction performance.</p>
<div id="fig:drug_disease:roc_pr" class="fignos">
<figure>
<img src="images/drug_disease_prediction/roc_pr_curves.svg" style="width:80.0%" alt="Figure 3: Drug-disease prediction performance for gene-based and LV-based approaches. The receiver operating characteristic (ROC) (left) and the precision-recall curves (right) for a gene-based and LV-based approach. AUC: area under the curve; AP: average precision." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>Drug-disease prediction performance for gene-based and LV-based approaches.</strong>
The receiver operating characteristic (ROC) (left) and the precision-recall curves (right) for a gene-based and LV-based approach.
AUC: area under the curve; AP: average precision.</figcaption>
</figure>
</div>
<p>It is important to note that the gene-trait associations and drug-induced expression profiles projected into the latent space represent a compressed version of the entire set of results.
Despite this information loss, the LV-based method outperformed the gene-based one with an area under the curve of 0.632 and an average precision of 0.858 (Figure <a href="#fig:drug_disease:roc_pr">3</a>).
The prediction results suggested that this low-dimensional space captures biologically meaningful patterns that can link pathophysiological processes with the mechanism of action of drugs.</p>
<p>We examined a specific drug-disease pair to determine whether the LVs driving the prediction were biologically plausible.
Nicotinic acid (niacin) is a B vitamin widely used clinically to treat lipid disorders, although there is controversy on its clinical utility in preventing cardiovascular disease <span class="citation" data-cites="bRPc66OD 13ZGxHjQ5 ZGvG75Bj">[<a href="#ref-bRPc66OD" role="doc-biblioref">58</a>,<a href="#ref-13ZGxHjQ5" role="doc-biblioref">59</a>,<a href="#ref-ZGvG75Bj" role="doc-biblioref">60</a>]</span>.
Niacin exerts its effects on multiple tissues, although its mechanisms are not well understood <span class="citation" data-cites="LVihFr3g 7OgaYjeL OOAkmacQ 13meq3Hgt">[<a href="#ref-LVihFr3g" role="doc-biblioref">61</a>,<a href="#ref-7OgaYjeL" role="doc-biblioref">62</a>,<a href="#ref-OOAkmacQ" role="doc-biblioref">63</a>,<a href="#ref-13meq3Hgt" role="doc-biblioref">64</a>]</span>.
This compound can increase high-density lipoprotein (HDL) by inhibiting an HDL catabolism receptor in the liver.
Niacin also inhibits diacylglycerol acyltransferase–2 (DGAT2), which decreases the production of low-density lipoproteins (LDL) either by modulating triglyceride synthesis in hepatocytes or by inhibiting adipocyte triglyceride lipolysis <span class="citation" data-cites="LVihFr3g">[<a href="#ref-LVihFr3g" role="doc-biblioref">61</a>]</span>.
Niacin was one of the drugs in the gold standard set indicated for atherosclerosis (AT) and coronary artery disease (CAD).
We observed that this compound was predicted by the gene-based and LV-based approach as a medical indication for coronary artery disease (CAD), with scores above the mean (0.51 and 0.96, respectively).
For AT, the LV-based approach predicted niacin as a therapeutic drug with a score of 0.52, whereas the gene-based method assigned a negative score of -0.01 (below the mean).
Since LVs represent interpretable features associated with specific cell types, we analyzed which LVs positively contributed to these predictions (i.e., with an opposite direction between niacin and the disease).
Notably, LV246 (Figure <a href="#fig:lv246">2</a>), expressed in adipose tissue and liver and associated with plasma lipids and high cholesterol (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv246">8</a>), was the 16th most important module in the prediction of niacin as a therapeutic drug for AT.
Besides the gold standard set, LV246 was among the top modules for other cardiovascular diseases, such as ischaemic heart disease (wide definition, 15th module) and high cholesterol (7th module).</p>
<div id="fig:lv116:cell_types" class="fignos">
<figure>
<img src="images/lvs_analysis/lv116/lv116-cell_types.svg" style="width:100.0%" alt="Figure 4: Top cell types/tissues where LV116’s genes are expressed. Values in the y-axis come from matrix \mathbf{B} in the MultiPLIER models (Figure 1b). In the x-axis, cell types/tissues are sorted by the maximum sample value. The figure shows a clear immune response with cell types under different stimuli.  MS: multiple sclerosis;   HSV: treated with herpes simplex virus;  WNV: infected with West Nile virus;  IFNa: treated with interferon-alpha;  HMDM: human peripheral blood mononuclear cell-derived macrophages;   Salm: infected with Salmonella typhimurium; Yers: infected with Yersinia pseudotuberculosis;  ISM: Interferon Signature Metric; SLE: Systemic lupus erythematosus." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Top cell types/tissues where LV116’s genes are expressed.</strong>
Values in the <span class="math inline">\(y\)</span>-axis come from matrix <span class="math inline">\(\mathbf{B}\)</span> in the MultiPLIER models (Figure <a href="#fig:entire_process">1</a>b).
In the <span class="math inline">\(x\)</span>-axis, cell types/tissues are sorted by the maximum sample value.
The figure shows a clear immune response with cell types under different stimuli.
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP045500 -->
MS: multiple sclerosis;
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP045569 -->
<!-- PBMCs: peripheral blood mononuclear cells; -->
HSV: treated with herpes simplex virus;
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP015670 -->
WNV: infected with West Nile virus;
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP062958 -->
IFNa: treated with interferon-alpha;
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP039361 -->
HMDM: human peripheral blood mononuclear cell-derived macrophages;
<!-- IPSDM: human induced pluripotent stem cell-derived macrophages; -->
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP056733 -->
Salm: infected with <em>Salmonella typhimurium</em>;
Yers: infected with <em>Yersinia pseudotuberculosis</em>;
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP062966 -->
ISM: Interferon Signature Metric;
SLE: Systemic lupus erythematosus.</figcaption>
</figure>
</div>
<p>The analysis of other top niacin-contributing LVs across different cardiovascular diseases revealed additional mechanisms of action.
For example, <em>GPR109A/HCAR2</em> encodes a G protein-coupled high-affinity niacin receptor in adipocytes and immune cells, including monocytes, macrophages, neutrophils and dendritic cells <span class="citation" data-cites="izbPw2kc wI0IjT3i">[<a href="#ref-izbPw2kc" role="doc-biblioref">65</a>,<a href="#ref-wI0IjT3i" role="doc-biblioref">66</a>]</span>.
It was initially thought that the antiatherogenic effects of niacin were solely due to the inhibition of lipolysis in adipose tissue.
However, it has been shown that nicotinic acid can reduce atherosclerosis progression independently of its antidyslipidemic activity by activating <em>GPR109A</em> in immune cells <span class="citation" data-cites="1Bz0jRHYo">[<a href="#ref-1Bz0jRHYo" role="doc-biblioref">67</a>]</span>, thus boosting anti-inflammatory processes <span class="citation" data-cites="1DblG8swn">[<a href="#ref-1DblG8swn" role="doc-biblioref">68</a>]</span>.
In addition, flushing, a common adverse effect of niacin, is also produced by the activation of GPR109A in Langerhans cells (macrophages of the skin).
This alternative mechanism for niacin could have been hypothesized by examining the cell types where the top-contributing modules are expressed:
for instance, LV116 and LV931 (Figure <a href="#fig:lv116:cell_types">4</a>, Supplementary Figure <a href="#fig:sup:lv931">17</a>, and Supplementary Tables <a href="#tbl:sup:multiplier_pathways:lv116">10</a> and <a href="#tbl:sup:multiplier_pathways:lv931">11</a>) were the top two modules for AT, with a strong signature in monocytes, macrophages, neutrophils, dendritic cells, among others.
In Figure <a href="#fig:lv116:cell_types">4</a>, it can be seen that LV116’s genes are expressed as an immune response when these cell types are under different stimuli, such as diarrhea caused by different pathogens <span class="citation" data-cites="RliFvowC">[<a href="#ref-RliFvowC" role="doc-biblioref">69</a>]</span>, samples from multiple sclerosis or systemic lupus erythematosus <span class="citation" data-cites="D5XBhzim EnoqU4ga">[<a href="#ref-D5XBhzim" role="doc-biblioref">70</a>,<a href="#ref-EnoqU4ga" role="doc-biblioref">71</a>]</span>, or infected with different viruses (such as herpes simplex <span class="citation" data-cites="11eausmiy">[<a href="#ref-11eausmiy" role="doc-biblioref">72</a>]</span>, West Nile virus <span class="citation" data-cites="mtMYROCN">[<a href="#ref-mtMYROCN" role="doc-biblioref">73</a>]</span>, <em>Salmonella typhimurium</em> <span class="citation" data-cites="1kgcHkGm">[<a href="#ref-1kgcHkGm" role="doc-biblioref">74</a>]</span>, among others).
These three LVs (LV246, LV116 and LV931) were among the top 20 modules contributing to the niacin prediction across different cardiovascular traits (Table <a href="#tbl:niacin:cardio:top_lvs">1</a>).</p>
<!-- niacin:cardiovascular:top_lvs:start DISABLE NOW, BUT HAS TO BE ADDED THE "end" version just below to update table -->
<div id="tbl:niacin:cardio:top_lvs" class="tablenos">
<table id="tbl:niacin:cardio:top_lvs">
<caption><span>Table 1:</span> LVs among the top 20 contributors to the prediction of niacin for five cardiovascular diseases. “Heart attack, angina, stroke or hypertension” refers to the UK Biobank data-field 6150. GWAS sample size: Atherosclerosis (361,194 in total and 566 cases), Chronic ischaemic heart disease (361,194 in total and 12,769 cases), Heart attack, angina, stroke or hypertension (360,420 in total and 253,565 cases), Ischaemic heart disease/wide definition (361,194 in total and 20,857 cases), High cholesterol/self-reported (361,141 in total and 43,957 cases). </caption>
<colgroup>
<col style="width: 9%" />
<col style="width: 30%" />
<col style="width: 60%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">LV</th>
<th style="text-align: left;">Cell type</th>
<th style="text-align: left;">Disease</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">LV116</td>
<td style="text-align: left;">Immune cells, skin</td>
<td style="text-align: left;">Atherosclerosis (ICD10 I70)</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Chronic ischaemic heart disease (ICD10 I25)</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Heart attack, angina, stroke or hypertension</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Ischaemic heart disease (wide definition)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">LV931</td>
<td style="text-align: left;">Immune cells</td>
<td style="text-align: left;">Atherosclerosis (ICD10 I70)</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Heart attack, angina, stroke or hypertension</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Ischaemic heart disease (wide definition)</td>
</tr>
<tr class="even">
<td style="text-align: left;">LV246</td>
<td style="text-align: left;">Adipose tissue, liver</td>
<td style="text-align: left;">Atherosclerosis (ICD10 I70)</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">High cholesterol (self-reported)</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Ischaemic heart disease (wide definition)</td>
</tr>
</tbody>
</table>
</div>
<p>Beyond cardiovascular traits, there are other potentially interesting LVs that could extend our understanding of the mechanisms of niacin.
For example, LV66, one of the top LVs affected by niacin (Supplementary Figure <a href="#fig:sup:lv66">18</a>), was mainly expressed in ovarian granulosa cells.
This compound has been very recently considered a potential therapeutic for ovarian diseases <span class="citation" data-cites="kLRErKXz TovvsrDr">[<a href="#ref-kLRErKXz" role="doc-biblioref">75</a>,<a href="#ref-TovvsrDr" role="doc-biblioref">76</a>]</span>, as it was found to promote follicle growth and inhibit granulosa cell apoptosis in animal models.</p>
<h3 id="lvs-reveal-trait-clusters-with-shared-transcriptomic-properties">LVs reveal trait clusters with shared transcriptomic properties</h3>
<div id="fig:clustering:design" class="fignos">
<figure>
<img src="images/clustering/clustering_design.svg" style="width:100.0%" alt="Figure 5: Cluster analysis on traits using the latent gene expression representation. a) The projection of TWAS results on 3,752 traits into the latent gene expression representation is the input data to the clustering process. A linear (PCA) and non-linear (UMAP) dimensionality reduction techniques were applied to the input data, and five different clustering algorithms processed all data versions. These algorithms derive partitions from the data using different parameters (such as the number of clusters), leading to an ensemble of 4,428 partitions. Then, a distance matrix is derived by counting how many times a pair of traits was grouped in different clusters across the ensemble. Finally, a consensus function is applied to the distance matrix to generate consolidated partitions with different numbers of clusters (from 2 to \sqrt{n}\approx 60). These final solutions were represented in the clustering tree (Figure 6). b) The clusters found by the consensus function were used as labels to train a decision tree classifier on the original input data, which detects the LVs that better differentiate groups of traits." />
<figcaption aria-hidden="true"><span>Figure 5:</span> <strong>Cluster analysis on traits using the latent gene expression representation.</strong>
<strong>a)</strong> The projection of TWAS results on 3,752 traits into the latent gene expression representation is the input data to the clustering process.
A linear (PCA) and non-linear (UMAP) dimensionality reduction techniques were applied to the input data, and five different clustering algorithms processed all data versions.
These algorithms derive partitions from the data using different parameters (such as the number of clusters), leading to an ensemble of 4,428 partitions.
Then, a distance matrix is derived by counting how many times a pair of traits was grouped in different clusters across the ensemble.
Finally, a consensus function is applied to the distance matrix to generate consolidated partitions with different numbers of clusters (from 2 to <span class="math inline">\(\sqrt{n}\approx\)</span> 60).
These final solutions were represented in the clustering tree (Figure <a href="#fig:clustering:tree">6</a>).
<strong>b)</strong> The clusters found by the consensus function were used as labels to train a decision tree classifier on the original input data, which detects the LVs that better differentiate groups of traits.</figcaption>
</figure>
</div>
<p>We used the projection of gene-trait associations into the latent space to find groups of clusters linked by the same transcriptional processes.
Since individual clustering algorithms have different biases (i.e., assumptions about the data structure), we designed a consensus clustering framework that combines solutions or partitions of traits generated by different methods (<a href="#sec:methods:clustering">Methods</a>).
Consensus or ensemble approaches have been recommended to avoid several pitfalls when performing cluster analysis on biological data <span class="citation" data-cites="14dCeRkua">[<a href="#ref-14dCeRkua" role="doc-biblioref">77</a>]</span>.
Since diversity in the ensemble is crucial for these methods, we generated different data versions which were processed using different methods with varying sets of parameters (Figure <a href="#fig:clustering:design">5</a>a).
Then, a consensus function combines the ensemble into a consolidated solution, which has been shown to outperform any individual member of the ensemble <span class="citation" data-cites="uw3AnEgA cuROQDFa">[<a href="#ref-uw3AnEgA" role="doc-biblioref">78</a>,<a href="#ref-cuROQDFa" role="doc-biblioref">79</a>]</span>.
Our clustering pipeline generated 15 final consensus clustering solutions (Supplementary Figure <a href="#fig:sup:clustering:agreement">15</a>).
The number of clusters of these partitions (between 5 to 29) was learned from the data by selecting the partitions with the largest agreement with the ensemble <span class="citation" data-cites="uw3AnEgA">[<a href="#ref-uw3AnEgA" role="doc-biblioref">78</a>]</span>.
Instead of selecting one of these final solutions with a specific number of clusters, we used a clustering tree <span class="citation" data-cites="xhtEAzx6">[<a href="#ref-xhtEAzx6" role="doc-biblioref">80</a>]</span> (Figure <a href="#fig:clustering:tree">6</a>) to examine stable groups of traits across multiple resolutions.
To understand which latent variables differentiated the group of traits, we trained a decision tree classifier on the input data <span class="math inline">\(\hat{\mathbf{M}}\)</span> using the clusters found as labels (Figure <a href="#fig:clustering:design">5</a>b, see <a href="#sec:methods:clustering">Methods</a>).</p>
<div id="fig:clustering:tree" class="fignos">
<figure>
<img src="images/clustering/clustering_tree.svg" style="width:100.0%" alt="Figure 6: Clustering tree using multiple resolutions for clusters of traits. Each row represents a partition/grouping of the traits, and each circle is a cluster from that partition. The number of clusters goes from 5 to 29. Arrows indicate how traits in one cluster move across clusters from different partitions. Most of the clusters are preserved across different resolutions, showing highly stable solutions even with independent runs of the clustering algorithm.  RDW: red cell (erythrocyte) distribution width; BMI: body mass index; WC: waist circumference; HC: hip circumference; RA: rheumatoid arthritis; SLE: systemic lupus erythematosus; HTN: Hypertension; IBD: inflammatory bowel disease; SCZ: Schizophrenia; CAD: Coronary artery disease; AD: Alzheimer’s disease;  Descriptions of traits by cluster ID (from left to right): 12: also includes lymphocyte count and allergies such as allergic rhinitis or eczema; 4: includes reticulocyte count and percentage, immature reticulocyte fraction, and high light scatter reticulocytes count and percentage; 2: includes mean corpuscular volume, mean corpuscular hemoglobin, mean reticulocyte volume, mean sphered cell volume; 5: includes erythrocyte count, hemoglobin concentration, and hematocrit percentage; 20: also includes weight, waist and hip circumference; 18: also includes body impedance measures and ankle spacing width; 19: also includes basal metabolic rate; 1: includes platelet count, crit, mean volume, and distribution width; 13: diabetes refers to age when diabetes was first diagnosed; 25: also includes vascular problems such as angina, deep vein thrombosis (DVT), intraocular pressure, eye and mouth problems, pulse rate, hand-grip strength, several measurements of physical activity, jobs involving heavy physical work, types of transport used, intake of vitamin/mineral supplements, and various types of body pain and medications for pain relief; 21: also includes attention deficit hyperactivity disorder (ADHD), number of years of schooling completed, bone density, and intracranial volume measurement; 28: includes diabetes, gout, arthrosis, and respiratory diseases (and related medications such as ramipril, allopurinol, and lisinopril), urine assays, female-specific factors (age at menarche, menopause, first/last live birth), and several environmental/behavioral factors such as intake of a range of food/drink items including alcohol, time spent outdoors and watching TV, smoking and sleeping habits, early-life factors (breastfed as a baby, maternal smoking around birth), education attainment, psychological and mental health, and health satisfaction; 11: also includes fasting blood glucose and insulin measurement; 16: lipids include high and low-density lipoprotein (HDL and LDL) cholesterol, triglycerides, and average number of methylene groups per a double bond; 14: includes myocardial infarction, coronary atherosclerosis, ischaemic heart disease (wide definition); 7: includes monocyte count and percentage; 24: includes lymphocyte count and percentage; 9: includes neutrophil count, neutrophil+basophil count, neutrophil+eosinophil count, granulocyte count, leukocyte count, and myeloid cell count; 3: includes eosinophil count, eosinophil percentage, and eosinophil+basophil count." />
<figcaption aria-hidden="true"><span>Figure 6:</span> <strong>Clustering tree using multiple resolutions for clusters of traits.</strong>
Each row represents a partition/grouping of the traits, and each circle is a cluster from that partition.
The number of clusters goes from 5 to 29.
Arrows indicate how traits in one cluster move across clusters from different partitions.
Most of the clusters are preserved across different resolutions, showing highly stable solutions even with independent runs of the clustering algorithm.
<!--  -->
RDW: red cell (erythrocyte) distribution width;
BMI: body mass index;
WC: waist circumference;
HC: hip circumference;
RA: rheumatoid arthritis;
SLE: systemic lupus erythematosus;
HTN: Hypertension;
IBD: inflammatory bowel disease;
SCZ: Schizophrenia;
CAD: Coronary artery disease;
AD: Alzheimer’s disease;
<!--  -->
<em>Descriptions of traits by cluster ID (from left to right):</em>
12: also includes lymphocyte count and allergies such as allergic rhinitis or eczema;
4: includes reticulocyte count and percentage, immature reticulocyte fraction, and high light scatter reticulocytes count and percentage;
2: includes mean corpuscular volume, mean corpuscular hemoglobin, mean reticulocyte volume, mean sphered cell volume;
5: includes erythrocyte count, hemoglobin concentration, and hematocrit percentage;
20: also includes weight, waist and hip circumference;
18: also includes body impedance measures and ankle spacing width;
19: also includes basal metabolic rate;
1: includes platelet count, crit, mean volume, and distribution width;
13: diabetes refers to age when diabetes was first diagnosed;
25: also includes vascular problems such as angina, deep vein thrombosis (DVT), intraocular pressure, eye and mouth problems, pulse rate, hand-grip strength, several measurements of physical activity, jobs involving heavy physical work, types of transport used, intake of vitamin/mineral supplements, and various types of body pain and medications for pain relief;
21: also includes attention deficit hyperactivity disorder (ADHD), number of years of schooling completed, bone density, and intracranial volume measurement;
28: includes diabetes, gout, arthrosis, and respiratory diseases (and related medications such as ramipril, allopurinol, and lisinopril), urine assays, female-specific factors (age at menarche, menopause, first/last live birth), and several environmental/behavioral factors such as intake of a range of food/drink items including alcohol, time spent outdoors and watching TV, smoking and sleeping habits, early-life factors (breastfed as a baby, maternal smoking around birth), education attainment, psychological and mental health, and health satisfaction;
11: also includes fasting blood glucose and insulin measurement;
16: lipids include high and low-density lipoprotein (HDL and LDL) cholesterol, triglycerides, and average number of methylene groups per a double bond;
14: includes myocardial infarction, coronary atherosclerosis, ischaemic heart disease (wide definition);
7: includes monocyte count and percentage;
24: includes lymphocyte count and percentage;
9: includes neutrophil count, neutrophil+basophil count, neutrophil+eosinophil count, granulocyte count, leukocyte count, and myeloid cell count;
3: includes eosinophil count, eosinophil percentage, and eosinophil+basophil count.</figcaption>
</figure>
</div>
<p>We found that phenotypes were grouped into five clear branches, defined by their first node at the top of the Figure <a href="#fig:clustering:tree">6</a>:
0) a “large” branch that includes most of the traits subdivided only starting at <span class="math inline">\(k\)</span>=16 (with asthma, subjective well-being traits, and nutrient intake clusters),
1) heel bone-densitometry measurements,
2) hematological assays on red blood cells,
3) physical measures, including spirometry and body impedance, and anthropometric traits with fat-free and fat mass measures in separate sub-branches, and
4) a “complex” branch including keratometry measurements, assays on white blood cells and platelets, skin and hair color traits, autoimmune disorders, and cardiovascular diseases (which also included other cardiovascular-related traits such as hand-grip strength <span class="citation" data-cites="aBVh8zt1">[<a href="#ref-aBVh8zt1" role="doc-biblioref">81</a>]</span>, and environmental/behavioral factors such as physical activity and diet) (see Supplementary Files 3-6 for clustering results).
Within these branches, results were relatively stable, with the same traits often clustered together across different resolutions.
Arrows between clusters show traits moving from one group to another, and this mainly happens between clusters within the “complex” branch (4) and between clusters from the “large” branch (0) to the “complex” branch.
This behavior is expected since complex diseases are usually associated with shared genetic and environmental factors and are thus hard to categorize into a single cluster.</p>
<div id="fig:clustering:heatmap" class="fignos">
<figure>
<img src="images/clustering/global_clustermap-plain.svg" style="width:100.0%" alt="Figure 7: Cluster-specific and general transcriptional processes associated with different diseases. The plot shows a submatrix of \hat{\mathbf{M}} for the main trait clusters at k=29, considering only LVs (rows) that are well-aligned with at least one pathway." />
<figcaption aria-hidden="true"><span>Figure 7:</span> <strong>Cluster-specific and general transcriptional processes associated with different diseases.</strong>
The plot shows a submatrix of <span class="math inline">\(\hat{\mathbf{M}}\)</span> for the main trait clusters at <span class="math inline">\(k\)</span>=29, considering only LVs (rows) that are well-aligned with at least one pathway.</figcaption>
</figure>
</div>
<p>Next, we analyzed which LVs were driving these clusters of traits.
For this, we trained decision tree classifiers on the input data using each cluster at <span class="math inline">\(k\)</span>=29 (bottom of Figure <a href="#fig:clustering:tree">6</a>) as labels (see <a href="#sec:methods:clustering">Methods</a>).
This procedure yielded the top LVs that were most discriminative for each cluster.
Several of these LVs were well-aligned to existing pathways (Figure <a href="#fig:clustering:heatmap">7</a>), whereas others were not aligned to prior knowledge but still expressed in relevant tissues (Supplementary Figure <a href="#fig:sup:clustering:novel:heatmap">16</a>).
In Figure <a href="#fig:clustering:heatmap">7</a>, it can be seen that some LVs were highly specific to certain traits, while others were associated with a wide range of different phenotypes, thus potentially involved in more general biological functions.
We used our regression framework to determine whether these LVs were significantly associated with different traits.
For example, LVs such as LV928 and LV30, which were well-aligned to early progenitors of the erythrocytes lineage <span class="citation" data-cites="Zk82GvJV">[<a href="#ref-Zk82GvJV" role="doc-biblioref">82</a>]</span> (Supplementary Tables <a href="#tbl:sup:multiplier_pathways:lv928">13</a> and <a href="#tbl:sup:multiplier_pathways:lv30">16</a>), were predominantly expressed in early differentiation stages of erythropoiesis (Supplementary Figures <a href="#fig:sup:lv928">19</a> and <a href="#fig:sup:lv30">20</a>) and strongly associated with different assays on red blood cells (FDR &lt; 0.05; Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv928">14</a>, <a href="#tbl:sup:emerge_assocs:lv928">15</a>, and <a href="#tbl:sup:emerge_assocs:lv30">18</a>).
In contrast, other LVs were highly specific, such as LV730, which is expressed in thrombocytes from different cancer samples (Supplementary Figures <a href="#fig:sup:lv730">21</a> and Supplementary Table <a href="#tbl:sup:multiplier_pathways:lv730">19</a>), and strongly associated with hematological assays on platelets (FDR &lt; 0.05, Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv730">20</a>);
or LV598, whose genes were expressed in corneal endothelial cells (Supplementary Figures <a href="#fig:sup:lv598">22</a> and Supplementary Table <a href="#tbl:sup:multiplier_pathways:lv598">22</a>) and associated with keratometry measurements (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv598">23</a>).</p>
<p>The sub-branches of autoimmune and cardiovascular diseases merged together at <span class="math inline">\(k=10\)</span> (middle of Figure <a href="#fig:clustering:tree">6</a>), so we expected to find LVs that specifically affect one or both of these types of diseases.
For example, LV57, expressed in T cells (Supplementary Figure <a href="#fig:sup:lv57">23</a> and Supplementary Table <a href="#tbl:sup:multiplier_pathways:lv57">25</a>), was the most strongly associated gene module with autoimmune disorders in PhenomeXcan (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv57">26</a>), with significant associations with hypothyroidism that were replicated in eMERGE (<a href="#tbl:sup:emerge_assocs:lv57">27</a>).
However, this LV was also strongly associated with deep venous thrombosis in both PhenomeXcan and eMERGE.
On the other hand, LV844 was more autoimmune-specific, with associations to polymyalgia rheumatica, type 1 diabetes, rheumatoid arthritis, and celiac disease in PhenomeXcan (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv844">29</a>).
However, these did not replicate in eMERGE.
This LV was expressed in a wide range of cell types, including blood, breast organoids, myeloma cells, lung fibroblasts, and different cell types from the brain (Supplementary Figure <a href="#fig:sup:lv844">24</a> and Supplementary Table <a href="#tbl:sup:multiplier_pathways:lv844">28</a>).</p>
<p>The cardiovascular sub-branch had 129 significant LV-trait associations in PhenomeXcan and 23 in eMERGE.
LV136, aligned with known collagen formation and muscle contraction pathways (Supplementary Table <a href="#tbl:sup:multiplier_pathways:lv136">31</a>), was associated with coronary artery disease and keratometry measurements in PhenomeXcan (Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv136">32</a>).
In eMERGE, this LV was associated with coronary atherosclerosis (phecode: 411.4) (Supplementary Table <a href="#tbl:sup:emerge_assocs:lv136">33</a>).
LV136 was expressed in a wide range of cell types, including fibroblasts, mesenchymal stem cells, osteoblasts, pancreatic stellate cells, cardiomyocytes, and adipocytes (Supplementary Figure <a href="#fig:sup:lv136">25</a>).
Within the cardiovascular sub-branch, we found neuropsychiatric and neurodevelopmental disorders such as Alzheimer’s disease, schizophrenia, and attention deficit hyperactivity disorder (ADHD).
These disorders were previously linked to the cardiovascular system <span class="citation" data-cites="10qjLoufR j2Sl4DAE 17LYMnG9n 13t4TuFeJ">[<a href="#ref-10qjLoufR" role="doc-biblioref">83</a>,<a href="#ref-j2Sl4DAE" role="doc-biblioref">84</a>,<a href="#ref-17LYMnG9n" role="doc-biblioref">85</a>,<a href="#ref-13t4TuFeJ" role="doc-biblioref">86</a>]</span> and share several risk factors, including hypertension, high cholesterol, obesity, smoking, among others <span class="citation" data-cites="D83Aqhga 9BGyO071">[<a href="#ref-D83Aqhga" role="doc-biblioref">87</a>,<a href="#ref-9BGyO071" role="doc-biblioref">88</a>]</span>.
However, our results grouped these diseases by potentially shared transcriptional processes expressed in specific tissues/cell types.
Alzheimer’s disease (not present in eMERGE), for instance, was significantly associated with LV21 in PhenomeXcan (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv21">35</a>).
LV21, a gene module not aligned to prior pathways, was strongly expressed in a variety of soft tissue sarcomas, monocytes/macrophages (including microglia from cortex samples), and aortic valves (Supplementary Figure <a href="#fig:sup:lv21">26</a> and Supplementary Table <a href="#tbl:sup:multiplier_pathways:lv21">34</a>).
This LV was also strongly associated with lipids and high cholesterol in PhenomeXcan and hyperlipidemia (phecode: 272.1) in eMERGE (Supplementary Table <a href="#tbl:sup:emerge_assocs:lv21">36</a>).
As discussed previously, macrophages play a key role in the reverse cholesterol transport and thus atherogenesis <span class="citation" data-cites="idlBgtFz">[<a href="#ref-idlBgtFz" role="doc-biblioref">89</a>]</span>, and lipid metabolism in microglia has been recently identified as an important factor in the development of neurodegenerative diseases <span class="citation" data-cites="18I4ish9s">[<a href="#ref-18I4ish9s" role="doc-biblioref">90</a>]</span>.</p>
<h2 id="discussion">Discussion</h2>
<p>We have introduced a novel computational strategy that integrates statistical associations from TWAS with groups of genes (gene modules) that have similar expression patterns across the same cell types.
Our key innovation is that we project gene-trait associations through a latent representation derived not strictly from measures of normal tissue but also from cell types under a variety of stimuli and at various developmental stages.
This improves interpretation by going beyond statistical associations to infer cell type-specific features of complex phenotypes.
Our approach can identify disease-relevant cell types from summary statistics, and several disease-associated gene modules were replicated in eMERGE.
Using a CRISPR screen to analyze lipid regulation, we found that our gene module-based approach can prioritize causal genes even when single gene associations are not detected.
We interpret these findings with an omnigenic perspective of “core” and “peripheral” genes, suggesting that the approach can identify genes that directly affect the trait with no mediated regulation of other genes and thus prioritize alternative and potentially more attractive therapeutic targets.</p>
<p>Using our gene module perspective, we also integrated drug-induced transcriptional profiles, which allowed us to connect diseases, drugs, and cell types.
We showed that the LV-based drug-repurposing approach outperformed the gene-based one when predicting drug-disease links for 322 drugs across 53 diseases.
Furthermore, and beyond statistical prediction, we focused on cardiovascular traits and a particular drug, niacin, to show that the approach connects pathophysiological processes with known mechanisms of action, including those in adipose tissue, immune cells, and ovarian granulosa cells.
Our LV-based approach could be helpful in generating novel hypotheses to evaluate potential mechanisms of action, or even adverse effects, of known or experimental drugs.</p>
<p>We found that the analysis of associations through latent representations provided reasonable groupings of diseases and traits affected by shared and distinct transcriptional mechanisms expressed in highly relevant tissues.
Our cluster analysis approach also detected the LVs that were most discriminative for each cluster.
Several of these LVs were also significantly associated with different traits.
Some LVs were strongly aligned with known pathways, but others (like LV57) were not, which might represent novel disease-relevant mechanisms.
In some cases, the features/LVs linked to phenotypes appear to be associated with specific cell types.
Associations with such cell type marker genes may reveal potentially causal cell types for a phenotype with more precision.
We observed modules expressed primarily in one tissue (such as adipose in LV246 or ovary in LV66).
Others appeared to be expressed in many contexts, which may capture pathways associated with related complex diseases.
For example, LV136 is associated with cardiovascular disease and measures of corneal biomechanics and is expressed in fibroblasts, osteoblasts, pancreas, liver, and cardiomyocytes, among others.
Other examples include LV844, expressed in whole blood samples and associated with a range of autoimmune diseases;
or LV57, which is clearly expressed in T cells and strongly associated with autoimmune and venous thromboembolism.
From an omnigenic point of view, these patterns might represent cases of “network pleiotropy,” where the same cell types mediate molecularly related traits.
To our knowledge, projection through a representation learned on complementary but distinct datasets is a novel approach to identifying cell type and pathway effects on complex phenotypes that is computationally simple to implement.</p>
<p>We also demonstrated that clustering trees, introduced initially as a means to examine developmental processes in single-cell data, provide a multi-resolution grouping of phenotypes based on latent variable associations.
We employed hard-partitioning algorithms (one trait belongs exclusively to one cluster) where the distance between two traits takes into account all gene modules.
However, it is also plausible for two complex diseases to share only a few biological processes instead of being similar across most of them.
Another important consideration is that our TWAS results were derived from a large set of GWAS of different sample sizes and qualities.
Although the potential issues derived from this data heterogeneity were addressed before performing our cluster analyses on traits, data preprocessing steps are always challenging and might not avoid bias altogether.
Considering groups of related diseases was previously shown to be more powerful in detecting shared genetic etiology <span class="citation" data-cites="LyJmyoQr veADXImD">[<a href="#ref-LyJmyoQr" role="doc-biblioref">91</a>,<a href="#ref-veADXImD" role="doc-biblioref">92</a>]</span>, and clustering trees provide a way to explore such relationships in the context of latent variables.</p>
<p>Finally, we developed an LV-based regression framework to detect whether gene modules are associated with a trait using TWAS <span class="math inline">\(p\)</span>-values.
We used PhenomeXcan as a discovery cohort across four thousand traits, and many LV-trait associations replicated in eMERGE.
In PhenomeXcan, we found 3,450 significant LV-trait associations (FDR &lt; 0.05) with 686 LVs (out of 987) associated with at least one trait and 1,176 traits associated with at least one LV.
In eMERGE, we found 196 significant LV-trait associations, with 116 LVs associated with at least one trait/phecode and 81 traits with at least one LV.
We only focused on a few disease types from our trait clusters, but the complete set of associations on other disease domains is available in our <a href="https://github.com/greenelab/phenoplier">Github repository</a> for future research.
As noted in <a href="#sec:methods:reg">Methods</a>, one limitation of the regression approach is that the gene-gene correlations are only approximately accurate, which could lead to false positives if the correlation among the top genes in a module is not precisely captured.
The regression model, however, is approximately well-calibrated, and we did not observe inflation when running the method in real data.</p>
<p>Our approach rests on the assumption that gene modules with coordinated expression patterns will also manifest coordinated pathological effects.
Our implementation in this work integrates two complementary approaches.
The first is MultiPLIER, which extracts latent variables from large expression datasets, and these LVs could represent either real transcriptional processes or technical factors (“batch effects”).
We used a previously published model derived from recount2, which was designed to analyze rare disorders but might not be the optimal latent representation for the wide range of complex diseases considered here.
Also, the underlying factorization method rests on linear combinations of variables, which could miss important and more complex co-expression patterns.
In addition, recount2, the training dataset used, has since been surpassed in size and scale by other resources <span class="citation" data-cites="Nz3IMEzd TPVeG4GP">[<a href="#ref-Nz3IMEzd" role="doc-biblioref">20</a>,<a href="#ref-TPVeG4GP" role="doc-biblioref">93</a>]</span>.
However, it is important to note that our models impose very few assumptions on the latent expression representation.
Therefore, we should be able to easily replace MultiPLIER with other similar approaches like GenomicSuperSignature <span class="citation" data-cites="X4fhSCkz">[<a href="#ref-X4fhSCkz" role="doc-biblioref">94</a>]</span>.
The second approach we used in this study is TWAS, where we are only considering the hypothesis that GWAS loci affect traits via changes in gene expression.
Other effects, such as coding variants disrupting protein-protein interactions, are not captured.
Additionally, TWAS has several limitations that can lead to false positives <span class="citation" data-cites="l6ogswV3 ndd3tW4g">[<a href="#ref-l6ogswV3" role="doc-biblioref">95</a>,<a href="#ref-ndd3tW4g" role="doc-biblioref">96</a>]</span>.
Like GWAS, which generally detects groups of associated variants in linkage disequilibrium (LD), TWAS usually identifies several genes within the same locus <span class="citation" data-cites="AxVJwanp Z8bvDdVq">[<a href="#ref-Z8bvDdVq" role="doc-biblioref">25</a>,<a href="#ref-AxVJwanp" role="doc-biblioref">97</a>]</span>.
This is due to sharing of GWAS variants in gene expression models, correlated expression of nearby genes, or even correlation of their predicted expression due to eQTLs in LD, among others <span class="citation" data-cites="l6ogswV3">[<a href="#ref-l6ogswV3" role="doc-biblioref">95</a>]</span>.
Our LV-based regression framework, however, accounts for these gene-gene correlations in TWAS reasonably well.</p>
<p>Our findings are concordant with previous studies showing that drugs with genetic support are more likely to succeed through the drug development pipeline <span class="citation" data-cites="REXpV7nA 17oeJ0CXy">[<a href="#ref-REXpV7nA" role="doc-biblioref">7</a>,<a href="#ref-17oeJ0CXy" role="doc-biblioref">30</a>]</span>.
In this case, projecting association results through latent variables better prioritized disease-treatment pairs than considering single-gene effects alone.
An additional benefit is that the latent variables driving predictions represent interpretable genetic features that can be examined to infer potential mechanisms of action.
Here we prioritized drugs for diseases with very different tissue etiologies, and a challenge of the approach is to select the most appropriate tissue model from TWAS to find reversed transcriptome patterns between genes and drug-induced perturbations.</p>
<p>Ultimately, the quality of the representations is essential to performance.
Here we used a representation derived from a factorization of bulk RNA-seq data.
Detailed perturbation datasets and single-cell profiling of tissues, with and without perturbagens, and at various stages of development provide an avenue to generate higher quality and more interpretable representations.
On the other hand, the key to interpretability is driven by the annotation of sample metadata.
New approaches to infer and annotate with structured metadata are promising and can be directly applied to existing data <span class="citation" data-cites="fnDaLjFy">[<a href="#ref-fnDaLjFy" role="doc-biblioref">98</a>]</span>.
Rapid improvements in both areas set the stage for latent variable projections to be widely applied to disentangle the genetic basis of complex human phenotypes.
By providing a new perspective for a mechanistic understanding of statistical associations from TWAS, our method can generate testable hypotheses for the post-GWAS functional characterization of complex diseases, which will likely be an area of great importance in the coming years.</p>
<h2 id="sec:methods">Methods and materials</h2>
<p>PhenoPLIER is a framework that combines different computational approaches to integrate gene-trait associations and drug-induced transcriptional responses with groups of functionally-related genes (referred to as gene modules or latent variables/LVs).
Gene-trait associations are computed using the PrediXcan family of methods, whereas latent variables are inferred by the MultiPLIER models applied on large gene expression compendia.
PhenoPLIER provides
1) a regression model to compute an LV-trait association,
2) a consensus clustering approach applied to the latent space to learn shared and distinct transcriptomic properties between traits, and
3) an interpretable, LV-based drug repurposing framework.
We provide the details of these methods below.</p>
<h3 id="sec:methods:predixcan">The PrediXcan family of methods for gene-based associations</h3>
<p>We used Summary-PrediXcan (S-PrediXcan) <span class="citation" data-cites="vLyTudUB">[<a href="#ref-vLyTudUB" role="doc-biblioref">99</a>]</span> and Summary-MultiXcan (S-MultiXcan) <span class="citation" data-cites="1FFzCXo1s">[<a href="#ref-1FFzCXo1s" role="doc-biblioref">24</a>]</span> as the gene-based statistical approaches, which belong to the PrediXcan family of methods <span class="citation" data-cites="Z8bvDdVq">[<a href="#ref-Z8bvDdVq" role="doc-biblioref">25</a>]</span>.
We broadly refer to these approaches as TWAS (transcription-wide association studies).
S-PrediXcan, the summary-based version of PrediXcan, computes the univariate association between a trait and a gene’s predicted expression in a single tissue.
In contrast, S-MultiXcan, the summary-based version of MultiXcan, computes the joint association between a gene’s predicted expression in all tissues and a trait.
S-PrediXcan and S-MultiXcan only need GWAS summary statistics instead of individual-level genotype and phenotype data.</p>
<p>Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information).
In the following, we refer to <span class="math inline">\(\mathbf{y}\)</span> as a vector of traits for <span class="math inline">\(n\)</span> individuals that is centered for convenience (so that no intercept is necessary);
<span class="math inline">\(\mathbf{\tilde{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}\)</span> is the gene’s predicted expression for all individuals in tissue <span class="math inline">\(l\)</span>, <span class="math inline">\(X_a\)</span> is the genotype of SNP <span class="math inline">\(a\)</span> and <span class="math inline">\(w_{a}\)</span> its weight in the tissue prediction model <span class="math inline">\(l\)</span>;
and <span class="math inline">\(\mathbf{t}_l\)</span> is the standardized version of <span class="math inline">\(\mathbf{\tilde{t}}_l\)</span> with mean equal to zero and standard deviation equal to one.</p>
<p>S-PrediXcan <span class="citation" data-cites="vLyTudUB">[<a href="#ref-vLyTudUB" role="doc-biblioref">99</a>]</span> is the summary version of PrediXcan <span class="citation" data-cites="Z8bvDdVq">[<a href="#ref-Z8bvDdVq" role="doc-biblioref">25</a>]</span>.
PrediXcan models the trait as a linear function of the gene’s expression on a single tissue using the univariate model</p>
<p><span id="eq:predixcan" class="eqnos"><span class="math display">\[
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
\]</span><span class="eqnos-number">(1)</span></span> </p>
<p>where <span class="math inline">\(\hat{\gamma}_l\)</span> is the estimated effect size or regression coefficient, and <span class="math inline">\(\bm{\epsilon}_l\)</span> are the error terms with variance <span class="math inline">\(\sigma_{\epsilon}^{2}\)</span>.
The significance of the association is assessed by computing the <span class="math inline">\(z\)</span>-score <span class="math inline">\(\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)\)</span> for a gene’s tissue model <span class="math inline">\(l\)</span>.
PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan <span class="math inline">\(z\)</span>-scores using only GWAS summary statistics with the expression</p>
<p><span id="eq:spredixcan" class="eqnos"><span class="math display">\[
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
\]</span><span class="eqnos-number">(2)</span></span> </p>
<p>where <span class="math inline">\(\hat{\sigma}_a\)</span> is the variance of SNP <span class="math inline">\(a\)</span>, <span class="math inline">\(\hat{\sigma}_l\)</span> is the variance of the predicted expression of a gene in tissue <span class="math inline">\(l\)</span>, and <span class="math inline">\(\hat{\beta}_a\)</span> is the estimated effect size of SNP <span class="math inline">\(a\)</span> from the GWAS.
In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) <span class="citation" data-cites="9Pr9idng">[<a href="#ref-9Pr9idng" role="doc-biblioref">4</a>]</span> as the reference panel.
Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the <span class="math inline">\(z\)</span>-scores in our drug repurposing approach (described below).</p>
<p>S-MultiXcan <span class="citation" data-cites="1FFzCXo1s">[<a href="#ref-1FFzCXo1s" role="doc-biblioref">24</a>]</span>, on the other hand, is the summary version of MultiXcan.
MultiXcan is more powerful than PrediXcan in detecting gene-trait associations, although it does not provide the direction of effects.
Its main output is the <span class="math inline">\(p\)</span>-value (obtained with an F-test) of the multiple tissue model</p>
<p><span id="eq:multixcan" class="eqnos"><span class="math display">\[
\begin{split}
\mathbf{y} &amp; = \sum_{l=1}^{p} \mathbf{t}_l g_l + \mathbf{e} \\
&amp; = \mathbf{T} \mathbf{g} + \mathbf{e},
\end{split}
\]</span><span class="eqnos-number">(3)</span></span> </p>
<p>where <span class="math inline">\(\mathbf{T}\)</span> is a matrix with <span class="math inline">\(p\)</span> columns <span class="math inline">\(\mathbf{t}_l\)</span>,
<span class="math inline">\(\hat{g}_l\)</span> is the estimated effect size for the predicted gene expression in tissue <span class="math inline">\(l\)</span> (and thus <span class="math inline">\(\mathbf{\hat{g}}\)</span> is a vector with <span class="math inline">\(p\)</span> estimated effect sizes <span class="math inline">\(\hat{g}_l\)</span>),
and <span class="math inline">\(\mathbf{e}\)</span> are the error terms with variance <span class="math inline">\(\sigma_{e}^{2}\)</span>.
Given the high correlation between predicted expression values for a gene across different tissues, MultiXcan uses the principal components (PCs) of <span class="math inline">\(\mathbf{T}\)</span> to avoid collinearity issues.
S-MultiXcan derives the joint regression estimates (effect sizes and their variances) in Equation (<a href="#eq:multixcan">3</a>) using the marginal estimates from S-PrediXcan in Equation (<a href="#eq:spredixcan">2</a>).
Under the null hypothesis of no association, <span class="math inline">\(\mathbf{\hat{g}}^{\top} \frac{\mathbf{T}^{\top}\mathbf{T}}{\sigma_{e}^{2}} \mathbf{\hat{g}} \sim \chi_{p}^{2}\)</span>, and therefore the significance of the association in S-MultiXcan is estimated with</p>
<p><span id="eq:smultixcan" class="eqnos"><span class="math display">\[
\begin{split}
\frac{\mathbf{\hat{g}}^{\top} (\mathbf{T}^{\top}\mathbf{T}) \mathbf{\hat{g}}}{\sigma_{e}^{2}} &amp; \approx \bm{\hat{\gamma}}^{\top} \frac{\sqrt{n-1}}{\sigma_{\epsilon}} \left(\frac{\mathbf{T}^{\top} \mathbf{T}}{n-1}\right)^{-1} \frac{\sqrt{n-1}}{\sigma_{\epsilon}} \bm{\hat{\gamma}} \\
&amp; = \mathbf{\hat{z}}^{\top} Cor(\mathbf{T})^{-1} \mathbf{\hat{z}},
\end{split}
\]</span><span class="eqnos-number">(4)</span></span> </p>
<p>where <span class="math inline">\(\mathbf{\hat{z}}\)</span> is a vector with <span class="math inline">\(p\)</span> <span class="math inline">\(z\)</span>-scores (Equation (<a href="#eq:spredixcan">2</a>)) for each tissue available for the gene,
and <span class="math inline">\(Cor(\mathbf{T})\)</span> is the autocorrelation matrix of <span class="math inline">\(\mathbf{T}\)</span>.
Since <span class="math inline">\(\mathbf{T}^{\top}\mathbf{T}\)</span> is singular for many genes, S-MultiXcan computes the pseudo-inverse <span class="math inline">\(Cor(\mathbf{T})^{+}\)</span> using the <span class="math inline">\(k\)</span> top PCs, and thus <span class="math inline">\(\mathbf{\hat{z}}^{\top} Cor(\mathbf{T})^{+} \mathbf{\hat{z}} \sim \chi_k^2\)</span>.
To arrive at this expression, S-MultiXcan uses the conservative approximation <span class="math inline">\(\sigma_{e}^{2} \approx \sigma_{\epsilon}^{2}\)</span>, that is, the variance of the error terms in the joint regression is approximately equal to the residual variance of the marginal regressions.
Another important point is that <span class="math inline">\(Cor(\mathbf{T})\)</span> is estimated using a global genotype covariance matrix, whereas marginal <span class="math inline">\(\hat{z}_l\)</span> in Equation (<a href="#eq:spredixcan">2</a>) are approximated using tissue-specific genotype covariances.
Although S-MultiXcan yields highly concordant estimates compared with MultiXcan, results are not perfectly correlated across genes <span class="citation" data-cites="1FFzCXo1s">[<a href="#ref-1FFzCXo1s" role="doc-biblioref">24</a>]</span>.
As we explain later, these differences are important for our LV-based regression model when computing the gene-gene correlation matrix.
We used S-MultiXcan results for our LV-based regression model and our cluster analyses of traits.</p>
<h3 id="sec:methods:twas">TWAS resources</h3>
<p>We used two large TWAS resources from different cohorts for discovery and replication, all obtained from European ancestries.
PhenomeXcan <span class="citation" data-cites="lY5ln3dB">[<a href="#ref-lY5ln3dB" role="doc-biblioref">42</a>]</span>, our discovery cohort, provides results on 4,091 traits across different categories.
Supplemenetary File 1 has all the details about the included GWAS, sample size and disease/trait categories.
In PhenomeXcan, these publicly available GWAS summary statistics were used to compute
1) gene-based associations with the PrediXcan family of methods (described before), and
2) a posterior probability of colocalization between GWAS loci and <em>cis</em>-eQTL with fastENLOC <span class="citation" data-cites="lY5ln3dB ndd3tW4g">[<a href="#ref-lY5ln3dB" role="doc-biblioref">42</a>,<a href="#ref-ndd3tW4g" role="doc-biblioref">96</a>]</span>.
We refer to the matrix of <span class="math inline">\(z\)</span>-scores from S-PrediXcan (Equation (<a href="#eq:spredixcan">2</a>)) across <span class="math inline">\(q\)</span> traits and <span class="math inline">\(m\)</span> genes in tissue <span class="math inline">\(t\)</span> as <span class="math inline">\(\mathbf{M}^{t} \in \mathbb{R}^{q \times m}\)</span>.
As explained later, matrices <span class="math inline">\(\mathbf{M}^{t}\)</span> were used in our LV-based drug repurposing framework since they provide direction of effects.
The S-MultiXcan results (22,515 gene associations across 4,091 traits) were used in our LV-based regression framework and our cluster analyses of traits.
For the cluster analyses, we used the <span class="math inline">\(p\)</span>-values converted to <span class="math inline">\(z\)</span>-scores: <span class="math inline">\(\mathbf{M}=\Phi^{-1}(1 - p/2)\)</span>, where <span class="math inline">\(\Phi^{-1}\)</span> is the probit function.
Higher <span class="math inline">\(z\)</span>-scores correspond to stronger associations.</p>
<p>Our discovery cohort was eMERGE <span class="citation" data-cites="wfqjCerX">[<a href="#ref-wfqjCerX" role="doc-biblioref">46</a>]</span>, where the same TWAS methods were run on 309 phecodes <span class="citation" data-cites="gZAOkumx">[<a href="#ref-gZAOkumx" role="doc-biblioref">27</a>]</span> across different categories (more information about traits are available in <span class="citation" data-cites="gZAOkumx">[<a href="#ref-gZAOkumx" role="doc-biblioref">27</a>]</span>).
We used these results to replicate the associations found with our LV-based regression framework in PhenomeXcan.</p>
<h3 id="sec:methods:multiplier">MultiPLIER and Pathway-level information extractor (PLIER)</h3>
<p>MultiPLIER <span class="citation" data-cites="14rnBunuZ">[<a href="#ref-14rnBunuZ" role="doc-biblioref">44</a>]</span> extracts patterns of co-expressed genes from recount2 <span class="citation" data-cites="6SPTvFXq">[<a href="#ref-6SPTvFXq" role="doc-biblioref">19</a>]</span> (without including GTEx samples), a large gene expression dataset.
The approach applies the pathway-level information extractor method (PLIER) <span class="citation" data-cites="Ki2ij7zE">[<a href="#ref-Ki2ij7zE" role="doc-biblioref">45</a>]</span>, which performs unsupervised learning using prior knowledge (canonical pathways) to reduce technical noise.
PLIER uses a matrix factorization approach that deconvolutes gene expression data into a set of latent variables (LV), where each LV represents a gene module.
The MultiPLIER models reduced the dimensionality in recount2 to 987 LVs.</p>
<p>Given a gene expression dataset <span class="math inline">\(\mathbf{Y}^{m \times c}\)</span> with <span class="math inline">\(m\)</span> genes and <span class="math inline">\(c\)</span> experimental conditions and a prior knowledge matrix <span class="math inline">\(\mathbf{C} \in \{0,1\}^{m \times p}\)</span> for <span class="math inline">\(p\)</span> MSigDB pathways <span class="citation" data-cites="1CbVoEpNJ">[<a href="#ref-1CbVoEpNJ" role="doc-biblioref">100</a>]</span> (so that <span class="math inline">\(\mathbf{C}_{ij} = 1\)</span> if gene <span class="math inline">\(i\)</span> belongs to pathway <span class="math inline">\(j\)</span>), PLIER finds <span class="math inline">\(\mathbf{U}\)</span>, <span class="math inline">\(\mathbf{Z}\)</span>, and <span class="math inline">\(\mathbf{B}\)</span> minimizing</p>
<p><span id="eq:met:plier_func" class="eqnos"><span class="math display">\[
||\mathbf{Y} - \mathbf{Z}\mathbf{B}||^{2}_{F} + \lambda_1 ||\mathbf{Z} - \mathbf{C}\mathbf{U}||^{2}_{F} + \lambda_2 ||\mathbf{B}||^{2}_{F} + \lambda_3 ||\mathbf{U}||_{L^1}
\]</span><span class="eqnos-number">(5)</span></span> </p>
<p>subject to <span class="math inline">\(\mathbf{U}&gt;0, \mathbf{Z}&gt;0\)</span>;
<span class="math inline">\(\mathbf{Z}^{m \times l}\)</span> are the gene loadings with <span class="math inline">\(l\)</span> latent variables,
<span class="math inline">\(\mathbf{B}^{l \times c}\)</span> is the latent space for <span class="math inline">\(c\)</span> conditions,
<span class="math inline">\(\mathbf{U}^{p \times l}\)</span> specifies which of the <span class="math inline">\(p\)</span> prior-information pathways in <span class="math inline">\(\mathbf{C}\)</span> are represented for each LV,
and <span class="math inline">\(\lambda_i\)</span> are different regularization parameters used in the training step.
<span class="math inline">\(\mathbf{Z}\)</span> is a low-dimensional representation of the gene space where each LV aligns as much as possible to prior knowledge, and it might represent either a known or novel gene module (i.e., a meaningful biological pattern) or noise.</p>
<p>For our drug repurposing and cluster analyses, we used this model to project gene-trait (from TWAS) and gene-drug associations (from LINCS L1000) into this low-dimensional gene module space.
For instance, TWAS associations <span class="math inline">\(\mathbf{M}\)</span> (either from S-PrediXcan or S-MultiXcan) were projected using</p>
<p><span id="eq:proj" class="eqnos"><span class="math display">\[
\hat{\mathbf{M}} = (\mathbf{Z}^{\top} \mathbf{Z} + \lambda_{2} \mathbf{I})^{-1} \mathbf{Z}^{\top} \mathbf{M},
\]</span><span class="eqnos-number">(6)</span></span> </p>
<p>where <span class="math inline">\(\hat{\mathbf{M}}^{l \times q}\)</span> is a matrix where traits are represented by gene modules instead of single genes.
As explained later, we used the same approach to project drug-induced transcriptional profiles in LINCS L1000 to obtain a representation of drugs using gene modules.</p>
<h3 id="sec:methods:reg">Regression model for LV-trait associations</h3>
<p>We adapted the gene-set analysis framework from MAGMA <span class="citation" data-cites="19XiXgYmd">[<a href="#ref-19XiXgYmd" role="doc-biblioref">101</a>]</span> to TWAS.
We used a competitive test to predict gene-trait associations from TWAS using gene weights from an LV, testing whether top-weighted genes for an LV are more strongly associated with the phenotype than other genes with relatively small or zero weights.
Thus, we fit the model</p>
<p><span id="eq:reg:model" class="eqnos"><span class="math display">\[
\mathbf{m}=\beta_{0} + \mathbf{s} \beta_{s} + \sum_{i} \mathbf{x}_{i} \beta_{i} + \bm{\epsilon},
\]</span><span class="eqnos-number">(7)</span></span> </p>
<p>where <span class="math inline">\(\mathbf{m}\)</span> is a vector of S-MultiXcan gene <span class="math inline">\(p\)</span>-values for a trait (with a <span class="math inline">\(-log_{10}\)</span> transformation);
<span class="math inline">\(\mathbf{s}\)</span> is a binary indicator vector with <span class="math inline">\(s_{\ell}=1\)</span> for the top 1% of genes with the largest loadings for LV <span class="math inline">\(\ell\)</span> (from <span class="math inline">\(\mathbf{Z}_{\ell}\)</span>) and zero otherwise;
<span class="math inline">\(\mathbf{x}_{i}\)</span> is a gene property used as a covariate;
<span class="math inline">\(\beta\)</span> are effect sizes (with <span class="math inline">\(\beta_{0}\)</span> as the intercept);
and <span class="math inline">\(\bm{\epsilon} \sim \mathrm{MVN}(0, \sigma^{2} \mathbf{R})\)</span> is a vector of error terms with a multivariate normal distribution (MVN) where <span class="math inline">\(\mathbf{R}\)</span> is the matrix of gene correlations.</p>
<p>The model tests the null hypothesis <span class="math inline">\(\beta_{s} = 0\)</span> against the one-sided hypothesis <span class="math inline">\(\beta_{s} &gt; 0\)</span>.
Therefore, <span class="math inline">\(\beta_{s}\)</span> reflects the difference in trait associations between genes that are part of LV <span class="math inline">\(\ell\)</span> and genes outside of it.
Following the MAGMA framework, we used two gene properties as covariates:
1) <em>gene size</em>, defined as the number of PCs retained in S-MultiXcan,
and 2) <em>gene density</em>, defined as the ratio of the number of PCs to the number of tissues available.</p>
<p>Since the error terms <span class="math inline">\(\bm{\epsilon}\)</span> could be correlated, we cannot assume they have independent normal distributions as in a standard linear regression model.
In the PrediXcan family of methods, the predicted expression of a pair of genes could be correlated if they share eQTLs or if these are in LD <span class="citation" data-cites="l6ogswV3">[<a href="#ref-l6ogswV3" role="doc-biblioref">95</a>]</span>.
Therefore, we used a generalized least squares approach to account for these correlations.
The gene-gene correlation matrix <span class="math inline">\(\mathbf{R}\)</span> was approximated by computing the correlations between the model sum of squares (SSM) for each pair of genes under the null hypothesis of no association.
These correlations are derived from the individual-level MultiXcan model (Equation (<a href="#eq:multixcan">3</a>)), where the predicted expression matrix <span class="math inline">\(\mathbf{T}_{i} \in \mathbb{R}^{n \times p_i}\)</span> of a gene <span class="math inline">\(i\)</span> across <span class="math inline">\(p_i\)</span> tissues is projected into its top <span class="math inline">\(k_i\)</span> PCs, resulting in matrix <span class="math inline">\(\mathbf{P}_{i} \in \mathbb{R}^{n \times k_i}\)</span>.
From the MAGMA framework, we know that the SSM for each gene is proportial to <span class="math inline">\(\mathbf{y}^{\top} \mathbf{P}_{i} \mathbf{P}_{i}^{\top} \mathbf{y}\)</span>.
Under the null hypothesis of no association, the covariances between the SSM of genes <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> is therefore given by <span class="math inline">\(2 \times \mathrm{Trace}(\mathbf{P}_{i}^{\top} \mathbf{P}_{j} \mathbf{P}_{j}^{\top} \mathbf{P}_{i})\)</span>.
The standard deviations of each SSM are given by <span class="math inline">\(\sqrt{2 \times k_{i}} \times (n - 1)\)</span>.
Therefore, the correlation between the SSMs for genes <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> can be written as follows:</p>
<p><span id="eq:reg:r" class="eqnos"><span class="math display">\[
\begin{split}
\mathbf{R}_{ij} &amp; = \frac{2 \times \mathrm{Tr}(\mathbf{P}_{i}^{\top} \mathbf{P}_{j} \mathbf{P}_{j}^{\top} \mathbf{P}_{i})}{\sqrt{2 \times k_{i}} \times \sqrt{2 \times k_{j}} \times (n - 1)^2} \\
&amp; = \frac{2 \times \mathrm{Tr}(Cor(\mathbf{P}_{i}, \mathbf{P}_{j}) \times Cor(\mathbf{P}_{j}, \mathbf{P}_{i}))}{\sqrt{2 \times k_{i}} \times \sqrt{2 \times k_{j}}},
\end{split}
\]</span><span class="eqnos-number">(8)</span></span> </p>
<p>where columns <span class="math inline">\(\mathbf{P}\)</span> are standardized,
<span class="math inline">\(\mathrm{Tr}\)</span> is the trace of a matrix,
and the cross-correlation matrix between PCs <span class="math inline">\(Cor(\mathbf{P}_{i}, \mathbf{P}_{j}) \in \mathbb{R}^{k_i \times k_j}\)</span> is given by</p>
<p><span id="eq:reg:cor_pp" class="eqnos"><span class="math display">\[
\begin{split}
Cor(\mathbf{P}_{i}, \mathbf{P}_{j}) &amp; = Cor(\mathbf{T}_{i} \mathbf{V}_{i}^{\top} \mathrm{diag}(\lambda_i)^{-1/2}, \mathbf{T}_{j} \mathbf{V}_{j}^{\top} \mathrm{diag}(\lambda_j)^{-1/2}) \\
&amp; = \mathrm{diag}(\lambda_i)^{-1/2} \mathbf{V}_{i} (\frac{\mathbf{T}_{i}^{\top} \mathbf{T}_{j}}{n-1}) \mathbf{V}_{j}^{\top} \mathrm{diag}(\lambda_j)^{-1/2},
\end{split}
\]</span><span class="eqnos-number">(9)</span></span> </p>
<p>where <span class="math inline">\(\frac{\mathbf{T}_{i}^{\top} \mathbf{T}_{j}}{n-1} \in \mathbb{R}^{p_i \times p_j}\)</span> is the cross-correlation matrix between the predicted expression levels of genes <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span>,
and columns of <span class="math inline">\(\mathbf{V}_{i}\)</span> and scalars <span class="math inline">\(\lambda_i\)</span> are the eigenvectors and eigenvalues of <span class="math inline">\(\mathbf{T}_{i}\)</span>, respectively.
S-MultiXcan keeps only the top eigenvectors using a condition number threshold of <span class="math inline">\(\frac{\max(\lambda_i)}{\lambda_i} &lt; 30\)</span>.
To estimate the correlation of predicted expression levels for genes <span class="math inline">\(i\)</span> in tissue <span class="math inline">\(k\)</span> and gene <span class="math inline">\(j\)</span> in tissue <span class="math inline">\(l\)</span>, <span class="math inline">\((\mathbf{t}_k^i, \mathbf{t}_l^j)\)</span> (<span class="math inline">\(\mathbf{t}_k^i\)</span> is the <span class="math inline">\(k\)</span>th column of <span class="math inline">\(\mathbf{T}_{i}\)</span>), we used <span class="citation" data-cites="1FFzCXo1s">[<a href="#ref-1FFzCXo1s" role="doc-biblioref">24</a>]</span></p>
<p><span id="eq:reg:corr_genes" class="eqnos"><span class="math display">\[
\begin{split}
\frac{(\mathbf{T}_{i}^{\top} \mathbf{T}_{j})_{kl}}{n-1} &amp; = Cor(\mathbf{t}_k^i, \mathbf{t}_l^j) \\
&amp; = \frac{ Cov(\mathbf{t}_k, \mathbf{t}_l) } { \sqrt{\widehat{\mathrm{var}}(\mathbf{t}_k) \widehat{\mathrm{var}}(\mathbf{t}_l)} } \\
&amp; = \frac{ Cov(\sum_{a \in \mathrm{model}_k} w_a^k X_a, \sum_{b \in \mathrm{model}_l} w_b^l X_b) }  {\sqrt{\widehat{\mathrm{var}}(\mathbf{t}_k) \widehat{\mathrm{var}}(\mathbf{t}_l)} } \\
&amp; = \frac{ \sum_{a \in \mathrm{model}_k \\ b \in \mathrm{model}_l} w_a^k w_b^l Cov(X_a, X_b)} {\sqrt{\widehat{\mathrm{var}}(\mathbf{t}_k) \widehat{\mathrm{var}}(\mathbf{t}_l)} } \\
&amp; = \frac{ \sum_{a \in \mathrm{model}_k \\ b \in \mathrm{model}_l} w_a^k w_b^l \Gamma_{ab}} {\sqrt{\widehat{\mathrm{var}}(\mathbf{t}_k) \widehat{\mathrm{var}}(\mathbf{t}_l)} },
\end{split}
\]</span><span class="eqnos-number">(10)</span></span> </p>
<p>where <span class="math inline">\(X_a\)</span> is the genotype of SNP <span class="math inline">\(a\)</span>,
<span class="math inline">\(w_a^k\)</span> is the weight of SNP <span class="math inline">\(a\)</span> for gene expression prediction in the tissue model <span class="math inline">\(k\)</span>,
and <span class="math inline">\(\Gamma = \widehat{\mathrm{var}}(\mathbf{X}) = (\mathbf{X} - \mathbf{\bar{X}})^{\top} (\mathbf{X} - \mathbf{\bar{X}}) / (n-1)\)</span> is the genotype covariance matrix using GTEx v8 as the reference panel, which is the same used in all TWAS methods described here.
The variance of the predicted expression values of gene <span class="math inline">\(i\)</span> in tissue <span class="math inline">\(k\)</span> is estimated as <span class="citation" data-cites="vLyTudUB">[<a href="#ref-vLyTudUB" role="doc-biblioref">99</a>]</span>:</p>
<p><span id="eq:reg:var_gene" class="eqnos"><span class="math display">\[
\begin{split}
\widehat{\mathrm{var}}(\mathbf{t}_k^i) &amp; = (\mathbf{W}^k)^\top \Gamma^k \mathbf{W}^k \\
&amp; = \sum_{a \in \mathrm{model}_k \\ b \in \mathrm{model}_k} w_a^k w_b^k \Gamma_{ab}^k.
\end{split}
\]</span><span class="eqnos-number">(11)</span></span> </p>
<p>Note that, since we used the MultiXcan regression model (Equation (<a href="#eq:multixcan">3</a>)), <span class="math inline">\(\mathbf{R}\)</span> is only an approximation of gene correlations in S-MultiXcan.
As explained before, S-MultiXcan approximates the joint regression parameters in MultiXcan using the marginal regression estimates from S-PrediXcan in (<a href="#eq:spredixcan">2</a>) with some simplifying assumptions and different genotype covariance matrices.
This complicates the derivation of an S-MultiXcan-specific solution to compute <span class="math inline">\(\mathbf{R}\)</span>.
To account for this, we used a submatrix <span class="math inline">\(\mathbf{R}_{\ell}\)</span> corresponding to genes that are part of LV <span class="math inline">\(\ell\)</span> only (top 1% of genes) instead of the entire matrix <span class="math inline">\(\mathbf{R}\)</span>.
This simplification is conservative since correlations are accounted for top genes only.
Our simulations (<a href="#sm:reg:null_sim">Supplementary Note 1</a>) show that the model is approximately well-calibrated and can correct for LVs with adjacent and highly correlated genes at the top (e.g., Figure <a href="#fig:reg:nulls:qqplot:lv234">9</a>).
The model can also detect LVs associated with relevant traits (Figure <a href="#fig:lv246">2</a> and Table <a href="#tbl:sup:phenomexcan_assocs:lv246">8</a>) that are replicated in a different cohort (Table <a href="#tbl:sup:emerge_assocs:lv246">9</a>).</p>
<p>In Equation (<a href="#eq:reg:corr_genes">10</a>), for each gene, we only considered tissue models present in S-PrediXcan results, as well as SNPs present in GWAS used as input for the TWAS approaches.
This is necessary to obtain more accurate correlations estimates <span class="citation" data-cites="1FFzCXo1s">[<a href="#ref-1FFzCXo1s" role="doc-biblioref">24</a>]</span>.
Therefore, we computed different correlation matrices for PhenomeXcan and eMERGE.
In PhenomeXcan, most of the GWAS (4,049) were obtained from the UK Biobank using the same pipeline and including the same set of SNPs, so a single correlation matrix was used for this set.
For the rest, we used a single correlation matrix for each group of traits that shared the same or most of the SNPs.</p>
<p>We ran our regression model for all 987 LVs across the 4,091 traits in PhenomeXcan.
For replication, we ran the model in the 309 phecodes in eMERGE.
We adjusted the <span class="math inline">\(p\)</span>-values using the Benjamini-Hochberg procedure.</p>
<h3 id="sec:methods:drug">LV-based drug repurposing approach</h3>
<p>For the drug-disease prediction, we derived an LV-based method based on a drug repositioning framework previously used for psychiatry traits <span class="citation" data-cites="17oeJ0CXy">[<a href="#ref-17oeJ0CXy" role="doc-biblioref">30</a>]</span>, where individual/single genes associated with a trait are anticorrelated with expression profiles for drugs.
We compared our LV-based method with this previously published, single-gene approach.
For the single-gene method, we computed a drug-disease score by multiplying each S-PrediXcan set of signed <span class="math inline">\(z\)</span>-scores in tissue <span class="math inline">\(t\)</span>, <span class="math inline">\(\mathbf{M}^t\)</span>, with another set of signed <span class="math inline">\(z\)</span>-scores from transcriptional responses profiled in LINCS L1000 <span class="citation" data-cites="F7lIlh2N">[<a href="#ref-F7lIlh2N" role="doc-biblioref">43</a>]</span>, <span class="math inline">\(\mathbf{L}^{c \times m}\)</span> (for <span class="math inline">\(c\)</span> compounds).
Here <span class="math inline">\(\mathbf{M}^t\)</span> contains information about whether a higher or lower predicted expression of a gene is associated with disease risk, whereas <span class="math inline">\(\mathbf{L}\)</span> indicates whether a drug increases or decreases the expression of a gene.
Therefore, these two matrices can be multiplied to compute a score for a drug-disease pair.
The result of this product is <span class="math inline">\(\mathbf{D}^{t,k}=-1 \cdot \mathbf{M}^{t,k} \mathbf{L}^\top\)</span>, where <span class="math inline">\(k\)</span> refers to the number of most significant gene associations in <span class="math inline">\(\mathbf{M}^t\)</span> for each trait.
As suggested in <span class="citation" data-cites="17oeJ0CXy">[<a href="#ref-17oeJ0CXy" role="doc-biblioref">30</a>]</span>, <span class="math inline">\(k\)</span> could be either all genes or the top 50, 100, 250, and 500; then, we averaged score ranks across all <span class="math inline">\(k\)</span> and obtained <span class="math inline">\(\mathbf{D}^t\)</span>.
Finally, for each drug-disease pair, we took the maximum prediction score across all tissues: <span class="math inline">\(\mathbf{D}_{ij} = \max \{ \mathbf{D}_{ij}^t \mid \forall t \}\)</span>.</p>
<p>The same procedure was used for the LV-based approach, where we projected <span class="math inline">\(\mathbf{M}^{t}\)</span> and <span class="math inline">\(\mathbf{L}\)</span> into the gene module latent space using Equation (<a href="#eq:proj">6</a>), leading to <span class="math inline">\(\hat{\mathbf{M}}^t\)</span> and <span class="math inline">\(\hat{\mathbf{L}}^{l \times c}\)</span>, respectively.
Finally, <span class="math inline">\(\mathbf{D}^{t,k}=-1 \cdot \hat{\mathbf{L}}^{\top} \hat{\mathbf{M}}^{t,k}\)</span>, where in this case <span class="math inline">\(k\)</span> could be all LVs or the top 5, 10, 25 and 50 (since we have an order of magnitude less LVs than genes).</p>
<p>Since the gold standard of drug-disease medical indications is described with Disease Ontology IDs (DOID) <span class="citation" data-cites="1FsruosUW">[<a href="#ref-1FsruosUW" role="doc-biblioref">102</a>]</span>, we mapped PhenomeXcan traits to the Experimental Factor Ontology <span class="citation" data-cites="9okjVu3s">[<a href="#ref-9okjVu3s" role="doc-biblioref">103</a>]</span> using <span class="citation" data-cites="16RTdMKxI">[<a href="#ref-16RTdMKxI" role="doc-biblioref">104</a>]</span>, and then to DOID.</p>
<h3 id="sec:methods:clustering">Consensus clustering of traits</h3>
<p>We performed two preprocessing steps on the S-MultiXcan results before the cluster analysis.
First, we combined results in <span class="math inline">\(\mathbf{M}\)</span> (with <span class="math inline">\(p\)</span>-values converted to <span class="math inline">\(z\)</span>-scores, as described before) for traits that mapped to the same Experimental Factor Ontology (EFO) <span class="citation" data-cites="9okjVu3s">[<a href="#ref-9okjVu3s" role="doc-biblioref">103</a>]</span> term using the Stouffer’s method: <span class="math inline">\(\sum w_i M_{ij} / \sqrt{\sum w_i^2}\)</span>, where <span class="math inline">\(w_i\)</span> is a weight based on the GWAS sample size for trait <span class="math inline">\(i\)</span>, and <span class="math inline">\(M_{ij}\)</span> is the <span class="math inline">\(z\)</span>-score for gene <span class="math inline">\(j\)</span>.
Second, we divided all <span class="math inline">\(z\)</span>-scores for each trait <span class="math inline">\(i\)</span> by their sum to reduce the effect of highly polygenic traits: <span class="math inline">\(M_{ij} / \sum M_{ij}\)</span>.
Finally, we projected this data matrix using Equation (<a href="#eq:proj">6</a>), obtaining <span class="math inline">\(\hat{\mathbf{M}}\)</span> with <span class="math inline">\(n\)</span>=3,752 traits and <span class="math inline">\(l\)</span>=987 LVs as the input of our clustering pipeline.</p>
<p>A partitioning of <span class="math inline">\(\hat{\mathbf{M}}\)</span> with <span class="math inline">\(n\)</span> traits into <span class="math inline">\(k\)</span> clusters is represented as a label vector <span class="math inline">\(\pi \in \mathbb{N}^n\)</span>.
Consensus clustering approaches consist of two steps:
1) the generation of an ensemble <span class="math inline">\(\Pi\)</span> with <span class="math inline">\(r\)</span> partitions of the dataset: <span class="math inline">\(\Pi=\{\pi_1, \pi_2, \ldots, \pi_r\}\)</span>,
and 2) the combination of the ensemble into a consolidated solution defined as:</p>
<p><span id="eq:consensus:obj_func" class="eqnos"><span class="math display">\[
\pi^* = \mathrm{arg}\,\underset{\hat{\pi}}{\max} Q(\{ \lvert \mathcal{L}^i \lvert \phi(\hat{\pi}_{\mathcal{L}^i}, \pi_{i \mathcal{L}^i}) \mid i \in \{1,\ldots,r\} \}),
\]</span><span class="eqnos-number">(12)</span></span> </p>
<p>where <span class="math inline">\(\mathcal{L}^i\)</span> is a set of data indices with known cluster labels for partition <span class="math inline">\(i\)</span>,
<span class="math inline">\(\phi\colon \mathbb{N}^n \times \mathbb{N}^n \to \mathbb{R}\)</span> is a function that measures the similarity between two partitions,
and <span class="math inline">\(Q\)</span> is a measure of central tendency, such as the mean or median.
We used the adjusted Rand index (ARI) <span class="citation" data-cites="e4AuoW8N">[<a href="#ref-e4AuoW8N" role="doc-biblioref">105</a>]</span> for <span class="math inline">\(\phi\)</span> and the median for <span class="math inline">\(Q\)</span>.
To obtain <span class="math inline">\(\pi^*\)</span>, we define a consensus function <span class="math inline">\(\Gamma\colon \mathbb{N}^{n \times r} \to \mathbb{N}^n\)</span> with <span class="math inline">\(\Pi\)</span> as the input.
We used consensus functions based on the evidence accumulation clustering (EAC) paradigm <span class="citation" data-cites="cuROQDFa">[<a href="#ref-cuROQDFa" role="doc-biblioref">79</a>]</span>, where <span class="math inline">\(\Pi\)</span> is first transformed into a distance matrix
<span class="math inline">\(\mathbf{D}_{ij} = d_{ij} / r\)</span>,
where <span class="math inline">\(d_{ij}\)</span> is the number of times traits <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> were grouped in different clusters across all <span class="math inline">\(r\)</span> partitions in <span class="math inline">\(\Pi\)</span>.
Then, <span class="math inline">\(\Gamma\)</span> can be any similarity-based clustering algorithm, which is applied on <span class="math inline">\(\mathbf{D}\)</span> to derive the final partition <span class="math inline">\(\pi^*\)</span>.</p>
<p>For the ensemble generation step, we used different algorithms to create a highly diverse set of partitions (see Figure <a href="#fig:clustering:design">5</a>) since diversity is an important property for ensembles <span class="citation" data-cites="8js8Q3pF rcTMvL18 t5p3UpxZ">[<a href="#ref-8js8Q3pF" role="doc-biblioref">106</a>,<a href="#ref-rcTMvL18" role="doc-biblioref">107</a>,<a href="#ref-t5p3UpxZ" role="doc-biblioref">108</a>]</span>.
We used three data representations: the raw dataset, its projection into the top 50 principal components, and the embedding learned by UMAP <span class="citation" data-cites="157h5hA34">[<a href="#ref-157h5hA34" role="doc-biblioref">109</a>]</span> using 50 components.
For each of these, we applied five clustering algorithms covering a wide range of different assumptions on the data structure: <span class="math inline">\(k\)</span>-means <span class="citation" data-cites="MxGpAiPu">[<a href="#ref-MxGpAiPu" role="doc-biblioref">110</a>]</span>, spectral clustering <span class="citation" data-cites="x3CT24TB">[<a href="#ref-x3CT24TB" role="doc-biblioref">111</a>]</span>, a Gaussian mixture model (GMM), hierarchical clustering, and DBSCAN <span class="citation" data-cites="JrL3iQea">[<a href="#ref-JrL3iQea" role="doc-biblioref">112</a>]</span>.
For <span class="math inline">\(k\)</span>-means, spectral clustering and GMM, we specified a range of <span class="math inline">\(k\)</span> between 2 and <span class="math inline">\(\sqrt{n} \approx 60\)</span>, and for each <span class="math inline">\(k\)</span> we generated five partitions using random seeds.
For hierarchical clustering, for each <span class="math inline">\(k\)</span>, we generated four partitions using common linkage criteria: ward, complete, average and single.
For DBSCAN, we combined different ranges for parameters <span class="math inline">\(\epsilon\)</span> (the maximum distance between two data points to be considered part of the same neighborhood) and <em>minPts</em> (the minimum number of data points in a neighborhood for a data point to be considered a core point), based on the procedure in <span class="citation" data-cites="FB7XPWl6">[<a href="#ref-FB7XPWl6" role="doc-biblioref">113</a>]</span>.
Specifically, we used <em>minPts</em> values from 2 to 125.
For each data representation (raw, PCA and UMAP), we determined a plausible range of <span class="math inline">\(\epsilon\)</span> values by observing the distribution of the mean distance of the <em>minPts</em>-nearest neighbors across all data points.
Since some combinations of <em>minPts</em> and <span class="math inline">\(\epsilon\)</span> might not produce a meaningful partition (for instance, when all points are detected as noisy or only one cluster is found), we resampled partitions generated by DBSCAN to ensure an equal representation of this algorithm in the ensemble.
This procedure generated a final ensemble of 4,428 partitions of 3,752 traits.</p>
<p>Finally, we used spectral clustering on <span class="math inline">\(\mathbf{D}\)</span> to derive the final consensus partitions.
<span class="math inline">\(\mathbf{D}\)</span> was first transformed into a similarity matrix by applying an RBF kernel <span class="math inline">\(\mathrm{exp}(-\gamma \mathbf{D}^2)\)</span> using four different values for <span class="math inline">\(\gamma\)</span> that we empirically determined to work best.
Therefore, for each <span class="math inline">\(k\)</span> between 2 and 60, we derived four consensus partitions and selected the one that maximized Equation (<a href="#eq:consensus:obj_func">12</a>).
We further filtered this set of 59 solutions to keep only those with an ensemble agreement larger than the 75th percentile (Supplementary Figure <a href="#fig:sup:clustering:agreement">15</a>), leaving a total of 15 final consensus partitions shown in Figure <a href="#fig:clustering:tree">6</a>.</p>
<p>The input data in our clustering pipeline undergoes several linear and nonlinear transformations, including PCA, UMAP and the ensemble transformation using the EAC paradigm (distance matrix <span class="math inline">\(\mathbf{D}\)</span>).
Although consensus clustering has clear advantages for biological data <span class="citation" data-cites="S7LBsfcF">[<a href="#ref-S7LBsfcF" role="doc-biblioref">114</a>]</span>, this set of data transformations complicates the interpretation of results.
To circumvent this, we used a supervised learning approach to detect which gene modules/LVs are the most important for each cluster of traits (Figure <a href="#fig:clustering:design">5</a>b).
Note that we did not use this supervised model for prediction but only to learn which features (LVs) were most discriminative for each cluster.
For this, we used the highest resolution partition (<span class="math inline">\(k\)</span>=29, although any could be used) to train a decision tree model using each of the clusters as labels and the projected data <span class="math inline">\(\hat{\mathbf{M}}\)</span> as the training samples.
For each <span class="math inline">\(k\)</span>, we built a set of binary labels with the current cluster’s traits as the positive class and the rest of the traits as the negative class.
Then, we selected the LV in the root node of the trained model only if its threshold was positive and larger than one standard deviation.
Next, we removed this LV from <span class="math inline">\(\hat{\mathbf{M}}\)</span> (regardless of being previously selected or not) and trained the model again.
We repeated this procedure 20 times to extract the top 20 LVs that better discriminate traits in a cluster from the rest.</p>
<p>In <a href="#sm:clustering:null_sim">Supplementary Note 2</a>, we performed several analyses under a null hypothesis of no structure in the data to verify that the clustering results detected by this pipeline were real.</p>
<h3 id="sec:methods:crispr">CRISPR-Cas9 screening</h3>
<p><strong>Cell culture.</strong>
HepG2 cells were obtained from ATCC (ATCC® HB-8065™), and maintained in Eagle’s Minimum Essential Medium with L-Glutamine (EMEM, Cat. 112-018-101, Quality Biology) supplemented with 10% Fetal Bovine Serum (FBS, Gibco, Cat.16000-044), and 1% Pen/Strep (Gibco, Cat.15140-122).
Cells were kept at 37oC in a humidity-controlled incubator with 5% CO2, and were maintained at a density not exceeding more than 80% confluency.</p>
<p><strong>Genome-wide lentiviral pooled CRISPR-Cas9 library.</strong>
3rd lentiviral generation, Broad GPP genome-wide Human Brunello CRISPR knockout Pooled library was provided by David Root and John Doench from Addgene (Cat. 73179-LV), and was used for HepG2 cell transduction.
It consists of 76,441 sgRNAs, and targets 19,114 genes in the human genome with an average of 4 sgRNAs per gene.
Each 20nt sgRNA cassette was inserted into the lentiCRIS-PRv2 backbone between U6 promoter and gRNA scaffold.
Through cell transduction, the lentiviral vectors which encode Cas9 were used to deliver the sgRNA cassette containing plasmids into cells during cell replication.
Unsuccessful transduced cells were excluded through puromycin selection.</p>
<p><strong>Lentiviral titer determination.</strong>
No-spin lentiviral transduction was utilized for the screen.
In a Collagen-I coated 6-wells plate, approximate 2.5 M cells were seeded each well in the presence of 8ug/ml polybrene (Millipore Sigma, Cat. TR-1003 G), and a different titrated virus volume (e.g., 0, 50, 100, 200, 250, and 400ul) was assigned to each well.
EMEM complete media was added to make the final volume of 1.24ml. 16-18hrs post-transduction, virus/polybrene-containing media was removed from each well.
Cells were washed twice with 1x DPBS and replaced with fresh EMEM.
At 24h, cells in each well were trypsinized, diluted (e.g.,1:10), and seeded in pairs of wells of 6-well plates. At 60hr post-transduction, cell media in each well was replaced with fresh EMEM. 2ug/ml of puromycin (Gibco, Cat. A1113803) was added to one well out of the pair. 2-5 days after puromycin selection, or the 0 virus well treated with puromycin had no survival of cells, cells in both wells with/without puromycin were collected and counted for viability.
Percentage of Infection (PI%) was obtained by comparing the cell numbers with/without puromycin selection within each pair.
By means of Poisson’s distribution theory, when transduction efficiency (PI%) is between 30-50%, which corresponds to an MOI (Multiplicity of Infection) of ~0.35-0.70. At MOI equal to or close to 0.3, around 95% of infected cells are predicted to have only one copy of the virus.
Therefore, a volume of virus (120ul) yielding 30-40% of transduction efficiency was chosen for further large-scale viral transduction.</p>
<p><strong>Lentiviral Transduction in HepG2 Using Brunello CRISPR Knockout Pooled Library.</strong>
In order to achieve a coverage (representation) of at least 500 cells per sgRNA, and at an MOI between 0.3-0.4 to ensure 95% of infected cells get only one viral particle per cell, ~200M cells were initiated for the screen.
Transduction was carried out in a similar fashion as described above.
Briefly, 2.5M cells were seeded in each well of 14 6-well plates, along with 8ug/ml of polybrene.
A volume of 120ul of the virus was added to each experimental well. 18hrs post-transduction, virus/PB mix medium was removed, and cells in each well were collected, counted, and pooled into T175 flasks.
At 60hr post-transduction, 2ug/ml of puromycin was added to each flask.
Mediums were changed every two days with fresh EMEM, topped with 2ug/ml puromycin.
Seven days after puromycin selection, cells were collected, pooled, counted, and replated.</p>
<p><strong>Fluorescent dye staining.</strong> 9 days after puromycin selection, cells were assigned to 2 groups. 20-30M cells were collected as Unsorted Control.
The cell pellet was spun down at 500 x g for 5min at 4oC.
The dry pellet was kept at -80oC for further genomic DNA isolation.
The rest of the cells (approximately 200M) were kept in 100mm dishes and stained with a fluorescent dye (LipidSpotTM 488, Biotium, Cat. 70065-T).
In Brief, LipidSpot 488 was diluted to 1:100 with DPBS.
4ml of staining solution was used for each dish and incubated at 37oC for 30min.
Cell images were captured through fluorescent microscope EVOS for GFP signal detection (Figure <a href="#fig:sup:crispr:fig1">S1</a>).</p>
<p><strong>Fluorescence-activated cell sorting (FACS).</strong>
Cells were immediately collected into 50ml tubes (From this point on, keep cells cold), and spun at 500 x g for 5min at 4oC.
After DPBS wash, cell pellets were resuspended with FACS Sorting Buffer (1x DPBS without Ca2+/Mg2+, 2.5mM EDTA, 25mM HEPES, 1% BSA.
The solution was filter sterilized, and kept at 4oC), pi-pet gently to make single cells.
The cell solution was then filtered through a cell strainer (Falcon, Cat. 352235) and was kept on ice, protected from light.
Collected cells were sorted on FACSJazz. 100um nozzle was used for sorting. ~20% of each GFP-High and GFP-Low (Figure <a href="#fig:sup:crispr:fig2">S2</a>) were collected into 15ml tubes.
After sorting, cells were immediately spun down.
Pellets were kept at -80oC for further genomic DNA isolation.</p>
<p><strong>Genomic DNA isolation and verification.</strong>
Three conditions of Genomic DNA (Un-Sorted Control, lentiV2 GFP-High, and lentiV2 GFP-Low) were extracted using QIAamp DNA Blood Mini Kit (Qiagen, Cat.51104), followed by UV Spectroscopy (Nanodrop) to access the quality and quantity of the gDNA.
A total of 80-160ug of gDNA was isolated for each condition.
sgRNA cassette and lentiviral specific transgene in isolated gDNA were verified through PCR (Figure <a href="#fig:sup:crispr:fig3">S3</a>).</p>
<p><strong>Illumina libraries generation and sequencing.</strong>
The fragment containing sgRNA cassette was amplified using P5 /P7 primers, as indicated in <span class="citation" data-cites="vNXTnmxp">[<a href="#ref-vNXTnmxp" role="doc-biblioref">115</a>]</span>, and primer sequences were adapted from Broad Institute protocol (Figure <a href="#fig:sup:crispr:table1">S4</a>).
Stagger sequence (0-8nt) was included in P5 and 8bp uniquely barcoded sequence in P7.
Primers were synthesized through Integrated DNA Technologies (IDT), and each primer was PAGE purified. 32 PCR reactions were set up for each condition.
Each 100ul PCR reaction consists of roughly 5ug of gDNA, 5ul of each 10uM P5 and P7. ExTaq DNA Polymerase (TaKaRa, Cat. RR001A) was used to amplify the amplicon.
PCR Thermal Cycler Parameters set as Initial at 95oC for 1min; followed by 24 cycles of Denaturation at 94oC for 30 seconds, Annealing at 52.5oC for 30 seconds, Extension at 72oC for 30 seconds.
A final Elongation at 72oC for 10 minutes. 285bp-293bp PCR products were expected (Figure <a href="#fig:sup:crispr:fig4">S5</a> A).
PCR products within the same condition were pooled and purified using SPRIselect beads (Beckman Coulter, Cat. B23318).
Purified Illumina libraries were quantitated on Qubit, and the quality of the library was analyzed on Bio-analyzer using High Sensitivity DNA Chip.
A single approximate 285bp peak was expected. (Figure <a href="#fig:sup:crispr:fig4">S5</a> B).
Final Illumina library samples were sequenced on Nova-seq 6000.
Samples were pooled and loaded on an SP flow cell, along with a 20% PhiX control v3 library spike-in.</p>
<h3 id="code-and-data-availability">Code and data availability</h3>
<p>The code and data to reproduce all the analyses in this work are available in <a href="https://github.com/greenelab/phenoplier">https://github.com/greenelab/phenoplier</a>.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-e0tRKjE5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>A large-scale analysis of tissue-specific pathology and gene expression of human disease genes and complexes.</strong> <div class="csl-block">Kasper Lage, Niclas Tue Hansen, EOlof Karlberg, Aron C Eklund, Francisco S Roque, Patricia K Donahoe, Zoltan Szallasi, Thomas Skøt Jensen, Søren Brunak</div> <em>Proceedings of the National Academy of Sciences of the United States of America</em> (2008-12-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19104045">https://www.ncbi.nlm.nih.gov/pubmed/19104045</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.0810772105">10.1073/pnas.0810772105</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19104045">19104045</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2606902">PMC2606902</a></div></div>
</div>
<div id="ref-CVF61Un5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>Understanding multicellular function and disease with human tissue-specific networks</strong> <div class="csl-block">Casey S Greene, Arjun Krishnan, Aaron K Wong, Emanuela Ricciotti, Rene A Zelaya, Daniel S Himmelstein, Ran Zhang, Boris M Hartmann, Elena Zaslavsky, Stuart C Sealfon, … Olga G Troyanskaya</div> <em>Nature Genetics</em> (2015-04-27) <a href="https://doi.org/f7dvkv">https://doi.org/f7dvkv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ng.3259">10.1038/ng.3259</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25915600">25915600</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4828725">PMC4828725</a></div></div>
</div>
<div id="ref-Om8ZhS06" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>Conditional and interaction gene-set analysis reveals novel functional pathways for blood pressure</strong> <div class="csl-block">Christiaan A de Leeuw, Sven Stringer, Ilona A Dekkers, Tom Heskes, Danielle Posthuma</div> <em>Nature Communications</em> (2018-09-14) <a href="https://doi.org/gd6d85">https://doi.org/gd6d85</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-018-06022-6">10.1038/s41467-018-06022-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30218068">30218068</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6138636">PMC6138636</a></div></div>
</div>
<div id="ref-9Pr9idng" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>The GTEx Consortium atlas of genetic regulatory effects across human tissues</strong><div class="csl-block">, François Aguet, Shankara Anand, Kristin G Ardlie, Stacey Gabriel, Gad A Getz, Aaron Graubert, Kane Hadley, Robert E Handsaker, Katherine H Huang, … Simona Volpi</div> <em>Science</em> (2020-09-11) <a href="https://doi.org/ghbnhr">https://doi.org/ghbnhr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.aaz1776">10.1126/science.aaz1776</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32913098">32913098</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7737656">PMC7737656</a></div></div>
</div>
<div id="ref-1DoyZS7y0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>Index and biological spectrum of human DNase I hypersensitive sites</strong> <div class="csl-block">Wouter Meuleman, Alexander Muratov, Eric Rynes, Jessica Halow, Kristen Lee, Daniel Bates, Morgan Diegel, Douglas Dunn, Fidencio Neri, Athanasios Teodosiadis, … John Stamatoyannopoulos</div> <em>Nature</em> (2020-07-29) <a href="https://doi.org/gg6dhp">https://doi.org/gg6dhp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-020-2559-3">10.1038/s41586-020-2559-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32728217">32728217</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7422677">PMC7422677</a></div></div>
</div>
<div id="ref-nhaocxmR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>Mechanisms of tissue and cell-type specificity in heritable traits and diseases</strong> <div class="csl-block">Idan Hekselman, Esti Yeger-Lotem</div> <em>Nature Reviews Genetics</em> (2020-01-08) <a href="https://doi.org/ggkx9v">https://doi.org/ggkx9v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41576-019-0200-9">10.1038/s41576-019-0200-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31913361">31913361</a></div></div>
</div>
<div id="ref-REXpV7nA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>The support of human genetic evidence for approved drug indications</strong> <div class="csl-block">Matthew R Nelson, Hannah Tipney, Jeffery L Painter, Judong Shen, Paola Nicoletti, Yufeng Shen, Aris Floratos, Pak Chung Sham, Mulin Jun Li, Junwen Wang, … Philippe Sanseau</div> <em>Nature Genetics</em> (2015-06-29) <a href="https://doi.org/f3mn52">https://doi.org/f3mn52</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ng.3314">10.1038/ng.3314</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26121088">26121088</a></div></div>
</div>
<div id="ref-PgEwSS4Q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>Are drug targets with genetic support twice as likely to be approved? Revised estimates of the impact of genetic support for drug mechanisms on the probability of drug approval</strong> <div class="csl-block">Emily A King, JWade Davis, Jacob F Degner</div> <em>PLOS Genetics</em> (2019-12-12) <a href="https://doi.org/gg957r">https://doi.org/gg957r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pgen.1008489">10.1371/journal.pgen.1008489</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31830040">31830040</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6907751">PMC6907751</a></div></div>
</div>
<div id="ref-15J98V2qM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>An integrated encyclopedia of DNA elements in the human genome</strong> <em>Nature</em> (2012-09) <a href="https://doi.org/bg9d">https://doi.org/bg9d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature11247">10.1038/nature11247</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22955616">22955616</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3439153">PMC3439153</a></div></div>
</div>
<div id="ref-sLkFMFZj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Integrative analysis of 111 reference human epigenomes</strong> <div class="csl-block">Anshul Kundaje, Wouter Meuleman, Jason Ernst, Misha Bilenky, Angela Yen, Alireza Heravi-Moussavi, Pouya Kheradpour, Zhizhuo Zhang, Jianrong Wang, … Manolis Kellis</div> <em>Nature</em> (2015-02-18) <a href="https://doi.org/f62jpn">https://doi.org/f62jpn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature14248">10.1038/nature14248</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25693563">25693563</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4530010">PMC4530010</a></div></div>
</div>
<div id="ref-SxuuTQTQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>An atlas of active enhancers across human cell types and tissues</strong> <div class="csl-block">Robin Andersson, Claudia Gebhard, Irene Miguel-Escalada, Ilka Hoof, Jette Bornholdt, Mette Boyd, Yun Chen, Xiaobei Zhao, Christian Schmidl, … Albin Sandelin</div> <em>Nature</em> (2014-03) <a href="https://doi.org/r35">https://doi.org/r35</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature12787">10.1038/nature12787</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24670763">24670763</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5215096">PMC5215096</a></div></div>
</div>
<div id="ref-xRGqPsT2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>Regulatory genomic circuitry of human disease loci by integrative epigenomics</strong> <div class="csl-block">Carles A Boix, Benjamin T James, Yongjin P Park, Wouter Meuleman, Manolis Kellis</div> <em>Nature</em> (2021-02-03) <a href="https://doi.org/ghzkhr">https://doi.org/ghzkhr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-020-03145-z">10.1038/s41586-020-03145-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33536621">33536621</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7875769">PMC7875769</a></div></div>
</div>
<div id="ref-WFslDIWl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types</strong> <div class="csl-block">Hilary K Finucane, Yakir A Reshef, Verneri Anttila, Kamil Slowikowski, Alexander Gusev, Andrea Byrnes, Steven Gazal, Po-Ru Loh, Caleb Lareau, Noam Shoresh, … </div> <em>Nature Genetics</em> (2018-04) <a href="https://doi.org/gdfjqt">https://doi.org/gdfjqt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41588-018-0081-4">10.1038/s41588-018-0081-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29632380">29632380</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5896795">PMC5896795</a></div></div>
</div>
<div id="ref-aIyQY5ZT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>The Post-GWAS Era: From Association to Function</strong> <div class="csl-block">Michael D Gallagher, Alice S Chen-Plotkin</div> <em>The American Journal of Human Genetics</em> (2018-05) <a href="https://doi.org/gdmftd">https://doi.org/gdmftd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ajhg.2018.04.002">10.1016/j.ajhg.2018.04.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29727686">29727686</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5986732">PMC5986732</a></div></div>
</div>
<div id="ref-z8MQTAnJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>Biological interpretation of genome-wide association studies using predicted gene functions</strong> <div class="csl-block">Tune H Pers, Juha M Karjalainen, Yingleong Chan, Harm-Jan Westra, Andrew R Wood, Jian Yang, Julian C Lui, Sailaja Vedantam, Stefan Gustafsson, … Lude Franke</div> <em>Nature Communications</em> (2015-01-19) <a href="https://doi.org/f3mwhd">https://doi.org/f3mwhd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ncomms6890">10.1038/ncomms6890</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25597830">25597830</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4420238">PMC4420238</a></div></div>
</div>
<div id="ref-O0e3EhY6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Relaxed purifying selection and possibly high rate of adaptation in primate lineage-specific genes.</strong> <div class="csl-block">James J Cai, Dmitri A Petrov</div> <em>Genome biology and evolution</em> (2010-07-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20624743">https://www.ncbi.nlm.nih.gov/pubmed/20624743</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/gbe/evq019">10.1093/gbe/evq019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20624743">20624743</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2997544">PMC2997544</a></div></div>
</div>
<div id="ref-18jYvPauB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>Elevated rates of protein secretion, evolution, and disease among tissue-specific genes.</strong> <div class="csl-block">Eitan E Winter, Leo Goodstadt, Chris P Ponting</div> <em>Genome research</em> (2004-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14707169">https://www.ncbi.nlm.nih.gov/pubmed/14707169</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/gr.1924004">10.1101/gr.1924004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14707169">14707169</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC314278">PMC314278</a></div></div>
</div>
<div id="ref-wNE0EQlN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>A large-scale analysis of tissue-specific pathology and gene expression of human disease genes and complexes</strong> <div class="csl-block">Kasper Lage, Niclas Tue Hansen, EOlof Karlberg, Aron C Eklund, Francisco S Roque, Patricia K Donahoe, Zoltan Szallasi, Thomas Skøt Jensen, Søren Brunak</div> <em>Proceedings of the National Academy of Sciences</em> (2008-12-30) <a href="https://doi.org/d5qcv9">https://doi.org/d5qcv9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.0810772105">10.1073/pnas.0810772105</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19104045">19104045</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2606902">PMC2606902</a></div></div>
</div>
<div id="ref-6SPTvFXq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>Reproducible RNA-seq analysis using recount2</strong> <div class="csl-block">Leonardo Collado-Torres, Abhinav Nellore, Kai Kammers, Shannon E Ellis, Margaret A Taub, Kasper D Hansen, Andrew E Jaffe, Ben Langmead, Jeffrey T Leek</div> <em>Nature Biotechnology</em> (2017-04) <a href="https://doi.org/gf75hp">https://doi.org/gf75hp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt.3838">10.1038/nbt.3838</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28398307">28398307</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6742427">PMC6742427</a></div></div>
</div>
<div id="ref-Nz3IMEzd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Massive mining of publicly available RNA-seq data from human and mouse</strong> <div class="csl-block">Alexander Lachmann, Denis Torre, Alexandra B Keenan, Kathleen M Jagodnik, Hoyjin J Lee, Lily Wang, Moshe C Silverstein, Avi Ma’ayan</div> <em>Nature Communications</em> (2018-04-10) <a href="https://doi.org/gc92dr">https://doi.org/gc92dr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-018-03751-6">10.1038/s41467-018-03751-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29636450">29636450</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5893633">PMC5893633</a></div></div>
</div>
<div id="ref-57TjOMEA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>Identification of therapeutic targets from genetic association studies using hierarchical component analysis</strong> <div class="csl-block">Hao-Chih Lee, Osamu Ichikawa, Benjamin S Glicksberg, Aparna A Divaraniya, Christine E Becker, Pankaj Agarwal, Joel T Dudley</div> <em>BioData Mining</em> (2020-06-17) <a href="https://doi.org/gjp5pf">https://doi.org/gjp5pf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s13040-020-00216-9">10.1186/s13040-020-00216-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32565911">32565911</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7301559">PMC7301559</a></div></div>
</div>
<div id="ref-yEdRP9Xx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>Novel Variance-Component TWAS method for studying complex human diseases with applications to Alzheimer’s dementia</strong> <div class="csl-block">Shizhen Tang, Aron S Buchman, Philip L De Jager, David A Bennett, Michael P Epstein, Jingjing Yang</div> <em>PLOS Genetics</em> (2021-04-02) <a href="https://doi.org/gjpr3j">https://doi.org/gjpr3j</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pgen.1009482">10.1371/journal.pgen.1009482</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33798195">33798195</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8046351">PMC8046351</a></div></div>
</div>
<div id="ref-1D63fEEPb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>Integrative approaches for large-scale transcriptome-wide association studies</strong> <div class="csl-block">Alexander Gusev, Arthur Ko, Huwenbo Shi, Gaurav Bhatia, Wonil Chung, Brenda WJH Penninx, Rick Jansen, Eco JC de Geus, Dorret I Boomsma, Fred A Wright, … Bogdan Pasaniuc</div> <em>Nature Genetics</em> (2016-02-08) <a href="https://doi.org/f3vf4p">https://doi.org/f3vf4p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ng.3506">10.1038/ng.3506</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26854917">26854917</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4767558">PMC4767558</a></div></div>
</div>
<div id="ref-1FFzCXo1s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>Integrating predicted transcriptome from multiple tissues improves association detection</strong> <div class="csl-block">Alvaro N Barbeira, Milton Pividori, Jiamao Zheng, Heather E Wheeler, Dan L Nicolae, Hae Kyung Im</div> <em>PLOS Genetics</em> (2019-01-22) <a href="https://doi.org/ghs8vx">https://doi.org/ghs8vx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pgen.1007889">10.1371/journal.pgen.1007889</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30668570">30668570</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6358100">PMC6358100</a></div></div>
</div>
<div id="ref-Z8bvDdVq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>A gene-based association method for mapping traits using reference transcriptome data</strong> <div class="csl-block">Eric R Gamazon, Heather E Wheeler, Kaanan P Shah, Sahar V Mozaffari, Keston Aquino-Michaels, Robert J Carroll, Anne E Eyler, Joshua C Denny, Dan L Nicolae, Nancy J Cox, … </div> <em>Nature Genetics</em> (2015-08-10) <a href="https://doi.org/f7p9zv">https://doi.org/f7p9zv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ng.3367">10.1038/ng.3367</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26258848">26258848</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4552594">PMC4552594</a></div></div>
</div>
<div id="ref-ktVcsmYD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><strong>UTMOST, a single and cross-tissue TWAS (Transcriptome Wide Association Study), reveals new ASD (Autism Spectrum Disorder) associated genes.</strong> <div class="csl-block">Cristina Rodriguez-Fontenla, Angel Carracedo</div> <em>Translational psychiatry</em> (2021-04-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33931583">https://www.ncbi.nlm.nih.gov/pubmed/33931583</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41398-021-01378-8">10.1038/s41398-021-01378-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33931583">33931583</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8087708">PMC8087708</a></div></div>
</div>
<div id="ref-gZAOkumx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>Multi-ancestry gene-trait connection landscape using electronic health record (EHR) linked biobank data</strong> <div class="csl-block">Binglan Li, Yogasudha Veturi, Anastasia Lucas, Yuki Bradford, Shefali S Verma, Anurag Verma, Joseph Park, Wei-Qi Wei, Qiping Feng, Bahram Namjou, … Marylyn D Ritchie</div> <em>Cold Spring Harbor Laboratory</em> (2021-10-26) <a href="https://doi.org/gnbdnb">https://doi.org/gnbdnb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2021.10.21.21265225">10.1101/2021.10.21.21265225</a></div></div>
</div>
<div id="ref-zwpq2IXD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>Shared and distinct genetic risk factors for childhood-onset and adult-onset asthma: genome-wide and transcriptome-wide studies.</strong> <div class="csl-block">Milton Pividori, Nathan Schoettler, Dan L Nicolae, Carole Ober, Hae Kyung Im</div> <em>The Lancet. Respiratory medicine</em> (2019-04-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31036433">https://www.ncbi.nlm.nih.gov/pubmed/31036433</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s2213-2600(19)30055-4">10.1016/s2213-2600(19)30055-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31036433">31036433</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6534440">PMC6534440</a></div></div>
</div>
<div id="ref-hSYqogYZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>Polygenic transcriptome risk scores (PTRS) can improve portability of polygenic risk scores across ancestries</strong> <div class="csl-block">Yanyu Liang, Milton Pividori, Ani Manichaikul, Abraham A Palmer, Nancy J Cox, Heather E Wheeler, Hae Kyung Im</div> <em>Genome Biology</em> (2022-01-13) <a href="https://doi.org/gqtdvn">https://doi.org/gqtdvn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s13059-021-02591-w">10.1186/s13059-021-02591-w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35027082">35027082</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8759285">PMC8759285</a></div></div>
</div>
<div id="ref-17oeJ0CXy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>Analysis of genome-wide association data highlights candidates for drug repositioning in psychiatry</strong> <div class="csl-block">Hon-Cheong So, Carlos Kwan-Long Chau, Wan-To Chiu, Kin-Sang Ho, Cho-Pong Lo, Stephanie Ho-Yue Yim, Pak-Chung Sham</div> <em>Nature Neuroscience</em> (2017-08-14) <a href="https://doi.org/gbrssh">https://doi.org/gbrssh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nn.4618">10.1038/nn.4618</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28805813">28805813</a></div></div>
</div>
<div id="ref-vpIDZCSa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>An Expanded View of Complex Traits: From Polygenic to Omnigenic</strong> <div class="csl-block">Evan A Boyle, Yang I Li, Jonathan K Pritchard</div> <em>Cell</em> (2017-06) <a href="https://doi.org/gcpgdz">https://doi.org/gcpgdz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2017.05.038">10.1016/j.cell.2017.05.038</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28622505">28622505</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536862">PMC5536862</a></div></div>
</div>
<div id="ref-pZZn28he" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>A global overview of pleiotropy and genetic architecture in complex traits</strong> <div class="csl-block">Kyoko Watanabe, Sven Stringer, Oleksandr Frei, Maša Umićević Mirkov, Christiaan de Leeuw, Tinca JC Polderman, Sophie van der Sluis, Ole A Andreassen, Benjamin M Neale, Danielle Posthuma</div> <em>Nature Genetics</em> (2019-08-19) <a href="https://doi.org/ggr84r">https://doi.org/ggr84r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41588-019-0481-0">10.1038/s41588-019-0481-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31427789">31427789</a></div></div>
</div>
<div id="ref-PDWEwciL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>Detection and interpretation of shared genetic influences on 42 human traits</strong> <div class="csl-block">Joseph K Pickrell, Tomaz Berisa, Jimmy Z Liu, Laure Ségurel, Joyce Y Tung, David A Hinds</div> <em>Nature Genetics</em> (2016-05-16) <a href="https://doi.org/f8ssw4">https://doi.org/f8ssw4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ng.3570">10.1038/ng.3570</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27182965">27182965</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5207801">PMC5207801</a></div></div>
</div>
<div id="ref-LXvTZzEA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>Trans Effects on Gene Expression Can Drive Omnigenic Inheritance</strong> <div class="csl-block">Xuanyao Liu, Yang I Li, Jonathan K Pritchard</div> <em>Cell</em> (2019-05) <a href="https://doi.org/gfz8bj">https://doi.org/gfz8bj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2019.04.014">10.1016/j.cell.2019.04.014</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31051098">31051098</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6553491">PMC6553491</a></div></div>
</div>
<div id="ref-jrAMOJCD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>Architecture of the human regulatory network derived from ENCODE data.</strong> <div class="csl-block">Mark B Gerstein, Anshul Kundaje, Manoj Hariharan, Stephen G Landt, Koon-Kiu Yan, Chao Cheng, Xinmeng Jasmine Mu, Ekta Khurana, Joel Rozowsky, Roger Alexander, … Michael Snyder</div> <em>Nature</em> (2012-09-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22955619">https://www.ncbi.nlm.nih.gov/pubmed/22955619</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature11245">10.1038/nature11245</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22955619">22955619</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4154057">PMC4154057</a></div></div>
</div>
<div id="ref-SjGoBywE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><strong>A narrow repertoire of transcriptional modules responsive to pyogenic bacteria is impaired in patients carrying loss-of-function mutations in MYD88 or IRAK4.</strong> <div class="csl-block">Laia Alsina, Elisabeth Israelsson, Matthew C Altman, Kristen K Dang, Pegah Ghandil, Laura Israel, Horst von Bernuth, Nicole Baldwin, Huanying Qin, Zongbo Jin, … Damien Chaussabel</div> <em>Nature immunology</em> (2014-10-26) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25344726">https://www.ncbi.nlm.nih.gov/pubmed/25344726</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ni.3028">10.1038/ni.3028</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25344726">25344726</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4281021">PMC4281021</a></div></div>
</div>
<div id="ref-S4e4WaP3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline"><strong>Cluster analysis and display of genome-wide expression patterns.</strong> <div class="csl-block">MB Eisen, PT Spellman, PO Brown, D Botstein</div> <em>Proceedings of the National Academy of Sciences of the United States of America</em> (1998-12-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9843981">https://www.ncbi.nlm.nih.gov/pubmed/9843981</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.95.25.14863">10.1073/pnas.95.25.14863</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9843981">9843981</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC24541">PMC24541</a></div></div>
</div>
<div id="ref-f2r8LLIn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline"><strong>Democratizing systems immunology with modular transcriptional repertoire analyses.</strong> <div class="csl-block">Damien Chaussabel, Nicole Baldwin</div> <em>Nature reviews. Immunology</em> (2014-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24662387">https://www.ncbi.nlm.nih.gov/pubmed/24662387</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nri3642">10.1038/nri3642</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24662387">24662387</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4118927">PMC4118927</a></div></div>
</div>
<div id="ref-VzZoy0BD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline"><strong>How does gene expression clustering work?</strong> <div class="csl-block">Patrik D'haeseleer</div> <em>Nature biotechnology</em> (2005-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16333293">https://www.ncbi.nlm.nih.gov/pubmed/16333293</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt1205-1499">10.1038/nbt1205-1499</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16333293">16333293</a></div></div>
</div>
<div id="ref-1BVbSrr6M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline"><strong>A comprehensive evaluation of module detection methods for gene expression data</strong> <div class="csl-block">Wouter Saelens, Robrecht Cannoodt, Yvan Saeys</div> <em>Nature Communications</em> (2018-03-15) <a href="https://doi.org/gc9x36">https://doi.org/gc9x36</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-018-03424-4">10.1038/s41467-018-03424-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29545622">29545622</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5854612">PMC5854612</a></div></div>
</div>
<div id="ref-S56q1qoc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline"><strong>A modular analysis framework for blood genomics studies: application to systemic lupus erythematosus.</strong> <div class="csl-block">Damien Chaussabel, Charles Quinn, Jing Shen, Pinakeen Patel, Casey Glaser, Nicole Baldwin, Dorothee Stichweh, Derek Blankenship, Lei Li, Indira Munagala, … Virginia Pascual</div> <em>Immunity</em> (2008-07-18) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18631455">https://www.ncbi.nlm.nih.gov/pubmed/18631455</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.immuni.2008.05.012">10.1016/j.immuni.2008.05.012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18631455">18631455</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2727981">PMC2727981</a></div></div>
</div>
<div id="ref-lY5ln3dB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline"><strong>PhenomeXcan: Mapping the genome to the phenome through the transcriptome</strong> <div class="csl-block">Milton Pividori, Padma S Rajagopal, Alvaro Barbeira, Yanyu Liang, Owen Melia, Lisa Bastarache, YoSon Park, GTEx Consortium, Xiaoquan Wen, Hae K Im</div> <em>Science Advances</em> (2020-09-11) <a href="https://doi.org/ghbvbf">https://doi.org/ghbvbf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/sciadv.aba2083">10.1126/sciadv.aba2083</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32917697">32917697</a></div></div>
</div>
<div id="ref-F7lIlh2N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline"><strong>A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles</strong> <div class="csl-block">Aravind Subramanian, Rajiv Narayan, Steven M Corsello, David D Peck, Ted E Natoli, Xiaodong Lu, Joshua Gould, John F Davis, Andrew A Tubelli, Jacob K Asiedu, … Todd R Golub</div> <em>Cell</em> (2017-11) <a href="https://doi.org/cgwt">https://doi.org/cgwt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2017.10.049">10.1016/j.cell.2017.10.049</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29195078">29195078</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5990023">PMC5990023</a></div></div>
</div>
<div id="ref-14rnBunuZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline"><strong>MultiPLIER: A Transfer Learning Framework for Transcriptomics Reveals Systemic Features of Rare Disease</strong> <div class="csl-block">Jaclyn N Taroni, Peter C Grayson, Qiwen Hu, Sean Eddy, Matthias Kretzler, Peter A Merkel, Casey S Greene</div> <em>Cell Systems</em> (2019-05) <a href="https://doi.org/gf75g5">https://doi.org/gf75g5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cels.2019.04.003">10.1016/j.cels.2019.04.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31121115">31121115</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6538307">PMC6538307</a></div></div>
</div>
<div id="ref-Ki2ij7zE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline"><strong>Pathway-level information extractor (PLIER) for gene expression data</strong> <div class="csl-block">Weiguang Mao, Elena Zaslavsky, Boris M Hartmann, Stuart C Sealfon, Maria Chikina</div> <em>Nature Methods</em> (2019-06-27) <a href="https://doi.org/gf75g6">https://doi.org/gf75g6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-019-0456-1">10.1038/s41592-019-0456-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31249421">31249421</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7262669">PMC7262669</a></div></div>
</div>
<div id="ref-wfqjCerX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline"><strong>The Electronic Medical Records and Genomics (eMERGE) Network: past, present, and future</strong> <div class="csl-block">Omri Gottesman, Helena Kuivaniemi, Gerard Tromp, WAndrew Faucett, Rongling Li, Teri A Manolio, Saskia C Sanderson, Joseph Kannry, Randi Zinberg, Melissa A Basford, … Marc S Williams</div> <em>Genetics in Medicine</em> (2013-10) <a href="https://doi.org/f5dwbt">https://doi.org/f5dwbt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/gim.2013.72">10.1038/gim.2013.72</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23743551">23743551</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3795928">PMC3795928</a></div></div>
</div>
<div id="ref-nmJxPpE5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline"><strong>The UK Biobank resource with deep phenotyping and genomic data</strong> <div class="csl-block">Clare Bycroft, Colin Freeman, Desislava Petkova, Gavin Band, Lloyd T Elliott, Kevin Sharp, Allan Motyer, Damjan Vukcevic, Olivier Delaneau, Jared O’Connell, … Jonathan Marchini</div> <em>Nature</em> (2018-10) <a href="https://doi.org/gfb7h2">https://doi.org/gfb7h2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-018-0579-z">10.1038/s41586-018-0579-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30305743">30305743</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786975">PMC6786975</a></div></div>
</div>
<div id="ref-1FVd2WW6G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline"><strong>Finding function: evaluation methods for functional genomic data</strong> <div class="csl-block">Chad L Myers, Daniel R Barrett, Matthew A Hibbs, Curtis Huttenhower, Olga G Troyanskaya</div> <em>BMC Genomics</em> (2006-07-25) <a href="https://doi.org/fg6wnk">https://doi.org/fg6wnk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2164-7-187">10.1186/1471-2164-7-187</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16869964">16869964</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1560386">PMC1560386</a></div></div>
</div>
<div id="ref-DN7TyZzb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline"><strong>The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens</strong> <div class="csl-block">Naihui Zhou, Yuxiang Jiang, Timothy R Bergquist, Alexandra J Lee, Balint Z Kacsoh, Alex W Crocker, Kimberley A Lewis, George Georghiou, Huy N Nguyen, Md Nafiz Hamid, … Iddo Friedberg</div> <em>Genome Biology</em> (2019-11-19) <a href="https://doi.org/ggnxpz">https://doi.org/ggnxpz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s13059-019-1835-8">10.1186/s13059-019-1835-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31744546">31744546</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6864930">PMC6864930</a></div></div>
</div>
<div id="ref-18TSqd1tG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline"><strong>Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression</strong> <div class="csl-block">Etienne Becht, Nicolas A Giraldo, Laetitia Lacroix, Bénédicte Buttard, Nabila Elarouci, Florent Petitprez, Janick Selves, Pierre Laurent-Puig, Catherine Sautès-Fridman, Wolf H Fridman, Aurélien de Reyniès</div> <em>Genome Biology</em> (2016-10-20) <a href="https://doi.org/f87sgf">https://doi.org/f87sgf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s13059-016-1070-5">10.1186/s13059-016-1070-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27765066">27765066</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5073889">PMC5073889</a></div></div>
</div>
<div id="ref-Z8WXLD67" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline"><strong>Fast gene set enrichment analysis</strong> <div class="csl-block">Gennady Korotkevich, Vladimir Sukhov, Nikolay Budin, Boris Shpak, Maxim N Artyomov, Alexey Sergushichev</div> <em>Cold Spring Harbor Laboratory</em> (2016-06-20) <a href="https://doi.org/gfpqhm">https://doi.org/gfpqhm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/060012">10.1101/060012</a></div></div>
</div>
<div id="ref-6PR8LEXK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline"><strong>DrugBank 4.0: shedding new light on drug metabolism</strong> <div class="csl-block">Vivian Law, Craig Knox, Yannick Djoumbou, Tim Jewison, An Chi Guo, Yifeng Liu, Adam Maciejewski, David Arndt, Michael Wilson, Vanessa Neveu, … David S Wishart</div> <em>Nucleic Acids Research</em> (2013-11-06) <a href="https://doi.org/f3mn6d">https://doi.org/f3mn6d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkt1068">10.1093/nar/gkt1068</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24203711">24203711</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3965102">PMC3965102</a></div></div>
</div>
<div id="ref-O21tn8vf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline"><strong>Systematic integration of biomedical knowledge prioritizes drugs for repurposing</strong> <div class="csl-block">Daniel Scott Himmelstein, Antoine Lizee, Christine Hessler, Leo Brueggeman, Sabrina L Chen, Dexter Hadley, Ari Green, Pouya Khankhanian, Sergio E Baranzini</div> <em>eLife</em> (2017-09-22) <a href="https://doi.org/cdfk">https://doi.org/cdfk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.26726">10.7554/elife.26726</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28936969">28936969</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640425">PMC5640425</a></div></div>
</div>
<div id="ref-1DJZvtwP1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline"><strong>Dhimmel/Lincs V2.0: Refined Consensus Signatures From Lincs L1000</strong> <div class="csl-block">Daniel Himmelstein, Leo Brueggeman, Sergio Baranzini</div> <em>Zenodo</em> (2016-03-08) <a href="https://doi.org/f3mqvr">https://doi.org/f3mqvr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.5281/zenodo.47223">10.5281/zenodo.47223</a></div></div>
</div>
<div id="ref-1ClBKizD7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline"><strong>Computational Repositioning of the Anticonvulsant Topiramate for Inflammatory Bowel Disease</strong> <div class="csl-block">Joel T Dudley, Marina Sirota, Mohan Shenoy, Reetesh K Pai, Silke Roedder, Annie P Chiang, Alex A Morgan, Minnie M Sarwal, Pankaj Jay Pasricha, Atul J Butte</div> <em>Science Translational Medicine</em> (2011-08-17) <a href="https://doi.org/bmh5ts">https://doi.org/bmh5ts</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/scitranslmed.3002648">10.1126/scitranslmed.3002648</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21849664">21849664</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3479650">PMC3479650</a></div></div>
</div>
<div id="ref-mZjkE1xU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline"><strong>Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data</strong> <div class="csl-block">Marina Sirota, Joel T Dudley, Jeewon Kim, Annie P Chiang, Alex A Morgan, Alejandro Sweet-Cordero, Julien Sage, Atul J Butte</div> <em>Science Translational Medicine</em> (2011-08-17) <a href="https://doi.org/c3fwxv">https://doi.org/c3fwxv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/scitranslmed.3001318">10.1126/scitranslmed.3001318</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21849665">21849665</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3502016">PMC3502016</a></div></div>
</div>
<div id="ref-10KA5jTBQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline"><strong>Dhimmel/Indications V1.0. Pharmacotherapydb: The Open Catalog Of Drug Therapies For Disease</strong> <div class="csl-block">Daniel S Himmelstein, Pouya Khankhanian, Christine S Hessler, Ari J Green, Sergio E Baranzini</div> <em>Zenodo</em> (2016-03-15) <a href="https://doi.org/f3mqwb">https://doi.org/f3mqwb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.5281/zenodo.47664">10.5281/zenodo.47664</a></div></div>
</div>
<div id="ref-bRPc66OD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline"><strong>Niacin in patients with low HDL cholesterol levels receiving intensive statin therapy.</strong><div class="csl-block">, William E Boden, Jeffrey L Probstfield, Todd Anderson, Bernard R Chaitman, Patrice Desvignes-Nickens, Kent Koprowicz, Ruth McBride, Koon Teo, William Weintraub</div> <em>The New England journal of medicine</em> (2011-11-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22085343">https://www.ncbi.nlm.nih.gov/pubmed/22085343</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1056/nejmoa1107579">10.1056/nejmoa1107579</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22085343">22085343</a></div></div>
</div>
<div id="ref-13ZGxHjQ5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline"><strong>Effects of extended-release niacin with laropiprant in high-risk patients.</strong><div class="csl-block">, Martin J Landray, Richard Haynes, Jemma C Hopewell, Sarah Parish, Theingi Aung, Joseph Tomson, Karl Wallendszus, Martin Craig, Lixin Jiang, … Jane Armitage</div> <em>The New England journal of medicine</em> (2014-07-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25014686">https://www.ncbi.nlm.nih.gov/pubmed/25014686</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1056/nejmoa1300955">10.1056/nejmoa1300955</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25014686">25014686</a></div></div>
</div>
<div id="ref-ZGvG75Bj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">60. </div><div class="csl-right-inline"><strong>Assessment of the Role of Niacin in Managing Cardiovascular Disease Outcomes: A Systematic Review and Meta-analysis.</strong> <div class="csl-block">Elvira D'Andrea, Spencer P Hey, Cherie L Ramirez, Aaron S Kesselheim</div> <em>JAMA network open</em> (2019-04-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30977858">https://www.ncbi.nlm.nih.gov/pubmed/30977858</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1001/jamanetworkopen.2019.2224">10.1001/jamanetworkopen.2019.2224</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30977858">30977858</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6481429">PMC6481429</a></div></div>
</div>
<div id="ref-LVihFr3g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">61. </div><div class="csl-right-inline"><strong>Mechanism of Action of Niacin</strong> <div class="csl-block">Vaijinath S Kamanna, Moti L Kashyap</div> <em>The American Journal of Cardiology</em> (2008-04) <a href="https://doi.org/c8zwdt">https://doi.org/c8zwdt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.amjcard.2008.02.029">10.1016/j.amjcard.2008.02.029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18375237">18375237</a></div></div>
</div>
<div id="ref-7OgaYjeL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">62. </div><div class="csl-right-inline"><strong>Niacin: an old lipid drug in a new NAD+ dress</strong> <div class="csl-block">Mario Romani, Dina Carina Hofer, Elena Katsyuba, Johan Auwerx</div> <em>Journal of Lipid Research</em> (2019-04) <a href="https://doi.org/gjpjft">https://doi.org/gjpjft</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1194/jlr.s092007">10.1194/jlr.s092007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30782960">30782960</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6446705">PMC6446705</a></div></div>
</div>
<div id="ref-OOAkmacQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">63. </div><div class="csl-right-inline"><strong>The therapeutic role of niacin in dyslipidemia management.</strong> <div class="csl-block">William E Boden, Mandeep S Sidhu, Peter P Toth</div> <em>Journal of cardiovascular pharmacology and therapeutics</em> (2013-12-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24363242">https://www.ncbi.nlm.nih.gov/pubmed/24363242</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1177/1074248413514481">10.1177/1074248413514481</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24363242">24363242</a></div></div>
</div>
<div id="ref-13meq3Hgt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">64. </div><div class="csl-right-inline"><strong>High-density lipoproteins in the prevention of cardiovascular disease: changing the paradigm.</strong> <div class="csl-block">S Tuteja, DJ Rader</div> <em>Clinical pharmacology and therapeutics</em> (2014-04-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24713591">https://www.ncbi.nlm.nih.gov/pubmed/24713591</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/clpt.2014.79">10.1038/clpt.2014.79</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24713591">24713591</a></div></div>
</div>
<div id="ref-izbPw2kc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">65. </div><div class="csl-right-inline"><strong>The nicotinic acid receptor GPR109A (HM74A or PUMA-G) as a new therapeutic target</strong> <div class="csl-block">S OFFERMANNS</div> <em>Trends in Pharmacological Sciences</em> (2006-07) <a href="https://doi.org/fgb4tr">https://doi.org/fgb4tr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.tips.2006.05.008">10.1016/j.tips.2006.05.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16766048">16766048</a></div></div>
</div>
<div id="ref-wI0IjT3i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">66. </div><div class="csl-right-inline"><strong>Langerhans Cells Release Prostaglandin D2 in Response to Nicotinic Acid</strong> <div class="csl-block">Dominique Maciejewski-Lenoir, Jeremy G Richman, Yaron Hakak, Ibragim Gaidarov, Dominic P Behan, Daniel T Connolly</div> <em>Journal of Investigative Dermatology</em> (2006-12) <a href="https://doi.org/dgxg75">https://doi.org/dgxg75</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/sj.jid.5700586">10.1038/sj.jid.5700586</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17008871">17008871</a></div></div>
</div>
<div id="ref-1Bz0jRHYo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">67. </div><div class="csl-right-inline"><strong>Nicotinic acid inhibits progression of atherosclerosis in mice through its receptor GPR109A expressed by immune cells</strong> <div class="csl-block">Martina Lukasova, Camille Malaval, Andreas Gille, Jukka Kero, Stefan Offermanns</div> <em>Journal of Clinical Investigation</em> (2011-03-01) <a href="https://doi.org/cqftcq">https://doi.org/cqftcq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1172/jci41651">10.1172/jci41651</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21317532">21317532</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3048854">PMC3048854</a></div></div>
</div>
<div id="ref-1DblG8swn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">68. </div><div class="csl-right-inline"><strong>Role of HDL, ABCA1, and ABCG1 Transporters in Cholesterol Efflux and Immune Responses</strong> <div class="csl-block">Laurent Yvan-Charvet, Nan Wang, Alan R Tall</div> <em>Arteriosclerosis, Thrombosis, and Vascular Biology</em> (2010-02) <a href="https://doi.org/ds23w6">https://doi.org/ds23w6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1161/atvbaha.108.179283">10.1161/atvbaha.108.179283</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19797709">19797709</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812788">PMC2812788</a></div></div>
</div>
<div id="ref-RliFvowC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">69. </div><div class="csl-right-inline"><strong>Shared and organism-specific host responses to childhood diarrheal diseases revealed by whole blood transcript profiling</strong> <div class="csl-block">Hannah A DeBerg, Mussaret B Zaidi, Matthew C Altman, Prasong Khaenam, Vivian H Gersuk, Freddy D Campos, Iza Perez-Martinez, Mario Meza-Segura, Damien Chaussabel, Jacques Banchereau, … Peter S Linsley</div> <em>PLOS ONE</em> (2018-01-29) <a href="https://doi.org/gcwgcr">https://doi.org/gcwgcr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0192082">10.1371/journal.pone.0192082</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29377961">29377961</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5788382">PMC5788382</a></div></div>
</div>
<div id="ref-D5XBhzim" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">70. </div><div class="csl-right-inline"><strong>Copy Number Loss of the Interferon Gene Cluster in Melanomas Is Linked to Reduced T Cell Infiltrate and Poor Patient Prognosis</strong> <div class="csl-block">Peter S Linsley, Cate Speake, Elizabeth Whalen, Damien Chaussabel</div> <em>PLoS ONE</em> (2014-10-14) <a href="https://doi.org/gk9k8s">https://doi.org/gk9k8s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0109760">10.1371/journal.pone.0109760</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25314013">25314013</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4196925">PMC4196925</a></div></div>
</div>
<div id="ref-EnoqU4ga" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">71. </div><div class="csl-right-inline"><strong>The Ro60 autoantigen binds endogenous retroelements and regulates inflammatory gene expression</strong> <div class="csl-block">T Hung, GA Pratt, B Sundararaman, MJ Townsend, C Chaivorapol, T Bhangale, RR Graham, W Ortmann, LA Criswell, GW Yeo, TW Behrens</div> <em>Science</em> (2015-10-23) <a href="https://doi.org/f7vs67">https://doi.org/f7vs67</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.aac7442">10.1126/science.aac7442</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26382853">26382853</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4691329">PMC4691329</a></div></div>
</div>
<div id="ref-11eausmiy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">72. </div><div class="csl-right-inline"><strong>Homo sapiens (ID 258384) - BioProject - NCBI</strong> <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA258384">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA258384</a></div>
</div>
<div id="ref-mtMYROCN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">73. </div><div class="csl-right-inline"><strong>Identification of Genes Critical for Resistance to Infection by West Nile Virus Using RNA-Seq Analysis</strong> <div class="csl-block">Feng Qian, Lisa Chung, Wei Zheng, Vincent Bruno, Roger Alexander, Zhong Wang, Xiaomei Wang, Sebastian Kurscheid, Hongyu Zhao, Erol Fikrig, … Ruth Montgomery</div> <em>Viruses</em> (2013-07-08) <a href="https://doi.org/f49d7g">https://doi.org/f49d7g</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/v5071664">10.3390/v5071664</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23881275">23881275</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3738954">PMC3738954</a></div></div>
</div>
<div id="ref-1kgcHkGm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">74. </div><div class="csl-right-inline"><strong>Mycobacterial infection induces a specific human innate immune response</strong> <div class="csl-block">John D Blischak, Ludovic Tailleux, Amy Mitrano, Luis B Barreiro, Yoav Gilad</div> <em>Scientific Reports</em> (2015-11-20) <a href="https://doi.org/f7zk5c">https://doi.org/f7zk5c</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/srep16882">10.1038/srep16882</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26586179">26586179</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4653619">PMC4653619</a></div></div>
</div>
<div id="ref-kLRErKXz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">75. </div><div class="csl-right-inline"><strong>Niacin Inhibits Apoptosis and Rescues Premature Ovarian Failure</strong> <div class="csl-block">Shufang Wang, Min Sun, Ling Yu, Yixuan Wang, Yuanqing Yao, Deqing Wang</div> <em>Cellular Physiology and Biochemistry</em> (2018) <a href="https://doi.org/gfqvcq">https://doi.org/gfqvcq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1159/000495051">10.1159/000495051</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30415247">30415247</a></div></div>
</div>
<div id="ref-TovvsrDr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">76. </div><div class="csl-right-inline"><strong>Chronic niacin administration ameliorates ovulation, histological changes in the ovary and adiponectin concentrations in a rat model of polycystic ovary syndrome</strong> <div class="csl-block">Negin Asadi, Mahin Izadi, Ali Aflatounian, Mansour Esmaeili-Dehaj, Mohammad Ebrahim Rezvani, Zeinab Hafizi</div> <em>Reproduction, Fertility and Development</em> (2021) <a href="https://doi.org/gjpjkt">https://doi.org/gjpjkt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1071/rd20306">10.1071/rd20306</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33751926">33751926</a></div></div>
</div>
<div id="ref-14dCeRkua" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">77. </div><div class="csl-right-inline"><strong>Avoiding common pitfalls when clustering biological data</strong> <div class="csl-block">Tom Ronan, Zhijie Qi, Kristen M Naegle</div> <em>Science Signaling</em> (2016-06-14) <a href="https://doi.org/gcvjr6">https://doi.org/gcvjr6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/scisignal.aad1932">10.1126/scisignal.aad1932</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27303057">27303057</a></div></div>
</div>
<div id="ref-uw3AnEgA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">78. </div><div class="csl-right-inline"><strong>Cluster Ensembles – A Knowledge Reuse Framework for Combining Multiple Partitions</strong> <div class="csl-block">Alexander Strehl, Ghosh Joydeep</div> <em>Journal of Machine Learning Research</em> (2002) <a href="https://www.jmlr.org/papers/v3/strehl02a.html">https://www.jmlr.org/papers/v3/strehl02a.html</a></div>
</div>
<div id="ref-cuROQDFa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">79. </div><div class="csl-right-inline"><strong>Combining multiple clusterings using evidence accumulation</strong> <div class="csl-block">Ana LN Fred, Anil K Jain</div> <em>IEEE Transactions on Pattern Analysis and Machine Intelligence</em> (2005-06) <a href="https://doi.org/bsknv6">https://doi.org/bsknv6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1109/tpami.2005.113">10.1109/tpami.2005.113</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15943417">15943417</a></div></div>
</div>
<div id="ref-xhtEAzx6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">80. </div><div class="csl-right-inline"><strong>Clustering trees: a visualization for evaluating clusterings at multiple resolutions</strong> <div class="csl-block">Luke Zappia, Alicia Oshlack</div> <em>GigaScience</em> (2018-07-01) <a href="https://doi.org/gfzqf5">https://doi.org/gfzqf5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/gigascience/giy083">10.1093/gigascience/giy083</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30010766">30010766</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6057528">PMC6057528</a></div></div>
</div>
<div id="ref-aBVh8zt1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">81. </div><div class="csl-right-inline"><strong>Prognostic value of grip strength: findings from the Prospective Urban Rural Epidemiology (PURE) study.</strong> <div class="csl-block">Darryl P Leong, Koon K Teo, Sumathy Rangarajan, Patricio Lopez-Jaramillo, Alvaro Avezum, Andres Orlandini, Pamela Seron, Suad H Ahmed, Annika Rosengren, Roya Kelishadi, … </div> <em>Lancet (London, England)</em> (2015-05-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25982160">https://www.ncbi.nlm.nih.gov/pubmed/25982160</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0140-6736(14)62000-6">10.1016/s0140-6736(14)62000-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25982160">25982160</a></div></div>
</div>
<div id="ref-Zk82GvJV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">82. </div><div class="csl-right-inline"><strong>Densely Interconnected Transcriptional Circuits Control Cell States in Human Hematopoiesis</strong> <div class="csl-block">Noa Novershtern, Aravind Subramanian, Lee N Lawton, Raymond H Mak, WNicholas Haining, Marie E McConkey, Naomi Habib, Nir Yosef, Cindy Y Chang, Tal Shay, … Benjamin L Ebert</div> <em>Cell</em> (2011-01) <a href="https://doi.org/cf5k92">https://doi.org/cf5k92</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2011.01.004">10.1016/j.cell.2011.01.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21241896">21241896</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049864">PMC3049864</a></div></div>
</div>
<div id="ref-10qjLoufR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">83. </div><div class="csl-right-inline"><strong>Depression as a predictor for coronary heart disease. a review and meta-analysis.</strong> <div class="csl-block">Reiner Rugulies</div> <em>American journal of preventive medicine</em> (2002-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12093424">https://www.ncbi.nlm.nih.gov/pubmed/12093424</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0749-3797(02)00439-7">10.1016/s0749-3797(02)00439-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12093424">12093424</a></div></div>
</div>
<div id="ref-j2Sl4DAE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">84. </div><div class="csl-right-inline"><strong>Mental Disorders Across the Adult Life Course and Future Coronary Heart Disease</strong> <div class="csl-block">Catharine R Gale, GDavid Batty, David PJ Osborn, Per Tynelius, Finn Rasmussen</div> <em>Circulation</em> (2014-01-14) <a href="https://doi.org/qm4">https://doi.org/qm4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1161/circulationaha.113.002065">10.1161/circulationaha.113.002065</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24190959">24190959</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4107269">PMC4107269</a></div></div>
</div>
<div id="ref-17LYMnG9n" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">85. </div><div class="csl-right-inline"><strong>Mortality gap for people with bipolar disorder and schizophrenia: UK-based cohort study 2000–2014</strong> <div class="csl-block">Joseph F Hayes, Louise Marston, Kate Walters, Michael B King, David PJ Osborn</div> <em>British Journal of Psychiatry</em> (2017-09) <a href="https://doi.org/gbwcjx">https://doi.org/gbwcjx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1192/bjp.bp.117.202606">10.1192/bjp.bp.117.202606</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28684403">28684403</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5579328">PMC5579328</a></div></div>
</div>
<div id="ref-13t4TuFeJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">86. </div><div class="csl-right-inline"><strong>Getting to the Heart of Alzheimer Disease</strong> <div class="csl-block">Joshua M Tublin, Jeremy M Adelstein, Federica del Monte, Colin K Combs, Loren E Wold</div> <em>Circulation Research</em> (2019-01-04) <a href="https://doi.org/gjzjgq">https://doi.org/gjzjgq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1161/circresaha.118.313563">10.1161/circresaha.118.313563</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30605407">30605407</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6319653">PMC6319653</a></div></div>
</div>
<div id="ref-D83Aqhga" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">87. </div><div class="csl-right-inline"><strong>The overlap between vascular disease and Alzheimer’s disease - lessons from pathology</strong> <div class="csl-block">Johannes Attems, Kurt A Jellinger</div> <em>BMC Medicine</em> (2014-11-11) <a href="https://doi.org/f6pjd4">https://doi.org/f6pjd4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s12916-014-0206-2">10.1186/s12916-014-0206-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25385447">25385447</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4226890">PMC4226890</a></div></div>
</div>
<div id="ref-9BGyO071" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">88. </div><div class="csl-right-inline"><strong>Cardiovascular Risk Factors for Alzheimer's Disease</strong> <div class="csl-block">Clive Rosendorff, Michal S Beeri, Jeremy M Silverman</div> <em>The American Journal of Geriatric Cardiology</em> (2007-03) <a href="https://doi.org/bpfw5d">https://doi.org/bpfw5d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1076-7460.2007.06696.x">10.1111/j.1076-7460.2007.06696.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17483665">17483665</a></div></div>
</div>
<div id="ref-idlBgtFz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">89. </div><div class="csl-right-inline"><strong>Reverse cholesterol transport and cholesterol efflux in atherosclerosis</strong> <div class="csl-block">R Ohashi, H Mu, X Wang, Q Yao, C Chen</div> <em>QJM: An International Journal of Medicine</em> (2005-10-28) <a href="https://doi.org/dn2fgt">https://doi.org/dn2fgt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/qjmed/hci136">10.1093/qjmed/hci136</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16258026">16258026</a></div></div>
</div>
<div id="ref-18I4ish9s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">90. </div><div class="csl-right-inline"><strong>Lipid and Lipoprotein Metabolism in Microglia</strong> <div class="csl-block">Bailey A Loving, Kimberley D Bruce</div> <em>Frontiers in Physiology</em> (2020-04-28) <a href="https://doi.org/gk92xd">https://doi.org/gk92xd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fphys.2020.00393">10.3389/fphys.2020.00393</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32411016">32411016</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7198855">PMC7198855</a></div></div>
</div>
<div id="ref-LyJmyoQr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">91. </div><div class="csl-right-inline"><strong>Shared genetic origin of asthma, hay fever and eczema elucidates allergic disease biology</strong> <div class="csl-block">Manuel A Ferreira, Judith M Vonk, Hansjörg Baurecht, Ingo Marenholz, Chao Tian, Joshua D Hoffman, Quinta Helmer, Annika Tillander, Vilhelmina Ullemar, … </div> <em>Nature Genetics</em> (2017-10-30) <a href="https://doi.org/gchg62">https://doi.org/gchg62</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ng.3985">10.1038/ng.3985</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29083406">29083406</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5989923">PMC5989923</a></div></div>
</div>
<div id="ref-veADXImD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">92. </div><div class="csl-right-inline"><strong>A genome-wide cross-trait analysis from UK Biobank highlights the shared genetic architecture of asthma and allergic diseases</strong> <div class="csl-block">Zhaozhong Zhu, Phil H Lee, Mark D Chaffin, Wonil Chung, Po-Ru Loh, Quan Lu, David C Christiani, Liming Liang</div> <em>Nature Genetics</em> (2018-05-21) <a href="https://doi.org/gdpmtn">https://doi.org/gdpmtn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41588-018-0121-0">10.1038/s41588-018-0121-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29785011">29785011</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5980765">PMC5980765</a></div></div>
</div>
<div id="ref-TPVeG4GP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">93. </div><div class="csl-right-inline"><strong>recount3: summaries and queries for large-scale RNA-seq expression and splicing</strong> <div class="csl-block">Christopher Wilks, Shijie C Zheng, Feng Yong Chen, Rone Charles, Brad Solomon, Jonathan P Ling, Eddie Luidy Imada, David Zhang, Lance Joseph, Jeffrey T Leek, … Ben Langmead</div> <em>Cold Spring Harbor Laboratory</em> (2021-05-23) <a href="https://doi.org/gj7cmq">https://doi.org/gj7cmq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2021.05.21.445138">10.1101/2021.05.21.445138</a></div></div>
</div>
<div id="ref-X4fhSCkz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">94. </div><div class="csl-right-inline"><strong>GenomicSuperSignature facilitates interpretation of RNA-seq experiments through robust, efficient comparison to public databases</strong> <div class="csl-block">Sehyun Oh, Ludwig Geistlinger, Marcel Ramos, Daniel Blankenberg, Marius van den Beek, Jaclyn N Taroni, Vincent J Carey, Casey S Greene, Levi Waldron, Sean Davis</div> <em>Nature Communications</em> (2022-06-27) <a href="https://doi.org/gqd7hm">https://doi.org/gqd7hm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-022-31411-3">10.1038/s41467-022-31411-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35760813">35760813</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9237024">PMC9237024</a></div></div>
</div>
<div id="ref-l6ogswV3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">95. </div><div class="csl-right-inline"><strong>Opportunities and challenges for transcriptome-wide association studies</strong> <div class="csl-block">Michael Wainberg, Nasa Sinnott-Armstrong, Nicholas Mancuso, Alvaro N Barbeira, David A Knowles, David Golan, Raili Ermel, Arno Ruusalepp, Thomas Quertermous, Ke Hao, … Anshul Kundaje</div> <em>Nature Genetics</em> (2019-03-29) <a href="https://doi.org/gf3hmr">https://doi.org/gf3hmr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41588-019-0385-z">10.1038/s41588-019-0385-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30926968">30926968</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6777347">PMC6777347</a></div></div>
</div>
<div id="ref-ndd3tW4g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">96. </div><div class="csl-right-inline"><strong>Probabilistic colocalization of genetic variants from complex and molecular traits: promise and limitations</strong> <div class="csl-block">Abhay Hukku, Milton Pividori, Francesca Luca, Roger Pique-Regi, Hae Kyung Im, Xiaoquan Wen</div> <em>The American Journal of Human Genetics</em> (2021-01) <a href="https://doi.org/gj58gg">https://doi.org/gj58gg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ajhg.2020.11.012">10.1016/j.ajhg.2020.11.012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33308443">33308443</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7820626">PMC7820626</a></div></div>
</div>
<div id="ref-AxVJwanp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">97. </div><div class="csl-right-inline"><strong>Transcriptome-wide association study of schizophrenia and chromatin activity yields mechanistic disease insights</strong> <div class="csl-block">Alexander Gusev, Nicholas Mancuso, Hyejung Won, Maria Kousi, Hilary K Finucane, Yakir Reshef, Lingyun Song, Alexias Safi, Steven McCarroll, Benjamin M Neale, … </div> <em>Nature Genetics</em> (2018-04) <a href="https://doi.org/gdfdf2">https://doi.org/gdfdf2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41588-018-0092-1">10.1038/s41588-018-0092-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29632383">29632383</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5942893">PMC5942893</a></div></div>
</div>
<div id="ref-fnDaLjFy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">98. </div><div class="csl-right-inline"><strong>Systematic tissue annotations of –omics samples by modeling unstructured metadata</strong> <div class="csl-block">Nathaniel T Hawkins, Marc Maldaver, Anna Yannakopoulos, Lindsay A Guare, Arjun Krishnan</div> <em>Cold Spring Harbor Laboratory</em> (2021-05-11) <a href="https://doi.org/gj2pkc">https://doi.org/gj2pkc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2021.05.10.443525">10.1101/2021.05.10.443525</a></div></div>
</div>
<div id="ref-vLyTudUB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">99. </div><div class="csl-right-inline"><strong>Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics</strong> <div class="csl-block">Alvaro N Barbeira, Scott P Dickinson, Rodrigo Bonazzola, Jiamao Zheng, Heather E Wheeler, Jason M Torres, Eric S Torstenson, Kaanan P Shah, Tzintzuni Garcia, … Hae Kyung Im</div> <em>Nature Communications</em> (2018-05-08) <a href="https://doi.org/gdjvp5">https://doi.org/gdjvp5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-018-03621-1">10.1038/s41467-018-03621-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29739930">29739930</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940825">PMC5940825</a></div></div>
</div>
<div id="ref-1CbVoEpNJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">100. </div><div class="csl-right-inline"><strong>The Molecular Signatures Database Hallmark Gene Set Collection</strong> <div class="csl-block">Arthur Liberzon, Chet Birger, Helga Thorvaldsdóttir, Mahmoud Ghandi, Jill P Mesirov, Pablo Tamayo</div> <em>Cell Systems</em> (2015-12) <a href="https://doi.org/gf78hq">https://doi.org/gf78hq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cels.2015.12.004">10.1016/j.cels.2015.12.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26771021">26771021</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4707969">PMC4707969</a></div></div>
</div>
<div id="ref-19XiXgYmd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">101. </div><div class="csl-right-inline"><strong>MAGMA: Generalized Gene-Set Analysis of GWAS Data</strong> <div class="csl-block">Christiaan A de Leeuw, Joris M Mooij, Tom Heskes, Danielle Posthuma</div> <em>PLOS Computational Biology</em> (2015-04-17) <a href="https://doi.org/gf92gp">https://doi.org/gf92gp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pcbi.1004219">10.1371/journal.pcbi.1004219</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25885710">25885710</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4401657">PMC4401657</a></div></div>
</div>
<div id="ref-1FsruosUW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">102. </div><div class="csl-right-inline"><strong>Human Disease Ontology 2018 update: classification, content and workflow expansion</strong> <div class="csl-block">Lynn M Schriml, Elvira Mitraka, James Munro, Becky Tauber, Mike Schor, Lance Nickle, Victor Felix, Linda Jeng, Cynthia Bearer, Richard Lichenstein, … Carol Greene</div> <em>Nucleic Acids Research</em> (2018-11-08) <a href="https://doi.org/ggx9wp">https://doi.org/ggx9wp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gky1032">10.1093/nar/gky1032</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30407550">30407550</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323977">PMC6323977</a></div></div>
</div>
<div id="ref-9okjVu3s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">103. </div><div class="csl-right-inline"><strong>Modeling sample variables with an Experimental Factor Ontology</strong> <div class="csl-block">James Malone, Ele Holloway, Tomasz Adamusiak, Misha Kapushesky, Jie Zheng, Nikolay Kolesnikov, Anna Zhukova, Alvis Brazma, Helen Parkinson</div> <em>Bioinformatics</em> (2010-03-03) <a href="https://doi.org/dsb6vt">https://doi.org/dsb6vt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btq099">10.1093/bioinformatics/btq099</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20200009">20200009</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2853691">PMC2853691</a></div></div>
</div>
<div id="ref-16RTdMKxI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">104. </div><div class="csl-right-inline"><strong>Mapping UK Biobank to the Experimental Factor Ontology (EFO)</strong> <div class="csl-block">EBISPOT</div> (2022-05-03) <a href="https://github.com/EBISPOT/EFO-UKB-mappings">https://github.com/EBISPOT/EFO-UKB-mappings</a></div>
</div>
<div id="ref-e4AuoW8N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">105. </div><div class="csl-right-inline"><strong>Comparing partitions</strong> <div class="csl-block">Lawrence Hubert, Phipps Arabie</div> <em>Journal of Classification</em> (1985-12) <a href="https://doi.org/bphmzh">https://doi.org/bphmzh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/bf01908075">10.1007/bf01908075</a></div></div>
</div>
<div id="ref-8js8Q3pF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">106. </div><div class="csl-right-inline"><strong>Diversity control for improving the analysis of consensus clustering</strong> <div class="csl-block">Milton Pividori, Georgina Stegmayer, Diego H Milone</div> <em>Information Sciences</em> (2016-09) <a href="https://doi.org/ghtqbk">https://doi.org/ghtqbk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ins.2016.04.027">10.1016/j.ins.2016.04.027</a></div></div>
</div>
<div id="ref-rcTMvL18" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">107. </div><div class="csl-right-inline"><strong>A Link-Based Approach to the Cluster Ensemble Problem</strong> <div class="csl-block">Natthakan Iam-On, Tossapon Boongoen, Simon Garrett, Chris Price</div> <em>IEEE Transactions on Pattern Analysis and Machine Intelligence</em> (2011-12) <a href="https://doi.org/cqgkh3">https://doi.org/cqgkh3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1109/tpami.2011.84">10.1109/tpami.2011.84</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21576752">21576752</a></div></div>
</div>
<div id="ref-t5p3UpxZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">108. </div><div class="csl-right-inline"><strong>Hybrid clustering solution selection strategy</strong> <div class="csl-block">Zhiwen Yu, Le Li, Yunjun Gao, Jane You, Jiming Liu, Hau-San Wong, Guoqiang Han</div> <em>Pattern Recognition</em> (2014-10) <a href="https://doi.org/ghtzwt">https://doi.org/ghtzwt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.patcog.2014.04.005">10.1016/j.patcog.2014.04.005</a></div></div>
</div>
<div id="ref-157h5hA34" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">109. </div><div class="csl-right-inline"><strong>UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction</strong> <div class="csl-block">Leland McInnes, John Healy, James Melville</div> <em>arXiv</em> (2020-09-21) <a href="https://arxiv.org/abs/1802.03426">https://arxiv.org/abs/1802.03426</a></div>
</div>
<div id="ref-MxGpAiPu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">110. </div><div class="csl-right-inline"><strong>k-means++: the advantages of careful seeding</strong> <div class="csl-block">David Arthur, Sergei Vassilvitskii</div> <em>Proceedings of the Eighteenth Annual ACM-SIAM Symposium on Discrete Algorithms</em> (2007) <a href="http://ilpubs.stanford.edu:8090/778/1/2006-13.pdf">http://ilpubs.stanford.edu:8090/778/1/2006-13.pdf</a></div>
</div>
<div id="ref-x3CT24TB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">111. </div><div class="csl-right-inline"><strong>On Spectral Clustering: Analysis and an algorithm</strong> <div class="csl-block">Andrew Ng, Michael Jordan, Yair Weiss</div> <em>Advances in Neural Information Processing Systems</em> (2001) <a href="https://ai.stanford.edu/~ang/papers/nips01-spectral.pdf">https://ai.stanford.edu/~ang/papers/nips01-spectral.pdf</a></div>
</div>
<div id="ref-JrL3iQea" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">112. </div><div class="csl-right-inline"><strong>A Density-Based Algorithm for Discovering Clusters in Large Spatial Databases with Noise</strong> <div class="csl-block">Martin Ester, Hans-Peter Kriegel, Jörg Sander, Xiaowei Xu</div> <em>Proceedings of the Second International Conference on Knowledge Discovery and Data Mining</em> (1996) <a href="https://www.aaai.org/Papers/KDD/1996/KDD96-037.pdf">https://www.aaai.org/Papers/KDD/1996/KDD96-037.pdf</a></div>
</div>
<div id="ref-FB7XPWl6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">113. </div><div class="csl-right-inline"><strong>Determination of Optimal Epsilon (Eps) Value on DBSCAN Algorithm to Clustering Data on Peatland Hotspots in Sumatra</strong> <div class="csl-block">Nadia Rahmah, Imas Sukaesih Sitanggang</div> <em>IOP Conference Series: Earth and Environmental Science</em> (2016-01) <a href="https://doi.org/gqr7z2">https://doi.org/gqr7z2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1088/1755-1315/31/1/012012">10.1088/1755-1315/31/1/012012</a></div></div>
</div>
<div id="ref-S7LBsfcF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">114. </div><div class="csl-right-inline"><strong>Avoiding common pitfalls when clustering biological data.</strong> <div class="csl-block">Tom Ronan, Zhijie Qi, Kristen M Naegle</div> <em>Science signaling</em> (2016-06-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27303057">https://www.ncbi.nlm.nih.gov/pubmed/27303057</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/scisignal.aad1932">10.1126/scisignal.aad1932</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27303057">27303057</a></div></div>
</div>
<div id="ref-vNXTnmxp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">115. </div><div class="csl-right-inline"><strong>Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9.</strong> <div class="csl-block">John G Doench, Nicolo Fusi, Meagan Sullender, Mudra Hegde, Emma W Vaimberg, Katherine F Donovan, Ian Smith, Zuzana Tothova, Craig Wilen, Robert Orchard, … David E Root</div> <em>Nature biotechnology</em> (2016-01-18) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26780180">https://www.ncbi.nlm.nih.gov/pubmed/26780180</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt.3437">10.1038/nbt.3437</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26780180">26780180</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4744125">PMC4744125</a></div></div>
</div>
<div id="ref-SiobXsoB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">116. </div><div class="csl-right-inline"><strong>Fine‐mapping and QTL tissue‐sharing information improves the reliability of causal gene identification</strong> <div class="csl-block">Alvaro N Barbeira, Owen J Melia, Yanyu Liang, Rodrigo Bonazzola, Gao Wang, Heather E Wheeler, François Aguet, Kristin G Ardlie, Xiaoquan Wen, Hae K Im</div> <em>Genetic Epidemiology</em> (2020-09-10) <a href="https://doi.org/gqsvf7">https://doi.org/gqsvf7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/gepi.22346">10.1002/gepi.22346</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32964524">32964524</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7693040">PMC7693040</a></div></div>
</div>
<div id="ref-GPHGnFRN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">117. </div><div class="csl-right-inline"><strong>Harmonization and Imputation Overview</strong> <div class="csl-block">hakyimlab</div> (2022-10-31) <a href="https://github.com/hakyimlab/summary-gwas-imputation">https://github.com/hakyimlab/summary-gwas-imputation</a></div>
</div>
<div id="ref-VKYS05n1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">118. </div><div class="csl-right-inline"><strong>Homo sapiens (ID 232177) - BioProject - NCBI</strong> <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA232177">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA232177</a></div>
</div>
</div>
<h2 id="acknowledgements">Acknowledgements</h2>
<p>Figure <a href="#fig:entire_process">1</a>a was created with BioRender.com.</p>
<h2 class="page_break_before" id="supplementary-material">Supplementary material</h2>
<h3 id="regression-model-for-lv-trait-associations">Regression model for LV-trait associations</h3>
<h4 id="sm:reg:null_sim">Supplementary Note 1: mean type I error rates and calibration of LV-based regression model</h4>
<p>We assessed our GLS model type I error rates (proportion of <span class="math inline">\(p\)</span>-values below 0.05) and calibration using a null model of random traits and genotype data from 1000 Genomes Phase III.
We selected 312 individuals with European ancestry, and then analyzed 1,000 traits drawn from a standard normal distribution <span class="math inline">\(\mathcal{N}(0,1)\)</span>.
We ran all the standard procedures for the TWAS approaches (S-PrediXcan and S-MultiXcan), including:
1) a standard GWAS using linear regression under an additive genetic model,
2) different GWAS processing steps, including harmonization and imputation procedures as defined in <span class="citation" data-cites="SiobXsoB">[<a href="#ref-SiobXsoB" role="doc-biblioref">116</a>]</span>,
3) S-PrediXcan and S-MultiXcan analyses.
Below we provide details for each of these steps.</p>
<p><strong>Step 1 - GWAS</strong>. We performed standard QC procedures such as
filtering out variants with missing call rates eexceeding 0.01, MAF below 1% or MAC below 20, and HWE below 1e-6,
and removing samples with high sex-discrepancy and high-relatedness (first and second degree).
We included sex and the top 20 principal components as covariates, performing the association test on 5,923,554 variants across all 1,000 random phenotypes.</p>
<p><strong>Step 2 - GWAS processing</strong>. These steps include harmonization of GWAS and imputation of <span class="math inline">\(z\)</span>-scores, which are part of the TWAS pipeline and are needed in order to ensure an acceptable overlap with SNPs in prediction models.
The scripts to run these steps are available in <span class="citation" data-cites="GPHGnFRN">[<a href="#ref-GPHGnFRN" role="doc-biblioref">117</a>]</span>.
These procedures were run for all 1,000 random phenotypes and generated a total number of 8,325,729 variants, including those with original and imputed <span class="math inline">\(z\)</span>-scores.</p>
<p><strong>Step 3 - TWAS</strong>. We processed the imputed GWAS with S-PrediXcan using the MASHR prediction models on 49 tissues from GTEx v8.
Then, S-MultiXcan was ran using the GWAS and S-PrediXcan outputs to generate gene-trait association <span class="math inline">\(p\)</span>-values.</p>
<p>Finally, we ran our GLS model (Equation (<a href="#eq:reg:model">7</a>)) to compute an association between each of the 987 LVs in MultiPLIER and the 1,000 S-MultiXcan results on random phenotypes.
For this, we built a gene correlation matrix specifically for this cohort (see <a href="#sec:methods:reg">Methods</a>).
Then, we compared the GLS results with an equivalent, baseline ordinarly least squares (OLS) model assuming independence between genes.
Figure <a href="#fig:reg:nulls:qqplots">8</a> compares the distribution of <span class="math inline">\(p\)</span>-values of the OLS and GLS models.
The GLS model has a slightly smaller mean type I error rate (0.0558, SD=0.0127) than the baseline OLS model (0.0584, SD=0.0140), and <span class="math inline">\(p\)</span>-values follow more closely the expected uniform distribution.
Importantly, the GLS model is able to correct for LVs with adjacent and highly correlated genes at the top such as LV234 (Figure <a href="#fig:reg:nulls:qqplot:lv234">9</a>), LV847 (Figure <a href="#fig:reg:nulls:qqplot:lv847">10</a>), LV45 (Figure <a href="#fig:reg:nulls:qqplot:lv45">11</a>), or LV800 (Figure <a href="#fig:reg:nulls:qqplot:lv800">12</a>), among others.
In contrast and as expected, the OLS model has higher mean type I errors and smaller-than-expected <span class="math inline">\(p\)</span>-values in all these cases.</p>
<p>We also detected other LVs with higher-than-expected mean type I errors for both the GLS and OLS models, although they don’t have a relatively large number of adjacent genes at the top.
One example is LV914, shown in Figure <a href="#fig:reg:nulls:qqplot:lv914">13</a>.
Inflation in these LVs might be explained by inaccuracies in correlation estimates between the individual-level MultiXcan model and its summary-based version (see Methods).
Therefore, we flagged those with a type I error rate larger than 0.07 (127 LVs) and excluded them from our main analyses.</p>
<div id="fig:reg:nulls:qqplots" class="fignos">
<figure>
<img src="images/gls/null_sims/models_qqplots.png" style="width:80.0%" alt="Figure 8: QQ-plots for OLS (baseline) and GLS (PhenoPLIER) models on random phenotypes." />
<figcaption aria-hidden="true"><span>Figure 8:</span> <strong>QQ-plots for OLS (baseline) and GLS (PhenoPLIER) models on random phenotypes.</strong></figcaption>
</figure>
</div>
<div id="fig:reg:nulls:qqplot:lv234" class="fignos">
<figure>
<img src="images/gls/null_sims/models_lv234.png" style="width:80.0%" alt="Figure 9: QQ-plots for LV234 on random phenotypes. Among the top 1% of genes in this LV, 17 are located in band 6p22.2, 5 in 6p22.1 and 3 in 7q11.23." />
<figcaption aria-hidden="true"><span>Figure 9:</span> <strong>QQ-plots for LV234 on random phenotypes.</strong>
Among the top 1% of genes in this LV, 17 are located in band 6p22.2, 5 in 6p22.1 and 3 in 7q11.23.</figcaption>
</figure>
</div>
<div id="fig:reg:nulls:qqplot:lv847" class="fignos">
<figure>
<img src="images/gls/null_sims/models_lv847.png" style="width:80.0%" alt="Figure 10: QQ-plots for LV847 on random phenotypes. Among the top 1% of genes in this LV, 15 are located in band 6p22.2, 5 in 6p22.1 and 2 in 15q26.1." />
<figcaption aria-hidden="true"><span>Figure 10:</span> <strong>QQ-plots for LV847 on random phenotypes.</strong>
Among the top 1% of genes in this LV, 15 are located in band 6p22.2, 5 in 6p22.1 and 2 in 15q26.1.</figcaption>
</figure>
</div>
<div id="fig:reg:nulls:qqplot:lv45" class="fignos">
<figure>
<img src="images/gls/null_sims/models_lv45.png" style="width:80.0%" alt="Figure 11: QQ-plots for LV45 on random phenotypes. Among the top 1% of genes in this LV, 12 are located in band 6p22.2, 6 in 6p22.1 and 3 in 1q23.3." />
<figcaption aria-hidden="true"><span>Figure 11:</span> <strong>QQ-plots for LV45 on random phenotypes.</strong>
Among the top 1% of genes in this LV, 12 are located in band 6p22.2, 6 in 6p22.1 and 3 in 1q23.3.</figcaption>
</figure>
</div>
<div id="fig:reg:nulls:qqplot:lv800" class="fignos">
<figure>
<img src="images/gls/null_sims/models_lv800.png" style="width:80.0%" alt="Figure 12: QQ-plots for LV800 on random phenotypes. Among the top 1% of genes in this LV, 16 are located in band 19q13.43, 9 in 19p13.2 and 9 in 19q13.31." />
<figcaption aria-hidden="true"><span>Figure 12:</span> <strong>QQ-plots for LV800 on random phenotypes.</strong>
Among the top 1% of genes in this LV, 16 are located in band 19q13.43, 9 in 19p13.2 and 9 in 19q13.31.</figcaption>
</figure>
</div>
<div id="fig:reg:nulls:qqplot:lv914" class="fignos">
<figure>
<img src="images/gls/null_sims/models_lv914.png" style="width:80.0%" alt="Figure 13: QQ-plots for LV914 on random phenotypes. Among the top 1% of genes in this LV, 2 are located in band 13q13.3, 2 in 7p15.2 and 2 in 19q13.2." />
<figcaption aria-hidden="true"><span>Figure 13:</span> <strong>QQ-plots for LV914 on random phenotypes.</strong>
Among the top 1% of genes in this LV, 2 are located in band 13q13.3, 2 in 7p15.2 and 2 in 19q13.2.</figcaption>
</figure>
</div>
<h4 id="lv-trait-associations-in-real-data">LV-trait associations in real data</h4>
<div id="fig:reg:real:qqplots" class="fignos">
<figure>
<img src="images/gls/real_data/qqplots.png" style="width:80.0%" alt="Figure 14: QQ-plots of LV-trait associations in real data. QQ-plot in PhenomeXcan (left, discovery cohort) across 4,091 traits and 987 LVs, and eMERGE (right, replication cohort) across 309 traits and 987 LVs." />
<figcaption aria-hidden="true"><span>Figure 14:</span> <strong>QQ-plots of LV-trait associations in real data.</strong>
QQ-plot in PhenomeXcan (left, discovery cohort) across 4,091 traits and 987 LVs, and eMERGE (right, replication cohort) across 309 traits and 987 LVs.</figcaption>
</figure>
</div>
<h3 id="crispr-cas9">CRISPR-Cas9</h3>
<h4 id="screening-steps">Screening steps</h4>
<div id="fig:sup:crispr:fig1" class="fignos">
<figure>
<img src="images/crispr/figure1.png" data-tag="S1" style="width:80.0%" alt="Figure S1: EVOS Fluorescent Microscope Image Capture. A. HepG2_lentiV2_Ctrl with no-viral transduction. B. HepG2_lentiV2 with viral transduction." />
<figcaption aria-hidden="true"><span>Figure S1:</span> <strong>EVOS Fluorescent Microscope Image Capture.</strong>
A. HepG2_lentiV2_Ctrl with no-viral transduction.
B. HepG2_lentiV2 with viral transduction.</figcaption>
</figure>
</div>
<div id="fig:sup:crispr:fig2" class="fignos">
<figure>
<img src="images/crispr/figure2.png" data-tag="S2" style="width:80.0%" alt="Figure S2: Fluorescence-Activated Cell Sorting Gate Setting. A. HepG2_UnStained WT. B. HepG2_lentiV2 with viral transduction." />
<figcaption aria-hidden="true"><span>Figure S2:</span> <strong>Fluorescence-Activated Cell Sorting Gate Setting.</strong>
A. HepG2_UnStained WT.
B. HepG2_lentiV2 with viral transduction.</figcaption>
</figure>
</div>
<div id="fig:sup:crispr:fig3" class="fignos">
<figure>
<img src="images/crispr/figure3.png" data-tag="S3" style="width:80.0%" alt="Figure S3: Verification of sgRNA cassette and lentiV2 transgene. A. 20nt sgRNA cassette was verified in lentiV2 transduced genomic DNA population, 163 bp PCR product obtained, while WT HepG2 didn’t possess the cassette, thus, no PCR product. B. lentiviral-specific transgene WPRE was verified in lentiV2 transduced genomic DNA population, while no transduced WT didn’t have the transgene, therefore, no 173 bp PCR product observed." />
<figcaption aria-hidden="true"><span>Figure S3:</span> <strong>Verification of sgRNA cassette and lentiV2 transgene.</strong>
A. 20nt sgRNA cassette was verified in lentiV2 transduced genomic DNA population, 163 bp PCR product obtained, while WT HepG2 didn’t possess the cassette, thus, no PCR product.
B. lentiviral-specific transgene WPRE was verified in lentiV2 transduced genomic DNA population, while no transduced WT didn’t have the transgene, therefore, no 173 bp PCR product observed.</figcaption>
</figure>
</div>
<div id="fig:sup:crispr:table1" class="fignos">
<figure>
<img src="images/crispr/table1.png" data-tag="S4" style="width:80.0%" alt="Figure S4: " />
<figcaption aria-hidden="true"><span>Figure S4:</span> </figcaption>
</figure>
</div>
<div id="fig:sup:crispr:fig4" class="fignos">
<figure>
<img src="images/crispr/figure4.png" data-tag="S5" style="width:80.0%" alt="Figure S5: Illumina library generation. A. Construct for generating illumina libraries. B. Final illumina library from HS DNA —showed a single ~285bp peak was generated." />
<figcaption aria-hidden="true"><span>Figure S5:</span> <strong>Illumina library generation.</strong>
A. Construct for generating illumina libraries.
B. Final illumina library from HS DNA —showed a single ~285bp peak was generated.</figcaption>
</figure>
</div>
<h4 id="gene-modules-enrichment-for-lipids-gene-sets">Gene modules enrichment for lipids gene-sets</h4>
<!-- lipids_gene_sets:modules_enriched_increase:start -->
<div id="tbl:sup:lipids_crispr:modules_enriched_increase" class="tablenos">
<table id="tbl:sup:lipids_crispr:modules_enriched_increase">
<caption><span>Table 2:</span> Gene modules (LVs) nominally enriched for the lipids-increasing gene-set from the CRISPR-screen (<em>P</em> &lt; 0.01). LVs significantly aligned with pathways (FDR &lt; 0.05) from the MultiPLIER models are shown in boldface.
<!-- lipids_gene_sets:modules_enriched_increase:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Gene module</th>
<th style="text-align: left;">Lipids gene-set</th>
<th style="text-align: left;">Leading edge</th>
<th style="text-align: left;">p-value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><strong>LV246</strong></td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;"><em>DGAT2</em>, <em>ACACA</em></td>
<td style="text-align: left;">0.0035</td>
</tr>
<tr class="even">
<td style="text-align: left;">LV702</td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;"><em>ACACA</em>, <em>DGAT2</em></td>
<td style="text-align: left;">0.0046</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><strong>LV607</strong></td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;"><em>ACACA</em>, <em>DGAT2</em></td>
<td style="text-align: left;">0.0058</td>
</tr>
<tr class="even">
<td style="text-align: left;">LV890</td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;"><em>ACACA</em>, <em>DGAT2</em></td>
<td style="text-align: left;">0.0067</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><strong>LV74</strong></td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;"><em>MBTPS1</em>, <em>DGAT2</em></td>
<td style="text-align: left;">0.0078</td>
</tr>
<tr class="even">
<td style="text-align: left;"><strong>LV865</strong></td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;"><em>ACACA</em>, <em>DGAT2</em></td>
<td style="text-align: left;">0.0092</td>
</tr>
<tr class="odd">
<td style="text-align: left;">LV841</td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;"><em>ACACA</em>, <em>DGAT2</em></td>
<td style="text-align: left;">0.0096</td>
</tr>
</tbody>
</table>
</div>
<!-- lipids_gene_sets:modules_enriched_decrease:start -->
<div id="tbl:sup:lipids_crispr:modules_enriched_decrease" class="tablenos">
<table id="tbl:sup:lipids_crispr:modules_enriched_decrease">
<caption><span>Table 3:</span> Gene modules (LVs) nominally enriched for the lipids-decreasing gene-set from the CRISPR-screen (<em>P</em> &lt; 0.01). LVs significantly aligned with pathways (FDR &lt; 0.05) from the MultiPLIER models are shown in boldface.
<!-- lipids_gene_sets:modules_enriched_decrease:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Gene module</th>
<th style="text-align: left;">Lipids gene-set</th>
<th style="text-align: left;">Leading edge</th>
<th style="text-align: left;">p-value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">LV520</td>
<td style="text-align: left;">decrease</td>
<td style="text-align: left;"><em>FBXW7</em>, <em>TCF7L2</em></td>
<td style="text-align: left;">0.0006</td>
</tr>
<tr class="even">
<td style="text-align: left;">LV801</td>
<td style="text-align: left;">decrease</td>
<td style="text-align: left;"><em>UBE2J2</em>, <em>TCF7L2</em></td>
<td style="text-align: left;">0.0022</td>
</tr>
<tr class="odd">
<td style="text-align: left;">LV512</td>
<td style="text-align: left;">decrease</td>
<td style="text-align: left;"><em>FBXW7</em>, <em>TCF7L2</em></td>
<td style="text-align: left;">0.0025</td>
</tr>
<tr class="even">
<td style="text-align: left;"><strong>LV612</strong></td>
<td style="text-align: left;">decrease</td>
<td style="text-align: left;"><em>PTEN</em>, <em>FBXW7</em></td>
<td style="text-align: left;">0.0036</td>
</tr>
<tr class="odd">
<td style="text-align: left;">LV41</td>
<td style="text-align: left;">decrease</td>
<td style="text-align: left;"><em>PCYT2</em>, <em>TCF7L2</em></td>
<td style="text-align: left;">0.0041</td>
</tr>
<tr class="even">
<td style="text-align: left;"><strong>LV838</strong></td>
<td style="text-align: left;">decrease</td>
<td style="text-align: left;"><em>UBE2J2</em>, <em>TCF7L2</em></td>
<td style="text-align: left;">0.0070</td>
</tr>
<tr class="odd">
<td style="text-align: left;">LV302</td>
<td style="text-align: left;">decrease</td>
<td style="text-align: left;"><em>TCF7L2</em>, <em>PTEN</em></td>
<td style="text-align: left;">0.0083</td>
</tr>
<tr class="even">
<td style="text-align: left;">LV959</td>
<td style="text-align: left;">decrease</td>
<td style="text-align: left;"><em>TCF7L2</em>, <em>PTEN</em></td>
<td style="text-align: left;">0.0092</td>
</tr>
</tbody>
</table>
</div>
<h3 id="consensus-clustering-of-traits">Consensus clustering of traits</h3>
<h4 id="sm:clustering:null_sim">Supplementary Note 2: Cluster analyses under the null hypothesis of no structure in the data</h4>
<p>For our clustering pipeline, we simulated different escenarios where there is no structure in the input data matrix <span class="math inline">\(\hat{\mathbf{M}}\)</span> (gene-trait associations from PhenomeXcan projected into the latent gene expression representation).
For this, we simulated two cases where any groupings of traits are removed:
1) the gene-trait association matrix <span class="math inline">\(\mathbf{M}\)</span> (from S-MultiXcan) does not have any meaningful structure to find groups of traits, while preserving the latent variables in <span class="math inline">\(\mathbf{Z}\)</span> from the MultiPLIER models;
and 2) the latent variables in matrix <span class="math inline">\(\mathbf{Z}\)</span> does not have any meaningful structure to find groups of traits, while preserving the gene-trait association matrix <span class="math inline">\(\mathbf{M}\)</span>.</p>
<p>For the first scenario, we shuffled genes in <span class="math inline">\(\mathbf{M}\)</span> for each trait, and this randomized matrix was then projected into the latent space.
For the second scenario, we projected matrix <span class="math inline">\(\mathbf{M}\)</span> into the latent space, and then shuffled LVs in <span class="math inline">\(\hat{\mathbf{M}}\)</span> for each trait.
For each of these scenarios, we ran exactly the same clustering pipeline we used for the real data (<a href="#sec:methods:clustering">Methods</a>), generating an ensemble of partitions that was later combined using the same consensus functions to derive the final partitions of traits.
Finally, we computed
1) stability statistics on the ensemble partitions from different algorithms
and 2) the agreement of the final consensus partition with the ensemble.</p>
<div id="fig:sup:clustering:agreement" class="fignos">
<figure>
<img src="images/clustering/selected_best_partitions_by_k.svg" style="width:100.0%" alt="Figure 15: Agreement of consensus partitions with ensemble. A real and two simulated scenarios with no data structure are shown. For each scenario, one final consensus partition was derived for each k from 2 to 60 (x-axis) following our clustering pipeline. For each partition, the agreement with the corresponding ensemble was computed using the ARI (y-axis). For the real data scenario, partitions with an agreement above the 75th percentile (dashed line) were selected for follow-up analyses in the main text." />
<figcaption aria-hidden="true"><span>Figure 15:</span> <strong>Agreement of consensus partitions with ensemble.</strong>
A real and two simulated scenarios with no data structure are shown.
For each scenario, one final consensus partition was derived for each <span class="math inline">\(k\)</span> from 2 to 60 (<span class="math inline">\(x\)</span>-axis) following our clustering pipeline.
For each partition, the agreement with the corresponding ensemble was computed using the ARI (<span class="math inline">\(y\)</span>-axis).
For the real data scenario, partitions with an agreement above the 75th percentile (dashed line) were selected for follow-up analyses in the main text.</figcaption>
</figure>
</div>
<p>The results of this analysis (Figure <a href="#fig:sup:clustering:agreement">15</a>) show that, under the two simulated null scenarios, the agreement of the consensus partitions with the ensemble is very close to zero.
This means, as expected, that there is no consensus among ensemble partitions generated with different clustering algorithms and data representations.
In contrast, using the real data, the consensus clustering approach finds trait pairs that are grouped together across the different members of the ensemble.
The partitions above the 75th percentile were considered in the main analyses, and are shown in the clustering tree in Figure <a href="#fig:clustering:tree">6</a>.</p>
<h4 id="cluster-specific-and-general-transcriptional-processes-associated-with-disease">Cluster-specific and general transcriptional processes associated with disease</h4>
<div id="fig:sup:clustering:novel:heatmap" class="fignos">
<figure>
<img src="images/clustering/global_clustermap-novel-plain.svg" style="width:100.0%" alt="Figure 16: Cluster-specific and general transcriptional processes associated with disease using novel LVs. The plot shows a submatrix of \hat{\mathbf{M}} for the main trait clusters at k=29, considering only LVs (rows) that are not aligned with any pathway. Standardized values from -6 (lighter color) to 21 (darker color)." />
<figcaption aria-hidden="true"><span>Figure 16:</span> <strong>Cluster-specific and general transcriptional processes associated with disease using novel LVs.</strong>
The plot shows a submatrix of <span class="math inline">\(\hat{\mathbf{M}}\)</span> for the main trait clusters at <span class="math inline">\(k\)</span>=29, considering only LVs (rows) that are not aligned with any pathway.
Standardized values from -6 (lighter color) to 21 (darker color).</figcaption>
</figure>
</div>
<h3 id="latent-variables-gene-modules-information">Latent variables (gene modules) information</h3>
<h4 id="lv603">LV603</h4>
<!-- LV603:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv603" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv603">
<caption><span>Table 4:</span> Pathways aligned to LV603 from the MultiPLIER models.
<!-- LV603:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">IRIS Neutrophil-Resting</td>
<td style="text-align: left;">0.91</td>
<td style="text-align: left;">4.51e-35</td>
</tr>
<tr class="even">
<td style="text-align: left;">SVM Neutrophils</td>
<td style="text-align: left;">0.98</td>
<td style="text-align: left;">1.43e-09</td>
</tr>
<tr class="odd">
<td style="text-align: left;">PID IL8CXCR2 PATHWAY</td>
<td style="text-align: left;">0.81</td>
<td style="text-align: left;">7.04e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">SIG PIP3 SIGNALING IN B LYMPHOCYTES</td>
<td style="text-align: left;">0.77</td>
<td style="text-align: left;">1.95e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV603:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv603" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv603">
<caption><span>Table 5:</span> Significant trait associations of LV603 in PhenomeXcan.
<!-- LV603:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 51%" />
<col style="width: 18%" />
<col style="width: 10%" />
<col style="width: 19%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Basophill percentage</td>
<td style="text-align: left;">349,861</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.19e‑10</td>
</tr>
<tr class="even">
<td style="text-align: left;">Basophill count</td>
<td style="text-align: left;">349,856</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.89e‑05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Treatment/medication code: ispaghula husk</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">327</td>
<td style="text-align: left;">1.36e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV603:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv603" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv603">
<caption><span>Table 6:</span> Significant trait associations of LV603 in eMERGE.
<!-- LV603:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 35%" />
<col style="width: 25%" />
<col style="width: 18%" />
<col style="width: 11%" />
<col style="width: 8%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">No significant associations</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv246">LV246</h4>
<!-- LV246:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv246" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv246" style="width:100%;">
<caption><span>Table 7:</span> Pathways aligned to LV246 from the MultiPLIER models.
<!-- LV246:multiplier_pathways:end --></caption>
<colgroup>
<col style="width: 79%" />
<col style="width: 8%" />
<col style="width: 12%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">REACTOME FATTY ACID TRIACYLGLYCEROL AND KETONE BODY METABOLISM</td>
<td style="text-align: left;">0.89</td>
<td style="text-align: left;">3.97e-16</td>
</tr>
<tr class="even">
<td style="text-align: left;">REACTOME METABOLISM OF LIPIDS AND LIPOPROTEINS</td>
<td style="text-align: left;">0.67</td>
<td style="text-align: left;">1.14e-08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">REACTOME TRIGLYCERIDE BIOSYNTHESIS</td>
<td style="text-align: left;">0.86</td>
<td style="text-align: left;">6.52e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">KEGG PYRUVATE METABOLISM</td>
<td style="text-align: left;">0.82</td>
<td style="text-align: left;">2.66e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">KEGG PROPANOATE METABOLISM</td>
<td style="text-align: left;">0.83</td>
<td style="text-align: left;">4.27e-03</td>
</tr>
</tbody>
</table>
</div>
<!-- LV246:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv246" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv246">
<caption><span>Table 8:</span> Significant trait associations of LV246 in PhenomeXcan.
<!-- LV246:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 69%" />
<col style="width: 11%" />
<col style="width: 6%" />
<col style="width: 12%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Triglycerides NMR</td>
<td style="text-align: left;">21,559</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.66e‑26</td>
</tr>
<tr class="even">
<td style="text-align: left;">LDL Cholesterol NMR</td>
<td style="text-align: left;">13,527</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.92e‑26</td>
</tr>
<tr class="odd">
<td style="text-align: left;">High cholesterol (self-reported)</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">43,957</td>
<td style="text-align: left;">1.08e‑24</td>
</tr>
<tr class="even">
<td style="text-align: left;">Cholesterol lowering medication</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">24,247</td>
<td style="text-align: left;">4.28e‑24</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Treatment/medication code: simvastatin</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">40,921</td>
<td style="text-align: left;">2.56e‑19</td>
</tr>
<tr class="even">
<td style="text-align: left;">CH2DB NMR</td>
<td style="text-align: left;">24,154</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.05e‑15</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Cholesterol lowering medication</td>
<td style="text-align: left;">165,340</td>
<td style="text-align: left;">38,057</td>
<td style="text-align: left;">9.58e‑15</td>
</tr>
<tr class="even">
<td style="text-align: left;">Treatment/medication code: atorvastatin</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">10,805</td>
<td style="text-align: left;">2.54e‑14</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Illnesses of mother: Alzheimer’s disease/dementia</td>
<td style="text-align: left;">331,041</td>
<td style="text-align: left;">28,507</td>
<td style="text-align: left;">2.76e‑08</td>
</tr>
<tr class="even">
<td style="text-align: left;">Illnesses of father: Alzheimer’s disease/dementia</td>
<td style="text-align: left;">312,666</td>
<td style="text-align: left;">15,022</td>
<td style="text-align: left;">2.76e‑08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Alzheimers Disease</td>
<td style="text-align: left;">54,162</td>
<td style="text-align: left;">17,008</td>
<td style="text-align: left;">1.10e‑07</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-butter spread type details: Flora Pro-Active or Benecol</td>
<td style="text-align: left;">190,094</td>
<td style="text-align: left;">29,048</td>
<td style="text-align: left;">5.63e‑07</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Illnesses of siblings: Alzheimer’s disease/dementia</td>
<td style="text-align: left;">279,062</td>
<td style="text-align: left;">1,609</td>
<td style="text-align: left;">6.16e‑07</td>
</tr>
<tr class="even">
<td style="text-align: left;">Any dementia</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">243</td>
<td style="text-align: left;">2.86e‑05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Illnesses of father: None of the above (group 1)</td>
<td style="text-align: left;">314,797</td>
<td style="text-align: left;">116,736</td>
<td style="text-align: left;">3.56e‑05</td>
</tr>
<tr class="even">
<td style="text-align: left;">Medication for cholesterol, blood pressure, diabetes, or take exogenous hormones (females)</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">133,338</td>
<td style="text-align: left;">1.10e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Treatment/medication code: lipitor 10mg tablet</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">2,584</td>
<td style="text-align: left;">1.55e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Treatment/medication code: rosuvastatin</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">2,227</td>
<td style="text-align: left;">1.37e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Illnesses of father: Heart disease</td>
<td style="text-align: left;">318,570</td>
<td style="text-align: left;">104,110</td>
<td style="text-align: left;">1.89e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Dementia</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">157</td>
<td style="text-align: left;">9.58e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mother still alive</td>
<td style="text-align: left;">355,029</td>
<td style="text-align: left;">140,246</td>
<td style="text-align: left;">1.76e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Job SOC coding: Librarians</td>
<td style="text-align: left;">91,149</td>
<td style="text-align: left;">1,248</td>
<td style="text-align: left;">3.22e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Alzheimer’s disease</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">119</td>
<td style="text-align: left;">3.61e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV246:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv246" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv246">
<caption><span>Table 9:</span> Significant trait associations of LV246 in eMERGE.
<!-- LV246:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 13%" />
<col style="width: 38%" />
<col style="width: 18%" />
<col style="width: 10%" />
<col style="width: 19%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">272.11</td>
<td style="text-align: left;">Hypercholesterolemia</td>
<td style="text-align: left;">40,786</td>
<td style="text-align: left;">14,138</td>
<td style="text-align: left;">4.40e‑09</td>
</tr>
<tr class="even">
<td style="text-align: left;">272.1</td>
<td style="text-align: left;">Hyperlipidemia</td>
<td style="text-align: left;">55,843</td>
<td style="text-align: left;">29,195</td>
<td style="text-align: left;">3.57e‑07</td>
</tr>
<tr class="odd">
<td style="text-align: left;">272</td>
<td style="text-align: left;">Disorders of lipoid metabolism</td>
<td style="text-align: left;">55,892</td>
<td style="text-align: left;">29,244</td>
<td style="text-align: left;">3.79e‑07</td>
</tr>
<tr class="even">
<td style="text-align: left;">292.3</td>
<td style="text-align: left;">Memory loss</td>
<td style="text-align: left;">48,785</td>
<td style="text-align: left;">2,094</td>
<td style="text-align: left;">1.80e‑02</td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv116">LV116</h4>
<!-- LV116:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv116" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv116">
<caption><span>Table 10:</span> Pathways aligned to LV116 from the MultiPLIER models.
<!-- LV116:multiplier_pathways:end --></caption>
<colgroup>
<col style="width: 76%" />
<col style="width: 9%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">REACTOME INTERFERON SIGNALING</td>
<td style="text-align: left;">0.84</td>
<td style="text-align: left;">3.48e-09</td>
</tr>
<tr class="even">
<td style="text-align: left;">SVM Macrophages M1</td>
<td style="text-align: left;">0.92</td>
<td style="text-align: left;">2.09e-05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">REACTOME INTERFERON ALPHA BETA SIGNALING</td>
<td style="text-align: left;">0.94</td>
<td style="text-align: left;">3.36e-05</td>
</tr>
<tr class="even">
<td style="text-align: left;">REACTOME CYTOKINE SIGNALING IN IMMUNE SYSTEM</td>
<td style="text-align: left;">0.67</td>
<td style="text-align: left;">1.53e-04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IRIS DendriticCell-LPSstimulated</td>
<td style="text-align: left;">0.65</td>
<td style="text-align: left;">1.09e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">KEGG CYTOSOLIC DNA SENSING PATHWAY</td>
<td style="text-align: left;">0.84</td>
<td style="text-align: left;">3.22e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">REACTOME NEGATIVE REGULATORS OF RIG I MDA5 SIGNALING</td>
<td style="text-align: left;">0.81</td>
<td style="text-align: left;">1.61e-02</td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv931">LV931</h4>
<div id="fig:sup:lv931" class="fignos">
<figure>
<img src="images/lvs_analysis/lv931/lv931-cell_types.svg" style="width:80.0%" alt="Figure 17: Cell types for LV931. " />
<figcaption aria-hidden="true"><span>Figure 17:</span> <strong>Cell types for LV931.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV931:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv931" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv931">
<caption><span>Table 11:</span> Pathways aligned to LV931 from the MultiPLIER models.
<!-- LV931:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">MIPS SPLICEOSOME</td>
<td style="text-align: left;">0.63</td>
<td style="text-align: left;">3.13e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">PID TGFBRPATHWAY</td>
<td style="text-align: left;">0.71</td>
<td style="text-align: left;">3.99e-02</td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv66">LV66</h4>
<div id="fig:sup:lv66" class="fignos">
<figure>
<img src="images/lvs_analysis/lv66/lv66-cell_types.svg" style="width:80.0%" alt="Figure 18: Cell types for LV66. " />
<figcaption aria-hidden="true"><span>Figure 18:</span> <strong>Cell types for LV66.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV66:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv66" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv66">
<caption><span>Table 12:</span> Pathways aligned to LV66 from the MultiPLIER models.
<!-- LV66:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">REACTOME METABOLISM OF LIPIDS AND LIPOPROTEINS</td>
<td style="text-align: left;">0.62</td>
<td style="text-align: left;">3.12e-04</td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv928">LV928</h4>
<div id="fig:sup:lv928" class="fignos">
<figure>
<img src="images/lvs_analysis/lv928/lv928-cell_types.svg" style="width:80.0%" alt="Figure 19: Cell types for LV928. " />
<figcaption aria-hidden="true"><span>Figure 19:</span> <strong>Cell types for LV928.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV928:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv928" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv928">
<caption><span>Table 13:</span> Pathways aligned to LV928 from the MultiPLIER models.
<!-- LV928:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">DMAP ERY3</td>
<td style="text-align: left;">0.81</td>
<td style="text-align: left;">1.16e-24</td>
</tr>
<tr class="even">
<td style="text-align: left;">DMAP ERY4</td>
<td style="text-align: left;">0.78</td>
<td style="text-align: left;">2.49e-17</td>
</tr>
</tbody>
</table>
</div>
<!-- LV928:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv928" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv928">
<caption><span>Table 14:</span> Significant trait associations of LV928 in PhenomeXcan.
<!-- LV928:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 55%" />
<col style="width: 16%" />
<col style="width: 10%" />
<col style="width: 17%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Mean sphered cell volume</td>
<td style="text-align: left;">344,729</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.60e‑20</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mean corpuscular haemoglobin concentration</td>
<td style="text-align: left;">350,468</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.42e‑17</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mean reticulocyte volume</td>
<td style="text-align: left;">344,728</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.77e‑17</td>
</tr>
<tr class="even">
<td style="text-align: left;">Reticulocyte count</td>
<td style="text-align: left;">344,729</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">2.28e‑10</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Reticulocyte percentage</td>
<td style="text-align: left;">344,728</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.37e‑09</td>
</tr>
<tr class="even">
<td style="text-align: left;">Red blood cell (erythrocyte) distribution width</td>
<td style="text-align: left;">350,473</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">2.90e‑09</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Reticulocyte Count</td>
<td style="text-align: left;">173,480</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.09e‑07</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mean corpuscular volume</td>
<td style="text-align: left;">350,473</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.46e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">High light scatter reticulocyte count</td>
<td style="text-align: left;">344,729</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.49e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Age at first episode of depression</td>
<td style="text-align: left;">61,033</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.33e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">High Light Scatter Reticulocyte Count</td>
<td style="text-align: left;">173,480</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.48e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mean corpuscular haemoglobin</td>
<td style="text-align: left;">350,472</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">4.02e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV928:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv928" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv928">
<caption><span>Table 15:</span> Significant trait associations of LV928 in eMERGE.
<!-- LV928:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 35%" />
<col style="width: 25%" />
<col style="width: 18%" />
<col style="width: 11%" />
<col style="width: 8%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">No significant associations</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv30">LV30</h4>
<div id="fig:sup:lv30" class="fignos">
<figure>
<img src="images/lvs_analysis/lv30/lv30-cell_types.svg" style="width:80.0%" alt="Figure 20: Cell types for LV30. " />
<figcaption aria-hidden="true"><span>Figure 20:</span> <strong>Cell types for LV30.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV30:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv30" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv30">
<caption><span>Table 16:</span> Pathways aligned to LV30 from the MultiPLIER models.
<!-- LV30:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">DMAP ERY3</td>
<td style="text-align: left;">0.95</td>
<td style="text-align: left;">5.62e-52</td>
</tr>
<tr class="even">
<td style="text-align: left;">DMAP ERY4</td>
<td style="text-align: left;">0.98</td>
<td style="text-align: left;">5.28e-51</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DMAP ERY5</td>
<td style="text-align: left;">0.98</td>
<td style="text-align: left;">1.96e-49</td>
</tr>
</tbody>
</table>
</div>
<!-- LV30:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv30" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv30">
<caption><span>Table 17:</span> Significant trait associations of LV30 in PhenomeXcan.
<!-- LV30:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 59%" />
<col style="width: 15%" />
<col style="width: 9%" />
<col style="width: 16%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Mean reticulocyte volume</td>
<td style="text-align: left;">344,728</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.09e‑32</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mean sphered cell volume</td>
<td style="text-align: left;">344,729</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.38e‑24</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Reticulocyte Count</td>
<td style="text-align: left;">173,480</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">6.28e‑18</td>
</tr>
<tr class="even">
<td style="text-align: left;">Reticulocyte percentage</td>
<td style="text-align: left;">344,728</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.27e‑17</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mean corpuscular haemoglobin concentration</td>
<td style="text-align: left;">350,468</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.62e‑17</td>
</tr>
<tr class="even">
<td style="text-align: left;">Reticulocyte count</td>
<td style="text-align: left;">344,729</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.62e‑17</td>
</tr>
<tr class="odd">
<td style="text-align: left;">High light scatter reticulocyte count</td>
<td style="text-align: left;">344,729</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">4.78e‑11</td>
</tr>
<tr class="even">
<td style="text-align: left;">High Light Scatter Reticulocyte Count</td>
<td style="text-align: left;">173,480</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">8.49e‑11</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Immature reticulocyte fraction</td>
<td style="text-align: left;">344,728</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">4.31e‑10</td>
</tr>
<tr class="even">
<td style="text-align: left;">High light scatter reticulocyte percentage</td>
<td style="text-align: left;">344,729</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.21e‑05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mean corpuscular volume</td>
<td style="text-align: left;">350,473</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">2.28e‑05</td>
</tr>
<tr class="even">
<td style="text-align: left;">Red blood cell (erythrocyte) distribution width</td>
<td style="text-align: left;">350,473</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.00e‑05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mean platelet (thrombocyte) volume</td>
<td style="text-align: left;">350,470</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">6.75e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mean corpuscular haemoglobin</td>
<td style="text-align: left;">350,472</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.90e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Illnesses of adopted mother: Chronic bronchitis/emphysema</td>
<td style="text-align: left;">2,938</td>
<td style="text-align: left;">238</td>
<td style="text-align: left;">1.92e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV30:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv30" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv30">
<caption><span>Table 18:</span> Significant trait associations of LV30 in eMERGE.
<!-- LV30:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 35%" />
<col style="width: 25%" />
<col style="width: 18%" />
<col style="width: 11%" />
<col style="width: 8%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">No significant associations</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv730">LV730</h4>
<div id="fig:sup:lv730" class="fignos">
<figure>
<img src="images/lvs_analysis/lv730/lv730-cell_types.svg" style="width:80.0%" alt="Figure 21: Cell types for LV730. " />
<figcaption aria-hidden="true"><span>Figure 21:</span> <strong>Cell types for LV730.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV730:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv730" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv730">
<caption><span>Table 19:</span> Pathways aligned to LV730 from the MultiPLIER models.
<!-- LV730:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">DMAP MEGA2</td>
<td style="text-align: left;">0.82</td>
<td style="text-align: left;">2.64e-05</td>
</tr>
</tbody>
</table>
</div>
<!-- LV730:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv730" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv730">
<caption><span>Table 20:</span> Significant trait associations of LV730 in PhenomeXcan.
<!-- LV730:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 62%" />
<col style="width: 14%" />
<col style="width: 8%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Platelet distribution width</td>
<td style="text-align: left;">350,470</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.13e‑10</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mean platelet (thrombocyte) volume</td>
<td style="text-align: left;">350,470</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.47e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Reason former drinker stopped drinking alcohol: Financial reasons</td>
<td style="text-align: left;">12,110</td>
<td style="text-align: left;">233</td>
<td style="text-align: left;">3.71e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV730:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv730" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv730">
<caption><span>Table 21:</span> Significant trait associations of LV730 in eMERGE.
<!-- LV730:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 35%" />
<col style="width: 25%" />
<col style="width: 18%" />
<col style="width: 11%" />
<col style="width: 8%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">No significant associations</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv598">LV598</h4>
<div id="fig:sup:lv598" class="fignos">
<figure>
<img src="images/lvs_analysis/lv598/lv598-cell_types.svg" style="width:80.0%" alt="Figure 22: Cell types for LV598. " />
<figcaption aria-hidden="true"><span>Figure 22:</span> <strong>Cell types for LV598.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV598:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv598" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv598">
<caption><span>Table 22:</span> Pathways aligned to LV598 from the MultiPLIER models.
<!-- LV598:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">PID SYNDECAN 1 PATHWAY</td>
<td style="text-align: left;">0.81</td>
<td style="text-align: left;">1.20e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">REACTOME COLLAGEN FORMATION</td>
<td style="text-align: left;">0.77</td>
<td style="text-align: left;">1.89e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV598:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv598" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv598">
<caption><span>Table 23:</span> Significant trait associations of LV598 in PhenomeXcan.
<!-- LV598:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 58%" />
<col style="width: 15%" />
<col style="width: 9%" />
<col style="width: 16%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Corneal resistance factor (right)</td>
<td style="text-align: left;">76,630</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">4.05e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Corneal resistance factor (left)</td>
<td style="text-align: left;">76,510</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.86e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">6mm strong meridian (left)</td>
<td style="text-align: left;">65,551</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">2.58e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Corneal hysteresis (right)</td>
<td style="text-align: left;">76,630</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.21e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">6mm strong meridian (right)</td>
<td style="text-align: left;">66,256</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">2.18e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Treatment/medication code: evening primrose oil product</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">814</td>
<td style="text-align: left;">2.58e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">6mm weak meridian (left)</td>
<td style="text-align: left;">65,551</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.67e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Hand grip strength (left)</td>
<td style="text-align: left;">359,704</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">4.15e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">3mm strong meridian (left)</td>
<td style="text-align: left;">75,398</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">4.74e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV598:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv598" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv598">
<caption><span>Table 24:</span> Significant trait associations of LV598 in eMERGE.
<!-- LV598:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 35%" />
<col style="width: 25%" />
<col style="width: 18%" />
<col style="width: 11%" />
<col style="width: 8%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">No significant associations</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv57">LV57</h4>
<div id="fig:sup:lv57" class="fignos">
<figure>
<img src="images/lvs_analysis/lv57/lv57-cell_types.svg" style="width:80.0%" alt="Figure 23: Cell types for LV57." />
<figcaption aria-hidden="true"><span>Figure 23:</span> <strong>Cell types for LV57.</strong></figcaption>
</figure>
</div>
<!-- LV57:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv57" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv57">
<caption><span>Table 25:</span> Pathways aligned to LV57 from the MultiPLIER models.
<!-- LV57:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">KEGG T CELL RECEPTOR SIGNALING PATHWAY</td>
<td style="text-align: left;">0.70</td>
<td style="text-align: left;">1.26e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">SVM T cells CD4 memory activated</td>
<td style="text-align: left;">0.79</td>
<td style="text-align: left;">2.59e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IRIS CD4Tcell-Th2-restimulated12hour</td>
<td style="text-align: left;">0.78</td>
<td style="text-align: left;">7.57e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">KEGG ALLOGRAFT REJECTION</td>
<td style="text-align: left;">1.00</td>
<td style="text-align: left;">1.09e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Custom Treg</td>
<td style="text-align: left;">0.98</td>
<td style="text-align: left;">1.37e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">PID NFAT TFPATHWAY</td>
<td style="text-align: left;">0.74</td>
<td style="text-align: left;">1.52e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IRIS MemoryTcell-RO-activated</td>
<td style="text-align: left;">0.70</td>
<td style="text-align: left;">2.87e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV57:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv57" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv57">
<caption><span>Table 26:</span> Significant trait associations of LV57 in PhenomeXcan.
<!-- LV57:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 75%" />
<col style="width: 9%" />
<col style="width: 5%" />
<col style="width: 9%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: deep venous thrombosis (dvt)</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">7,237</td>
<td style="text-align: left;">1.76e‑13</td>
</tr>
<tr class="even">
<td style="text-align: left;">Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Blood clot in the leg (DVT)</td>
<td style="text-align: left;">360,527</td>
<td style="text-align: left;">7,386</td>
<td style="text-align: left;">1.22e‑12</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Diagnoses - main ICD10: I80 Phlebitis and thrombophlebitis</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">2,289</td>
<td style="text-align: left;">7.62e‑12</td>
</tr>
<tr class="even">
<td style="text-align: left;">DVT of lower extremities</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">2,116</td>
<td style="text-align: left;">1.27e‑09</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Venous thromboembolism</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">4,620</td>
<td style="text-align: left;">2.28e‑08</td>
</tr>
<tr class="even">
<td style="text-align: left;">DVT of lower extremities and pulmonary embolism</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">4,319</td>
<td style="text-align: left;">4.36e‑08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Inflammatory Bowel Disease</td>
<td style="text-align: left;">34,652</td>
<td style="text-align: left;">12,882</td>
<td style="text-align: left;">1.95e‑05</td>
</tr>
<tr class="even">
<td style="text-align: left;">hypothyroidism (self-reported)</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">17,574</td>
<td style="text-align: left;">3.84e‑05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Medication: levothyroxine sodium</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">14,689</td>
<td style="text-align: left;">8.43e‑05</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mouth/teeth dental problems: Mouth ulcers</td>
<td style="text-align: left;">359,841</td>
<td style="text-align: left;">36,831</td>
<td style="text-align: left;">1.02e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Crohns Disease</td>
<td style="text-align: left;">20,833</td>
<td style="text-align: left;">5,956</td>
<td style="text-align: left;">1.02e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Facial ageing</td>
<td style="text-align: left;">330,409</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.04e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Ulcerative Colitis</td>
<td style="text-align: left;">27,432</td>
<td style="text-align: left;">6,968</td>
<td style="text-align: left;">1.27e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Hair colour (natural, before greying): Black</td>
<td style="text-align: left;">360,270</td>
<td style="text-align: left;">15,809</td>
<td style="text-align: left;">1.99e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Hair colour (natural, before greying): Light brown</td>
<td style="text-align: left;">360,270</td>
<td style="text-align: left;">147,560</td>
<td style="text-align: left;">4.69e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV57:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv57" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv57">
<caption><span>Table 27:</span> Significant trait associations of LV57 in eMERGE.
<!-- LV57:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 12%" />
<col style="width: 42%" />
<col style="width: 16%" />
<col style="width: 10%" />
<col style="width: 17%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">286</td>
<td style="text-align: left;">Coagulation defects</td>
<td style="text-align: left;">50,182</td>
<td style="text-align: left;">2,976</td>
<td style="text-align: left;">1.33e‑11</td>
</tr>
<tr class="even">
<td style="text-align: left;">452</td>
<td style="text-align: left;">Other venous embolism and thrombosis</td>
<td style="text-align: left;">40,476</td>
<td style="text-align: left;">3,816</td>
<td style="text-align: left;">1.52e‑05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">452.2</td>
<td style="text-align: left;">Deep vein thrombosis [DVT]</td>
<td style="text-align: left;">38,791</td>
<td style="text-align: left;">2,131</td>
<td style="text-align: left;">4.47e‑05</td>
</tr>
<tr class="even">
<td style="text-align: left;">244.4</td>
<td style="text-align: left;">Hypothyroidism NOS</td>
<td style="text-align: left;">53,968</td>
<td style="text-align: left;">9,284</td>
<td style="text-align: left;">1.12e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">244</td>
<td style="text-align: left;">Hypothyroidism</td>
<td style="text-align: left;">54,404</td>
<td style="text-align: left;">9,720</td>
<td style="text-align: left;">1.42e‑02</td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv844">LV844</h4>
<div id="fig:sup:lv844" class="fignos">
<figure>
<img src="images/lvs_analysis/lv844/lv844-cell_types.svg" style="width:80.0%" alt="Figure 24: Cell types for LV844." />
<figcaption aria-hidden="true"><span>Figure 24:</span> <strong>Cell types for LV844.</strong></figcaption>
</figure>
</div>
<!-- LV844:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv844" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv844">
<caption><span>Table 28:</span> Pathways aligned to LV844 from the MultiPLIER models.
<!-- LV844:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">KEGG ANTIGEN PROCESSING AND PRESENTATION</td>
<td style="text-align: left;">0.80</td>
<td style="text-align: left;">1.35e-03</td>
</tr>
</tbody>
</table>
</div>
<!-- LV844:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv844" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv844">
<caption><span>Table 29:</span> Significant trait associations of LV844 in PhenomeXcan.
<!-- LV844:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 61%" />
<col style="width: 14%" />
<col style="width: 8%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: polymyalgia rheumatica</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">753</td>
<td style="text-align: left;">5.22e‑06</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-cancer illness code, self-reported: type 1 diabetes</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">318</td>
<td style="text-align: left;">4.71e‑05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Type 1 diabetes with ketoacidosis</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">168</td>
<td style="text-align: left;">1.03e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Age diabetes diagnosed</td>
<td style="text-align: left;">16,166</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.86e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Milk type used: Other type of milk</td>
<td style="text-align: left;">360,806</td>
<td style="text-align: left;">4,213</td>
<td style="text-align: left;">5.48e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-cancer illness code, self-reported: appendicitis</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">3,058</td>
<td style="text-align: left;">6.12e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Diabetic ketoacidosis</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">234</td>
<td style="text-align: left;">7.13e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Rheumatoid Arthritis</td>
<td style="text-align: left;">80,799</td>
<td style="text-align: left;">19,234</td>
<td style="text-align: left;">7.46e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Type 1 diabetes without complications</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">247</td>
<td style="text-align: left;">1.05e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Started insulin within one year diagnosis of diabetes</td>
<td style="text-align: left;">16,415</td>
<td style="text-align: left;">1,999</td>
<td style="text-align: left;">1.30e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Insulin medication (males)</td>
<td style="text-align: left;">165,340</td>
<td style="text-align: left;">2,248</td>
<td style="text-align: left;">3.61e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Medication: insulin product</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">3,545</td>
<td style="text-align: left;">5.48e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Insulin medication (females)</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">1,476</td>
<td style="text-align: left;">7.93e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Type 1 diabetes</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">583</td>
<td style="text-align: left;">1.04e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Diagnoses - main ICD10: E10 Insulin-dependent diabetes mellitus</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">470</td>
<td style="text-align: left;">1.08e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Treatment/medication code: sulfasalazine</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">710</td>
<td style="text-align: left;">1.10e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">malabsorption/coeliac disease (self-reported)</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">1,587</td>
<td style="text-align: left;">3.12e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Job coding: school inspector, education inspector</td>
<td style="text-align: left;">89,866</td>
<td style="text-align: left;">238</td>
<td style="text-align: left;">3.71e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Seropositive rheumatoid arthritis</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">327</td>
<td style="text-align: left;">3.86e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-cancer illness code, self-reported: rheumatoid arthritis</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">4,017</td>
<td style="text-align: left;">4.21e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Age hayfever or allergic rhinitis diagnosed by doctor</td>
<td style="text-align: left;">20,904</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">4.44e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Other/unspecified seropositiverheumatoid arthritis</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">299</td>
<td style="text-align: left;">4.88e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV844:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv844" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv844">
<caption><span>Table 30:</span> Significant trait associations of LV844 in eMERGE.
<!-- LV844:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 35%" />
<col style="width: 25%" />
<col style="width: 18%" />
<col style="width: 11%" />
<col style="width: 8%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">No significant associations</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv136">LV136</h4>
<div id="fig:sup:lv136" class="fignos">
<figure>
<img src="images/lvs_analysis/lv136/lv136-cell_types.svg" style="width:80.0%" alt="Figure 25: Cell types for LV136.  Pulmonary microvascular endothelial cells were exposed to hypoxia for 24 hours or more [118];" />
<figcaption aria-hidden="true"><span>Figure 25:</span> <strong>Cell types for LV136.</strong>
<!--  -->
Pulmonary microvascular endothelial cells were exposed to hypoxia for 24 hours or more <span class="citation" data-cites="VKYS05n1">[<a href="#ref-VKYS05n1" role="doc-biblioref">118</a>]</span>;</figcaption>
</figure>
</div>
<!-- LV136:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv136" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv136">
<caption><span>Table 31:</span> Pathways aligned to LV136 from the MultiPLIER models.
<!-- LV136:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">PID INTEGRIN1 PATHWAY</td>
<td style="text-align: left;">0.88</td>
<td style="text-align: left;">9.35e-06</td>
</tr>
<tr class="even">
<td style="text-align: left;">KEGG ECM RECEPTOR INTERACTION</td>
<td style="text-align: left;">0.80</td>
<td style="text-align: left;">7.29e-05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">REACTOME COLLAGEN FORMATION</td>
<td style="text-align: left;">0.87</td>
<td style="text-align: left;">2.00e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">REACTOME MUSCLE CONTRACTION</td>
<td style="text-align: left;">0.75</td>
<td style="text-align: left;">1.49e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV136:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv136" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv136">
<caption><span>Table 32:</span> Significant trait associations of LV136 in PhenomeXcan.
<!-- LV136:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 61%" />
<col style="width: 14%" />
<col style="width: 8%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Coronary atherosclerosis</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">14,334</td>
<td style="text-align: left;">1.84e‑09</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chronic ischaemic heart disease (ICD10 I25)</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">12,769</td>
<td style="text-align: left;">3.52e‑09</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Ischaemic heart disease (wide definition)</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">20,857</td>
<td style="text-align: left;">3.95e‑08</td>
</tr>
<tr class="even">
<td style="text-align: left;">Coronary Artery Disease</td>
<td style="text-align: left;">184,305</td>
<td style="text-align: left;">60,801</td>
<td style="text-align: left;">4.18e‑08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">3mm strong meridian (right)</td>
<td style="text-align: left;">75,410</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">5.54e‑05</td>
</tr>
<tr class="even">
<td style="text-align: left;">6mm strong meridian (left)</td>
<td style="text-align: left;">65,551</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.35e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Corneal resistance factor (right)</td>
<td style="text-align: left;">76,630</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">2.02e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">6mm strong meridian (right)</td>
<td style="text-align: left;">66,256</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">2.58e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Heart attack</td>
<td style="text-align: left;">360,420</td>
<td style="text-align: left;">8,288</td>
<td style="text-align: left;">3.75e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Myocardial infarction</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">7,018</td>
<td style="text-align: left;">4.85e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Myocardial infarction, strict</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">7,018</td>
<td style="text-align: left;">4.85e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">3mm strong meridian (left)</td>
<td style="text-align: left;">75,398</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">6.65e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">heart attack/myocardial infarction (self-reported)</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">8,239</td>
<td style="text-align: left;">1.07e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">6mm weak meridian (left)</td>
<td style="text-align: left;">65,551</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.10e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">6mm weak meridian (right)</td>
<td style="text-align: left;">66,256</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.61e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Acute myocardial infarction (ICD10 I21)</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">5,948</td>
<td style="text-align: left;">2.24e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">3mm weak meridian (right)</td>
<td style="text-align: left;">75,410</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.69e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">3mm weak meridian (left)</td>
<td style="text-align: left;">75,398</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.96e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Intra-ocular pressure, Goldmann-correlated (right)</td>
<td style="text-align: left;">76,630</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">8.64e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">6mm asymmetry angle (right)</td>
<td style="text-align: left;">41,390</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.03e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Corneal resistance factor (left)</td>
<td style="text-align: left;">76,510</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.03e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Other specified disorders of muscle</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">257</td>
<td style="text-align: left;">1.09e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Major coronary heart disease event excluding revascularizations</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">10,157</td>
<td style="text-align: left;">2.44e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Major coronary heart disease event</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">10,157</td>
<td style="text-align: left;">2.44e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: angina</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">11,370</td>
<td style="text-align: left;">4.53e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Eye problems/disorders: Glaucoma</td>
<td style="text-align: left;">117,890</td>
<td style="text-align: left;">5,092</td>
<td style="text-align: left;">4.94e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV136:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv136" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv136">
<caption><span>Table 33:</span> Significant trait associations of LV136 in eMERGE.
<!-- LV136:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 14%" />
<col style="width: 33%" />
<col style="width: 19%" />
<col style="width: 11%" />
<col style="width: 20%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">411.4</td>
<td style="text-align: left;">Coronary atherosclerosis</td>
<td style="text-align: left;">52,836</td>
<td style="text-align: left;">13,715</td>
<td style="text-align: left;">1.42e‑03</td>
</tr>
</tbody>
</table>
</div>
<h4 id="lv21">LV21</h4>
<div id="fig:sup:lv21" class="fignos">
<figure>
<img src="images/lvs_analysis/lv21/lv21-cell_types.svg" style="width:80.0%" alt="Figure 26: Cell types for LV21. " />
<figcaption aria-hidden="true"><span>Figure 26:</span> <strong>Cell types for LV21.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV21:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv21" class="tablenos">
<table id="tbl:sup:multiplier_pathways:lv21">
<caption><span>Table 34:</span> Pathways aligned to LV21 from the MultiPLIER models.
<!-- LV21:multiplier_pathways:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">No pathways significantly enriched</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</div>
<!-- LV21:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv21" class="tablenos">
<table id="tbl:sup:phenomexcan_assocs:lv21">
<caption><span>Table 35:</span> Significant trait associations of LV21 in PhenomeXcan.
<!-- LV21:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 61%" />
<col style="width: 14%" />
<col style="width: 8%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">LDL Cholesterol NMR</td>
<td style="text-align: left;">13,527</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.08e‑12</td>
</tr>
<tr class="even">
<td style="text-align: left;">HDL Cholesterol NMR</td>
<td style="text-align: left;">19,270</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.03e‑11</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Alzheimers Disease</td>
<td style="text-align: left;">54,162</td>
<td style="text-align: left;">17,008</td>
<td style="text-align: left;">1.96e‑09</td>
</tr>
<tr class="even">
<td style="text-align: left;">Triglycerides NMR</td>
<td style="text-align: left;">21,559</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">2.05e‑09</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Illnesses of mother: Alzheimer’s disease/dementia</td>
<td style="text-align: left;">331,041</td>
<td style="text-align: left;">28,507</td>
<td style="text-align: left;">1.36e‑08</td>
</tr>
<tr class="even">
<td style="text-align: left;">Illnesses of father: Alzheimer’s disease/dementia</td>
<td style="text-align: left;">312,666</td>
<td style="text-align: left;">15,022</td>
<td style="text-align: left;">3.15e‑08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Illnesses of siblings: Alzheimer’s disease/dementia</td>
<td style="text-align: left;">279,062</td>
<td style="text-align: left;">1,609</td>
<td style="text-align: left;">1.55e‑07</td>
</tr>
<tr class="even">
<td style="text-align: left;">Any dementia</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">243</td>
<td style="text-align: left;">5.63e‑07</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Treatment/medication code: simvastatin</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">40,921</td>
<td style="text-align: left;">2.88e‑06</td>
</tr>
<tr class="even">
<td style="text-align: left;">Cholesterol lowering medication</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">24,247</td>
<td style="text-align: left;">4.45e‑05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">High cholesterol (self-reported)</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">43,957</td>
<td style="text-align: left;">9.90e‑05</td>
</tr>
<tr class="even">
<td style="text-align: left;">Cholesterol lowering medication</td>
<td style="text-align: left;">165,340</td>
<td style="text-align: left;">38,057</td>
<td style="text-align: left;">4.86e‑04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Dementia</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">157</td>
<td style="text-align: left;">9.80e‑04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Alzheimer’s disease</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">119</td>
<td style="text-align: left;">1.33e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mean reticulocyte volume</td>
<td style="text-align: left;">344,728</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.76e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Father’s age at death</td>
<td style="text-align: left;">266,231</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">6.68e‑03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Illnesses of mother: None of the above (group 1)</td>
<td style="text-align: left;">332,611</td>
<td style="text-align: left;">138,291</td>
<td style="text-align: left;">1.42e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">ECG, phase time</td>
<td style="text-align: left;">53,998</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">1.60e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Treatment/medication code: atorvastatin</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">10,805</td>
<td style="text-align: left;">2.92e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mean sphered cell volume</td>
<td style="text-align: left;">344,729</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">3.33e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: cellulitis</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">232</td>
<td style="text-align: left;">3.40e‑02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Medication for cholesterol, blood pressure or diabetes (males)</td>
<td style="text-align: left;">165,340</td>
<td style="text-align: left;">110,372</td>
<td style="text-align: left;">3.66e‑02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mother still alive</td>
<td style="text-align: left;">355,029</td>
<td style="text-align: left;">140,246</td>
<td style="text-align: left;">4.96e‑02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV21:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv21" class="tablenos">
<table id="tbl:sup:emerge_assocs:lv21">
<caption><span>Table 36:</span> Significant trait associations of LV21 in eMERGE.
<!-- LV21:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 13%" />
<col style="width: 38%" />
<col style="width: 18%" />
<col style="width: 10%" />
<col style="width: 19%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">272.1</td>
<td style="text-align: left;">Hyperlipidemia</td>
<td style="text-align: left;">55,843</td>
<td style="text-align: left;">29,195</td>
<td style="text-align: left;">4.22e‑03</td>
</tr>
<tr class="even">
<td style="text-align: left;">272</td>
<td style="text-align: left;">Disorders of lipoid metabolism</td>
<td style="text-align: left;">55,892</td>
<td style="text-align: left;">29,244</td>
<td style="text-align: left;">4.50e‑03</td>
</tr>
</tbody>
</table>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true },
    "TeX": {
      Macros: {
        bm: ["\\boldsymbol{#1}",1]
      }
    }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
