## Abstract {.page_break_before}

<!-- Tissue specificity is a key feature of human diseases. In this context, determining how genes influence these complex phenotypes requires mechanistically understanding expression regulation across different cell types, which in turn should lead to improved treatments. Integrating functional and GWAS data has improved the identification of these transcriptional mechanisms which, when dysregulated, commonly result in tissue- and cell lineage-specific pathology. However, widespread gene pleiotropy and polygenic traits reveal the highly interconnected nature of transcriptional networks, which complicates the interpretation of genetic effects and hampers translational efforts. We have developed a polygenic approach that maps both gene-trait associations and drug-transcriptional responses into a common representation based on tissue-specific gene co-expression patterns.

We integrated thousands of gene-trait associations (using TWAS from PhenomeXcan) and transcriptional profiles of drugs (LINCS L1000) into a low-dimensional representation learned from public gene expression data on tens of thousands of RNA-seq samples (recount2). This low-dimensional space comprised features representing groups of genes (gene modules) with coordinated expression across different tissues and cell types. When mapping gene-trait associations to this reduced expression space, we observed that diseases were significantly associated with gene modules expressed in relevant cell types, such as hypothyroidism with T cells and thyroid, coronary artery disease with cardiomyocytes, hypertension and lipids with adipose tissue, and heart problems with heart ventricle and muscle cells. We replicated gene module associations with cardiovascular and autoimmune diseases in the Electronic Medical Records and Genomics (eMERGE) network. We also performed a CRISPR-screen to analyze lipid regulation in HepG2 cells and observed more consistent trait associations with modules than we observe with individual genes. Compared to a single-gene approach, our module-based method also better predicted FDA-approved drug-disease links by capturing tissue-specific pathophysiological mechanisms linked with the mechanisms of action of drugs (e.g. niacin with cardiovascular traits via a known immune mechanism). Exploring the phenotype-module space also revealed stable trait clusters across different resolutions, including a complex branch involving lipids with cardiovascular, autoimmune, and neuropsychiatric disorders. We offer a novel gene module approach to enhance the understanding of complex diseases and their therapeutic modalities. -->
