## Abstract {.page_break_before}

Understanding how dysregulated transcriptional processes result in tissue-specific pathology requires a mechanistic interpretation of expression regulation across different cell types.
It has been shown that this insight is key for the development of new therapies.
These mechanisms can be identified with transcriptome-wide association studies (TWAS), which have represented a significant step forward to test the mediating role of gene expression in GWAS associations.
However, it is hard to disentangle causal cell types using eQTL data alone, and other methods generally do not use the large amounts of publicly available RNA-seq data.
Here we introduce PhenoPLIER, a polygenic approach that maps both gene-trait associations and pharmacological perturbation data into a common latent representation for a joint analysis.
This representation is based on modules of genes with similar expression patterns across the same tissues.
We observed that diseases were significantly associated with gene modules expressed in relevant cell types, and our approach was accurate in predicting known drug-disease pairs and inferring mechanisms of action.
Furthermore, using a CRISPR screen to analyze lipid regulation, we show that PhenoPLIER is more robust to prioritize drug targets than a standard single-gene method.
By incorporating groups of co-expressed genes, PhenoPLIER can contextualize genetic associations and reveal alternative targets for pharmacologic intervention when single disease-associated genes are not detected or are not druggable.
