## Introduction

<!--
# Things to remember:

1. There is some introduction in the BDS conference abstract, and also in the one for the CSHL that we didn't submit.
1. something like "identify the precise cell type, not only the tissue"

# Important papers


- mention here the funichelo (or something like that) about paper that eQTL are not useful to identify tissue of origin
From this paper, say that we shared the same principle by using gene expression data to infer context, and TWAS to find gene-disease associations (because of great sample size) (another way of saying it is imputed transcriptomes from GWAS)

- From Casey's R01:
"Tissue specificity is also a key feature of human disease and genes with tissue-specific expression are
enriched for disease associations 8–10"
LIST THOSE PAPERS HERE:
   - 8. Cai, J. J. & Petrov, D. A. Relaxed purifying selection and possibly high rate of adaptation in
   primate lineage-specific genes. Genome Biol. Evol. 2, 393–409 (2010).
   - 9. Winter, E. E., Goodstadt, L. & Ponting, C. P. Elevated rates of protein secretion, evolution, and
   disease among tissue-specific genes. Genome Res. 14, 54–61 (2004).
   - 10. Lage, K., Hansen, N. T., Karlberg, E. O., Eklund, A. C., Roque, F. S., Donahoe, P. K., Szallasi, Z.,
   Jensen, T. S. & Brunak, S. A large-scale analysis of tissue-specific pathology and gene
   expression of human disease genes and complexes. Proc. Natl. Acad. Sci. 105, 20870–20875
   (2008).

- Predicting genotype-specific gene regulatory networks
https://www.biorxiv.org/content/10.1101/2021.01.18.427134v1?rss=1
I like the ideas mentioned in the abstract and introduction of this paper.

- Identification of therapeutic targets from genetic association studies using hierarchical component analysis
Highly related paper.
See notes here https://drive.google.com/file/d/1C-6OGDxo3lG4_1b33prplsZ22r5hwjkq/view
Cite this paper about GWAS component, gene modules, and S-PrediXcan.
Some limitations of this method:
1. They find co-expression modules in GTEx only, we do it across recount2.
1. They use weighted correlation network analysis (WGCNA) to find modules,
   which seems simple.  We use PLIER with prior knowledge.
1. This is how they find a module: ``The eigen-gene component was then computed
   as the first principal component of the expression matrix of co-expressed
   genes''
1. ``Among 44 tissues analyzed, we generally detected 213±89 co-expression
   modules in one tissue.''
-->


Tissue specificity is a key feature of human disease and genes with tissue-specific expression are enriched for disease associations [@pmid:20624743; @pmid:14707169; @doi:10.1073/pnas.0810772105].
<!--  -->
Identifying the function of genes involves understanding the regulatory mechanisms that affect their expression across different tissues and cell types [@doi:10.1126/science.aaz1776; @doi:10.1038/s41586-020-2559-3; @doi:10.1038/s41576-019-0200-9].
<!--  -->
Large compendia describing key elements of regulatory DNA have been recently released or updated, which comprise chromatin-state annotations, high-resolution enhancers [@doi:10.1038/s41586-020-03145-z], DNase I hypersensitive sites maps [@doi:10.1038/s41586-020-2559-3], and the characterization of genetic effects on gene expression across different tissues [@doi:10.1126/science.aaz1776].
<!-- were recen [] and  have been made available to unravel pathophysiological mechanisms. -->
<!-- This is extremely important  given that most disease-associated loci are non-coding. -->
<!--  -->
The integration with genome-wide association studies (GWAS) on thousands of common diseases could dramatically improve the identification of these transcriptional mechanisms that, when dysregulated, often result in tissue- and cell lineage-specific pathology.


Owing to the readily available gene expression data across several tissues [@doi:10.1038/nbt.3838; @doi:10.1038/s41467-018-03751-6; @doi:10.1126/science.aaz1776], a popular approach to identify these biological processes is transcription-wide association studies (TWAS), which integrates expression quantitative trait loci (eQTL) data to provide a mechanistic interpretation for genome-wide association studies (GWAS).
<!-- Transcription-wide association studies (TWAS) integrate expression quantitative trait loci (eQTL) data to provide a mechanistic interpretation for genome-wide association studies (GWAS). -->
This is done by testing whether perturbations in gene regulatory mechanisms mediate the association between genetic variants and human diseases [@doi:10.1371/journal.pgen.1009482; @doi:10.1038/ng.3506; @doi:10.1371/journal.pgen.1007889; @doi:10.1038/ng.3367].
<!--  -->
However, TWAS has not been useful to detect tissue-specific effects [@doi:10.1016/j.ajhg.2017.01.031; @doi:10.1038/s41588-018-0081-4], since eQTLs are generally shared across tissues.
<!-- TODO: Question: Y el paper de mashr? https://www.nature.com/articles/s41588-018-0268-8#Sec12 -->
<!--  -->
Alternative statistical approaches that connect GWAS with gene expression data can infer disease-relevant tissues and cell types [@doi:10.1038/s41588-018-0081-4; @doi:10.1016/j.ajhg.2011.09.002; @doi:10.1093/bioinformatics/btu326; @doi:10.1038/ng.3598; @doi:10.1038/ncomms6890], but they generally apply enrichment analysis techniques that do not account for widespread gene correlations due to technical noise (i.e. "batch effects") [@doi:10.1038/nrg2825; @doi:10.1038/s41592-019-0456-1].
<!--  -->
In addition, they generally rely on small sets of expression data compared with the total RNA-seq samples available today [@doi:10.1038/s41467-018-03751-6; @doi:10.1038/nbt.3838].


<!-- mention number of tissues/cell types/ samples in recount2 -->

Here we propose a polygenic method that maps gene associations from TWAS on >4,000 traits [PHENOMEXCAN] into a latent representation learned from public gene expression repositories on >70,000 RNA-seq human samples [MULTIPLIER; RECOUNT2].
This low-dimensional space comprises latent variables representing gene modules with coordinated expression across different tissues and cell types.
We used a computational approach that can reduce technical noise by learning patterns that align to prior knowledge.
When mapping gene-trait associations to this reduced expression space, we found that traits and diseases are associated with gene modules expressed in relevant cell types.
Our approach is also more robust in finding meaningful module-trait associations even when individual genes involved in lipids metabolism do not reach genome-wide significance in lipids-related traits.
We also show that our module-based approach is more accurate in predicting known drug-disease association than using single gene-traits associations, and that our approach could be also useful to study mechanisms of action of drugs.
Finally, we performed cluster analysis on traits mapped to this latent representation, with clusters highly stable across different resolutions.
We found common and specific transcriptional processes associated with autoimmune and cardiovascular diseases.

<!--
Some ideas I still have to think about/read more:

- From the tissue specificity review, I liked the idea that "disease-associated genes are typically expressed widely across tissues".
Maybe I can something like: "One approach to disentangle this complex transcriptional mechinery is to consider genes that are co-expressed across different cell types".
- Take a look at the Biology of Genomes abstract; there are some ideas there that I might consider to mention here. I left a paragraph below, a first version of this introduction.
 -->





<!--
Adjusted from the Biology of Genomes abstract:

Transcription-wide association studies (TWAS) provide a mechanistic interpretation of genome-wide association studies (GWAS) by testing whether perturbations in gene regulatory mechanisms mediate the association between genetic variants and human diseases [@doi:10.1371/journal.pgen.1009482; @doi:10.1038/ng.3506; @doi:10.1371/journal.pgen.1007889; @doi:10.1038/ng.3367].
TWAS methods integrate expression quantitative trait loci (eQTL) with GWAS to compute gene-level associations and prioritize potential therapeutic targets [@doi:10.1038/s41588-019-0385-z; @doi:10.1038/nn.4618].
Yet the typical TWAS approach looks at one gene at a time, missing the fact that genes act in concert to carry out different functions.
Methods that consider gene-sets of functionally related genes can have more power to detect associations even when individual genes are not genome-wide significant [@doi:10.1073/pnas.0506580102; @doi:10.1371/journal.pcbi.1004219; @doi:10.1158/0008-5472.CAN-09-4502; @doi:10.1038/nrg2884; @doi:10.1038/s41467-018-06022-6], however, they are typically confined to already known pathways and do not incorporate the effects of eQTLs in the transcriptome.

Another part:

Pathway analysis considers groups of functionally related genes when studying a phenotype of interest, and it has been shown that this gene-set-based approach can have more power to detect associations even when individual genes are not gnenome-wide significant [@doi:10.1158/0008-5472.CAN-09-4502; @doi:10.1038/nrg2884; @doi:].

-->

`Notes:`{.red}

::: {style="color: red"}
- We need to say more about the clustering of traits, and select a good example to summarize our results.
- I'm not including eMERGE replication here because I still have to work on that part.

Minor:

- I don't like much the idea of mentioning "module-trait associations", since that conveys the idea of a p-value, which we are not giving.
- Reviewers might ask for the sample sizes of GWAS.
Some of them, in PhenoPLIER, are really underpowered, with very few cases;
others are from large studies.
We should be carefull when picking our examples in the Results section.
:::
