## Introduction

Tissue specificity is a key feature of human disease, and genes with tissue-specific expression are enriched for disease associations [@pmid:20624743; @pmid:14707169; @doi:10.1073/pnas.0810772105].
<!--  -->
Identifying the function of genes involves understanding the regulatory mechanisms that affect their expression across different tissues and cell types [@doi:10.1126/science.aaz1776; @doi:10.1038/s41586-020-2559-3; @doi:10.1038/s41576-019-0200-9].
<!--  -->
Large compendia describing key elements of regulatory DNA have been recently released or updated, which comprise chromatin-state annotations, high-resolution enhancers [@doi:10.1038/s41586-020-03145-z], DNase I hypersensitive sites maps [@doi:10.1038/s41586-020-2559-3], and the characterization of genetic effects on gene expression across different tissues [@doi:10.1126/science.aaz1776].
<!--  -->
The integration with genome-wide association studies (GWAS) on thousands of common diseases could dramatically improve the identification of these transcriptional mechanisms that, when dysregulated, often result in tissue- and cell lineage-specific pathology.


Owing to the readily available gene expression data across several tissues [@doi:10.1038/nbt.3838; @doi:10.1038/s41467-018-03751-6; @doi:10.1126/science.aaz1776; @doi:10.1186/s13040-020-00216-9], a popular approach to identify these biological processes is transcription-wide association studies (TWAS), which integrates expression quantitative trait loci (eQTL) data to provide a mechanistic interpretation for genome-wide association studies (GWAS).
<!--  -->
This is done by testing whether perturbations in gene regulatory mechanisms mediate the association between genetic variants and human diseases [@doi:10.1371/journal.pgen.1009482; @doi:10.1038/ng.3506; @doi:10.1371/journal.pgen.1007889; @doi:10.1038/ng.3367].
<!--  -->
However, TWAS has not been useful to detect tissue-specific effects [@doi:10.1016/j.ajhg.2017.01.031; @doi:10.1038/s41588-018-0081-4], since eQTLs are generally shared across tissues.
<!--  -->
Alternative statistical approaches that connect GWAS with gene expression data can infer disease-relevant tissues and cell types [@doi:10.1038/s41588-018-0081-4; @doi:10.1016/j.ajhg.2011.09.002; @doi:10.1093/bioinformatics/btu326; @doi:10.1038/ng.3598; @doi:10.1038/ncomms6890; @doi:10.1038/ng.3981], but they generally apply enrichment analysis techniques that do not account for widespread gene correlations due to technical noise (i.e. "batch effects") [@doi:10.1038/nrg2825; @doi:10.1038/s41592-019-0456-1].
<!--  -->
In addition, they generally rely on small sets of expression data compared with the total RNA-seq samples available today [@doi:10.1038/s41467-018-03751-6; @doi:10.1038/nbt.3838].


Here we propose a polygenic method that maps gene associations from TWAS on >4,000 traits [@doi:10.1126/sciadv.aba2083] into a latent representation learned from public gene expression repositories on tens of thousands of RNA-seq human samples [@doi:10.1016/j.cels.2019.04.003; @doi:10.1038/nbt.3838].
<!--  -->
This low-dimensional space comprises latent variables representing gene modules with coordinated expression across different tissues and cell types.
We used a computational approach that can reduce technical noise by learning patterns that align with prior knowledge.
<!--  -->
When mapping gene-trait associations to this reduced expression space, we found that traits and diseases are associated with gene modules expressed in relevant cell types.
Several of these associations found in PhenomeXcan also replicated in 309 curated disease phenotypes from the Electronic Medical Records and Genomics (eMERGE) network phase III [@doi:10.1038/gim.2013.72].
<!--  -->
Our approach is robust in finding meaningful module-trait associations even when individual genes involved in lipids metabolism do not reach genome-wide significance in lipids-related traits.
<!--  -->
We also show that our module-based approach is more accurate in predicting known drug-disease medical indications than using single gene-traits associations, potentially representing a more potent drug repositioning framework.
<!--  -->
Finally, we performed cluster analysis on traits mapped to this latent representation, with clusters highly stable across different resolutions.
We were able to identify highly disease-specific gene modules expressed in relevant tissues for a wide range of complex diseases, including autoimmune, cardiovascular, and mental disorders.
