## Introduction

<!--
Things to remember:

1. Define GTEx
-->

<!--
I like the abstract and introduction of this paper below, I might borrow some ideas:

Predicting genotype-specific gene regulatory networks
https://www.biorxiv.org/content/10.1101/2021.01.18.427134v1?rss=1
-->

<!--
From the Biological Data Science abstract:

Transcription-wide association studies (TWAS) provide a mechanistic
interpretation of genome-wide association studies (GWAS) results by testing
whether perturbations in gene regulatory mechanisms mediate the association
between genetic variants and human diseases. TWAS methods integrate expression
quantitative trait loci (eQTL) with GWAS results to calculate gene-level
associations and prioritize therapeutic targets. However, the typical TWAS
approach looks at one gene at a time, missing the fact that genes act in
concert to carry out different functions. We propose to interpret TWAS results
from a gene-module level (co-expression pattern), which can suggest alternative
treatment options when directly disease-associated genes are not druggable.
-->

<!--
Cite this paper about GWAS component, gene modules, and S-PrediXcan:
https://drive.google.com/file/d/1C-6OGDxo3lG4_1b33prplsZ22r5hwjkq/view

Some limitations of this method:

1. They find co-expression modules in GTEx only, we do it across recount2.
1. They use weighted correlation network analysis (WGCNA) to find modules,
   which seems simple.  We use PLIER with prior knowledge.
1. This is how they find a module: ``The eigen-gene component was then computed
   as the first principal component of the expression matrix of co-expressed
   genes''
1. ``Among 44 tissues analyzed, we generally detected 213Â±89 co-expression
   modules in one tissue.''
-->

