## Introduction

<!--
# Things to remember:

1. There is some introduction in the BDS conference abstract, and also in the one for the CSHL that we didn't submit.
1. something like "identify the precise cell type, not only the tissue"

# Important papers


- mention here the funichelo (or something like that) about paper that eQTL are not useful to identify tissue of origin
From this paper, say that we shared the same principle by using gene expression data to infer context, and TWAS to find gene-disease associations (because of great sample size) (another way of saying it is imputed transcriptomes from GWAS)

- From Casey's R01:
"Tissue specificity is also a key feature of human disease and genes with tissue-specific expression are
enriched for disease associations 8–10"
LIST THOSE PAPERS HERE:
   - 8. Cai, J. J. & Petrov, D. A. Relaxed purifying selection and possibly high rate of adaptation in
   primate lineage-specific genes. Genome Biol. Evol. 2, 393–409 (2010).
   - 9. Winter, E. E., Goodstadt, L. & Ponting, C. P. Elevated rates of protein secretion, evolution, and
   disease among tissue-specific genes. Genome Res. 14, 54–61 (2004).
   - 10. Lage, K., Hansen, N. T., Karlberg, E. O., Eklund, A. C., Roque, F. S., Donahoe, P. K., Szallasi, Z.,
   Jensen, T. S. & Brunak, S. A large-scale analysis of tissue-specific pathology and gene
   expression of human disease genes and complexes. Proc. Natl. Acad. Sci. 105, 20870–20875
   (2008).

- Predicting genotype-specific gene regulatory networks
https://www.biorxiv.org/content/10.1101/2021.01.18.427134v1?rss=1
I like the ideas mentioned in the abstract and introduction of this paper.

- Identification of therapeutic targets from genetic association studies using hierarchical component analysis
Highly related paper.
See notes here https://drive.google.com/file/d/1C-6OGDxo3lG4_1b33prplsZ22r5hwjkq/view
Cite this paper about GWAS component, gene modules, and S-PrediXcan.
Some limitations of this method:
1. They find co-expression modules in GTEx only, we do it across recount2.
1. They use weighted correlation network analysis (WGCNA) to find modules,
   which seems simple.  We use PLIER with prior knowledge.
1. This is how they find a module: ``The eigen-gene component was then computed
   as the first principal component of the expression matrix of co-expressed
   genes''
1. ``Among 44 tissues analyzed, we generally detected 213±89 co-expression
   modules in one tissue.''
-->
