### Integrating gene expression patterns with transcription-wide association studies

::: {style="color: red"}
Here we introduce our framework to perform the integration.

 1. Brief background on MultiPLIER and PhenomeXcan.
 1. Explanation of the framework depicted in Figure @fig:entire_process and its components.
 1. Description of initial set of results matching previous findings (neutrophil counts).
:::


![
**Schematic of the PhenoPLIER framework.**
<!--  -->
The integration process (middle) between MultiPLIER [@doi:10.1016/j.cels.2019.04.003] (top) and
PhenomeXcan [@doi:10.1101/833210] (bottom) outputs matrix $\mathbf{T}$. LV603 was previously
found to be strongly associated with neutrophil estimates when projecting a dataset of systemic
lupus erythematosus (SLE) whole blood (WB) into MultiPLIER. After integration, this LV is also
highly associated with blood count traits, with neutrophil counts at the top.
<!--  -->
](images/entire_process/entire_process.svg?sanitize=true "PhenoPLIER
framework"){#fig:entire_process height=5in}

Recently, a transfer learning approach called MultiPLIER [@doi:10.1016/j.cels.2019.04.003] was proposed where an
unsupervised model is trained on large gene expression compendia to extract patterns of co-expressed
genes. These gene modules can be later used in smaller rare disease datasets to effectively
identify biological processes related to disease severity. It was shown how these gene sets aligned
well with known biological pathways, as well as accurately predict cell type composition even though
it was not designed specifically for that.
<!--  -->
Although the models built using MultiPLIER aligned well to known biological factors, most of the
gene modules identified are completely unknown. Since genes in these modules vary together, it's
expected that they may also function together [@pubmed:16869964; @doi:10.1186/s13059-019-1835-8].
<!--  -->
To test whether patterns in the expression space match those in the TWAS space, we designed an
initial integration framework (Figure @fig:entire_process), where the general procedure
consists in projecting
<!--  -->
MultiXcan
<!--  -->
results (from PhenomeXcan resources) to the low-dimensional gene representation learned by MultiPLIER:

$$\mathbf{T} = (\mathbf{Z}^{\top} \mathbf{Z} + \lambda_{2} \mathbf{I})^{-1} \mathbf{Z}^{\top} \mathbf{M},$$ {#eq:projection}

where $\mathbf{M}$ is a matrix with the standardized effect sizes estimated by PhenomeXcan-MultiXcan,
and the rest of the variables come from MultiPLIER models: $\mathbf{Z}$ is the gene
loadings matrix (with genes in the rows and latent variables (LVs) in the columns), $\lambda_2$ is the
regularization parameter used in the training step, and $\mathbf{I}$ the identity matrix.
<!--  -->
Similarly to the original idea of transferring knowledge from a source domain to a target domain,
these steps can be thought of as the projection of the TWAS space into the gene expression latent
space, which results in matrix $\mathbf{T}$: all LVs are now described by traits from
PhenomeXcan, thus we can now infer the function of gene modules in $\mathbf{Z}$ from their association
to traits in $\mathbf{M}$.
<!--  -->
For all results analyzed in this section the projection procedure to obtain $\mathbf{T}$ was exactly
as follows:
1) keep genes in common between $\mathbf{Z}$ and $\mathbf{M}$ only (this led to 6,452 genes);
2) analyze the strongest signal from each LV in $\mathbf{Z}$ by taking the top 1\% of genes with
nonzero loadings values ($\mathbf{Z}_{i,j} > 0$, where $i$ are genes and $j$ latent variables);
this led to a median of 27 genes per LV;
3) project PhenomeXcan-MultiXcan results into the gene expression latent space using
Equation @eq:projection.
