### PhenoPLIER: an integration framework based on gene co-expression patterns

![
**Schematic of the PhenoPLIER framework.**
**a)** High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (genetic variants).
PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations.
**b)** The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top).
This process generates matrices $\mathbf{\hat{L}}$ and $\mathbf{\hat{M}}$, where LVs now describe each drug/trait.
In the middle, we show the three main computational components provided by PhenoPLIER to perform this integration:
1) an LV-based regression model,
2) a clustering framework to learn groups of traits,
and 3) an LV-based drug repurposing approach.
**c)** LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603 after projecting gene-trait associations in PhenomeXcan).
Genes that are part of LV603 were expressed in relevant cell types (top).
PBMC: peripheral blood mononuclear cells;
mDCs: myeloid dendritic cells.
](images/entire_process/entire_process.svg "PhenoPLIER framework"){#fig:entire_process width="100%"}


PhenoPLIER is a computational framework that combines gene-trait and gene-drug associations with gene modules expressed in specific contexts (Figure {@fig:entire_process}a).
The approach uses a latent representation (with latent variables or LVs representing gene modules) derived from a large gene expression compendium (Figure {@fig:entire_process}b, top) to integrate TWAS with drug-induced transcriptional responses (Figure {@fig:entire_process}b, bottom).
It consists of three main components (Figure {@fig:entire_process}b, middle): 1) an LV-based regression model to calculate an association between an LV and a trait, 2) a clustering framework to identify groups of traits with shared transcriptomic properties, and 3) an LV-based drug repurposing approach to link diseases to potential treatments.
We conducted simulations for our regression model ([Supplementary Note 1](#sm:reg:null_sim)) and clustering framework ([Supplementary Note 2](#sm:clustering:null_sim)) to ensure proper calibration and expected results when no association is present.


We used two statistical methods, Summary-MultiXcan (S-MultiXcan) and Summary-PrediXcan (S-PrediXcan), to derive TWAS results from PhenomeXcan [@doi:10.1126/sciadv.aba2083] and the eMERGE network [@doi:10.1101/2021.10.21.21265225], which were used as discovery and replication cohorts, respectively ([Methods](#sec:methods:twas)).
The PhenomeXcan provides gene-trait associations for 4,091 diseases and traits from the UK Biobank [@doi:10.1038/s41586-018-0579-z] and other studies, while the analyses on eMERGE were performed across 309 phecodes.
S-MultiXcan associations were used for the regression and clustering components, and S-PrediXcan associations were used for the drug repurposing component.
In addition, we also used colocalization results, which give the probability of overlap between the GWAS and eQTL signals.
For the drug-repurposing approach, we used transcriptional responses to small molecule perturbations from LINCS L1000 [@doi:10.1016/j.cell.2017.10.049], which comprises 1,170 compounds.


The MultiPLIER models were used to obtain a latent gene expression representation, which was derived by applying a matrix factorization method (the pathway-level information extractor or PLIER) to recount2 - a uniformly-curated collection of transcript-level gene expression quantified by RNA-seq in a large, diverse set of samples.
These samples were collected across a range of disease states, cell types, differentiation stages, and various stimuli (see [Methods](#sec:methods:multiplier)).
The MultiPLIER models optimized data reconstruction and the alignment of LVs (987) with prior knowledge/pathways.


<!--  -->
Our analysis of the latent variable LV603 revealed a link between transcriptional mechanisms and complex traits.
We found that LV603 was associated with a known neutrophil pathway and highly correlated with neutrophil count estimates from whole blood RNA-seq profiles [@doi:10.1186/s13059-016-1070-5].
We used PhenoPLIER to project different data modalities into the same space, and found that neutrophil counts, basophil counts, and basophil percentages were among the top 10 traits associated with this LV.
Additionally, we observed that the genes in LV603 were expressed in relevant cell types (Figure {@fig:entire_process}c).
These results suggest that groups of functionally related and co-expressed genes can be linked to groups of trait-associated genes.


PhenoPLIER enables users to answer a range of questions related to genetic studies, functional genomics, and gene co-expression.
These include whether disease-associated genes belong to modules expressed in particular tissues and cell types, whether cell type-specific modules are associated with different diseases, and if there is a subset of module genes that are closer to the definition of “core” genes.
Additionally, PhenoPLIER can be used to identify drugs that perturb transcriptional mechanisms and suggest potential mechanisms of action.
This can be useful for therapeutic targets and drug repurposing, as well as for clustering complex traits.
