### PhenoPLIER: an integration framework based on gene co-expression patterns

`NOTES:`{.Red}
- in methods I refer to PhenomeXcan as discovery cohort
- and eMERGE as replication cohort

![
**Schematic of the PhenoPLIER framework.**
**a)** High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (genetic variants).
PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations.
**b)** The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top).
The process generates matrix $\mathbf{\hat{M}}$, where each trait/drug is now described by LVs (only traits are shown).
**c)** LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603).
Genes in LV603 were expressed in relevant cell types (top).
PBMC: peripheral blood mononuclear cells;
mDCs: myeloid dendritic cells.
](images/entire_process/entire_process.svg "PhenoPLIER framework"){#fig:entire_process width="100%"}


PhenoPLIER combines TWAS and drug-induced transcriptional responses with gene co-expression patterns by projecting gene-trait and gene-drug associations onto a latent gene expression representation (Figure @fig:entire_process).
We used PhenomeXcan [@doi:10.1126/sciadv.aba2083], a TWAS resource for the UK Biobank [@doi:10.1038/s41586-018-0579-z] and other cohorts with 4,091 different diseases and traits, and the transcriptional responses to small molecule perturbations from LINCS L1000 [@doi:10.1016/j.cell.2017.10.049] comprising 1,170 compounds.
We obtained a latent gene expression representation from MultiPLIER [@doi:10.1016/j.cels.2019.04.003], an unsupervised learning approach applied to recount2 [@doi:10.1038/nbt.3838] -- a uniformly-curated collection of transcript-level gene expression quantified by RNA-seq in a large, diverse set of samples collected across a range of disease state, cell types differentiation stages, and various stimuli (Methods).
MultiPLIER extracted 987 latent variables (LV) by optimizing data reconstruction but also the alignment of LVs with prior knowledge/pathways (Methods).
Each LV or gene module represents a group of weighted genes expressed together in the same tissues and cell types as a functional unit.
Since LVs might represent a functional set of genes regulated by the same transcriptional program [@doi:10.1186/1471-2164-7-187; @doi:10.1186/s13059-019-1835-8], we conjecture that the projection of TWAS and pharmacologic perturbations data into this latent space could provide context for their interpretation.
PhenoPLIER converts gene associations to an LV score:
all genes' standardized effect sizes for a trait (from TWAS) or differential expression values for a drug (from pharmacologic perturbation data) are multiplied by the LV genes' weights and summed, producing a single value.
This process links different traits and drugs to LVs representing cell types and tissues, even at specific developmental stages or under distinct stimuli.
Examining these LVs is possible because the MultiPLIER's models link to samples, which may be annotated for experimental conditions (represented by matrix $\mathbf{B}$ in Figure @fig:entire_process b, top) in which genes in an LV are expressed.
To further analyze LV-trait associations, we adapted the gene-property approach from MAGMA [@doi:10.1371/journal.pcbi.1004219] to compute an LV-trait $p$-value by fitting a regression model with LV weights as predictors of a trait's gene associations.
Thus PhenoPLIER allows the user to address specific questions, namely:
do disease-associated genes belong to modules expressed in specific tissues and cell types?
Are these cell type-specific modules associated with different diseases (thus, potentially representing a "network pleiotropy" example from an omnigenic point of view [@doi:10.1016/j.cell.2017.05.038])?
Is there a subset of module's genes that is closer to the definition of "core" genes (i.e., directly affecting the trait with no mediated regulation of other genes [@doi:10.1016/j.cell.2019.04.014]) and thus represent alternative and potentially better candidate targets?
Are drugs perturbing these transcriptional mechanisms, and if so, can they suggest potential mechanisms of action?


In the original MultiPLIER study, the authors reported that one of the latent variables, identified as LV603, was associated with a known neutrophil pathway and highly correlated with neutrophil count estimates from whole blood RNA-seq profiles [@doi:10.1186/s13059-016-1070-5].
We analyzed LV603 using PhenoPLIER (Figure @fig:entire_process c) and found that
1) neutrophil counts and other white blood cell traits were ranked among the top 10 traits out of 4,091 (Figure @fig:entire_process c, bottom) and significantly associated with this LV (Supplementary Table @tbl:sup:phenomexcan_assocs:lv603) when using the gene-property approach,
and 2) LV603's genes were expressed in highly relevant cell types (Figure @fig:entire_process c, top).
These results suggested that groups of functionally related and co-expressed genes tend to correspond to groups of trait-associated genes.
Thus, the gene expression space (which has no GTEx samples) and the TWAS space (with gene models trained using GTEx v8) contained shared patterns, and the approach can link transcriptional mechanisms from large and diverse dataset collections to complex traits.
