### PhenoPLIER integrates TWAS with gene co-expression patterns

::: {style="color: red"}
Here we introduce the idea and some preliminary results showing that it works.
:::

![
**Schematic of the PhenoPLIER framework.**
<!--  -->
**a)** The integration process between gene co-expression patterns from
MultiPLIER [@doi:10.1016/j.cels.2019.04.003] (top) and TWAS results from
PhenomeXcan [@doi:10.1126/sciadv.aba2083]. It projects gene-based
association results on \~4,000 traits to a latent space learned from a
large gene expression compendium (recount2 [@doi:10.1038/nbt.3838]). This
generates matrix $\mathbf{\hat{M}}$, where each trait is now described by
latent variables/gene modules.
<!--  -->
**b)** After the integration process, we found that neutrophil counts and
other white blood cells were ranked among the top 10 traits for an LV that was
termed a neutrophil signature in the original MultiPLIER study.
`Improve cell types/tissues figure`{.red}
<!--  -->
](images/entire_process/entire_process.svg?sanitize=true "PhenoPLIER
framework"){#fig:entire_process height=5in}


MultiPLIER [@doi:10.1016/j.cels.2019.04.003] is a recent computational strategy
that extracts patterns of co-expressed genes from large gene-expression
datasets. The approach uses an unsupervised matrix factorization method that was
employed to extract latent variables from recount2 [@doi:10.1038/nbt.3838].
<!--  -->
The latent variables (LVs), essentially gene modules, then revealed biological
processes associated with disease severity in rare disease datasets that were
too small for effective model training. These gene sets aligned well with known
biological pathways and predicted cell type composition, even though the
approach was not explicitly designed for this goal.


Although the authors showed that certain patterns learned by MultiPLIER resemble
known biology, most of the LVs identified are completely unknown. Since genes in
these modules vary together in certain cell types and tissues, it's expected
that they may also function together [@doi:10.1186/1471-2164-7-187;
@doi:10.1186/s13059-019-1835-8].
<!--  -->
To test whether patterns in the expression space match those in the TWAS space,
we used PhenomeXcan [@doi:10.1126/sciadv.aba2083], a massive transcriptome-wide
association studies (TWAS) resource
obtained from the UK Biobank [@doi:10.1038/s41586-018-0579-z] and other cohorts
(Figure @fig:entire_process).
These results were projected to the low-dimensional gene representation learned
by MultiPLIER using:

$$\hat{\mathbf{M}} = (\mathbf{Z}^{\top} \mathbf{Z} + \lambda_{2} \mathbf{I})^{-1} \mathbf{Z}^{\top} \mathbf{M},$$ {#eq:proj}

where $\mathbf{M}^{n \times t}$ has gene-trait associations from MultiXcan
[@doi:10.1371/journal.pgen.1007889] (standardized effect sizes) for $n$ genes
and $t$ traits, $\mathbf{Z}^{n \times l}$ are the gene loadings with $l$ latent
variables, $\lambda_2$ is the regularization parameter used in the training
step, and $\hat{\mathbf{M}}^{l \times t}$ is finally the projection of
$\mathbf{M}$ into the latent space: all traits in PhenomeXcan are now described
by LVs, thus we can potentially infer the effects of gene modules on different
human traits. Since the MultiPLIER models also provide the conditions (such as
cell types and tissues) in which genes in a module are concurringly expressed,
our approach would also allow to infer the context in which the gene module
potentially affect a trait or disease.


::: {style="color: red"}
- Now talk about the right side of Figure 1
- Mention null test of LV603 with neutrophils
- Mention LINCS and the possibility to use other datatypes as well with
  MultiPLIER?
:::
