### PhenoPLIER: an integration framework based on gene co-expression patterns

![
**Schematic of the PhenoPLIER framework.**
**a)** High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (genetic variants).
PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations.
**b)** The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top).
This process generates matrices $\mathbf{\hat{L}}$ and $\mathbf{\hat{M}}$, where LVs now describe each drug/trait.
In the middle, we show the three main computational components provided by PhenoPLIER to perform this integration:
1) an LV-based regression model,
2) a clustering framework to learn groups of traits,
and 3) an LV-based drug repurposing approach.
**c)** LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603 after projecting gene-trait associations in PhenomeXcan).
Genes that are part of LV603 were expressed in relevant cell types (top).
PBMC: peripheral blood mononuclear cells;
mDCs: myeloid dendritic cells.
](images/entire_process/entire_process.svg "PhenoPLIER framework"){#fig:entire_process width="100%"}


PhenoPLIER is a flexible computational framework that combines gene-trait and gene-drug associations with gene modules expressed in specific contexts (Figure {@fig:entire_process}a).
The approach uses a latent representation (with latent variables, or LVs, representing gene modules) to integrate TWAS with drug-induced transcriptional responses (Figure {@fig:entire_process}b).
This approach consists of three main components (Figure {@fig:entire_process}b, see [Methods](#sec:methods)): 1) an LV-based regression model to compute an association between an LV and a trait, 2) a clustering framework to learn groups of traits with shared transcriptomic properties, and 3) an LV-based drug repurposing approach that links diseases to potential treatments.
We conducted extensive simulations for our regression model ([Supplementary Note 1](#sm:reg:null_sim)) and clustering framework ([Supplementary Note 2](#sm:clustering:null_sim)) to ensure proper calibration and expected results under a model of no association.


We used two cohorts for our study: PhenomeXcan [@doi:10.1126/sciadv.aba2083] and the eMERGE network [@doi:10.1101/2021.10.21.21265225].
PhenomeXcan provides gene-trait associations for 4,091 different diseases and traits from the UK Biobank [@doi:10.1038/s41586-018-0579-z], while the eMERGE network was used to analyze 309 phecodes.
Two statistical methods (see [Methods](#sec:methods:predixcan)) were used to derive TWAS results: Summary-MultiXcan (S-MultiXcan) associations for regression and clustering, and Summary-PrediXcan (S-PrediXcan) associations for drug repurposing.
Additionally, colocalization results were used to measure the overlap between GWAS and eQTL signals.
For drug repurposing, we used transcriptional responses to small molecule perturbations from LINCS L1000 [@doi:10.1016/j.cell.2017.10.049], which includes 1,170 compounds.


The MultiPLIER models [@doi:10.1016/j.cels.2019.04.003] were derived from a matrix factorization method (the pathway-level information extractor or PLIER [@doi:10.1038/s41592-019-0456-1]) applied to recount2 [@doi:10.1038/nbt.3838], a uniformly-curated collection of transcript-level gene expression quantified by RNA-seq in a large, diverse set of samples collected across a range of disease states, cell types, differentiation stages, and various stimuli (see [Methods](#sec:methods:multiplier)).
This resulted in a latent gene expression representation with 987 latent variables (LVs) optimized for data reconstruction and alignment with prior knowledge/pathways.


<!--  -->
By using PhenoPLIER, we can project different data modalities into the same space and convert gene associations to a single latent variable (LV) score.
This LV score is a weighted sum of all genes' standardized effect sizes for a trait (from TWAS) or differential expression values for a drug (from pharmacologic perturbation data).
Instead of looking at individual genes, this process links different traits and drugs to functionally-related groups of genes or LVs.
Moreover, PhenoPLIER can use annotations about the specific conditions where the group of genes is expressed, such as cell types and tissues, even at specific developmental stages, disease stages or under distinct stimuli. 

As an example, in the original MultiPLIER study, the authors reported that one of the latent variables, identified as LV603, was associated with a known neutrophil pathway and highly correlated with neutrophil count estimates from whole blood RNA-seq profiles.
When we analyzed LV603 using PhenoPLIER, we found that 1) neutrophil counts and other white blood cell traits were ranked among the top 10 traits out of 4,091, and basophils count and percentage were significantly associated with this LV when using our regression method, and 2) LV603's genes were expressed in highly relevant cell types.
These results suggest that groups of functionally related and co-expressed genes tend to correspond to groups of trait-associated genes, and the approach can link transcriptional mechanisms from large and diverse dataset collections to complex traits.


PhenoPLIER allows the user to address specific questions related to the genetic basis of diseases.
For example, it can be used to determine if disease-associated genes belong to modules expressed in specific tissues and cell types, whether these cell type-specific modules are associated with different diseases, and whether there is a subset of module's genes that might be considered "core" genes which directly affect the trait.
Furthermore, it can be used to identify drugs that perturb transcriptional mechanisms and suggest potential mechanisms of action.
