### PhenoPLIER: an integration framework based on gene co-expression patterns

![
**Schematic of the PhenoPLIER framework.**
**a)** High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (genetic variants).
PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations.
**b)** The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top).
This process generates matrices $\mathbf{\hat{L}}$ and $\mathbf{\hat{M}}$, where LVs now describe each drug/trait.
In the middle, we show the three main computational components provided by PhenoPLIER to perform this integration:
1) an LV-based regression model,
2) a clustering framework to learn groups of traits,
and 3) an LV-based drug repurposing approach.
**c)** LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603 after projecting gene-trait associations in PhenomeXcan).
Genes that are part of LV603 were expressed in relevant cell types (top).
PBMC: peripheral blood mononuclear cells;
mDCs: myeloid dendritic cells.
](images/entire_process/entire_process.svg "PhenoPLIER framework"){#fig:entire_process width="100%"}


PhenoPLIER is a computational framework that combines gene-trait and gene-drug associations with gene modules expressed in specific contexts (Figure {@fig:entire_process}a).
The approach uses a latent representation (with latent variables or LVs representing gene modules) derived from a large gene expression compendium (Figure {@fig:entire_process}b, top) to integrate TWAS with drug-induced transcriptional responses (Figure {@fig:entire_process}b, bottom) for a joint analysis.
This approach consists of three main components (Figure {@fig:entire_process}b, middle).
These components include a regression model to compute an association between an LV and a trait, a clustering framework to learn groups of traits with shared transcriptomic properties, and an LV-based drug repurposing approach that links diseases to potential treatments (see [Methods](#sec:methods)).
We conducted simulations for our regression model ([Supplementary Note 1](#sm:reg:null_sim)) and clustering framework ([Supplementary Note 2](#sm:clustering:null_sim)) to ensure accuracy and expected results under a model of no association.


We used TWAS results from PhenomeXcan [@doi:10.1126/sciadv.aba2083] and the eMERGE network [@doi:10.1101/2021.10.21.21265225] as discovery and replication cohorts, respectively ([Methods](#sec:methods:twas)).
PhenomeXcan provides gene-trait associations for 4,091 different diseases and traits from the UK Biobank [@doi:10.1038/s41586-018-0579-z], while the eMERGE network was analyzed across 309 phecodes.
We used two statistical methods for the TWAS results (see [Methods](#sec:methods:predixcan)): Summary-MultiXcan (S-MultiXcan) associations for the regression and clustering components, and Summary-PrediXcan (S-PrediXcan) associations for the drug repurposing component.
We also used colocalization results to assess the probability of overlap between the GWAS and eQTL signals.
For drug repurposing, we used transcriptional responses to small molecule perturbations from LINCS L1000 [@doi:10.1016/j.cell.2017.10.049], comprising 1,170 compounds.


The MultiPLIER models were derived by applying a matrix factorization method (the pathway-level information extractor or PLIER [@doi:10.1038/s41592-019-0456-1]) to recount2 [@doi:10.1038/nbt.3838], a uniformly-curated collection of transcript-level gene expression quantified by RNA-seq.
This collection includes samples collected across a range of disease states, cell types, differentiation stages, and various stimuli.
The MultiPLIER models extracted 987 latent variables (LVs) by optimizing data reconstruction and alignment of LVs with prior knowledge/pathways (see [Methods](#sec:methods:multiplier)).


<!--  -->
We hypothesized that projecting TWAS and pharmacologic perturbations data into the same latent space, which represents a functional set of genes regulated by the same transcriptional program, could provide a better mechanistic understanding [@doi:10.1186/1471-2164-7-187; @doi:10.1186/s13059-019-1835-8].
To do this, PhenoPLIER converts gene associations to a latent variable (LV) score by multiplying each gene's standardized effect size for a trait (from TWAS) or differential expression value for a drug (from pharmacologic perturbation data) by the LV genes' weights and summing these values.
This process links different traits and drugs to functionally-related groups of genes or LVs, instead of looking at individual genes.
Additionally, PhenoPLIER uses annotations about the specific conditions where the group of genes is expressed, such as cell types and tissues, even at specific developmental stages, disease stages or under distinct stimuli.
This information can improve the interpretability of results.
We used this approach to analyze LV603, which was reported in the original MultiPLIER study to be associated with a known neutrophil pathway and highly correlated with neutrophil count estimates from whole blood RNA-seq profiles [@doi:10.1186/s13059-016-1070-5].
Our analysis revealed that 1) neutrophil counts and other white blood cell traits were ranked among the top 10 traits out of 4,091 (Figure {@fig:entire_process}c, bottom), and basophils count and percentage were significantly associated with this LV when using our regression method (Supplementary Table @tbl:sup:phenomexcan_assocs:lv603), and 2) LV603's genes were expressed in highly relevant cell types (Figure {@fig:entire_process}c, top).
These results suggest that groups of functionally related and co-expressed genes tend to correspond to groups of trait-associated genes, and the approach can link transcriptional mechanisms from large and diverse dataset collections to complex traits.


PhenoPLIER allows users to answer specific questions related to genetic studies, functional genomics, and gene co-expression.
These questions include whether disease-associated genes belong to modules expressed in specific tissues and cell types, if these cell type-specific modules are associated with different diseases, and if there is a subset of module's genes that is closer to the definition of "core" genes and thus represents alternative and potentially better therapeutic targets.
Additionally, the tool can be used to identify drugs that are perturbing transcriptional mechanisms and suggest potential mechanisms of action.
