### PhenoPLIER: an integration framework based on gene co-expression patterns

![
**Schematic of the PhenoPLIER framework.**
**a)** High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (genetic variants).
PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations.
**b)** The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top).
This process generates matrices $\mathbf{\hat{L}}$ and $\mathbf{\hat{M}}$, where LVs now describe each drug/trait.
In the middle, we show the three main computational components provided by PhenoPLIER to perform this integration:
1) an LV-based regression model,
2) a clustering framework to learn groups of traits,
and 3) an LV-based drug repurposing approach.
**c)** LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603 after projecting gene-trait associations in PhenomeXcan).
Genes that are part of LV603 were expressed in relevant cell types (top).
PBMC: peripheral blood mononuclear cells;
mDCs: myeloid dendritic cells.
](images/entire_process/entire_process.svg "PhenoPLIER framework"){#fig:entire_process width="100%"}


PhenoPLIER is a computational framework that combines gene-trait and gene-drug associations with gene modules expressed in specific contexts (Figure {@fig:entire_process}a).
This approach uses a latent representation derived from a gene expression compendium (Figure {@fig:entire_process}b, top) to integrate TWAS with drug-induced transcriptional responses (Figure {@fig:entire_process}b, bottom) for a joint analysis.
The framework consists of three components (Figure {@fig:entire_process}b, middle; see [Methods](#sec:methods)): 1) an LV-based regression model to compute an association between an LV and a trait; 2) a clustering framework to identify groups of traits with shared transcriptomic properties; and 3) an LV-based drug repurposing approach that links diseases to potential treatments.
We conducted simulations for our regression model ([Supplementary Note 1](#sm:reg:null_sim)) and clustering framework ([Supplementary Note 2](#sm:clustering:null_sim)) to ensure proper calibration and expected results under a model of no association.


We used two cohorts to project genetic associations: PhenomeXcan [@doi:10.1126/sciadv.aba2083] and the eMERGE network [@doi:10.1101/2021.10.21.21265225].
PhenomeXcan provides gene-trait associations for 4,091 diseases and traits from the UK Biobank [@doi:10.1038/s41586-018-0579-z], whereas the eMERGE network was used for 309 phecodes.
We used two statistical methods for the TWAS results (see [Methods](#sec:methods:predixcan)): S-MultiXcan associations for the regression and clustering components, and S-PrediXcan associations for the drug repurposing component.
Additionally, we used colocalization results to assess the probability of overlap between GWAS and eQTL signals.
For drug repurposing, we used transcriptional responses to small molecule perturbations from LINCS L1000 [@doi:10.1016/j.cell.2017.10.049], which comprises 1,170 compounds.


The MultiPLIER models were derived by applying a matrix factorization method (the pathway-level information extractor or PLIER [@doi:10.1038/s41592-019-0456-1]) to recount2 [@doi:10.1038/nbt.3838], a uniformly-curated collection of transcript-level gene expression quantified by RNA-seq in a large, diverse set of samples.
This collection covers a range of disease states, cell types, differentiation stages, and various stimuli (see [Methods](#sec:methods:multiplier)).
The MultiPLIER models extracted 987 latent variables (LVs) by optimizing data reconstruction, as well as the alignment of LVs with prior knowledge/pathways.


We used PhenoPLIER to project genetic associations and pharmacologic perturbations data into a latent space of gene modules, or LVs.
Each LV represents a group of weighted genes expressed together in the same tissues and cell types as a functional unit.
By multiplying the effect sizes of each gene associated with a trait or differential expression values for a drug with the LV's gene weights, PhenoPLIER converts gene associations to an LV score.
This process links different traits and drugs to functionally-related groups of genes or LVs, rather than individual genes.
Additionally, PhenoPLIER uses annotations about the specific conditions in which the group of genes is expressed, such as cell types and tissues, even at specific developmental stages, disease stages or under distinct stimuli.
This improves the interpretability of the results.
For example, one of the latent variables, LV603, was associated with a known neutrophil pathway and highly correlated with neutrophil count estimates from whole blood RNA-seq profiles.
Using PhenoPLIER, we found that neutrophil counts and other white blood cell traits were ranked among the top 10 traits, and basophils count and percentage were significantly associated with this LV.
Additionally, LV603's genes were expressed in highly relevant cell types.
These results suggest that groups of functionally related and co-expressed genes correspond to groups of trait-associated genes, and the approach can link transcriptional mechanisms from large and diverse dataset collections to complex traits.


PhenoPLIER allows users to answer specific questions related to genetic studies and functional genomics.
For example, it can be used to determine if disease-associated genes belong to modules expressed in specific tissues and cell types.
In addition, it can be used to investigate if these cell type-specific modules are associated with different diseases, potentially representing a "network pleiotropy" example from an omnigenic point of view [@doi:10.1016/j.cell.2017.05.038].
It can also be used to identify a subset of module's genes that is closest to the definition of "core" genes (i.e., directly affecting the trait with no mediated regulation of other genes [@doi:10.1016/j.cell.2019.04.014]) and thus represent alternative and potentially better candidate targets.
Additionally, it can be used to investigate if drugs perturb these transcriptional mechanisms, and if they suggest potential mechanisms of action.
