### Genes causally involved in lipids accumulation are associated with relevant traits and tissues

We found 271 genes associated with lipids accumulation by using a genome-wide lentiviral pooled CRISPR-Cas9 library targeting 19,114 genes in the human genome (see Methods).
From these, we identified two gene-sets that either caused a decrease (96 genes in total, with eight high-confidence genes: *BLCAP*, *FBXW7*, *INSIG2*, *PCYT2*, *PTEN*, *SOX9*, *TCF7L2*, *UBE2J2*) or an increase of lipids (175 genes in total, with six high-confidence genes: *ACACA*, *DGAT2*, *HILPDA*, *MBTPS1*, *SCAP*, *SRPR*) (Supplementary File 1).
Next, we used these two gene-sets to assess whether single gene-trait associations in PhenomeXcan recapitulated lipids-related traits.
We show that our gene module-based approach 1) is robust to identify genetic associations even when known single-gene associations are not significant, 2) and that it can contextualize these results by identifying tissue and cell type-specific associations.


First, using these two gene-sets, we assessed the genes' effects on all phenotypes by adding their $p$-values (as $z$-scores) and obtaining a ranked list of traits.
The top associated traits for genes in the decreasing-lipids gene-set were highly relevant to lipid levels, such as hypertension, diastolic and systolic blood pressure, and vascular diseases, including also asthma and lung function.
We performed the same operation for our gene module-based approach by considering 24 modules nominally enriched (unadjusted $p$-value < 0.05) with the decreasing-lipids gene-set (Gene-set enrichment analysis).
In this case, we also found highly lipids-relevant traits among the top 25, including hypertension, blood pressure, specific cardiometabolic diseases like atherosclerosis, and celiac disease.
This is particularly relevant because every 24 modules aggregated a specific weighted combination of almost 3,000 genes' $z$-scores across all traits.
Thus, aggregating the effects of this number of genes and obtaining top-ranked lipids-relevant traits is unlikely to happen by chance ($p$<0.01), suggesting that gene co-expression patterns are concordant with gene-trait associations.


![
**Tissues and traits associated with a gene module related to lipids metabolism (LV246).**
<!--  -->
**a)** Top cell types/tissues where genes in LV246 are expressed on.
Values in the $y$-axis come from matrix $\mathbf{B}$ in the MultiPLIER models (Figure @fig:entire_process a).
In the $x$-axis, cell types/tissues are sorted by the maximum value.
<!--  -->
**b)** Gene-trait associations (S-MultiXcan) and colocalization (fastENLOC) for the top traits in LV246.
The top 40 genes in LV246 are shown, sorted by their module weight, from largest (top gene *SCD*) to smallest (gene *FAR2*).
<!--  -->
SGBS: Simpson Golabi Behmel Syndrome;
CH2DB: CH<sub>2</sub> groups to double bonds ratio;
NMR: nuclear magnetic resonance;
HDL: high-density lipoprotein.
<!--  -->
](images/lvs_analysis/lv246/lv246.svg "LV246 TWAS plot"){#fig:lv246 width="100%"}


When we considered the increasing-lipids set, genes and modules (unadjusted $p$-value < 0.05) were associated with a more diverse set of traits, such as blood count tests, whole-body bioelectrical impedance measures, severe asthma, lung function, and rheumatoid arthritis.
Additionally, gene modules were also associated with blood lipids, arterial stiffness, intraocular pressure, handgrip strength, and celiac disease.
Genes in LV246, which was significantly aligned with lipids metabolism and triglyceride biosynthesis pathways in MultiPLIER (Supplementary Table @tbl:sup:multiplier_pathways:lv246), were mainly co-expressed in adipose tissue (Figure @fig:lv246).
This LV was significantly associated with blood lipids and hypercholesterolemia (Supplementary Table @tbl:sup:phenomexcan_assocs:lv246, and gene associations/colocalization are shown in Figure @fig:lv246 b).
Two high-confidence genes from our CRISPR screening, *DGAT2* and *ACACA* (responsible for encoding enzymes for triglycerides and fatty acid synthesis), were among the top genes of LV246.
However, as it can be seen in Figure @fig:lv246 b, these two genes are not strongly associated with any of the top traits for this LV;
other members of this module, such as *SCD*, *LPL*, *FADS2*, *HMGCR* and *LDLR*, were instead significantly associated and colocalized with lipid-relevant traits.
This suggests that a module-based perspective can contextualize and reprioritize TWAS hits using modules of functionally related genes.


`Notes:`{.red}

- MENTION MODULE THAT ARE REALLY SIGNIFICANTLY ENRICHED AND PLOTS OF TISSUES/CELL TYPES/ PHENOTYPES?

::: {style="color: red"}
- Add -log10(p-value) in the legend of figure.
:::
