### LVs link genes that alter lipid accumulation with relevant traits and tissues

Our first experiment attempted to answer whether genes in a disease-relevant LV could represent alternative drug targets.
For this, the first step was to obtain a set of genes strongly associated with a phenotype of interest.
Therefore, we performed a fluorescence-based CRISPR-Cas9 in the HepG2 cell line and identified 462 genes associated with lipid regulation (Methods).
From these, we selected two high-confidence gene-sets that either caused a decrease or increase of lipids:
a lipids-decreasing gene-set with eight genes: *BLCAP*, *FBXW7*, *INSIG2*, *PCYT2*, *PTEN*, *SOX9*, *TCF7L2*, *UBE2J2*;
and a lipids-increasing gene-set with six genes: *ACACA*, *DGAT2*, *HILPDA*, *MBTPS1*, *SCAP*, *SRPR* (Supplementary File 1).


<!-- From these, we selected four gene-sets that either caused a decrease or increase of lipids with varying degrees of confidence:
a median-confidence lipids-decreasing gene-set had 96 genes in total, with a subset of eight high-confidence genes: *BLCAP*, *FBXW7*, *INSIG2*, *PCYT2*, *PTEN*, *SOX9*, *TCF7L2*, *UBE2J2*;
a median-confidence lipids-increasing gene-set had 175 genes in total, with a subset of eight high-confidence genes: *ACACA*, *DGAT2*, *HILPDA*, *MBTPS1*, *SCAP*, *SRPR* (Supplementary File 1). -->
<!-- Using Fast Gene Set Enrichment Analysis (FGSEA) [@doi:10.1101/060012], we found four LVs significantly enriched for the medium-confidence lipid-altering gene-sets (FDR < 0.05) (Supplementary Table @tbl:sup:lipids_crispr:modules_enriched).
Using the gene-property method, we found that three of these LVs were significantly associated with cardiovascular traits both in PhenomeXcan and eMERGE (Supplementary Tables @tbl:sup:phenomexcan_assocs:lv707, @tbl:sup:emerge_assocs:lv707, @tbl:sup:phenomexcan_assocs:lv905, @tbl:sup:emerge_assocs:lv905, @tbl:sup:phenomexcan_assocs:lv915, @tbl:sup:emerge_assocs:lv915). -->


<!-- Next, we analyzed all 987 LVs using Fast Gene Set Enrichment Analysis (FGSEA) [@doi:10.1101/060012], and found 15 LVs nominally enriched (unadjusted *P* < 0.01) with these lipid-altering gene-sets (Supplementary Tables @tbl:sup:lipids_crispr:modules_enriched_increase and @tbl:sup:lipids_crispr:modules_enriched_decrease).
To maximize interpretability, we only considered LVs that were well-aligned with at least one pathway from the MultiPLIER models (FDR < 0.05) and those where reliable sample metadata about tissue/cell type was available. -->
<!-- We found four LVs nominally enriched (unadjusted *P* < 0.01) with the lipids-increasing gene-set (Supplementary Table @tbl:sup:lipids_crispr:modules_enriched_increase) and two LVs nominally enriched with the lipids-decreasing gene-set (Supplementary Table @tbl:sup:lipids_crispr:modules_enriched_decrease).
By applying our adapted gene-property method, we found that `XXX`{.red} of these LVs were significantly associated with `XXX`{.red} traits both in PhenomeXcan and eMERGE (Supplementary Tables @, @, @, @, @, @).
`Talk briefly about the other LVs`{.red} -->
<!--  -->
<!--  -->
<!-- First, for each lipid-altering gene-set, we assessed the genes' effects on all phenotypes by adding their $p$-values (from S-MultiXcan, transformed to $z$-scores, with no direction of effect and standardized to account for large sample sizes) and obtaining a ranked list of traits. -->
<!-- The top associated traits for genes in the decreasing-lipids gene-set were highly relevant to lipid levels, such as hypertension, diastolic and systolic blood pressure, and vascular diseases, also including asthma and lung function (Supplementary Table @tbl:sup:single_genes:lipids_decreasing:top_traits). -->
<!-- We performed the same operation for our LV-based approach by considering 24 LVs nominally enriched (unadjusted $p$-value < 0.05) with the decreasing-lipids gene-set by using Fast Gene Set Enrichment Analysis (FGSEA) [@doi:10.1101/060012]. -->
<!-- In this case, we also found lipid-related traits among the top 25, including hypertension, blood pressure, cardiometabolic diseases like atherosclerosis, and celiac disease (Supplementary Table @tbl:sup:modules:lipids_decreasing:top_traits). -->


![
**Tissues and traits associated with a gene module related to lipid metabolism (LV246).**
<!--  -->
**a)** Top cell types/tissues where LV246's genes are expressed in.
Values in the $y$-axis come from matrix $\mathbf{B}$ in the MultiPLIER models (Figure @fig:entire_process b, see Methods).
In the $x$-axis, cell types/tissues are sorted by the maximum sample value.
<!--  -->
**b)** Gene-trait associations (S-MultiXcan; threshold at -log($p$)=10) and colocalization probability (fastENLOC) for the top traits in LV246.
The top 40 genes in LV246 are shown, sorted by their LV weight (matrix $\mathbf{Z}$), from largest (the top gene *SCD*) to smallest (*FAR2*);
*DGAT2* and *ACACA*, in boldface, are two of the six high-confidence genes in the lipids-increasing gene-set from the CRISPR-screen.
Cardiovascular-related traits are in boldface.
<!--  -->
SGBS: Simpson Golabi Behmel Syndrome;
CH2DB: CH<sub>2</sub> groups to double bonds ratio;
HDL: high-density lipoprotein;
RCP: locus regional colocalization probability.
<!--  -->
](images/lvs_analysis/lv246/lv246.svg "LV246 TWAS plot"){#fig:lv246 width="100%"}


Next, we analyzed all 987 LVs using Fast Gene Set Enrichment Analysis (FGSEA) [@doi:10.1101/060012], and found 15 LVs nominally enriched (unadjusted *P* < 0.01) with these lipid-altering gene-sets (Supplementary Tables @tbl:sup:lipids_crispr:modules_enriched_increase and @tbl:sup:lipids_crispr:modules_enriched_decrease).
<!-- When we considered the increasing-lipids gene-set, genes and LVs were associated with a more diverse set of traits, such as blood count tests, impedance measures, and bone-densitometry (Supplementary Tables @tbl:sup:single_genes:lipids_increasing:top_traits and @tbl:sup:modules:lipids_increasing:top_traits). -->
<!-- FGSEA found 27 LVs nominally enriched for the increasing-lipids gene-set which were associated with the same traits, and additionally to lung function, arterial stiffness, intraocular pressure, handgrip strength, rheumatoid arthritis, and celiac disease. -->
Among those with reliable sample metadata, LV246, the top LV associated with the lipids-increasing gene-set, contained genes mainly co-expressed in adipose tissue (Figure @fig:lv246 a), which plays a key role in coordinating and regulating lipid metabolism.
<!-- Gene-trait associations and colocalization for the top genes in this LV are shown in Figure @fig:lv246 b). -->
Using the gene-property analysis method, we found that gene weights for this LV were predictive of gene associations for plasma lipids and high cholesterol (Supplementary Table @tbl:sup:phenomexcan_assocs:lv246).
Two high-confidence genes from our CRISPR screening, *DGAT2* and *ACACA*, are responsible for encoding enzymes for triglycerides and fatty acid synthesis and were among the highest-weighted genes of LV246 (Figure @fig:lv246 b, in boldface).
However, in contrast to other members of this LV, *DGAT2* and *ACACA* were not strongly associated nor colocalized with any of the cardiovascular-related traits and thus would not have been prioritized by TWAS alone;
other members of LV246, such as *SCD*, *LPL*, *FADS2*, *HMGCR*, and *LDLR*, were instead significantly associated and colocalized with lipid-related traits.
This lack of association/colocalization of two high-confidence genes from our CRISPR-screen might be explained from an omnigenic point of view [@doi:10.1016/j.cell.2019.04.014].
Assuming that the TWAS models for *DGAT2* and *ACACA* capture all common *cis*-eQTLs (the only genetic component of gene expression that TWAS can capture) and there are no rare *cis*-eQTLs, these two genes might represent "core" genes (i.e., they directly affect the trait with no mediated regulation of other genes), and many of the rest in the LV are "peripheral" genes that *trans*-regulate them.
These results suggested that our LV-based approach can contextualize genetic associations and prioritize alternative therapeutic targets, mainly when TWAS cannot detect them or hits are not druggable.
