### PhenoPLIER: an integration framework based on gene co-expression patterns

![
**Schematic of the PhenoPLIER framework.**
**a)** High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (genetic variants).
PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations.
**b)** The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top).
This process generates matrices $\mathbf{\hat{L}}$ and $\mathbf{\hat{M}}$, where LVs now describe each drug/trait.
In the middle, we show the three main computational components provided by PhenoPLIER to perform this integration:
1) an LV-based regression model,
2) a clustering framework to learn groups of traits,
and 3) an LV-based drug repurposing approach.
**c)** LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603 after projecting gene-trait associations in PhenomeXcan).
Genes that are part of LV603 were expressed in relevant cell types (top).
PBMC: peripheral blood mononuclear cells;
mDCs: myeloid dendritic cells.
](images/entire_process/entire_process.svg "PhenoPLIER framework"){#fig:entire_process width="100%"}


PhenoPLIER is a computational framework that combines gene-trait and gene-drug associations with gene modules expressed in specific contexts (Figure {@fig:entire_process}a).
It uses a latent representation derived from a gene expression compendium (Figure {@fig:entire_process}b, top) to integrate TWAS with drug-induced transcriptional responses (Figure {@fig:entire_process}b, bottom).
It consists of three components (Figure {@fig:entire_process}b, middle): a LV-based regression model to compute an association between an LV and a trait, a clustering framework to identify traits with shared transcriptomic properties, and an LV-based drug repurposing approach to link diseases to potential treatments.
Extensive simulations were performed for the regression model ([Supplementary Note 1](#sm:reg:null_sim)) and clustering framework ([Supplementary Note 2](#sm:clustering:null_sim)) to ensure proper calibration and expected results under a model of no association.


We used TWAS results from PhenomeXcan [@doi:10.1126/sciadv.aba2083] and the eMERGE network [@doi:10.1101/2021.10.21.21265225] as discovery and replication cohorts, respectively ([Methods](#sec:methods:twas)).
We applied two statistical methods: Summary-MultiXcan (S-MultiXcan) and Summary-PrediXcan (S-PrediXcan) ([Methods](#sec:methods:predixcan)).
S-MultiXcan was used for the regression and clustering components, and S-PrediXcan was used for the drug repurposing component.
We also used colocalization results, which provide a probability of overlap between the GWAS and eQTL signals.
For drug repurposing, we used transcriptional responses to small molecule perturbations from LINCS L1000 [@doi:10.1016/j.cell.2017.10.049] comprising 1,170 compounds.


The latent gene expression representation was obtained from the MultiPLIER models [@doi:10.1016/j.cels.2019.04.003], which were derived by applying a matrix factorization method (the pathway-level information extractor or PLIER [@doi:10.1038/s41592-019-0456-1]) to recount2 [@doi:10.1038/nbt.3838] -- a uniformly-curated collection of transcript-level gene expression quantified by RNA-seq in a large, diverse set of samples collected across a range of disease states, cell types differentiation stages, and various stimuli (see [Methods](#sec:methods:multiplier)).
The MultiPLIER models extracted 987 LVs by optimizing data reconstruction but also the alignment of LVs with prior knowledge/pathways.


We used PhenoPLIER to project genetic associations and pharmacologic perturbations data into a latent space, where each latent variable (LV) or gene module represents a group of weighted genes expressed together in the same tissues and cell types as a functional unit.
We found that LVs can link transcriptional mechanisms from large and diverse dataset collections to complex traits.
For example, LV603 was associated with a known neutrophil pathway and highly correlated with neutrophil count estimates from whole blood RNA-seq profiles.
We found that 1) neutrophil counts and other white blood cell traits were ranked among the top 10 traits out of 4,091, and basophils count and percentage were significantly associated with this LV (Supplementary Table @tbl:sup:phenomexcan_assocs:lv603), and 2) LV603's genes were expressed in highly relevant cell types (Figure {@fig:entire_process}c).


PhenoPLIER allows users to answer questions about the relationship between disease-associated genes and gene expression modules in specific tissues and cell types.
It also helps to identify if cell type-specific modules are associated with different diseases from an omnigenic perspective.
Additionally, it can identify a subset of module genes that are closer to the definition of "core" genes, which could represent better candidate targets.
Finally, PhenoPLIER can identify drugs that perturb transcriptional mechanisms and suggest potential mechanisms of action.
