### PhenoPLIER: an integration framework based on gene co-expression patterns

![
**Schematic of the PhenoPLIER framework.**
**a)** High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (genetic variants).
PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations.
**b)** The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top).
This process generates matrices $\mathbf{\hat{L}}$ and $\mathbf{\hat{M}}$, where LVs now describe each drug/trait.
In the middle, we show the three main computational components provided by PhenoPLIER to perform this integration:
1) an LV-based regression model,
2) a clustering framework to learn groups of traits,
and 3) an LV-based drug repurposing approach.
**c)** LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603 after projecting gene-trait associations in PhenomeXcan).
Genes that are part of LV603 were expressed in relevant cell types (top).
PBMC: peripheral blood mononuclear cells;
mDCs: myeloid dendritic cells.
](images/entire_process/entire_process.svg "PhenoPLIER framework"){#fig:entire_process width="100%"}


PhenoPLIER is a computational framework that integrates gene-trait and gene-drug associations with gene modules expressed in specific contexts (Figure {@fig:entire_process}a).
It uses a latent representation derived from a large gene expression compendium to combine TWAS with drug-induced transcriptional responses (Figure {@fig:entire_process}b).
This approach has three main components (Figure {@fig:entire_process}b, middle): 1) an LV-based regression model to compute an association between an LV and a trait; 2) a clustering framework to learn groups of traits with shared transcriptomic properties; and 3) an LV-based drug repurposing approach that links diseases to potential treatments (see [Methods](#sec:methods)).
We conducted simulations for our regression model ([Supplementary Note 1](#sm:reg:null_sim)) and clustering framework ([Supplementary Note 2](#sm:clustering:null_sim)) to ensure proper calibration and expected results under a model of no association.


We used two cohorts for our study: PhenomeXcan [@doi:10.1126/sciadv.aba2083] and the eMERGE network [@doi:10.1101/2021.10.21.21265225].
The former provides gene-trait associations for over 4,000 diseases and traits from the UK Biobank [@doi:10.1038/s41586-018-0579-z], while the latter consists of 309 phecodes.
We used two statistical methods (see [Methods](#sec:methods:predixcan)) to derive TWAS results: 1) Summary-MultiXcan (S-MultiXcan) associations for the regression and clustering components, and 2) Summary-PrediXcan (S-PrediXcan) associations for the drug repurposing component.
We also used colocalization results to measure the probability of overlap between GWAS and eQTL signals.
For the drug-repurposing approach, we used transcriptional responses to small molecule perturbations from LINCS L1000 [@doi:10.1016/j.cell.2017.10.049], which comprises 1,170 compounds.


The latent gene expression representation was obtained from the MultiPLIER models [@doi:10.1016/j.cels.2019.04.003], which were derived by applying a matrix factorization method (the pathway-level information extractor or PLIER [@doi:10.1038/s41592-019-0456-1]) to recount2 [@doi:10.1038/nbt.3838] -- a uniformly-curated collection of transcript-level gene expression quantified by RNA-seq in a large, diverse set of samples collected across a range of disease states, cell types differentiation stages, and various stimuli (see [Methods](#sec:methods:multiplier)).
The MultiPLIER models extracted 987 LVs by optimizing data reconstruction but also the alignment of LVs with prior knowledge/pathways.


The latent variables (LVs) in our study represent a group of weighted genes expressed together as a functional unit.
We hypothesized that projecting TWAS and pharmacologic perturbations data into this latent space could provide a better mechanistic understanding.
PhenoPLIER converts gene associations to an LV score by multiplying the genes' standardized effect sizes for a trait (from TWAS) or differential expression values for a drug (from pharmacologic perturbation data) by the LV genes' weights and summing them to produce a single value.
This process links different traits and drugs to functionally-related groups of genes or LVs, and can be improved by adding annotations about the specific conditions in which the group of genes is expressed, such as cell types and tissues, even at specific developmental stages, disease stages or under distinct stimuli.
For example, one of the latent variables, LV603, was associated with a known neutrophil pathway and highly correlated with neutrophil count estimates from whole blood RNA-seq profiles.
We found that it was associated with several white blood cell traits and basophils count and percentage, and its genes were expressed in highly relevant cell types.
These results suggest that functionally related and co-expressed genes can be linked to groups of trait-associated genes, allowing us to project transcriptional mechanisms from large and diverse dataset collections to complex traits.


PhenoPLIER enables users to answer several key questions.
For example, do genes associated with a certain disease belong to certain modules expressed in particular tissues and cell types? Are these cell type-specific modules associated with different diseases, potentially representing an example of "network pleiotropy" from an omnigenic perspective [@doi:10.1016/j.cell.2017.05.038]? Is there a subset of module genes that can be considered "core" genes (genes directly affecting the trait without the mediation of other genes [@doi:10.1016/j.cell.2019.04.014]) and, thus, represent possible alternative and better targets? Can drugs affect these transcriptional mechanisms, and can they suggest potential mechanisms of action?
