### Our gene module-based approach more accurately predicts known disease therapeutics

We systematically evaluated our gene module-based approach from a translational perspective by assessing whether it could more accurately predict known treatments for disease.
For this, we used the transcriptional responses to small molecule perturbations profiled in LINCS L1000 [@doi:10.1016/j.cell.2017.10.049], which were further processed and mapped to DrubBank IDs [@doi:10.1093/nar/gkt1068; @doi:10.7554/eLife.26726; @doi:10.5281/zenodo.47223].
Based on the established drug repurposing strategy that looks for reversed transcriptome patterns between genes and drug-induced perturbations [@doi:10.1126/scitranslmed.3002648; @doi:10.1126/scitranslmed.3001318], we used a framework for prioritizing drug candidates that uses imputed transcriptomes from GWAS [@doi:10.1038/nn.4618].
For this, we computed a drug-disease score by anti-correlating the $z$-scores for a disease (from TWAS) and the $z$-scores for a drug (from LINCS) across sets of genes of different size (see Methods).
Therefore, a positive score for a drug-disease pair indicates that a higher (lower) predicted expression of disease-associated genes are down (up)-regulated by the drug, thus predicting a potential treatment.
Similarly for the gene module approach, we estimated how pharmacological perturbations affected the gene module activity by projecting expression profiles of drugs into our latent representation (Equation @eq:proj).
We used a manually-curated gold standard of drug-disease medical indications [@doi:10.7554/eLife.26726; @doi:10.5281/zenodo.47664] across 53 diseases and 322 compounds to evaluate and compare the prediction performance.


![
**Drug-disease prediction performace for gene-based and module-based approaches.**
<!--  -->
The receiver operating characteristic (ROC) (left) and the precision-recall curves (right) for a gene-based and our module-based approach.
<!--  -->
AUC: area under the curve; AP: average precision.
<!--  -->
](images/drug_disease_prediction/roc_pr_curves.svg "ROC-PR curves for drug-disease prediction"){#fig:drug_disease:roc_pr width="100%"}


The ROC and precision-recall (PR) curves comparing both approaches are shown in Figure @fig:drug_disease:roc_pr.
Our proposed gene module-based method outperformed the gene-based one with an area under the curve of 0.632 and an average precision of 0.858.
<!--  -->
It is important to note that the gene-trait associations and drug-induced expression profiles projected into the latent space represent a compressed version of the entire set of results.
The prediction results show that this low-dimensional space captures biologically meaningful patterns that can better represent and link pathophysiological processes with the mechanisms of action of drugs.
In the following, we show examples to better understand these results by focusing on specific drug-disease pairs where both methods disagreed.


<!-- #### Niacin and cardiovascular diseases -->

Nicotinic acid (niacin), a B<sub>3</sub> vitamin widely used clinically to treat lipid disorders, exerts its effects on a number tissues, and although not all its mechanisms have been documented [@doi:10.1016/j.amjcard.2008.02.029; @doi:10.1194/jlr.S092007], this compound can increase high density lipoprotein (HDL) by inhibiting an HDL catabolism receptor in liver.
Niacin also inhibits diacylglycerol acyltransferaseâ€“2 (DGAT2), which decrease production of low density lipoproteins (LDL) by modulating triglyceride synthesis in hepatocytes, or by inhibiting adipocyte triglyceride lipolysis.
Niacin was categorized in our gold standard as a disease-modifying indication for atherosclerosis (AT) and coronary artery disease (CAD), and not for pancreatitis.
For pancreatitis, both the gene-based and module-based methods assigned a negative score (below their averages), which agrees with the gold standard in that niacin does not therapeutically change the biology of the disease.
For AT and CAD, the module-based approach predicted niacin as a therapeutic drug by scoring them with 0.52 and 0.96 (above the mean), whereas the gene-based method assigned negative scores of -0.09 and -0.16 (below the mean), respectively.
To understand why the predictions by the module-based method were different, we obtained the list of LVs that positively contributed to the score by looking at large positive (negative) LV values for the disease and large negative (positive) LV values for the drug of interest.
Notably, LV246 (analyzed previously) was among the top 20 modules contributing to the prediction of niacin as a therapeutic drug for AT.
As shown in Figure Figure @fig:lv246, this module is mainly expressed in adipose cells and hepatocytes, its top genes encode important enzymes involved in lipid biosynthesis, and several of them are significantly associated and colocalized with cardiovascular-related traits in our results. For instance,
*SCD* (10q24.31) is associated with hypercholesterolemia ($P=1.9\mathrm{e}{-5}$) and its GWAS and eQTL signals are fully colocalized ($\mathrm{RCP}=1.0$);
*LPL* (8p21.3), which is known to be linked to different disorders of lipoprotein metabolism, is strongly associated with hypercholesterolemia ($P=7.5\mathrm{e}{-17}$ and $\mathrm{RCP}=0.26$), and family history of heart disease ($P=1.7\mathrm{e}{-5}$ and $\mathrm{RCP}=0.22$);
other genes associated with hypercholesterolemia in this module are
*FADS2* (11q12.2) ($P=9.42\mathrm{e}{-5}$ and $\mathrm{RCP}=0.623$),
*HMGCR* (5q13.3) ($P=1.3\mathrm{e}{-42}$ and $\mathrm{RCP}=0.23$),
and *LDLR* (19p13.2) ($P=9.9\mathrm{e}{-136}$ and $\mathrm{RCP}=0.41$).


![
**Cell types where the top 10 modules contributing for Niacin-atherosclerosis prediction are expressed.**
<!--  -->
Average expression ($y$-axis) across different cell types ($x$-axis) for the top 10 latent variables/modules with a positive contribution for the niacin-AT prediction.
The figure shows a clear signature for immune cells, driven mainly by the top 2 modules: LV116 and LV931 (see supplementary material).
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP045569 -->
PBMCs: peripheral blood mononuclear cells;
HSV: treated with herpes simplex virus;
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP015670 -->
WNV: Infected with West Nile virus;
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP062958 -->
IFNa: interferon-alpha treatment;
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP039361 -->
HMDM: human peripheral blood mononuclear cell0derived macrophage;
IPSDM: human induced pluripotent stem cell-derived macrophages;
<!-- https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP015670 -->
](images/drug_disease_prediction/niacin-at-modules_cell_types.svg "Cell types of top modules for Niacin-AT prediction"){#fig:drug_disease:niacin width="80%"}


The analysis of the other related modules revealed additional known mechanisms of action of niacin.
For example, GPR109A/HCAR2 is a G protein-coupled high-affinity niacin receptor in adipocytes and immune cells, including in monocytes, macrophages, neutrophils and dendritic cells [@doi:10.1016/j.tips.2006.05.008; @doi:10.1038/sj.jid.5700586].
It was initially thought that the antiatherogenic effects of niacin were due to inhibition of lipolysis in adipose tissue.
However, it has been shown that nicotinic acid can reduce atherosclerosis progression independently of its antidyslipidemic activity through the activation of GPR109A in immune cells [@doi:10.1172/JCI41651], thus boosting anti-inflammatory processes and reversing cholesterol transport [@doi:10.1161/ATVBAHA.108.179283].
As shown in Figure @fig:drug_disease:niacin, this alternative mechanism for niacin could have been hypothesized by examining the cell types where modules positively contributing to the niacin-AT prediction are expressed.


The projection of these two types of data into a common latent gene module-based representation could provide a more powerful framework for drug repositioning using data from genetic studies.
Additionally, our approach could be also helpful to better understand the mechanism of pharmacological effect of known or experimental drugs.
For example, one of the top modules affected by niacin (LV66) is mainly expressed in ovarian granulosa cells.
This compound has been very recently analyzed as a potential therapeutic for ovarian diseases [@doi:10.1159/000495051; @doi:10.1071/RD20306], which was found to promote follicle growth and inhibit granulosa cell apoptosis in animal models.
Our proposed approach could serve to generate novel hypothesis to evaluate potential mechanisms of action of different drugs.


`Notes:`{.red}

::: {style="color: red"}
- It could be good to discuss cases where the gene-based approach performed better.
- This section references the CRISPR section that was not yet merged.
- (THIS IS THE MAIN THING) It might be necessary to use Cmap build 02 also here, since there are some concerns about the LINCS imputation pipeline (https://think-lab.github.io/d/185/).
- For the gene based approach, I'm using all 22k genes. I should try also to restrict to MultiPLIER genes only.
- I don't have to stick to the gold standard, I can take a look at all predictions (actually, I would need to run this). Maybe we can provide the predictions for all traits in PhenomeXcan for download.
- It would be great to have an expert in cardiovascular diseases and lipid disorder review this part.
- For Figure XXX, I'm removing cancer-related cell lines, since I think they add noise. Is that fine?
- Is it clear the message in Figure fig:drug_disease:niacin? An alternative is to just show the most interesting LVs instead of averaging all and show the top cell types as it is now.

Ideas/minor:

- We could also test for enrichment of drugs that are indicated for each disorder, or included in clinical trials.
- There are number of other analyses we could do here: for instance, if we keep only LVs aligned with pathways, what happens with prediction performance? If it goes down, it means that among not-aligned LVs we have useful information to link diseases and drugs.
It would be nice to be able to state that.
- Another idea could be to use the LVs detected with the decision trees and the clustering results.
If it gets better, it could mean that considering groups of diseases (our clusters) more accurately captures the shared mechanisms between them, and considering single disease entities could be more prone to noise.
:::
