### PhenoPLIER improves drug-disease prediction

![
**Drug-disease prediction.**
<!--  -->
Explain.
<!--  -->
AUC: area under the curve; AP: average precision.
<!--  -->
](images/drug_disease_prediction/auc_pr.png "AUC-PR of drug-disease
associations"){#fig:drug_disease:auc_pr width="100%"}


We showed that the latent representation learned in a large gene expression data
set is helpful to link novel gene sets to complex diseases and the cell types
where they are expressed.
<!--  -->
We reasoned that this representation might be also useful to more
accurately predict the potential therapeutic effects of drugs. If the gene
patterns captured by MultiPLIER represent real and possibily unknown biological
pathways, then our approach might actually produce a more accurate estimation of
the effects of a perturbed molecular mechanisms on disease, and also how
pharmacological perturbations affect the process activity.
<!--  -->
This would, in addition to connecting gene modules with their context-specific
effects on complex traits, identify which compounds might provide an avenue to
alter a dysregulated process activity for therapeutical utility.


To test this hypothesis, we used a gold standard of drug-disease medical
indications [@doi:10.7554/eLife.26726; @doi:10.5281/zenodo.47664] to evaluate
and compare the prediction performance of both the original gene-disease
associations from PhenomeXcan, and its projection representing gene
module-disease associations.
<!--  -->
To test this, we used the transcriptional responses to small molecule
perturbations profiled in LINCS L1000 [@doi:10.1016/j.cell.2017.10.049], which
were further processed to obtain consensus signatures and map to DrubBank IDs
[@doi:10.1093/nar/gkt1068; @doi:10.7554/eLife.26726; @doi:10.5281/zenodo.47223].
<!--  -->
To compute a drug-disease score, we followed a similar procedure used previously
[@doi:10.1038/nn.4618] to anti-correlates gene-traits associations from TWAS and
expression profiles of drugs using their signed $z$-scores (see the
supplementary material). Here we used the dot product between gene-traits
$z$-scores and the consensus $z$-scores in LINCS L1000, which led to a score for
each drug-disease pair. (`Add more details?`{.red}).
<!--  -->
To obtain a drug-disease association for the gene module-mapped TWAS results, we
first projected LINCS L1000 data into this latent representation using Equation
(@eq:proj), thus leading to a matrix with the expression profiles of drugs
mapped to latent variables. This can be interpreted as the effects of compounds
on gene modules activity. Then, similarly as before, we anti-correlated gene
module-traits scores and module expression profiles of drugs.

`(Add number of drugs, diseases, and final number of mappings)`{.red}
<!--  -->
<!-- Both $\matr{D}$ and $\hat{\matr{D}}$ contain scores for 1170 drugs and 4091 traits,
which were further mapped to 56 unique Disease Ontology IDs. -->
<!--  -->
<!-- Then I assessed the performance of these two approaches using a manually curated list of
drug-disease associations~\autocite{Himmelstein2017, HimmelsteinPharmaDB2016}.
97 diseases and 601 compounds in PharmarcotherapyDB~\autocite{Himmelstein2017,
HimmelsteinPharmaDB2016},
which contains 755 disease-modifying therapies and 243
non-indications. After integrating this gold-standard list with the predictions, I
obtained 458 positive and 136 negative cases for evaluation.
-->


<!-- Finally, I obtained drug-disease predictions using both the original (gene-based) and
projected (gene module-based) associations: $\matr{D}=\matr{L}^{\top}\matr{M}$ and
$\hat{\matr{D}}=\hat{\matr{L}}^{\top}\hat{\matr{M}}$, respectively. -->
<!--  -->

The ROC and Precision-Recall (PR) curves comparing both approaches are shown in
Figure @fig:drug_disease:auc_pr. Notably, the gene module-based approach
proposed here clearly outperformed the gene-based one, with an area under the
curve (AUC) of 0.615 vs 0.573, and an average precision (AP) of 0.851 vs 0.841.
<!--  -->
This of particularly striking given that our approach is a reduced or compressed
version of the complete gene-based ones, suggesting that a gene module
perspective can be more informative, for instance, for drug-repurposing
scenarios using genetic studies.



::: {style="color: red"}
Another analysis here could be:

- Try the prediction again by keeping well-aligned LVs only. Do we get the same
  prediction performance?
:::

<!-- ::: {style="color: red"}
Papers to read:
- "SubtypeDrug: a software package for prioritization of candidate cancer
  subtype-specific drugs" https://doi.org/10.1093/bioinformatics/btab011 It
  might be interesting to see how the predict drug-disease associations when
  they say "evaluation of drug-disease reverse association"
::: -->
