### Our gene module-based approach more accurately predicts known disease therapeutics

We systematically evaluated our gene module-based approach from a translational perspective by assessing whether it could more accurately predict known treatments for disease.
For this, we used the transcriptional responses to small molecule perturbations profiled in LINCS L1000 [@doi:10.1016/j.cell.2017.10.049], which were further processed and mapped to DrubBank IDs [@doi:10.1093/nar/gkt1068; @doi:10.7554/eLife.26726; @doi:10.5281/zenodo.47223].
Based on the established drug repurposing strategy that looks for reversed transcriptome patterns between genes and drug-induced perturbations [@doi:10.1126/scitranslmed.3002648; @doi:10.1126/scitranslmed.3001318], we used a framework for prioritizing drug candidates that uses imputed transcriptomes from GWAS [@doi:10.1038/nn.4618].
Thus, we computed a drug-disease score by anti-correlating the $z$-scores for a disease (from TWAS) and the $z$-scores for a drug (from LINCS) across different sets of genes (see supplementary material).
Similarly for the gene module approach, we estimated how pharmacological perturbations affected the gene module activity by projecting expression profiles of drugs into our latent representation (Equation @eq:proj).
We used a gold standard of drug-disease medical indications [@doi:10.7554/eLife.26726; @doi:10.5281/zenodo.47664] across 97 diseases and 601 compounds to evaluate and compare the prediction performance.


![
**Drug-disease prediction performace for gene-based and module-based approaches.**
<!--  -->
The receiver operating characteristic (ROC) (left) and the precision-recall curves (right) for a gene-based and our module-based approach.
<!--  -->
AUC: area under the curve; AP: average precision.
<!--  -->
](images/drug_disease_prediction/roc_pr_curves.svg "ROC-PR curves for drug-disease prediction"){#fig:drug_disease:roc_pr width="100%"}


The ROC and precision-recall (PR) curves comparing both approaches are shown in Figure @fig:drug_disease:roc_pr.
Our proposed gene module-based method clearly outperformed the gene-based one with an area under the curve of 0.632 vs 0.578, and an average precision of 0.858 vs 0.847.
<!--  -->
Notably, the gene-trait associations and drug-induced expression profiles projected into the latent space represent a compressed version of the entire set of results.
The prediction results show that this low-dimentional space captures biologically meaningful patterns that can better represent and link pathophysiological processes with the mechanisms of action of drugs.
The projection of these two types of data into this common latent representation could provide a more powerful framework for drug repositioning using genetic data.


Take one disease, like hypertension (to keep the same topic), and see how both approaches performed.
It would be great to explain an association. For instance, we take some of the top drugs for hypertension, and explain why there were correctly identified, or incorrectly identified.

another example could be Dexamethasone and Betamethasone for allergic rhinitis, which is detected by the module-based approach, but not by the gene-based approach
- in which cell types/tissues are the 
I could make a search for this (detected by one, not detected by the other)
- same for atherosclerosis and Niacin

another analysis is why one drug is not detected, for instance, Desloratadine for allergic rhinitis
what happens if I remove an important LV to detect a drug-disease association?

I don't have to stick to the gold standard, I can take a look at all predictions (actually, I would need to run this)


`Notes:`{.red}

::: {style="color: red"}
- It might be necessary to use Cmap build 02 also here, since there are some concerns about the LINCS imputation pipeline (https://think-lab.github.io/d/185/).
- For the gene based approach, I'm using all 22k genes. I should try also to restrict to MultiPLIER genes only.

Ideas/minor:

- We could also test for enrichment of drugs that are indicated for each disorder, or included in clinical trials.
- There are number of other analyses we could do here: for instance, if we keep only LVs aligned with pathways, what happens with prediction performance? If it goes down, it means that among not-aligned LVs we have useful information to link diseases and drugs.
It would be nice to be able to state that.
- Another idea could be to use the LVs detected with the decision trees and the clustering results.
If it gets better, it could mean that considering groups of diseases (our clusters) more accurately captures the shared mechanisms between them, and considering single disease entities could be more prone to noise.
:::
