### LVs reveal trait clusters with shared transcriptomic properties

![
**Cluster analysis on traits using the latent gene expression representation.**
**a)** The projection of TWAS results on 3,752 traits into the latent gene expression representation is the input data to the clustering process.
A linear (PCA) and non-linear (UMAP) dimensionality reduction techniques were applied to the input data, and five different clustering algorithms processed all data versions.
These algorithms derive partitions from the data using different parameters (such as the number of clusters), leading to an ensemble of 4,428 partitions.
Then, a distance matrix is derived by counting how many times a pair of traits was grouped in different clusters across the ensemble.
Finally, a consensus function is applied to the distance matrix to generate consolidated partitions with different numbers of clusters (from 2 to $\sqrt{n}\approx$ 60).
These final solutions were represented in the clustering tree (Figure @fig:clustering:tree).
**b)** The clusters found by the consensus function were used as labels to train a decision tree classifier on the original input data, which detects the LVs that better differentiate groups of traits.
](images/clustering/clustering_design.svg "Cluster analysis on traits"){#fig:clustering:design width="100%"}


We used a projection of gene-trait associations into a latent space to find groups of clusters linked by the same transcriptional processes.
To avoid potential pitfalls when performing cluster analysis on biological data [@doi:10.1126/scisignal.aad1932], we used a consensus clustering framework that combines solutions or partitions of traits generated by different methods with varying sets of parameters ([Methods](#sec:methods:clustering)).
This approach has been shown to outperform individual methods [@Strehl2002; @doi:10.1109/TPAMI.2005.113].
Our clustering pipeline generated 15 final consensus clustering solutions (Supplementary Figure @fig:sup:clustering:agreement).
To examine stable groups of traits across multiple resolutions, we used a clustering tree [@doi:10.1093/gigascience/giy083] (Figure @fig:clustering:tree).
We then trained a decision tree classifier on the input data $\hat{\mathbf{M}}$ using the clusters found as labels (Figure {@fig:clustering:design}b, see [Methods](#sec:methods:clustering)).


![
**Clustering tree using multiple resolutions for clusters of traits.**
Each row represents a partition/grouping of the traits, and each circle is a cluster from that partition.
The number of clusters goes from 5 to 29.
Arrows indicate how traits in one cluster move across clusters from different partitions.
Most of the clusters are preserved across different resolutions, showing highly stable solutions even with independent runs of the clustering algorithm.
<!--  -->
RDW: red cell (erythrocyte) distribution width;
BMI: body mass index;
WC: waist circumference;
HC: hip circumference;
RA: rheumatoid arthritis;
SLE: systemic lupus erythematosus;
HTN: Hypertension;
IBD: inflammatory bowel disease;
SCZ: Schizophrenia;
CAD: Coronary artery disease;
AD: Alzheimer's disease;
<!--  -->
*Descriptions of traits by cluster ID (from left to right):*
12: also includes lymphocyte count and allergies such as allergic rhinitis or eczema;
4: includes reticulocyte count and percentage, immature reticulocyte fraction, and high light scatter reticulocytes count and percentage;
2: includes mean corpuscular volume, mean corpuscular hemoglobin, mean reticulocyte volume, mean sphered cell volume;
5: includes erythrocyte count, hemoglobin concentration, and hematocrit percentage;
20: also includes weight, waist and hip circumference;
18: also includes body impedance measures and ankle spacing width;
19: also includes basal metabolic rate;
1: includes platelet count, crit, mean volume, and distribution width;
13: diabetes refers to age when diabetes was first diagnosed;
25: also includes vascular problems such as angina, deep vein thrombosis (DVT), intraocular pressure, eye and mouth problems, pulse rate, hand-grip strength, several measurements of physical activity, jobs involving heavy physical work, types of transport used, intake of vitamin/mineral supplements, and various types of body pain and medications for pain relief;
21: also includes attention deficit hyperactivity disorder (ADHD), number of years of schooling completed, bone density, and intracranial volume measurement;
28: includes diabetes, gout, arthrosis, and respiratory diseases (and related medications such as ramipril, allopurinol, and lisinopril), urine assays, female-specific factors (age at menarche, menopause, first/last live birth), and several environmental/behavioral factors such as intake of a range of food/drink items including alcohol, time spent outdoors and watching TV, smoking and sleeping habits, early-life factors (breastfed as a baby, maternal smoking around birth), education attainment, psychological and mental health, and health satisfaction;
11: also includes fasting blood glucose and insulin measurement;
16: lipids include high and low-density lipoprotein (HDL and LDL) cholesterol, triglycerides, and average number of methylene groups per a double bond;
14: includes myocardial infarction, coronary atherosclerosis, ischaemic heart disease (wide definition);
7: includes monocyte count and percentage;
24: includes lymphocyte count and percentage;
9: includes neutrophil count, neutrophil+basophil count, neutrophil+eosinophil count, granulocyte count, leukocyte count, and myeloid cell count;
3: includes eosinophil count, eosinophil percentage, and eosinophil+basophil count.
](images/clustering/clustering_tree.svg "Clustering tree on groups of traits"){#fig:clustering:tree width="100%"}


We identified five clear branches of phenotypes, which were represented by the first node in Figure @fig:clustering:tree.
These included a "large" branch with most of the traits, which were only subdivided starting at $k$=16 (with clusters for asthma, subjective well-being traits, and nutrient intake).
The other branches were heel bone-densitometry measurements, hematological assays on red blood cells, physical measures (including spirometry and body impedance, and anthropometric traits with fat-free and fat mass measures in separate sub-branches), and a "complex" branch that included keratometry measurements, assays on white blood cells and platelets, skin and hair color traits, autoimmune disorders, and cardiovascular diseases (including other cardiovascular-related traits such as hand-grip strength [@pmid:25982160], and environmental/behavioral factors such as physical activity and diet).
Across different resolutions, the same traits often clustered together within these branches.
We observed arrows between clusters that showed traits moving from one group to another, particularly between clusters within the "complex" branch (4) and between clusters from the "large" branch (0) to the "complex" branch.
This behavior is expected, since complex diseases are usually associated with shared genetic and environmental factors and are thus hard to categorize into a single cluster.
For more details, see Supplementary Files 3-6.


![
**Cluster-specific and general transcriptional processes associated with different diseases.**
The plot shows a submatrix of $\hat{\mathbf{M}}$ for the main trait clusters at $k$=29, considering only LVs (rows) that are well-aligned with at least one pathway.
](images/clustering/global_clustermap-plain.svg "Heatmap with gene modules and traits"){#fig:clustering:heatmap width="100%"}


We used a decision tree classifier to analyze which latent variables (LVs) were driving the clusters of traits.
The classifier was trained on the input data with each cluster at $k$=29 (bottom of Figure @fig:clustering:tree) as labels (see [Methods](#sec:methods:clustering)).
The top LVs that were most discriminative for each cluster were identified.
Several of these LVs were aligned to existing pathways (Figure @fig:clustering:heatmap), while others were not, but still expressed in relevant tissues (Supplementary Figure @fig:sup:clustering:novel:heatmap).
Figure @fig:clustering:heatmap showed that some LVs were highly specific to certain traits, while others were associated with a wide range of different phenotypes, potentially involved in more general biological functions.
We used our regression framework to determine whether these LVs were significantly associated with different traits.
For example, LV928 and LV30 were aligned to early progenitors of the erythrocytes lineage [@doi:10.1016/j.cell.2011.01.004] (Supplementary Tables @tbl:sup:multiplier_pathways:lv928 and @tbl:sup:multiplier_pathways:lv30), expressed in early differentiation stages of erythropoiesis (Supplementary Figures @fig:sup:lv928 and @fig:sup:lv30), and strongly associated with red blood cell assays (FDR < 0.05; Supplementary Tables @tbl:sup:phenomexcan_assocs:lv928, @tbl:sup:emerge_assocs:lv928, and @tbl:sup:emerge_assocs:lv30).
Other LVs were highly specific, such as LV730, expressed in thrombocytes from different cancer samples (Supplementary Figures @fig:sup:lv730 and Supplementary Table @tbl:sup:multiplier_pathways:lv730) and strongly associated with hematological assays on platelets (FDR < 0.05, Supplementary Table @tbl:sup:phenomexcan_assocs:lv730); or LV598, whose genes were expressed in corneal endothelial cells (Supplementary Figures @fig:sup:lv598 and Supplementary Table @tbl:sup:multiplier_pathways:lv598) and associated with keratometry measurements (Supplementary Table @tbl:sup:phenomexcan_assocs:lv598).


At $k=10$, the sub-branches of autoimmune and cardiovascular diseases merged together (middle of Figure @fig:clustering:tree).
We expected to find LVs that would be specifically associated with one or both of these types of diseases.
For example, LV57 was expressed in T cells (Supplementary Figure @fig:sup:lv57 and Supplementary Table @tbl:sup:multiplier_pathways:lv57) and was the most strongly associated gene module with autoimmune disorders in PhenomeXcan (Supplementary Table @tbl:sup:phenomexcan_assocs:lv57).
This LV had significant associations with hypothyroidism that were replicated in eMERGE (@tbl:sup:emerge_assocs:lv57).
It was also strongly associated with deep venous thrombosis in both PhenomeXcan and eMERGE.
On the other hand, LV844 was more specific to autoimmune diseases, with associations to polymyalgia rheumatica, type 1 diabetes, rheumatoid arthritis, and celiac disease in PhenomeXcan (Supplementary Table @tbl:sup:phenomexcan_assocs:lv844).
These did not replicate in eMERGE.
This LV was expressed in many cell types, including blood, breast organoids, myeloma cells, lung fibroblasts, and different cell types from the brain (Supplementary Figure @fig:sup:lv844 and Supplementary Table @tbl:sup:multiplier_pathways:lv844).


The cardiovascular sub-branch of our study had 129 significant LV-trait associations in PhenomeXcan and 23 in eMERGE.
LV136 was associated with coronary artery disease and keratometry measurements in PhenomeXcan, and with coronary atherosclerosis in eMERGE.
This LV was expressed in a variety of cell types, including fibroblasts, mesenchymal stem cells, osteoblasts, pancreatic stellate cells, cardiomyocytes, and adipocytes.
We also found neuropsychiatric and neurodevelopmental disorders within this sub-branch, such as Alzheimer's disease, schizophrenia, and attention deficit hyperactivity disorder (ADHD).
These diseases were linked to the cardiovascular system and share several risk factors, such as hypertension, high cholesterol, obesity, and smoking.
Our results suggest that these diseases may be grouped by potentially shared transcriptional processes expressed in specific tissues/cell types.
For example, Alzheimer's disease (not present in eMERGE) was significantly associated with LV21 in PhenomeXcan.
This LV was strongly expressed in a variety of soft tissue sarcomas, monocytes/macrophages (including microglia from cortex samples), and aortic valves.
It was also strongly associated with lipids and high cholesterol in PhenomeXcan, and with hyperlipidemia in eMERGE.
Macrophages play a key role in the reverse cholesterol transport and thus atherogenesis, and lipid metabolism in microglia has been identified as an important factor in the development of neurodegenerative diseases.
