## Discussion

We present a novel computational strategy that integrates statistical associations from TWAS with groups of genes that have similar expression patterns across the same cell types.
This approach improves interpretation by providing a latent representation derived not only from measures of normal tissue but also from cell types under various stimuli and at different developmental stages.
Our method can identify disease-relevant cell types from summary statistics, and several disease-associated gene modules were replicated in eMERGE.
Using a CRISPR screen to analyze lipid regulation, we found that our gene module-based approach can prioritize causal genes even when single gene associations are not detected.
This suggests that the approach can identify genes that directly affect the trait, without any mediated regulation of other genes, and thus prioritize alternative and potentially more attractive therapeutic targets.


By using a gene module perspective, we integrated drug-induced transcriptional profiles to connect diseases, drugs, and cell types.
We found that the LV-based drug-repurposing approach was more successful than the gene-based one in predicting drug-disease links for 322 drugs across 53 diseases.
To further demonstrate the approach, we focused on cardiovascular traits and a specific drug, niacin, and showed that it connects pathophysiological processes with known mechanisms of action, such as those in adipose tissue, immune cells, and ovarian granulosa cells.
This LV-based approach could be useful in forming hypotheses to evaluate potential mechanisms of action, or even adverse effects, of known or experimental drugs.


Our analysis of associations through latent representations provided reasonable groupings of diseases and traits affected by shared and distinct transcriptional mechanisms expressed in relevant tissues.
Cluster analysis revealed the latent variables (LVs) that were most discriminative for each cluster.
Several of these LVs were significantly associated with different traits.
Some were strongly aligned with known pathways, while others (like LV57) were not, potentially representing novel disease-relevant mechanisms.
The features/LVs linked to phenotypes were associated with specific cell types, which may reveal causal cell types for a phenotype more precisely.
Modules were expressed primarily in one tissue (e.g.
adipose in LV246 or ovary in LV66) while others were expressed in many contexts, potentially capturing pathways associated with related complex diseases.
Examples include LV136, expressed in fibroblasts, osteoblasts, pancreas, liver and cardiomyocytes, and associated with cardiovascular disease and measures of corneal biomechanics; LV844, expressed in whole blood samples and associated with a range of autoimmune diseases; and LV57, expressed in T cells and strongly associated with autoimmune and venous thromboembolism.
This approach of projecting genetic associations through gene expression patterns may represent cases of "network pleiotropy," where the same cell types mediate molecularly related traits.
This novel approach to identifying cell type and pathway effects on complex phenotypes is computationally simple to implement.


<!-- In this portion, we used S-MultiXcan associations, which only provide the association strength between a gene and a trait, but with no direction of effect.
This does mean that traits are grouped based on associated genes, but genes could have opposite effects on traits within the same cluster. -->
We have demonstrated that clustering trees, initially used to examine developmental processes in single-cell data, can be employed to group phenotypes based on latent variable associations.
Hard-partitioning algorithms (where one trait exclusively belongs to one cluster) were used to calculate the distance between two traits, taking into account all gene modules.
However, it is possible for two complex diseases to share only a few biological processes instead of being similar across most of them.
Our TWAS results were derived from a large set of GWAS of varying sample sizes and qualities, and while data preprocessing steps were taken to address potential issues, bias may still remain.
Previous studies have shown that considering groups of related diseases is more powerful in detecting shared genetic etiology [@doi:10.1038/ng.3985; @doi:10.1038/s41588-018-0121-0], and clustering trees provide a way to explore such relationships in the context of latent variables.


Finally, we developed a regression framework based on latent variables (LVs) to detect whether gene modules are associated with a trait.
This framework was tested using PhenomeXcan and eMERGE cohorts, and the results showed 3,450 and 196 significant LV-trait associations respectively (FDR < 0.05) with 686 and 116 LVs associated with at least one trait/phecode.
Additionally, 1,176 and 81 traits were associated with at least one LV in PhenomeXcan and eMERGE respectively.
While we focused on a few disease types from our trait clusters, the complete set of associations on other disease domains is available in our Github repository for further research.
One limitation of the regression approach is that the gene-gene correlations are only approximately accurate, which could lead to false positives.
However, the regression model is approximately well-calibrated, and we did not observe inflation when running the method in real data.


<!-- Larger datasets and methods designed to learn representations with this application in mind could further refine the approach and are a promising avenue for future research. -->
Our approach assumes that gene modules with coordinated expression patterns will also manifest coordinated pathological effects.
To achieve this, we used two complementary approaches: MultiPLIER and TWAS.
MultiPLIER extracts latent variables from large expression datasets, which may represent either real transcriptional processes or technical factors ("batch effects").
We used a previously published model derived from recount2, which was designed to analyze rare disorders; however, it may not be the optimal latent representation for the wide range of complex diseases considered here.
The underlying factorization method relies on linear combinations of variables, which could miss more complex co-expression patterns.
We should be able to easily replace MultiPLIER with other similar approaches, such as GenomicSuperSignature.
TWAS considers the hypothesis that GWAS loci affect traits via changes in gene expression, but other effects, such as coding variants disrupting protein-protein interactions, are not captured.
Additionally, TWAS has several limitations that can lead to false positives.
Our LV-based regression framework, however, accounts for gene-gene correlations in TWAS reasonably well.


Our findings are consistent with previous studies, which suggest that drugs with genetic support are more likely to be successful in the drug development process [@doi:10.1038/ng.3314; @doi:10.1038/nn.4618].
By projecting association results through latent variables, we were able to prioritize disease-treatment pairs more effectively than if we had only considered single-gene effects.
Additionally, the latent variables driving these predictions can be examined to infer potential mechanisms of action.
In this case, we prioritized drugs for diseases with very different tissue etiologies.
However, a challenge of this approach is selecting the most appropriate tissue model from TWAS to identify gene expression patterns between genes and drug-induced perturbations.


Ultimately, the quality of the representations is essential for successful performance.
In this study, a representation derived from a factorization of bulk RNA-seq data was used.
To generate higher quality and more interpretable representations, detailed perturbation datasets and single-cell profiling of tissues, with and without perturbagens, and at various stages of development can be used.
Additionally, the annotation of sample metadata is key to interpretability.
New approaches to infer and annotate with structured metadata are promising and can be directly applied to existing data [@doi:10.1101/2021.05.10.443525].
With rapid improvements in both these areas, latent variable projections can be widely applied to understand the genetic basis of complex human phenotypes.
Our method provides a new perspective for a mechanistic understanding of statistical associations from TWAS, which can generate testable hypotheses for the post-GWAS functional characterization of complex diseases.
This is likely to be an important area of research in the coming years.
