## Discussion

We present a new computational method that combines statistical associations from TWAS with gene modules showing similar expression patterns in the same cell types.
Our approach goes beyond simple statistical associations to understand the cell type-specific features of complex phenotypes.
We demonstrated its utility by replicating several disease-associated gene modules in eMERGE, and by using a CRISPR screen to prioritize causal genes in lipid regulation, even when single gene associations were not detected.
Our findings suggest that this method can identify genes that have direct effects on the trait, and prioritize alternative therapeutic targets.


By using a gene module perspective, we integrated drug-induced transcriptional profiles and connected diseases, drugs, and cell types.
Our LV-based drug-repurposing approach showed better performance than the gene-based one when predicting drug-disease links for 322 drugs across 53 diseases.
We further focused on cardiovascular traits and the drug niacin to demonstrate that the approach connects pathophysiological processes with known mechanisms of action, such as those in adipose tissue, immune cells, and ovarian granulosa cells.
Our LV-based approach could be useful for generating new hypotheses to evaluate potential mechanisms of action, or even adverse effects, of known or experimental drugs.


We found that the analysis of associations through latent representations provided groupings of diseases and traits that are affected by shared and distinct transcriptional mechanisms expressed in relevant tissues.
Our cluster analysis approach also detected the latent variables (LVs) that were most discriminative for each cluster.
Several of these LVs were significantly associated with different traits and some were strongly aligned with known pathways, while others (like LV57) were not, which could represent novel disease-relevant mechanisms.
In some cases, the features/LVs linked to phenotypes appeared to be associated with specific cell types.
Associations with such cell type marker genes may reveal potentially causal cell types for a phenotype with more precision.
We observed modules expressed primarily in one tissue (such as adipose in LV246 or ovary in LV66) and others that were expressed in many contexts, which may capture pathways associated with related complex diseases.
For example, LV136 was associated with cardiovascular disease and measures of corneal biomechanics and was expressed in fibroblasts, osteoblasts, pancreas, liver, and cardiomyocytes, among others.
Other examples include LV844, expressed in whole blood samples and associated with a range of autoimmune diseases, or LV57, which was clearly expressed in T cells and associated with autoimmune and venous thromboembolism.
From an omnigenic point of view, these patterns might represent cases of "network pleiotropy," where the same cell types mediate molecularly related traits.
Our approach of projecting genetic associations through gene expression patterns to identify cell type and pathway effects on complex phenotypes is novel and computationally simple to implement.


We demonstrated that clustering trees, which were initially used to examine developmental processes in single-cell data, can provide a multi-resolution grouping of phenotypes based on latent variable associations.
We employed hard-partitioning algorithms, where the distance between two traits takes into account all gene modules.
However, it is possible for two complex diseases to share only a few biological processes instead of being similar across most of them.
Our TWAS results were derived from a large set of GWAS of different sample sizes and qualities, and we addressed potential issues from this data heterogeneity.
Although data preprocessing steps are always challenging, considering groups of related diseases has been shown to be more powerful in detecting shared genetic etiology [@doi:10.1038/ng.3985; @doi:10.1038/s41588-018-0121-0], and clustering trees provide a way to explore such relationships in the context of latent variables.


Finally, we developed a regression framework based on latent variables (LVs) to detect whether gene modules are associated with a trait.
We used PhenomeXcan as a discovery cohort across four thousand traits, and many LV-trait associations replicated in eMERGE.
In PhenomeXcan, we found 3,450 significant LV-trait associations (FDR < 0.05) with 686 LVs (out of 987) associated with at least one trait and 1,176 traits associated with at least one LV.
In eMERGE, we found 196 significant LV-trait associations, with 116 LVs associated with at least one trait/phecode and 81 traits with at least one LV.
We only focused on a few disease types from our trait clusters, but the complete set of associations is available in our Github repository for future research.
One limitation of this regression approach is that the gene-gene correlations are only approximately accurate, which could lead to false positives if the correlation among the top genes in a module is not precisely captured.
However, the regression model is approximately well-calibrated, and we did not observe any inflation when running the method in real data.


Our approach assumes that gene modules with coordinated expression patterns will have coordinated pathological effects.
To implement this, we used two complementary approaches.
The first is MultiPLIER, a method for extracting latent variables from large expression datasets.
These latent variables could represent either real transcriptional processes or technical factors ("batch effects").
We used a previously published model derived from recount2, but this may not be optimal for the range of complex diseases considered here.
The underlying factorization method is based on linear combinations of variables, which may miss important and more complex co-expression patterns.
In addition, recount2 has since been surpassed in size and scale by other resources.
However, our models impose few assumptions on the latent expression representation, so we can easily replace MultiPLIER with other similar approaches like GenomicSuperSignature. 

The second approach is TWAS, which only considers the hypothesis that GWAS loci affect traits via changes in gene expression.
Other effects, such as coding variants disrupting protein-protein interactions, are not captured.
Additionally, TWAS has several limitations that can lead to false positives.
As with GWAS, TWAS usually identifies several genes within the same locus due to sharing of GWAS variants in gene expression models, correlated expression of nearby genes, or correlation of their predicted expression due to eQTLs in linkage disequilibrium, among others.
Our latent variable-based regression framework accounts for these gene-gene correlations in TWAS well.


Our findings are in agreement with previous studies demonstrating that drugs with genetic backing are more likely to be successful in the drug development process [@doi:10.1038/ng.3314; @doi:10.1038/nn.4618].
By projecting association data through latent variables, we were able to prioritize disease-treatment pairs more effectively than by considering single-gene effects alone.
An additional advantage is that the latent variables used for predictions are interpretable genetic characteristics that can be analyzed to infer potential mechanisms of action.
We prioritized drugs for diseases with distinct tissue etiologies, but a challenge of this approach is to select the most appropriate tissue model from TWAS in order to identify gene expression patterns between genes and drug-induced alterations.


Ultimately, the quality of the representations is essential to performance.
Here, we used a representation derived from a factorization of bulk RNA-seq data.
To generate higher quality and more interpretable representations, detailed perturbation datasets and single-cell profiling of tissues, with and without perturbagens, and at various stages of development, can be used.
To increase interpretability, the key is to annotate sample metadata.
New approaches to infer and annotate with structured metadata are promising and can be applied to existing data [@doi:10.1101/2021.05.10.443525].
With rapid improvements in both areas, latent variable projections can be widely applied to understand the genetic basis of complex human phenotypes.
Our method provides a new perspective for a mechanistic understanding of statistical associations from TWAS, which can generate testable hypotheses for the post-GWAS functional characterization of complex diseases.
This is likely to be an area of great importance in the coming years.
