## Discussion

We present a novel computational strategy that combines statistical associations from TWAS with gene modules, which are groups of genes with similar expression patterns across the same cell types.
Our approach is unique in that we project gene-trait associations through a latent representation of cell types under various stimuli and at different developmental stages.
This allows us to go beyond statistical associations and infer cell type-specific features of complex phenotypes.
Our approach can identify disease-relevant cell types from summary statistics, and several disease-associated gene modules were replicated in eMERGE.
We used a CRISPR screen to analyze lipid regulation and found that our gene module-based approach can prioritize causal genes even when single gene associations are not detected.
These results suggest that our approach can identify genes that directly affect the trait without any mediated regulation of other genes, thereby prioritizing alternative and potentially more attractive therapeutic targets.


By integrating drug-induced transcriptional profiles, we were able to link diseases, drugs, and cell types.
Our analysis showed that the LV-based drug-repurposing approach was more effective than the gene-based one for predicting drug-disease links for 322 drugs across 53 diseases.
We further applied our approach to cardiovascular traits and niacin, demonstrating that it was able to connect pathophysiological processes to known mechanisms of action in adipose tissue, immune cells, and ovarian granulosa cells.
This approach could be useful for generating hypotheses to evaluate potential mechanisms of action or adverse effects of known or experimental drugs.


Our analysis revealed associations between genetic patterns and complex diseases and traits.
We found that clustering of these patterns provided reasonable groupings of diseases and traits affected by shared and distinct transcriptional mechanisms expressed in relevant tissues.
We identified latent variables (LVs) that were most discriminative for each cluster.
Some of these LVs were associated with known pathways, while others (e.g.
LV57) may represent novel disease-relevant mechanisms.
We observed modules expressed primarily in one tissue (such as adipose in LV246 or ovary in LV66), and others expressed in many contexts.
For example, LV136 was associated with cardiovascular disease and measures of corneal biomechanics, and was expressed in fibroblasts, osteoblasts, pancreas, liver, and cardiomyocytes.
Similarly, LV844 was expressed in whole blood samples and associated with a range of autoimmune diseases, while LV57 was strongly expressed in T cells and associated with autoimmune and venous thromboembolism.
Our approach of projecting genetic associations through gene expression patterns is a novel way to identify cell type and pathway effects on complex phenotypes, and is computationally simple to implement.


We demonstrated that clustering trees, which were initially used to study developmental processes in single-cell data, can provide a multi-resolution grouping of phenotypes based on latent variable associations.
We used hard-partitioning algorithms (where one trait exclusively belongs to one cluster) with a distance between two traits that takes into account all gene modules.
However, it is also possible for two complex diseases to share only a few biological processes instead of being similar across most of them.
Our TWAS results were derived from a large set of GWAS with different sample sizes and qualities, and we addressed potential issues from data heterogeneity before performing our cluster analyses.
Previous research has shown that considering groups of related diseases is more powerful in detecting shared genetic etiology [@doi:10.1038/ng.3985; @doi:10.1038/s41588-018-0121-0], and clustering trees provide a way to explore such relationships in the context of latent variables.


Finally, we developed a regression framework based on latent variables (LVs) to detect whether gene modules are associated with a trait using TWAS $p$-values.
We used PhenomeXcan as a discovery cohort for 4,000 traits, and many LV-trait associations were replicated in eMERGE.
In PhenomeXcan, we found 3,450 significant associations (FDR < 0.05) with 686 LVs associated with at least one trait and 1,176 traits associated with at least one LV.
In eMERGE, we found 196 significant associations, with 116 LVs associated with at least one trait/phecode and 81 traits with at least one LV.
We focused on a few disease types from our trait clusters, but the complete set of associations on other disease domains is available in our [Github repository](https://github.com/greenelab/phenoplier).
One limitation of the regression approach is that the gene-gene correlations are only approximately accurate, which could lead to false positives if the correlation among the top genes in a module is not precisely captured.
However, the regression model is approximately well-calibrated, and we did not observe inflation when running the method in real data.


Our approach in this work integrates two complementary approaches to project genetic associations through gene expression patterns.
The first is MultiPLIER, which extracts latent variables from large expression datasets, and these latent variables could represent either real transcriptional processes or technical factors.
We used a previously published model derived from recount2, which was designed for rare disorders.
This factorization method is based on linear combinations of variables, which may miss more complex co-expression patterns.
We can easily replace MultiPLIER with other similar approaches like GenomicSuperSignature.
The second approach is TWAS, which only considers the hypothesis that GWAS loci affect traits via changes in gene expression.
This method has several limitations, such as false positives and the identification of several genes within the same locus.
Our latent variable-based regression framework, however, accounts for these gene-gene correlations in TWAS reasonably well.


Our findings are consistent with those of previous studies, which showed that drugs with genetic support are more likely to be successful in the drug development process [@doi:10.1038/ng.3314; @doi:10.1038/nn.4618].
Projecting association results through latent variables enhanced the prioritization of disease-treatment pairs compared to considering single-gene effects alone.
Additionally, the latent variables driving predictions are interpretable genetic features that can be examined to infer potential mechanisms of action.
In this study, we prioritized drugs for diseases with different tissue etiologies.
However, a challenge of this approach is selecting the most appropriate tissue model from TWAS to identify transcriptome patterns between genes and drug-induced perturbations.


Ultimately, the quality of the representations is essential for performance.
Here, we used a representation derived from a factorization of bulk RNA-seq data.
To generate higher quality and more interpretable representations, detailed perturbation datasets and single-cell profiling of tissues with and without perturbagens at various stages of development can be employed.
Furthermore, the key to interpretability is driven by the annotation of sample metadata.
New approaches to infer and annotate with structured metadata are promising and can be directly applied to existing data [@doi:10.1101/2021.05.10.443525].
Rapid improvements in both areas will enable latent variable projections to be widely applied to decipher the genetic basis of complex human phenotypes.
Our method provides a new perspective for a mechanistic understanding of statistical associations from TWAS, which can generate testable hypotheses for the post-GWAS functional characterization of complex diseases.
This is likely to be an area of great importance in the coming years.
