## Discussion

We have developed a novel computational strategy that combines TWAS statistical associations with gene modules that have similar expression patterns across cell types.
Our innovation is that we project gene-trait associations through a latent representation derived from cell types under various stimuli and developmental stages.
This improves interpretation by going beyond statistical associations to infer cell type-specific features of complex phenotypes.
Our approach can identify disease-relevant cell types from summary statistics, and several disease-associated gene modules were replicated in eMERGE.
Using a CRISPR screen to analyze lipid regulation, we found that our gene module-based approach can prioritize causal genes even when single gene associations are not detected.
These findings support an omnigenic perspective of "core" and "peripheral" genes, suggesting that the approach can identify genes that directly affect the trait with no mediated regulation of other genes and thus prioritize alternative and potentially more effective therapeutic targets.


Using our gene module perspective, we integrated drug-induced transcriptional profiles to connect diseases, drugs, and cell types.
We found that the LV-based drug-repurposing approach outperformed the gene-based one when predicting drug-disease links for 322 drugs across 53 diseases.
To further demonstrate the utility of the approach, we focused on cardiovascular traits and a particular drug, niacin, to show that it can connect pathophysiological processes with known mechanisms of action, including those in adipose tissue, immune cells, and ovarian granulosa cells.
This LV-based approach could be helpful in generating novel hypotheses to evaluate potential mechanisms of action, or even adverse effects, of known or experimental drugs.


We found that the analysis of associations through latent representations provided reasonable groupings of diseases and traits affected by shared and distinct transcriptional mechanisms expressed in relevant tissues.
Our cluster analysis approach identified the latent variables (LVs) that were most discriminative for each cluster.
Several of these LVs were associated with different traits, some of which were strongly aligned with known pathways, while others (such as LV57) were not, which may represent novel disease-relevant mechanisms.
In some cases, the features/LVs linked to phenotypes appear to be associated with specific cell types, potentially revealing causal cell types for a phenotype with more precision.
We observed modules expressed primarily in one tissue (e.g.
adipose in LV246 or ovary in LV66), as well as others expressed in many contexts, which may capture pathways associated with related complex diseases.
For example, LV136, expressed in fibroblasts, osteoblasts, pancreas, liver, and cardiomyocytes, was associated with cardiovascular disease and measures of corneal biomechanics.
Other examples include LV844, expressed in whole blood samples and associated with a range of autoimmune diseases; and LV57, which is clearly expressed in T cells and associated with autoimmune and venous thromboembolism.
Our approach of projecting genetic associations through gene expression patterns is a novel approach to identifying cell type and pathway effects on complex phenotypes, which is computationally simple to implement.


<!-- In this portion, we used S-MultiXcan associations, which only provide the association strength between a gene and a trait, but with no direction of effect.
This does mean that traits are grouped based on associated genes, but genes could have opposite effects on traits within the same cluster. -->
Another important consideration is that our TWAS results were derived from a large set of GWAS of different sample sizes and qualities.
Although the potential issues derived from this data heterogeneity were addressed before performing our cluster analyses on traits, data preprocessing steps are always challenging and might not avoid bias altogether.
Considering groups of related diseases was previously shown to be more powerful in detecting shared genetic etiology [@doi:10.1038/ng.3985; @doi:10.1038/s41588-018-0121-0], and clustering trees provide a way to explore such relationships in the context of latent variables.
We showed that clustering trees can be used to group phenotypes based on latent variable associations.
We used hard-partitioning algorithms, which assigned each trait exclusively to one cluster.
The distance between two traits was calculated based on all gene modules.
This approach may be useful when two complex diseases share only a few biological processes, rather than having similarities across most of them.


Finally, we developed a regression framework based on latent variables (LVs) to detect whether gene modules are associated with a trait using TWAS $p$-values.
We used PhenomeXcan as a discovery cohort across four thousand traits, and many of the LV-trait associations replicated in eMERGE.
In PhenomeXcan, we found 3,450 significant LV-trait associations (FDR < 0.05) with 686 LVs associated with at least one trait and 1,176 traits associated with at least one LV.
In eMERGE, we found 196 significant LV-trait associations with 116 LVs associated with at least one trait/phecode and 81 traits with at least one LV.
We only focused on a few disease types from our trait clusters, but a complete set of associations on other disease domains is available in our Github repository for future research.
One limitation of the regression approach is that the gene-gene correlations are only approximately accurate, which could lead to false positives if the correlation among the top genes in a module is not precisely captured.
However, the regression model is approximately well-calibrated and we did not observe inflation when running the method in real data.


<!-- Larger datasets and methods designed to learn representations with this application in mind could further refine the approach and are a promising avenue for future research. -->
Our approach in this work integrates two complementary approaches.
The first is MultiPLIER, which extracts latent variables (LVs) from expression datasets.
The model used was derived from recount2, which was designed to analyze rare disorders.
However, it can be easily replaced with other similar approaches, such as GenomicSuperSignature.
The second approach is TWAS, which assumes that GWAS loci affect traits via changes in gene expression.
Limitations of TWAS can lead to false positives, and it usually identifies several genes within the same locus due to sharing of GWAS variants, correlated expression of nearby genes, or correlation of predicted expression due to eQTLs in linkage disequilibrium (LD).
Our LV-based regression framework accounts for these gene-gene correlations in TWAS.


Our findings are in line with previous studies, which showed that drugs with genetic support are more likely to be successful during the drug development process [@doi:10.1038/ng.3314; @doi:10.1038/nn.4618].
In this case, projecting association results through latent variables was more effective than considering the effects of single genes.
An additional advantage is that the latent variables driving the predictions can be interpreted to infer potential mechanisms of action.
In our study, we prioritized drugs for diseases with different tissue etiologies.
However, a challenge of this approach is to select the most appropriate tissue model from TWAS to identify the transcriptome patterns between genes and drug-induced perturbations.


Ultimately, the quality of the representations is essential to performance.
Here, we used a representation derived from a factorization of bulk RNA-seq data.
To generate higher quality and more interpretable representations, detailed perturbation datasets and single-cell profiling of tissues with and without perturbagens at various stages of development can be used.
Additionally, the key to interpretability is driven by the annotation of sample metadata.
New approaches to infer and annotate with structured metadata are promising and can be directly applied to existing data [@doi:10.1101/2021.05.10.443525].
These rapid improvements in both areas set the stage for latent variable projections to be widely applied to disentangle the genetic basis of complex human phenotypes.
Our method provides a new perspective for a mechanistic understanding of statistical associations from TWAS, and can generate testable hypotheses for the post-GWAS functional characterization of complex diseases.
This is likely to be an area of great importance in the coming years.
