## Discussion

We have introduced a novel computational approach that can map existing TWAS results into a latent gene expression representation to infer cell type-specific effects for complex diseases.
Leveraging one of the largest gene expression compendia, we integrated gene co-expression patterns on tens of thousands of RNA-seq samples covering different tissues and cell types under a variety of stimuli and developmental stages.
We showed that this latent representation is robust to capture relevant associations even when known single gene-trait effects are not detected.
Moreover, projecting gene-trait and gene-drug associations into this common representation can better link pathophysiological processes with the mechanisms of action of drugs, what led to a more accurate prediction of known disease treatments.
Finally, we showed how the approach could be used to group diseases and traits affected by the same transcriptional processes, which showed disease-specific modules expressed in highly relevant tissues.


The key difference of our approach with other alternatives is that we can incorporate patterns learned in tens of thousands of RNA-seq samples across a variety of cell types, stimuli and developmental stages.
<!--  -->
As we showed that gene modules are expressed in very specific cell types and this has the advange of identifying potentially causal cell types with more precision.
We provide examples of modules clearly expressed in one type of cell (such as adipose in LV246, thyroid in LV155 or ovary in LV66), whereas others seem to be activated across different contexts and thus capture a more realistic multi-tissue scenario for complex diseases (for example, LV136 is associated with coronary artery disease and expressed in fibroblasts, osteoblasts, pancreas, liver, and cardiomyocytes).
To our knowledge, this is a novel approach for the identification of tissue and cell type-specific effects that is computationally simple to implement.


Our approach assumes that gene modules with coordinated expression controlled by a perturbed transcriptional process will also manifest coordinated pathological effects on relevant diseases.
This leads to several limitations related to
1) the unsupervised learning process that detects modules of co-expressed genes (MultiPLIER),
and 2) the statistical methods that link SNP-trait to gene-trait associations through eQTL (TWAS/PhenomeXcan).
<!--  -->
First, MultiPLIER applies a matrix factorization algorithm to extract latent variables/modules across a large expression dataset.
These modules could represent real transcriptional processes or technical factors ("batch effects").
Although the deconvolution process successfully uses prior knowledge (pathways) to segregate noisy from real biological patterns, several detected modules could be either novel or just noise.
On the other hand, the underlying factorization method is restricted to linear patterns, what could miss important and more complex co-expression patterns.
<!--  -->
Second, TWAS approaches have several limitation that can lead to false positives [@doi:10.1038/s41588-019-0385-z; @doi:10.1016/j.ajhg.2020.11.012].
Similar to GWAS that usually detects groups of associated variants in LD (linkage disequilibrium), TWAS usually identifies several genes within the same locus [@doi:10.1038/s41588-018-0092-1; @doi:10.1038/ng.3367].
This is due to sharing of GWAS variants in gene expression models, to correlated expression of nearby genes, or even correlation of their predicted expression due to eQTLs in LD, among others [@doi:10.1038/s41588-019-0385-z].
<!--  -->
These two factors can combine to produce spurious module-trait associations, but our results suggest that the first aspect (module detection) could be the most problematic and probably more challenging to address.


Our approach is also in line with previous studies showing that drugs with genetic support are more likely to succeed through the drug development pipeline [@doi:10.1038/ng.3314; @doi:10.1038/nn.4618].
We show that a gene module perspetive can more accurately predict known therapeutics, and potentially serve as a more effective drug repositioning framework than considering single-gene effects.
This outcome is explained by the biologically meaningful patterns captured by the latent gene expression space.
Projecting single gene-trait associations and gene-compound profiles into this common representation translated into a better link between pathophysiological processes with the mechanisms of action of drugs.


Our consensus clustering pipeline combined different algorithms and assumptions on data structure to deliver highly stable groups of traits differentiated by common transcriptional processes.
Considering groups of related diseases was previously shown to be more powerful to detect shared genetic etiology [@doi:10.1038/ng.3985; @doi:10.1038/s41588-018-0121-0];
Our low-dimensional expression representation was used to detect and interpret, in an unsupervised manner, groups of diseases suspected to share common pathophysiological mechanisms, such as cardiovascular, autoimmune and mental diseases.
The approach, however, also has several limitation: 1) the data we used for grouping traits, and 2) the fit of the clustering algorithms used for highly multi-factorial traits.
First, we used the S-MultiXcan associations, which only provide the association strenght between a gene and a trait, but with no direction of effect.
This makes groups of traits harder to interpret, since we can only infer that transcriptional processes are associated with traits in the same cluster, but they could actually have opposite effects.
Second, we employed hard-partitioning algorithms (one trait belongs exclusively to one cluster): the distance between two traits takes into account all gene modules.
However, it is likely for two complex diseases to share just a few biological processes instead of being similar across all of them.
These more complex scenarios cannot be captured by the our clustering pipeline, as it can be seen for traits in the "complex" branch (with cardiovascular, mental and autoimmune diseases) which are hard to categorize into a single cluster or branch.
This limitation was circumvented at some degree by using the hierarchical structure provided by clustering trees.


The proposed approach will benefit of larger gene expression datasets on different tissues and cell types measured on highly heterogeneous conditions.
High-quality attributes for RNA-seq samples are key for a successful application of our method, and we welcome current efforts to improve this situation [@doi:10.1101/2021.05.10.443525].
This will enable the identification of more precise tissue- and cell lineage-specific effects on complex diseases.

<!--
- talk about the omnigenic model!
-->
