## Discussion

We present a novel computational strategy that integrates statistical associations from TWAS with groups of genes (gene modules) that have similar expression patterns across the same cell types.
Our approach improves the interpretation of genetic studies by using a latent representation derived not only from measures of normal tissue, but also from cell types under different stimuli and at various developmental stages.
This allows us to identify disease-relevant cell types from summary statistics, and several disease-associated gene modules were replicated in eMERGE.
Using a CRISPR screen to analyze lipid regulation, we found that our gene module-based approach can prioritize causal genes even when single gene associations are not detected.
We interpret these findings with an omnigenic perspective of "core" and "peripheral" genes, suggesting that this approach can identify genes that directly affect the trait with no mediated regulation of other genes, thus prioritizing alternative and potentially more attractive therapeutic targets for functional genomics, gene co-expression, and drug repurposing.


Our study combined genetic studies with functional genomics to project genetic associations through gene expression patterns.
We showed that this approach is effective in connecting diseases, drugs, and cell types.
We demonstrated that the LV-based drug-repurposing approach is better than the gene-based one when predicting drug-disease links for 322 drugs and 53 diseases.
We also focused on cardiovascular traits and a particular drug, niacin, to demonstrate how this approach can link pathophysiological processes to known mechanisms of action, such as those in adipose tissue, immune cells, and ovarian granulosa cells.
Our LV-based approach could be beneficial in generating hypotheses to evaluate potential mechanisms of action, or even adverse effects, of known or experimental drugs.


Our analysis revealed reasonable groupings of diseases and traits connected to shared and distinct transcriptional mechanisms expressed in relevant tissues.
We also identified latent variables (LVs) that were most discriminative for each cluster.
Some of these LVs were linked to known pathways, while others (e.g.
LV57) may represent novel disease-relevant mechanisms.
In some cases, the features/LVs associated with phenotypes were related to specific cell types, which could help to identify causal cell types with more precision.
We observed modules primarily expressed in one tissue (e.g.
adipose in LV246 or ovary in LV66), as well as those expressed in many contexts, which may capture pathways associated with related complex diseases.
For example, LV136 was associated with cardiovascular disease and corneal biomechanics, and was expressed in multiple tissues, including fibroblasts, osteoblasts, pancreas, liver, and cardiomyocytes.
Other examples include LV844, expressed in whole blood samples and associated with a range of autoimmune diseases, and LV57, which is clearly expressed in T cells and associated with autoimmune and venous thromboembolism.
This approach of projecting associations through a representation learned on complementary but distinct datasets is a novel way to identify cell type and pathway effects on complex phenotypes and is computationally simple to implement.


We demonstrated that clustering trees, initially used to study single-cell data, can provide a multi-resolution grouping of phenotypes based on latent variable associations.
We used hard-partitioning algorithms (where each trait belongs to one cluster) and the distance between two traits takes into account all gene modules.
It is also possible for two complex diseases to share only a few biological processes, rather than being similar across most of them.
Our TWAS results were derived from a large set of GWAS of different sample sizes and qualities, so potential issues from this data heterogeneity were addressed before performing the cluster analyses.
It has been previously shown that considering groups of related diseases is more powerful in detecting shared genetic etiology [@doi:10.1038/ng.3985; @doi:10.1038/s41588-018-0121-0], and clustering trees provide a way to explore such relationships in the context of latent variables.


Finally, we developed a linear regression-based framework to detect associations between gene modules and traits using TWAS $p$-values.
We used PhenomeXcan as a discovery cohort across four thousand traits, and many of the associations replicated in eMERGE.
In PhenomeXcan, we found 3,450 significant associations (FDR < 0.05) with 686 gene modules associated with at least one trait and 1,176 traits associated with at least one module.
In eMERGE, we found 196 significant associations, with 116 modules associated with at least one trait/phecode and 81 traits with at least one module.
We only focused on a few disease types from our trait clusters, but the complete set of associations is available in our Github repository for future research.
One limitation of the regression approach is that the gene-gene correlations are only approximately accurate, which could lead to false positives if the correlation among the top genes in a module is not precisely captured.
However, the regression model is approximately well-calibrated, and we did not observe inflation when running the method in real data.


Our approach assumes that gene modules with coordinated expression patterns will have coordinated pathological effects.
We used two complementary approaches to implement this.
The first was MultiPLIER, which extracts latent variables from large expression datasets.
These LVs could represent either real transcriptional processes or technical factors ("batch effects").
We used a previously published model derived from recount2, which was designed to analyze rare disorders.
However, this method relies on linear combinations of variables, which could miss more complex co-expression patterns.
Additionally, recount2 has since been surpassed in size and scale by other resources.
Nevertheless, our models impose few assumptions on the latent expression representation, so we can easily replace MultiPLIER with other similar approaches, such as GenomicSuperSignature.
The second approach was TWAS, which considers the hypothesis that GWAS loci affect traits via changes in gene expression.
However, this approach has several limitations that can lead to false positives.
Additionally, TWAS usually identifies several genes within the same locus due to sharing of GWAS variants in gene expression models, correlated expression of nearby genes, or even correlation of their predicted expression due to eQTLs in LD.
Our LV-based regression framework accounts for these gene-gene correlations in TWAS reasonably well.


Our findings are consistent with previous research, which suggests that drugs with genetic support are more likely to be successful during the drug development process [@doi:10.1038/ng.3314; @doi:10.1038/nn.4618].
Projecting association results through latent variables was more effective in prioritizing disease-treatment pairs than examining single-gene effects alone.
Additionally, the latent variables driving predictions are interpretable and can be used to infer potential mechanisms of action.
In this study, we prioritized drugs for diseases with different tissue etiologies.
A challenge of the approach is selecting the most appropriate tissue model from TWAS to identify gene expression patterns between genes and drug-induced perturbations.


Ultimately, the quality of the representations is essential for performance.
Here, we used a representation derived from a factorization of bulk RNA-seq data.
To generate higher quality and more interpretable representations, detailed perturbation datasets and single-cell profiling of tissues with and without perturbagens at various stages of development can be used.
Additionally, the key to interpretability is driven by the annotation of sample metadata.
New approaches to infer and annotate with structured metadata (e.g.
@doi:10.1101/2021.05.10.443525) are promising and can be applied to existing data.
As rapid improvements are made in both areas, it sets the stage for latent variable projections to be widely applied to disentangle the genetic basis of complex human phenotypes.
Our method provides a new perspective for a mechanistic understanding of statistical associations from TWAS and can generate testable hypotheses for the post-GWAS functional characterization of complex diseases, which will likely be an important area in the coming years.
