## Discussion

We have developed a new computational strategy that combines statistical associations from TWAS with gene modules that have similar expression patterns in the same cell types.
Our innovation is that we project gene-trait associations through a latent representation that includes not only normal tissue, but also cell types exposed to various stimuli and developmental stages.
This enables us to go beyond statistical associations to infer cell type-specific features of complex phenotypes.
Our approach can identify disease-relevant cell types from summary statistics, and some of the disease-associated gene modules were replicated in eMERGE.
Using CRISPR screening to analyze lipid regulation, we found that our gene module-based approach can prioritize causal genes even when single gene associations are not detected.
Our findings suggest an omnigenic perspective of "core" and "peripheral" genes, which could lead to the identification of genes that directly affect the trait without being mediated by other genes and thus prioritize alternative, potentially more attractive therapeutic targets.


Our gene module perspective allowed us to integrate drug-induced transcriptional profiles, connecting diseases, drugs, and cell types.
We found that the LV-based drug-repurposing approach was more effective than the gene-based one in predicting drug-disease links for 322 drugs across 53 diseases.
To further demonstrate its utility, we focused on cardiovascular traits and a specific drug, niacin, to illustrate how the approach can link pathophysiological processes to known mechanisms of action, including those in adipose tissue, immune cells, and ovarian granulosa cells.
Our LV-based approach may be useful for generating novel hypotheses to evaluate potential mechanisms of action, or even adverse effects, of known or experimental drugs.


We found that our analysis of genetic associations through latent representations provided reasonable groupings of diseases and traits affected by shared and distinct transcriptional mechanisms expressed in relevant tissues.
Our cluster analysis approach also detected the latent variables (LVs) that were most discriminative for each cluster.
Several of these LVs were significantly associated with different traits, and some were strongly aligned with known pathways, while others (e.g.
LV57) were not, which may represent novel, disease-relevant mechanisms.
In some cases, the features/LVs linked to phenotypes were associated with specific cell types, which may help reveal causal cell types for a phenotype more precisely.
We observed modules expressed primarily in one tissue (such as adipose in LV246 or ovary in LV66), as well as those expressed in many contexts, which may capture pathways associated with related complex diseases.
For example, LV136 is associated with cardiovascular disease, corneal biomechanics, and is expressed in various tissues including fibroblasts, osteoblasts, pancreas, liver, and cardiomyocytes.
Other examples include LV844, expressed in whole blood samples and associated with a range of autoimmune diseases; and LV57, which is expressed in T cells and strongly associated with autoimmune and venous thromboembolism.
Our approach of projecting genetic associations through gene expression patterns may be a novel way to identify cell type and pathway effects on complex phenotypes, and it is computationally simple to implement.


We demonstrated how clustering trees can be used to examine the associations between phenotypes based on latent variables.
Clustering trees employ hard-partitioning algorithms, meaning that each trait belongs exclusively to one cluster, and the distance between two traits takes into account all gene modules.
It is possible for two complex diseases to share only a few biological processes instead of being similar across most of them.
The results of our TWAS analyses were derived from a large set of GWAS of different sample sizes and qualities, and we took steps to address any potential issues due to data heterogeneity.
Previous studies have shown that considering groups of related diseases is more powerful in detecting shared genetic etiology [@doi:10.1038/ng.3985; @doi:10.1038/s41588-018-0121-0], and clustering trees provide a way to explore such relationships in the context of latent variables.


We developed a linear regression-based framework to detect if gene modules are associated with a trait using TWAS $p$-values.
We tested it in PhenomeXcan, a discovery cohort of four thousand traits, and in eMERGE, a replication cohort.
In PhenomeXcan, we found 3,450 significant LV-trait associations (FDR < 0.05) with 686 out of 987 LVs associated with at least one trait, and 1,176 traits associated with at least one LV.
In eMERGE, we found 196 significant LV-trait associations, with 116 LVs associated with at least one trait/phecode and 81 traits with at least one LV.
We focused on a few disease types from our trait clusters, and the complete set of associations is available in our Github repository for future research.
Our approach is limited in that the gene-gene correlations are only approximately accurate, which could lead to false positives if the correlation among the top genes in a module is not precisely captured.
However, the regression model is approximately well-calibrated and we did not observe inflation when running the method in real data.


Our approach in this work combines two complementary approaches.
The first is MultiPLIER, which extracts latent variables from large expression datasets.
This can represent either real transcriptional processes or technical factors ("batch effects").
We used a previously published model derived from recount2, which was designed to analyze rare disorders.
However, this approach imposes very few assumptions on the latent expression representation, so it can be easily replaced with other similar approaches like GenomicSuperSignature.
The second approach is TWAS, which only considers the hypothesis that GWAS loci affect traits via changes in gene expression.
This method has several limitations that can lead to false positives, and usually identifies several genes within the same locus due to sharing of GWAS variants, correlated expression of nearby genes, or even correlation of their predicted expression due to eQTLs in linkage disequilibrium.
Our LV-based regression framework, however, accounts for these gene-gene correlations in TWAS reasonably well.


Our findings are consistent with the results of previous studies, which showed that drugs with genetic support are more likely to succeed in the drug development process [@doi:10.1038/ng.3314; @doi:10.1038/nn.4618].
Our approach of projecting association results through latent variables was more effective at prioritizing disease-treatment pairs than considering single-gene effects alone.
Additionally, these latent variables can be studied to infer potential mechanisms of action.
We used this approach to prioritize drugs for diseases with different tissue etiologies, although a challenge is selecting the most appropriate tissue model from TWAS to identify gene expression patterns between drugs and disease-induced perturbations.


Ultimately, the quality of the representations is essential to performance.
Here, we used a representation derived from a factorization of bulk RNA-seq data.
To generate higher quality and more interpretable representations, detailed perturbation datasets and single-cell profiling of tissues with and without perturbagens at various stages of development can be used.
On the other hand, the key to interpretability is driven by the annotation of sample metadata.
New approaches to infer and annotate with structured metadata are promising and can be applied to existing data [@doi:10.1101/2021.05.10.443525].
These rapid improvements in both areas set the stage for latent variable projections to be widely applied to disentangle the genetic basis of complex human phenotypes.
By providing a new perspective for a mechanistic understanding of statistical associations from TWAS, our method can generate testable hypotheses for the post-GWAS functional characterization of complex diseases, which will be an important area in the coming years.
