## Discussion

<!--
-summarize results and why it is important
"We have shown that..." something like that
RE-READ INTRODUCTION TO SEE WHAT WE PROMISED THERE
- for the clustering part, we are using S-MultiXcan results which have no direction of effects; this only allows us to infer that traits in the same cluster might be affected by the same processes, but we have less information to infer directionality

- We depend a lot on good resources and annotations of gene expression data.
For example, LV136 was associated with CAD and fibroblasts were at the top, but with a high variance/variability: where are those fibroblasts from? where they treated with different compounds, sampled under different conditions?
Resources to improve annotation of gene expression data will be fundamental to under these questions.

- our approach uses a hard clustering method, which exclusively assigns a trait to a cluster.
this limits the ability to capture more complex relationships where one transcriptional process might be affecting several different traits.
the behavior of traits moving to different clusters or subbranches shows this complexity
this happens becase the models we use take into account all latent variables when computing a distance between a pair of traits
a more flexible approach, such as biclustering methods, might be helpful to detect these more complex scenarios

- advantages of our approach (more robust to noise, method is intuitive and computationally simple to implement, etc)

- TWAS are subject to many vulnerabilities; we show in this paper that a gene module perspective can reduce noisy associations and more clearly reveal meaningful processes underlying disease.
- The approach is promising for a drug repurposing scenario, and also study potential adverse effects.
- In line with previous studies showing that drugs with genetic support are more likely to succeed [https://www.nature.com/articles/ng.3314 https://www.nature.com/articles/nn.4618], we show that a gene module perspetive can more accurately predict known therapeutics, and potentially serve as a more effective drug repositioning framework.

- if there is module that is missing in our models, we will have less performance

- Although we focused on TWAS and PrediXcan, our approach does not depend on a particular gene-based test.

- the importance to invest more resources in finding gene co-expression patterns to increase the number of potential gene modules
- one disadvantage is that we have a limited amount of genes, only 6.000 out of 22k in phenomexcan;
more sophisticated methods for gene co-expression pattern discovery are needed to expand the implications of these analyses and cover more contexts.
- this limited amount of genes makes our approach biased to already known pathways and mechanisms
- maybe something saying that we work at the gene-level, not SNP-level, not sure; in the heritability paper (nat gen) they said one limitation of their approach is that they need a matched LD reference panel. I think we don't need that reference panel.
-->
