## Methods and materials {#sec:methods}

PhenoPLIER is a framework that combines different computational approaches to integrate gene-trait associations and drug-induced transcriptional responses with groups of functionally-related genes (referred to as gene modules or latent variables/LVs).
Gene-trait associations are computed using the PrediXcan family of methods \cite{Gazal2018}, whereas latent variables are inferred by the MultiPLIER models \cite{Gazal2019} applied on large gene expression compendia.
PhenoPLIER provides (1) a regression model to compute an LV-trait association, (2) a consensus clustering approach applied to the latent space to learn shared and distinct transcriptomic properties between traits, and (3) an interpretable, LV-based drug repurposing framework.
We provide the details of these methods below.

The regression model is based on the equation $$\mathbf{y} = \mathbf{X}\boldsymbol{\beta} + \boldsymbol{\epsilon}$$ {#id}, where $\mathbf{y}$ is a vector of trait values, $\mathbf{X}$ is a matrix of latent variables, $\boldsymbol{\beta}$ is a vector of regression coefficients, and $\boldsymbol{\epsilon}$ is a vector of noise terms.
The MultiPLIER model \cite{Gazal2019} is used to infer the latent variables $\mathbf{X}$ from gene expression data.
The consensus clustering approach \cite{Strehl2002} is used to learn shared and distinct transcriptomic properties between traits by applying a clustering algorithm to the latent space.
Finally, the interpretable, LV-based drug repurposing framework \cite{Gazal2019} is used to identify drug


### The PrediXcan family of methods for gene-based associations {#sec:methods:predixcan}

The models used in S-PrediXcan and S-MultiXcan are defined as follows:

We used Summary-PrediXcan (S-PrediXcan) [@doi:10.1038/s41467-018-03621-1] and Summary-MultiXcan (S-MultiXcan) [@doi:10.1371/journal.pgen.1007889], belonging to the PrediXcan family of methods [@doi:10.1038/ng.3367], as the gene-based statistical approaches.
We broadly refer to these approaches as TWAS (transcription-wide association studies).
S-PrediXcan, the summary-based version of PrediXcan, computes the univariate association between a trait and a gene's predicted expression in a single tissue.
S-MultiXcan, the summary-based version of MultiXcan, on the other hand, computes the joint association between a gene's predicted expression in all tissues and a trait.
Both S-PrediXcan and S-MultiXcan require only GWAS summary statistics instead of individual-level genotype and phenotype data.
The models used in S-PrediXcan and S-MultiXcan are defined as follows:
$$
\begin{aligned}
y_{i} &= \beta_{0} + \beta_{g} g_{i} + \beta_{z} z_{i} + \epsilon_{i} \\
g_{i} &= \gamma_{0} + \gamma_{z} z_{i} + \eta_{i}
\end{aligned}
$$ {#eq1}
where $y_{i}$ is the phenotype of the $i$-th individual, $g_{i}$ is the expression of the gene of interest in the

We briefly provide the details of the TWAS methods necessary to explain our regression framework later (see the referenced articles for more information).
Let $\mathbf{y}$ be a vector of traits for $n$ individuals, centered for convenience (so that no intercept is necessary).
For each tissue $l$, $\mathbf{\tilde{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression, where $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$.
$\mathbf{t}_l$ is the standardized version of $\mathbf{\tilde{t}}_l$, with mean equal to zero and standard deviation equal to one.

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367], which models the trait as a linear function of the gene's expression on a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$.
The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$.
Whereas PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS.
In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel.
Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).

S-MultiXcan [@doi:10.1371/journal.pgen.1007889] is a summary version of MultiXcan, which is more powerful than PrediXcan in detecting gene-trait associations, but does not provide the direction of effects.
Its main output is the $p$-value (obtained with an F-test) of the multiple tissue model given by Equation (@eq:multixcan):
$$
\begin{split}
\mathbf{y} & = \sum_{l=1}^{p} \mathbf{t}_l g_l + \mathbf{e} \\
& = \mathbf{T} \mathbf{g} + \mathbf{e},
\end{split}
$$ {#eq:multixcan}
where $\mathbf{T}$ is a matrix with $p$ columns $\mathbf{t}_l$, $\hat{g}_l$ is the estimated effect size for the predicted gene expression in tissue $l$ (and thus $\mathbf{\hat{g}}$ is a vector with $p$ estimated effect sizes $\hat{g}_l$), and $\mathbf{e}$ are the error terms with variance $\sigma_{e}^{2}$.
To avoid collinearity issues due to the high correlation between predicted expression values for a gene across different tissues, MultiXcan uses the principal components (PCs) of $\mathbf{T}$. 

S-MultiXcan derives the joint regression estimates (effect sizes and their variances) in Equation (@eq:multixcan) using the marginal estimates from S-PrediXcan in Equation (@eq:spredixcan).
Under the null hypothesis of no association, $\mathbf{\hat{g}}^{\top} \frac{\mathbf{T}^{\top}\mathbf{T}}{\sigma_{e}^{2}} \mathbf{\hat{g}} \sim \chi_{p}^{2}$, and therefore the significance of the association in S-MultiXcan is estimated with Equation (@eq:smultixcan):
$$
\begin{split}
\frac{\mathbf{\hat{g}}^{\top} (\mathbf{T}^{\top}\mathbf{T}) \mathbf{\hat{g}}}{\sigma_{e}^{2}} & \approx \bm{\hat{\gamma}}^{\top} \frac{\sqrt{n-1}}{\sigma_{\epsilon}} \left(\frac{\mathbf{T}^{\top} \mathbf{T}}{n-1}\right)^{-1} \frac{\sqrt{n-1}}{\sigma_{\epsilon}} \bm{\hat{\gamma}} \\
& = \mathbf{\hat{z}}^{\top} Cor(\mathbf{T})^{-1} \mathbf{\hat{z}},
\end{split}
$$ {#eq:smultixcan}
where $\mathbf{\hat{z}}$ is a vector with $p$ $z$-scores (Equation (@eq:spredixcan)) for each tissue available for the gene, and $Cor(\mathbf{T})$ is the autocorrelation matrix of $\mathbf{T}$.
To compute this expression, S-MultiXcan uses the conservative approximation $\sigma_{e}^{2} \approx \sigma_{\epsilon}^{2}$, that is, the variance of the error terms in the joint regression is approximately equal to the residual variance of the marginal regressions.
It also estimates $Cor(\mathbf{T})^{+}$ using the $k$ top PCs, and thus $\mathbf{\hat{z}}^{\top} Cor(\mathbf{T})^{+} \mathbf{\hat{z}} \sim \chi_k^2$.
Results from S-MultiXcan are highly concordant compared with MultiXcan, although not perfectly correlated across genes [@doi:10.1371/journal.pgen.1007889].
We used S-MultiXcan results for our LV-based regression model and our cluster analyses of traits.


### TWAS resources {#sec:methods:twas}

We used two large TWAS resources from different cohorts for discovery and replication, all obtained from European ancestries.
PhenomeXcan [@doi:10.1126/sciadv.aba2083], our discovery cohort, provides results on 4,091 traits across different categories (details in Supplementary File 1).
In PhenomeXcan, publicly available GWAS summary statistics were used to compute gene-based associations with the PrediXcan family of methods (described previously) and a posterior probability of colocalization between GWAS loci and *cis*-eQTL with fastENLOC [@doi:10.1126/sciadv.aba2083; @doi:10.1016/j.ajhg.2020.11.012].
We refer to the matrix of $z$-scores from S-PrediXcan (Equation (@eq:spredixcan)) across $q$ traits and $m$ genes in tissue $t$ as $\mathbf{M}^{t} \in \mathbb{R}^{q \times m}$, which provides direction of effects and was used in our LV-based drug repurposing framework.
The S-MultiXcan results (22,515 gene associations across 4,091 traits) were used in our LV-based regression framework and our cluster analyses of traits.
For the latter, we converted the $p$-values to $z$-scores: $\mathbf{M}=\Phi^{-1}(1 - p/2)$, where $\Phi^{-1}$ is the probit function.
Higher $z$-scores correspond to stronger associations.

Our discovery cohort was eMERGE [@doi:10.1038/gim.2013.72], where the same TWAS methods were run on 309 phecodes [@doi:10.1101/2021.10.21.21265225] across different categories (more information about traits are available in [@doi:10.1101/2021.10.21.21265225]).
We used these results to replicate the associations found with our LV-based regression framework in PhenomeXcan.


### MultiPLIER and Pathway-level information extractor (PLIER) {#sec:methods:multiplier}

MultiPLIER extracts patterns of co-expressed genes from Recount2 [@doi:10.1038/nbt.3838], a large gene expression dataset, using the Pathway-Level Information Extractor (PLIER) method [@doi:10.1038/s41592-019-0456-1].
PLIER is an unsupervised learning approach that incorporates prior knowledge (canonical pathways) to reduce technical noise.
It employs a matrix factorization approach to deconvolute gene expression data into a set of latent variables (LVs), with each LV representing a gene module.
Applying MultiPLIER to Recount2 reduced the dimensionality to 987 LVs.

Given a gene expression dataset $\mathbf{Y}$ with $m$ genes and $c$ experimental conditions, and a prior knowledge matrix $\mathbf{C}$ for $p$ MSigDB pathways [@doi:10.1016/j.cels.2015.12.004], PLIER minimizes the following equation:

$$
||\mathbf{Y} - \mathbf{Z}\mathbf{B}||^{2}_{F} + \lambda_1 ||\mathbf{Z} - \mathbf{C}\mathbf{U}||^{2}_{F} + \lambda_2 ||\mathbf{B}||^{2}_{F} + \lambda_3 ||\mathbf{U}||_{L^1}
$$ {#eq:met:plier_func}

subject to $\mathbf{U}>0, \mathbf{Z}>0$.
Here, $\mathbf{Z}$ is a low-dimensional representation of the gene space with $l$ latent variables, $\mathbf{B}$ is the latent space for $c$ conditions, $\mathbf{U}$ specifies which of the $p$ prior-information pathways in $\mathbf{C}$ are represented for each latent variable, and $\lambda_i$ are different regularization parameters used in the training step.
This equation finds $\mathbf{U}$, $\mathbf{Z}$, and $\mathbf{B}$, and $\mathbf{Z}$ may represent either a known or novel gene module (i.e., a meaningful biological pattern) or noise.

We used the model to project genetic associations (from TWAS) and drug-induced transcriptional profiles (from LINCS L1000) into the low-dimensional gene module space.
Specifically, TWAS associations $\mathbf{M}$ (either from S-PrediXcan or S-MultiXcan) were projected using the equation:

$$
\hat{\mathbf{M}} = (\mathbf{Z}^{\top} \mathbf{Z} + \lambda_{2} \mathbf{I})^{-1} \mathbf{Z}^{\top} \mathbf{M},
$$ {#eq:proj}

where $\hat{\mathbf{M}}^{l \times q}$ is a matrix in which traits are represented by gene modules instead of single genes.
We used the same approach to project drug-induced transcriptional profiles in LINCS L1000 to obtain a representation of drugs using gene modules.


### Regression model for LV-trait associations {#sec:methods:reg}

We adapted the gene-set analysis framework from MAGMA [@doi:10.1371/journal.pcbi.1004219] to TWAS.
We tested whether the genes with the largest loadings for a latent variable (LV) were more strongly associated with a phenotype than other genes with relatively small or zero weights, by fitting the model:

$$
\mathbf{m}=\beta_{0} + \mathbf{s} \beta_{s} + \sum_{i} \mathbf{x}_{i} \beta_{i} + \bm{\epsilon},
$$ {#eq:reg:model}

where $\mathbf{m}$ is a vector of S-MultiXcan gene $p$-values for a trait (with a $-log_{10}$ transformation); $\mathbf{s}$ is a binary indicator vector, with $s_{\ell}=1$ for the top 1% of genes with the largest loadings for LV $\ell$ (from $\mathbf{Z}_{\ell}$) and zero otherwise; $\mathbf{x}_{i}$ is a gene property used as a covariate; $\beta$ are effect sizes (with $\beta_{0}$ as the intercept); and $\bm{\epsilon} \sim \mathrm{MVN}(0, \sigma^{2} \mathbf{R})$ is a vector of error terms with a multivariate normal distribution (MVN), where $\mathbf{R}$ is the matrix of gene correlations.

We then tested the association of each gene with the trait using a linear model with the two covariates and the LV effect $\beta_{s}$ (Equation \ref{eq:1}):

We tested the null hypothesis $\beta_{s} = 0$ against the one-sided hypothesis $\beta_{s} > 0$ using a linear model with two covariates and the LV effect $\beta_{s}$ (Equation \ref{eq:1}):

$$ y_{g, t} = \beta_{0} + \beta_{g} + \beta_{t} + \beta_{s} + \beta_{gsize} \cdot gsize_{g} + \beta_{gden} \cdot gden_{g} + \epsilon_{g, t} \;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;

We used a generalized least squares approach to account for correlations between the error terms $\bm{\epsilon}$ in the PrediXcan family of methods.
The gene-gene correlation matrix $\mathbf{R}$ was approximated by computing the correlations between the model sum of squares (SSM) for each pair of genes under the null hypothesis of no association.
To estimate the correlation of predicted expression levels for genes $i$ in tissue $k$ and gene $j$ in tissue $l$, we used the cross-correlation matrix between the predicted expression levels of genes $i$ and $j$ and the eigenvectors and eigenvalues of $\mathbf{T}_{i}$.
The variance of the predicted expression values of gene $i$ in tissue $k$ was estimated using the genotype covariance matrix and the weights of SNPs for gene expression prediction in the tissue model $k$.

We used the MultiXcan regression model (Equation (@eq:multixcan)) to approximate gene correlations in S-MultiXcan.
To do this, we used the marginal regression estimates from S-PrediXcan (Equation (@eq:spredixcan)) with some simplifying assumptions and different genotype covariance matrices.
To account for this, we used a submatrix $\mathbf{R}_{\ell}$ corresponding to genes that are part of LV $\ell$ only (top 1% of genes) instead of the entire matrix $\mathbf{R}$.
This simplification is conservative since correlations are accounted for top genes only.
Our simulations (Supplementary Note 1; #sm:reg:null_sim) show that the model is approximately well-calibrated and can correct for LVs with adjacent and highly correlated genes at the top (e.g., Figure @fig:reg:nulls:qqplot:lv234).
The model can also detect LVs associated with relevant traits (Figure @fig:lv246 and Table @tbl:sup:phenomexcan_assocs:lv246) that are replicated in a different cohort (Table @tbl:sup:emerge_assocs:lv246).

In Equation (@eq:reg:corr_genes), for each gene, we only considered tissue models present in S-PrediXcan results, as well as SNPs present in GWAS used as input for the TWAS approaches.
This is necessary to obtain more accurate correlation estimates [@doi:10.1371/journal.pgen.1007889].
Therefore, we computed different correlation matrices for PhenomeXcan and eMERGE.
For PhenomeXcan, we used a single correlation matrix for the 4,049 GWAS obtained from the UK Biobank, which were generated using the same pipeline and included the same set of SNPs.
For the remaining GWAS, we used a single correlation matrix for each group of traits that shared the same or most of the SNPs.

We ran our regression model for all 987 LVs across the 4,091 traits in PhenomeXcan.
For replication, we ran the model in the 309 phecodes in eMERGE.
We adjusted the $p$-values using the Benjamini-Hochberg procedure.


### LV-based drug repurposing approach {#sec:methods:drug}

For the drug-disease prediction, we derived a linear vector (LV)-based method based on a drug repositioning framework previously used for psychiatry traits [@doi:10.1038/nn.4618].
We compared this LV-based method with the single-gene approach from the same framework.
For the single-gene method, we computed a drug-disease score by multiplying each S-PrediXcan set of signed $z$-scores in tissue $t$, $\mathbf{M}^t$, with another set of signed $z$-scores from transcriptional responses profiled in LINCS L1000 [@doi:10.1016/j.cell.2017.10.049], $\mathbf{L}^{c \times m}$ (for $c$ compounds).
Here, $\mathbf{M}^t$ contains information about whether a higher or lower predicted expression of a gene is associated with disease risk, whereas $\mathbf{L}$ indicates whether a drug increases or decreases the expression of a gene.
Therefore, these two matrices can be multiplied to compute a score for a drug-disease pair, resulting in $\mathbf{D}^{t,k}=-1 \cdot \mathbf{M}^{t,k} \mathbf{L}^\top$, where $k$ refers to the number of most significant gene associations in $\mathbf{M}^t$ for each trait.
As suggested in [@doi:10.1038/nn.4618], $k$ could be either all genes or the top 50, 100, 250, and 500; then, we averaged the score ranks across all $k$ and obtained $\mathbf{D}^t$.
Finally, for each drug-disease pair, we took the maximum prediction score across all tissues: $\mathbf{D}_{ij} = \max \{ \mathbf{D}_{ij}^t \mid \forall t \}$.


The same procedure was used for the LV-based approach, where we projected $\mathbf{M}^{t}$ and $\mathbf{L}$ into the gene module latent space using Equation (@eq:proj), leading to $\hat{\mathbf{M}}^t$ and $\hat{\mathbf{L}}^{l \times c}$, respectively.
Finally, $\mathbf{D}^{t,k}=-1 \cdot \hat{\mathbf{L}}^{\top} \hat{\mathbf{M}}^{t,k}$, where in this case $k$ could be all LVs or the top 5, 10, 25 and 50 (since we have an order of magnitude less LVs than genes).


Since the gold standard of drug-disease medical indications is described with Disease Ontology IDs (DOID) [@doi:10.1093/nar/gky1032], we mapped PhenomeXcan traits to the Experimental Factor Ontology [@doi:10.1093/bioinformatics/btq099] using [@url:https://github.com/EBISPOT/EFO-UKB-mappings], and then to DOID.


### Consensus clustering of traits {#sec:methods:clustering}

We performed two preprocessing steps on the S-MultiXcan results before the cluster analysis.
First, we combined results in $\mathbf{M}$ (with $p$-values converted to $z$-scores, as described before) for traits that mapped to the same Experimental Factor Ontology (EFO) [@doi:10.1093/bioinformatics/btq099] term using the Stouffer's method: $$\frac{\sum w_i M_{ij}}{\sqrt{\sum w_i^2}},$$ where $w_i$ is a weight based on the GWAS sample size for trait $i$, and $M_{ij}$ is the $z$-score for gene $j$.
Second, we divided all $z$-scores for each trait $i$ by their sum to reduce the effect of highly polygenic traits: $$\frac{M_{ij}}{\sum M_{ij}}.$$ Finally, we projected this data matrix using Equation (@eq:proj), obtaining $\hat{\mathbf{M}}$ with $n$=3,752 traits and $l$=987 Latent Variables (LVs) as the input of our clustering pipeline.


A partitioning of $\hat{\mathbf{M}}$ with $n$ traits into $k$ clusters is represented as a label vector $\pi \in \mathbb{N}^n$.
Consensus clustering approaches consist of two steps.
First, an ensemble $\Pi$ with $r$ partitions of the dataset is generated: $\Pi=\{\pi_1, \pi_2, \ldots, \pi_r\}$.
Second, a consolidated solution is combined from the ensemble, defined as:

$$
\pi^* = \mathrm{arg}\,\underset{\hat{\pi}}{\max} Q(\{ \lvert \mathcal{L}^i \lvert \phi(\hat{\pi}_{\mathcal{L}^i}, \pi_{i \mathcal{L}^i}) \mid i \in \{1,\ldots,r\} \}),
$$ {#eq:consensus:obj_func}

where $\mathcal{L}^i$ is a set of data indices with known cluster labels for partition $i$, $\phi\colon \mathbb{N}^n \times \mathbb{N}^n \to \mathbb{R}$ is a function that measures the similarity between two partitions, and $Q$ is a measure of central tendency, such as the mean or median.
We used the adjusted Rand index (ARI) [@doi:10.1007/BF01908075] for $\phi$ and the median for $Q$.
To obtain $\pi^*$, we define a consensus function $\Gamma\colon \mathbb{N}^{n \times r} \to \mathbb{N}^n$ with $\Pi$ as the input.
We employed consensus functions based on the evidence accumulation clustering (EAC) paradigm [@doi:10.1109/TPAMI.2005.113], which first transforms $\Pi$ into a distance matrix $\mathbf{D}_{ij} = d_{ij} / r$, where $d_{ij}$ is the number of times traits $i$ and $j$ were grouped in different clusters across all $r$ partitions in $\Pi$.
Then, $\Gamma$ is any similarity-based clustering algorithm, which is applied on $\mathbf{D}$ to derive the final partition $\pi^*$.


We generated a highly diverse set of partitions for the ensemble generation step (see Figure @fig:clustering:design).
To do this, we used three data representations: the raw dataset, its projection into the top 50 principal components, and the embedding learned by UMAP [@arxiv:1802.03426] using 50 components.
For each of these, we applied five clustering algorithms: $k$-means [@Arthur2007], spectral clustering [@Ng2001], a Gaussian mixture model (GMM), hierarchical clustering, and DBSCAN [@Ester1996].
For $k$-means, spectral clustering and GMM, we specified a range of $k$ values between 2 and $\sqrt{n} \approx 60$, and for each $k$ we generated five partitions using random seeds.
For hierarchical clustering, for each $k$, we generated four partitions using common linkage criteria: ward, complete, average and single.
For DBSCAN, we combined different ranges for parameters $\epsilon$ (the maximum distance between two data points to be considered part of the same neighborhood) and *minPts* (the minimum number of data points in a neighborhood for a data point to be considered a core point), based on the procedure in [@doi:10.1088/1755-1315/31/1/012012].
Specifically, we used *minPts* values from 2 to 125 and determined a plausible range of $\epsilon$ values by observing the distribution of the mean distance of the *minPts*-nearest neighbors across all data points.
To ensure an equal representation of DBSCAN in the ensemble, we resampled partitions generated by this algorithm.
This procedure generated a final ensemble of 4,428 partitions of 3,752 traits, which is important for diversity since it is an important property for ensembles [@doi:10.1016/j.ins.2016.04.027; @doi:10.1109/TPAMI.2011.84; @doi:10.1016/j.patcog.2014.04.005].


We applied spectral clustering on $\mathbf{D}$ to derive the final consensus partitions.
$\mathbf{D}$ was first transformed into a similarity matrix by applying an RBF kernel $\mathrm{exp}(-\gamma \mathbf{D}^2)$, where $\gamma$ is a parameter that we empirically determined to work best.
We used four different values for $\gamma$.
For each $k$ between 2 and 60, we derived four consensus partitions and selected the one that maximized the objective function in Equation (@eq:consensus:obj_func).
We further filtered this set of 59 solutions to keep only those with an ensemble agreement larger than the 75th percentile (Supplementary Figure @fig:sup:clustering:agreement), leaving a total of 15 final consensus partitions shown in Figure @fig:clustering:tree.

The input data for our clustering pipeline underwent several linear and nonlinear transformations, including principal component analysis (PCA), uniform manifold approximation and projection (UMAP) and the ensemble transformation using the eigengene-based consensus clustering (EAC) paradigm (distance matrix $\mathbf{D}$).
Although consensus clustering has clear advantages for biological data [@pmid:27303057], these data transformations complicate interpretation of the results.
To address this, we used a supervised learning approach to detect which gene modules/latent variables (LVs) were the most important for each cluster of traits (Figure {@fig:clustering:design}b).
We did not use this supervised model for prediction, but only to learn which features (LVs) were the most discriminative for each cluster.
For this, we used the highest resolution partition ($k$=29, although any could be used) to train a decision tree model, using each of the clusters as labels and the projected data $\hat{\mathbf{M}}$ as the training samples.
For each $k$, we constructed a set of binary labels, with the current cluster's traits as the positive class and the rest of the traits as the negative class.
Then, we selected the LV in the root node of the trained model only if its threshold was positive and larger than one standard deviation.
Next, we removed this LV from $\hat{\mathbf{M}}$ (regardless of whether it was previously selected or not) and trained the model again.
We repeated this procedure 20 times to extract the top 20 LVs that best discriminated traits in a cluster from the rest.

In [Supplementary Note 2](#sm:clustering:null_sim), we performed several analyses under a null hypothesis of no structure in the data to verify that the clustering results detected by this pipeline were real.


### CRISPR-Cas9 screening {#sec:methods:crispr}

HepG2 cells were obtained from ATCC (ATCC® HB-8065™) and cultured in Eagle's Minimum Essential Medium with L-Glutamine (EMEM; Quality Biology, Cat.
112-018-101) supplemented with 10% Fetal Bovine Serum (FBS; Gibco, Cat.
16000-044) and 1% Pen/Strep (Gibco, Cat.
15140-122).
The cells were maintained at 37°C in a humidity-controlled incubator with 5% CO2 and kept at a density not exceeding 80% confluency.

A third-generation lentiviral CRISPR-Cas9 pooled library (Cat.
73179-LV) provided by David Root and John Doench from Addgene was used for HepG2 cell transduction.
This library consists of 76,441 sgRNAs that target 19,114 genes in the human genome, with an average of 4 sgRNAs per gene.
The sgRNA cassette was inserted into the lentiCRIS-PRv2 backbone between the U6 promoter and the gRNA scaffold.
Lentiviral vectors encoding Cas9 were used to deliver the sgRNA cassette-containing plasmids into cells during cell replication.
Unsuccessful transduced cells were then excluded through puromycin selection.

Lentiviral titer determination was performed using no-spin lentiviral transduction.
Cells were seeded in a Collagen-I coated 6-well plate with 8 μg/ml of polybrene (Millipore Sigma, Cat.
TR-1003 G) and a different titrated virus volume (e.g., 0, 50, 100, 200, 250, and 400 μl) assigned to each well.
The final volume of 1.24 ml was made up with EMEM complete media.
After 16-18 hours of post-transduction, the virus/polybrene-containing media was removed and cells were washed twice with 1x DPBS.
The cells were then trypsinized, diluted (e.g., 1:10), and seeded in pairs of wells of 6-well plates.
At 60 hours post-transduction, the cell media in each well was replaced with fresh EMEM and 2 μg/ml of puromycin (Gibco, Cat.
A1113803) was added to one well out of the pair.
After 2-5 days, the cells in both wells with/without puromycin were collected and counted for viability.
The Percentage of Infection (PI%) was calculated by comparing the cell numbers with/without puromycin selection within each pair, using Poisson's distribution theory.
When the transduction efficiency (PI%) was between 30-50%, corresponding to an MOI (Multiplicity of Infection) of ~0.35-0.70, a virus volume of 120 μl yielding 30-40% of transduction efficiency was chosen for further large-scale viral transduction.

We transduced HepG2 cells with a lentiviral Brunello CRISPR knockout pooled library to achieve a coverage of at least 500 cells per sgRNA, at a multiplicity of infection (MOI) of 0.3-0.4 to ensure that 95% of infected cells received only one viral particle per cell.
We seeded 2.5 million cells in each well of 14 6-well plates, along with 8 μg/ml of polybrene.
We then added 120 μl of the virus to each experimental well and 18 h post-transduction, we removed the virus/PB mix medium and collected, counted, and pooled the cells in each well into T175 flasks.
At 60 h post-transduction, we added 2 μg/ml of puromycin to each flask, and changed the mediums every two days with fresh EMEM, topped with 2 μg/ml puromycin.
Seven days after puromycin selection, we collected, pooled, counted, and replated the cells.

Nine days after puromycin selection, cells were assigned to two groups.
For the Unsorted Control group, 20-30 million cells were collected and spun down at 500 x g for five minutes at 4°C.
The dry pellet was kept at -80°C for further genomic DNA isolation.
The remaining cells (approximately 200 million) were stained with a fluorescent dye (LipidSpotTM 488, Biotium, Cat.
70065-T).
This dye was diluted to 1:100 with DPBS and 4 ml of the staining solution was used for each dish.
The cells were incubated at 37°C for 30 minutes, after which images of the cells were captured through fluorescent microscope EVOS for GFP signal detection (Figure @fig:sup:crispr:fig1).

Cells were sorted by fluorescence-activated cell sorting (FACS) using a 100 μm nozzle.
After collection into 50 mL tubes, cells were spun at 500 x g for 5 min at 4 °C.
They were then washed with DPBS and resuspended in FACS sorting buffer (1x DPBS without Ca2+/Mg2+, 2.5 mM EDTA, 25 mM HEPES, 1% BSA, filter-sterilized and kept at 4 °C).
The cell solution was then filtered through a cell strainer (Falcon, Cat.
352235) and kept on ice, protected from light.
Approximately 20% of each GFP-High and GFP-Low population (Figure @fig:sup:crispr:fig2) were collected into 15 mL tubes.
After sorting, cells were immediately spun down and the pellets were kept at -80 °C for further genomic DNA isolation.

Genomic DNA from three conditions (Un-Sorted Control, lentiV2 GFP-High, and lentiV2 GFP-Low) was isolated using the QIAamp DNA Blood Mini Kit (Qiagen, Cat.51104).
UV Spectroscopy (Nanodrop) was used to assess the quality and quantity of the gDNA, with a total of 80-160ug of gDNA isolated for each condition.
The presence of the sgRNA cassette and lentiviral specific transgene in the isolated gDNA was verified using PCR (Figure @fig:sup:crispr:fig3).

Illumina libraries were generated and sequenced for the fragment containing the sgRNA cassette.
Primers were designed according to the protocol from the Broad Institute (Figure @fig:sup:crispr:table1) and synthesized by Integrated DNA Technologies (IDT).
Each primer contained a 0-8nt stagger sequence in the P5 and 8bp uniquely barcoded sequence in the P7.
PCR reactions (100ul each) were set up for each condition, with 5ug of gDNA, 5ul of each 10uM P5 and P7, and ExTaq DNA Polymerase (TaKaRa, Cat.
RR001A).
The PCR parameters were an initial step at 95°C for 1min, followed by 24 cycles of denaturation at 94°C for 30 seconds, annealing at 52.5°C for 30 seconds, and extension at 72°C for 30 seconds, with a final elongation at 72°C for 10 minutes.
The expected PCR product size was 285bp-293bp (Figure @fig:sup:crispr:fig4 A).
The PCR products from the same condition were then pooled and purified using SPRIselect beads (Beckman Coulter, Cat.
B23318).
The quality and quantity of the library was assessed using Qubit and Bio-analyzer with a High Sensitivity DNA Chip, respectively.
A single approximate 285bp peak was expected (Figure @fig:sup:crispr:fig4 B).
Finally, the Illumina library samples were pooled and sequenced on Nova-seq 6000, with a 20% PhiX control v3 library spike-in.


### Code and data availability

The code and data to reproduce all the analyses in this work are available in [https://github.com/greenelab/phenoplier](https://github.com/greenelab/phenoplier).
