## Methods

### MultiPLIER and Pathway-level information extractor (PLIER)

MultiPLIER [@doi:10.1016/j.cels.2019.04.003] extracts patterns of co-expressed genes from recount2 [@doi:10.1038/nbt.3838], a large gene expression dataset.
The approach applies the pathway-level information extractor method (PLIER) [@doi:10.1038/s41592-019-0456-1], which performs unsupervised learning using prior knowledge (cannonical pathways) to reduce technical noise.
Via a matrix factorization approach, PLIER deconvolutes the gene expression data into a set of latent variables (LV), where each represents a gene module (i.e. a set of genes with coordinated expression patterns).
This reduced the data dimensionality into 987 latent variables.

Given a gene expression dataset $\mathbf{Y}^{n \times p}$ with $n$ genes and $p$ conditions and a prior knowledge matrix $\mathbf{C} \in \{0,1\}^{n \times m}$ for $m$ gene sets (so that $\mathbf{C}_{ij} = 1$ if gene $i$ belongs to gene set $j$), (e.g., gene sets from MSigDB [@doi:10.1016/j.cels.2015.12.004]), PLIER finds $\mathbf{U}$, $\mathbf{Z}$, and $\mathbf{B}$ minimizing

$$
||\mathbf{Y} - \mathbf{Z}\mathbf{B}||^{2}_{F} + \lambda_1 ||\mathbf{Z} - \mathbf{C}\mathbf{U}||^{2}_{F} + \lambda_2 ||\mathbf{B}||^{2}_{F} + \lambda_3 ||\mathbf{U}||_{L^1}
$$ {#eq:met:plier_func}

subject to $\mathbf{U}>0, \mathbf{Z}>0$;
$\mathbf{Z}^{n \times l}$ are the gene loadings with $l$ latent variables,
$\mathbf{B}^{l \times p}$ is the latent space for $p$ conditions,
$\mathbf{U}^{m \times l}$ specifies which of the $m$ prior-information gene sets in $\mathbf{C}$ are represented for each LV,
and $\lambda_i$ are different regularization parameters used in the training step.
<!--  -->
$\mathbf{Z}$ is a low-dimensional representation of the gene space where each LV aligns as much as possible to prior knowledge and it might represent a known or novel gene module (i.e., a meaningful biological pattern) or noise.


### CRISPR-Cas9 screening

`Add details`{.red}


### Consensus clustering of traits in PhenomeXcan

We used a consensus clustering approach based on evidence accumulation [@doi:10.1109/TPAMI.2005.113] to find groups of traits similarly affected by the same transcriptional processes.
This approach generates an ensemble of data partitions/solutions by applying standard clustering methods on the data, and then combining these partitions into a consolidated solution.
It has been shown that these combined solutions are more robust than individual members of the ensemble and more accurately represent the underlying structure of data [@Strehl2002; @doi:10.1109/TKDE.2014.2316512; @doi:10.1109/TKDE.2019.2903410].


An important property for a successful application of a consensus clustering approach is the diversity of the ensemble, understood as the level of disagreement between the clustering solutions in the ensemble [@doi:10.1016/j.ins.2016.04.027; @doi:10.1109/TPAMI.2011.84; @doi:10.1016/j.patcog.2014.04.005].
A diverse set of solutions can be generated by using different data representations, applying clustering algorithms with distinct assumptions, or varying the set of algorithm's parameters (such as the number of clusters or the initial random seeds).
Here, we performed cluster analysis using five different clustering algorithms ($k$-means, spectral clustering, hierarchical clustering, DBSCAN and a Gaussian mixture model) on three different data representations (the original data with 987 latent variables, its projection into the top 50 principal components, and the embedding learned by UMAP [@arxiv:1802.03426] using 50 components).
The clustering methods used cover a wide range of different assumptions on cluster shapes and a varying set of parameters such as $k$ (the number of clusters) from 2 to 60, the width of the Gaussian kernel in spectral clustering, and other method-specific parameters (the full details are available in the source code repository).

The process generated an ensemble with 4,428 clustering solutions for all traits.
This ensemble was used to derive a coassociation matrix between traits by counting the number of times a pair of traits was clustered together.
A consensus function based on evidence accumulation [@doi:10.1109/TPAMI.2005.113] was applied on the coassociation matrix to derive combined solution with $k$ from 2 to 60 (59 consolidated solutions in total).
Then, for each $k$, we used the adjusted Rand index to compare each consensus partition with the entire ensemble, selecting the one that maximized the agreement with it.
We further filtered this set to keep only those partitions with an agreement larger than the 75th percentile, leaving a total of 15 final consensus partitions (with $k$ from 5 to 26) shown in Figure @fig:clustering:tree with a clustering tree [@doi:10.1093/gigascience/giy083].


`Notes:`{.red}

::: {style="color: red"}
- This section needs more details.
- Add some formulas to more exactly explain how we represent the ensemble and the function of the consensus function that is maximized.
- Maybe a small table with the five clustering algorithms and the parameters used.
- The adjusted Rand index formula? This might not be necessary.
- Incorporate some figures I generated showing the partitions passing the 75th threshold.
:::


#### Clusters interpretation

::: {style="color: red"}
- Add a reference to the github repo.
- We trained a decision tree classifier using the clusters as labels.
- For each cluster, we created a set of labels including traits belonging to the cluster as the positive class, and all the rest as the negative class.
- Then we trained a decision tree classifier on the input data (traits x latent variables) and this set of labels.
- We took the latent variable at the root node, only if the treshold used for the division was positive and larger than 1.0.
- Then, we removed this latent variable from the input data and repeated the processes 20 times, thus obtaining at most 20 latent variables related to the cluster of traits.
- This allowed us to identify different latent variables (representing potential transcriptome processes) that similarly affected a single cluster of traits.
:::

![
**Pathways associated with gene modules in Figure @fig:clustering:heatmap.**
<!--  -->
`(Early draft version; instead of this figure, we might want to just add a table with information about LVs)`{.red}
<!--  -->
This figure is equivalent to Figure @fig:clustering:heatmap but, instead of cell types, labels on the right show all the associated pathways for each latent variable.
<!--  -->
](images/clustering/global_clustermap-pathways.svg "Heatmap with gene modules and traits"){#fig:clustering:heatmap_pathways width="100%"}


### Drug-disease predictions

**NOT FINISHED**

We used the dot product of the S-PrediXcan $z$-score for each gene-disease pair, and the $z$-score for each gene-drug pair in LINCS L1000, multiplied by -1.

To obtain a drug-disease association for the gene module-mapped TWAS results, we first projected LINCS L1000 data into this latent representation using Equation (@eq:proj), thus leading to a matrix with the expression profiles of drugs mapped to latent variables.
This can be interpreted as the effects of compounds on gene modules activity.
Then, similarly as before, we anti-correlated gene module-traits scores and module expression profiles of drugs.
