<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Milton Pividori" />
  <meta name="author" content="Marylyn D. Ritchie" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="dcterms.date" content="2021-05-24" />
  <meta name="keywords" content="gene co-expression, MultiPLIER, PhenomeXcan, TWAS" />
  <title>Integrating transcriptome-wide association studies with gene co-expression patterns</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Integrating transcriptome-wide association studies with gene co-expression patterns" />
  <meta name="citation_title" content="Integrating transcriptome-wide association studies with gene co-expression patterns" />
  <meta property="og:title" content="Integrating transcriptome-wide association studies with gene co-expression patterns" />
  <meta property="twitter:title" content="Integrating transcriptome-wide association studies with gene co-expression patterns" />
  <meta name="dc.date" content="2021-05-24" />
  <meta name="citation_publication_date" content="2021-05-24" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Milton Pividori" />
  <meta name="citation_author_institution" content="Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0002-3035-4403" />
  <meta name="twitter:creator" content="@miltondp" />
  <meta name="citation_author" content="Marylyn D. Ritchie" />
  <meta name="citation_author_institution" content="Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0002-1208-1720" />
  <meta name="twitter:creator" content="@MarylynRitchie" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, CO, 80045, USA" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, 80045, USA" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@GreeneScientist" />
  <link rel="canonical" href="https://greenelab.github.io/phenoplier_manuscript/" />
  <meta property="og:url" content="https://greenelab.github.io/phenoplier_manuscript/" />
  <meta property="twitter:url" content="https://greenelab.github.io/phenoplier_manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/phenoplier_manuscript/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/phenoplier_manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/phenoplier_manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/phenoplier_manuscript/v/ce74a8ca02f42e2f086f6e2885127d8b842be658/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/phenoplier_manuscript/v/ce74a8ca02f42e2f086f6e2885127d8b842be658/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/phenoplier_manuscript/v/ce74a8ca02f42e2f086f6e2885127d8b842be658/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  
  <!-- pandoc-eqnos: equation style -->
  <style>
    .eqnos { display: inline-block; position: relative; width: 100%; }
    .eqnos br { display: none; }
    .eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
</head>
<body>
<header id="title-block-header">
<h1 class="title">Integrating transcriptome-wide association studies with gene co-expression patterns</h1>
</header>
<p><span class="banner lightblue"><i class="fas fa-info-circle fa-lg"></i> <strong>Draft manuscript</strong><br>
Text in <span style="color: red">red</span>/<span class="red">red</span> are internal comments</span></p>
<p><small><em>
This manuscript
(<a href="https://greenelab.github.io/phenoplier_manuscript/v/ce74a8ca02f42e2f086f6e2885127d8b842be658/">permalink</a>)
was automatically generated
from <a href="https://github.com/greenelab/phenoplier_manuscript/tree/ce74a8ca02f42e2f086f6e2885127d8b842be658">greenelab/phenoplier_manuscript@ce74a8c</a>
on May 24, 2021.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Milton Pividori</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3035-4403">0000-0002-3035-4403</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/miltondp">miltondp</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/miltondp">miltondp</a><br>
<small>
Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
· Funded by The Gordon and Betty Moore Foundation GBMF 4552; The National Human Genome Research Institute (R01 HG010067)
</small></p></li>
<li><p><strong>Marylyn D. Ritchie</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1208-1720">0000-0002-1208-1720</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/MarylynRitchie">MarylynRitchie</a><br>
<small>
Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Casey S. Greene</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/GreeneScientist">GreeneScientist</a><br>
<small>
Center for Health AI, University of Colorado School of Medicine, Aurora, CO, 80045, USA; Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, 80045, USA
· Funded by The Gordon and Betty Moore Foundation (GBMF 4552); The National Human Genome Research Institute (R01 HG010067); The National Cancer Institute (R01 CA237170)
</small></p></li>
</ul>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<!-- 

 -->
<h2 id="introduction">Introduction</h2>
<p>Tissue specificity is a key feature of human disease, and genes with tissue-specific expression are enriched for disease associations <span class="citation" data-cites="O0e3EhY6 18jYvPauB wNE0EQlN">[<a href="#ref-O0e3EhY6" role="doc-biblioref">1</a>,<a href="#ref-18jYvPauB" role="doc-biblioref">2</a>,<a href="#ref-wNE0EQlN" role="doc-biblioref">3</a>]</span>.
<!--  -->
Identifying the function of genes involves understanding the regulatory mechanisms that affect their expression across different tissues and cell types <span class="citation" data-cites="9Pr9idng 1DoyZS7y0 nhaocxmR">[<a href="#ref-9Pr9idng" role="doc-biblioref">4</a>,<a href="#ref-1DoyZS7y0" role="doc-biblioref">5</a>,<a href="#ref-nhaocxmR" role="doc-biblioref">6</a>]</span>.
<!--  -->
Large compendia describing key elements of regulatory DNA have been recently released or updated, which comprise chromatin-state annotations, high-resolution enhancers <span class="citation" data-cites="xRGqPsT2">[<a href="#ref-xRGqPsT2" role="doc-biblioref">7</a>]</span>, DNase I hypersensitive sites maps <span class="citation" data-cites="1DoyZS7y0">[<a href="#ref-1DoyZS7y0" role="doc-biblioref">5</a>]</span>, and the characterization of genetic effects on gene expression across different tissues <span class="citation" data-cites="9Pr9idng">[<a href="#ref-9Pr9idng" role="doc-biblioref">4</a>]</span>.
<!--  -->
The integration with genome-wide association studies (GWAS) on thousands of common diseases could dramatically improve the identification of these transcriptional mechanisms that, when dysregulated, often result in tissue- and cell lineage-specific pathology.</p>
<p>Owing to the readily available gene expression data across several tissues <span class="citation" data-cites="6SPTvFXq Nz3IMEzd 9Pr9idng 57TjOMEA">[<a href="#ref-9Pr9idng" role="doc-biblioref">4</a>,<a href="#ref-6SPTvFXq" role="doc-biblioref">8</a>,<a href="#ref-Nz3IMEzd" role="doc-biblioref">9</a>,<a href="#ref-57TjOMEA" role="doc-biblioref">10</a>]</span>, a popular approach to identify these biological processes is transcription-wide association studies (TWAS), which integrates expression quantitative trait loci (eQTL) data to provide a mechanistic interpretation for genome-wide association studies (GWAS).
<!--  -->
This is done by testing whether perturbations in gene regulatory mechanisms mediate the association between genetic variants and human diseases <span class="citation" data-cites="yEdRP9Xx 1D63fEEPb 1FFzCXo1s Z8bvDdVq">[<a href="#ref-yEdRP9Xx" role="doc-biblioref">11</a>,<a href="#ref-1D63fEEPb" role="doc-biblioref">12</a>,<a href="#ref-1FFzCXo1s" role="doc-biblioref">13</a>,<a href="#ref-Z8bvDdVq" role="doc-biblioref">14</a>]</span>.
<!--  -->
However, TWAS has not been useful to detect tissue-specific effects <span class="citation" data-cites="4iMVo07x WFslDIWl">[<a href="#ref-4iMVo07x" role="doc-biblioref">15</a>,<a href="#ref-WFslDIWl" role="doc-biblioref">16</a>]</span>, since eQTLs are generally shared across tissues.
<!--  -->
Alternative statistical approaches that connect GWAS with gene expression data can infer disease-relevant tissues and cell types <span class="citation" data-cites="WFslDIWl nga58tSh kXUuzq6I kUDav9OX z8MQTAnJ 12w2U7X9m">[<a href="#ref-WFslDIWl" role="doc-biblioref">16</a>,<a href="#ref-nga58tSh" role="doc-biblioref">17</a>,<a href="#ref-kXUuzq6I" role="doc-biblioref">18</a>,<a href="#ref-kUDav9OX" role="doc-biblioref">19</a>,<a href="#ref-z8MQTAnJ" role="doc-biblioref">20</a>,<a href="#ref-12w2U7X9m" role="doc-biblioref">21</a>]</span>, but they generally apply enrichment analysis techniques that do not account for widespread gene correlations due to technical noise (i.e. “batch effects”) <span class="citation" data-cites="mPnIAH38 Ki2ij7zE">[<a href="#ref-mPnIAH38" role="doc-biblioref">22</a>,<a href="#ref-Ki2ij7zE" role="doc-biblioref">23</a>]</span>.
<!--  -->
In addition, they generally rely on small sets of expression data compared with the total RNA-seq samples available today <span class="citation" data-cites="Nz3IMEzd 6SPTvFXq">[<a href="#ref-6SPTvFXq" role="doc-biblioref">8</a>,<a href="#ref-Nz3IMEzd" role="doc-biblioref">9</a>]</span>.</p>
<p>Here we propose a polygenic method that maps gene associations from TWAS on &gt;4,000 traits <span class="citation" data-cites="lY5ln3dB">[<a href="#ref-lY5ln3dB" role="doc-biblioref">24</a>]</span> into a latent representation learned from public gene expression repositories on tens of thousands of RNA-seq human samples <span class="citation" data-cites="14rnBunuZ 6SPTvFXq">[<a href="#ref-6SPTvFXq" role="doc-biblioref">8</a>,<a href="#ref-14rnBunuZ" role="doc-biblioref">25</a>]</span>.
<!--  -->
This low-dimensional space comprises latent variables representing gene modules with coordinated expression across different tissues and cell types.
We used a computational approach that can reduce technical noise by learning patterns that align with prior knowledge.
<!--  -->
When mapping gene-trait associations to this reduced expression space, we found that traits and diseases are associated with gene modules expressed in relevant cell types.
Several of these associations found in PhenomeXcan also replicated in 309 curated disease phenotypes from the Electronic Medical Records and Genomics (eMERGE) network phase III <span class="citation" data-cites="wfqjCerX">[<a href="#ref-wfqjCerX" role="doc-biblioref">26</a>]</span>.
<!--  -->
Our approach is robust in finding meaningful module-trait associations even when individual genes involved in lipids metabolism do not reach genome-wide significance in lipids-related traits.
<!--  -->
We also show that our module-based approach is more accurate in predicting known drug-disease medical indications than using single gene-traits associations, potentially representing a more potent drug repositioning framework.
<!--  -->
Finally, we performed cluster analysis on traits mapped to this latent representation, with clusters highly stable across different resolutions.
We were able to identify highly disease-specific gene modules expressed in relevant tissues for a wide range of complex diseases, including autoimmune, cardiovascular, and mental disorders.</p>
<h2 id="results">Results</h2>
<!--

Some papers that might be interesting:

https://www.nature.com/articles/s41591-020-01221-5
Air pollution linked to neurodegeneration markers

-->
<h3 id="framework-for-the-integration-of-twas-with-gene-co-expression-patterns">Framework for the integration of TWAS with gene co-expression patterns</h3>
<div id="fig:entire_process" class="fignos">
<figure>
<img src="images/entire_process/entire_process.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 1:</span> <strong>Schematic of the PhenoPLIER framework.</strong>
<!--  -->
<strong>a)</strong> The integration process between gene co-expression patterns from MultiPLIER (top) and TWAS results from PhenomeXcan.
PhenoPLIER projects gene-based association results on thousands of traits to a latent space learned from large gene expression datasets.
The process generates matrix <span class="math inline">\(\mathbf{\hat{M}}\)</span>, where each trait is now described by latent variables (LV) or gene modules.
<!--  -->
<strong>b)</strong> After the integration process, we found that neutrophil counts and other white blood cells (bottom) were ranked among the top 10 traits for an LV that was termed a neutrophil signature in the original MultiPLIER study.
Genes in this LV are expressed in relevant cell types (top).
<!--  -->
PBMC: peripheral blood mononuclear cells;
mDCs: myeloid dendritic cells.
<!--  --></figcaption>
</figure>
</div>
<p>In Figure <a href="#fig:entire_process">1</a>, we show the main components of PhenoPLIER, our framework to integrate TWAS and gene co-expression patterns.
The framework combines TWAS results with gene co-expression patterns by projecting gene-trait associations on a latent gene expression representation.
We used PhenomeXcan <span class="citation" data-cites="lY5ln3dB">[<a href="#ref-lY5ln3dB" role="doc-biblioref">24</a>]</span>, a massive TWAS resource on the UK Biobank <span class="citation" data-cites="nmJxPpE5">[<a href="#ref-nmJxPpE5" role="doc-biblioref">27</a>]</span> and other cohorts that provides results for 4,091 different diseases and traits.
The latent gene expression representation was obtained by applying MultiPLIER <span class="citation" data-cites="14rnBunuZ">[<a href="#ref-14rnBunuZ" role="doc-biblioref">25</a>]</span>, an unsupervised learning approach, on recount2 <span class="citation" data-cites="6SPTvFXq">[<a href="#ref-6SPTvFXq" role="doc-biblioref">8</a>]</span>.
Each of the 987 latent variables (LV) represents a gene module, essentially a group of genes with coordinated expression patterns (i.e., expressed together in the same tissues and cell types).
Since modules might represent a functional set of genes regulated by the same transcriptional program <span class="citation" data-cites="1FVd2WW6G DN7TyZzb">[<a href="#ref-1FVd2WW6G" role="doc-biblioref">28</a>,<a href="#ref-DN7TyZzb" role="doc-biblioref">29</a>]</span>, the projection of TWAS results into this latent space might provide context for their interpretation (see Methods).
<!--  -->
Our approach translates gene-trait associations to a gene module-trait score, allowing us to explore how different traits and diseases related to modules expressed in specific cell types and tissues, even under different developmental stages or stimuli.
<!--  -->
This is possible because the MultiPLIER models also provide the experimental conditions (represented by matrix <span class="math inline">\(\mathbf{B}\)</span> in Figure <a href="#fig:entire_process">1</a> a) in which genes in a module are concurringly expressed.</p>
<p>In the original MultiPLIER study, the authors found an LV significantly associated with a known neutrophil pathway and highly correlated with neutrophil estimates from gene expression.
<!--  -->
We analyzed this LV using our approach (Figure <a href="#fig:entire_process">1</a> b), and found that
1) neutrophil counts and other white blood cell traits from PhenomeXcan were ranked among the top 10 traits for this LV,
and 2) that the genes in this LV were expressed in neutrophil and other relevant cells.
<!--  -->
Moreover, using a generalized least squares approach similar to the gene-property analysis in MAGMA <span class="citation" data-cites="19XiXgYmd">[<a href="#ref-19XiXgYmd" role="doc-biblioref">30</a>]</span>, we found that gene weights in this LV were predictive of gene associations for neutrophil count and percentage (FDR &lt; 0.01) (see Methods).
<!--  -->
These initial results strongly suggested that shared patterns exist in the gene expression space (which has no GTEx samples) and the TWAS space (with gene models trained using GTEx v8);
the approach also allows inferring the context-specific effects of gene modules on complex traits.
<!--  -->
We also show how the approach can aid translational efforts by mapping pharmacological perturbations to this latent space, enabling us to observe which compounds affect the transcriptional activity of gene modules.</p>
<h3 id="genes-causally-involved-in-lipids-accumulation-are-associated-with-relevant-traits-and-tissues">Genes causally involved in lipids accumulation are associated with relevant traits and tissues</h3>
<p>We found 271 genes associated with lipids accumulation by using a genome-wide lentiviral pooled CRISPR-Cas9 library targeting 19,114 genes in the human genome (see Methods).
From these, we identified two gene-sets that either caused a decrease (96 genes in total, with eight high-confidence genes: <em>BLCAP</em>, <em>FBXW7</em>, <em>INSIG2</em>, <em>PCYT2</em>, <em>PTEN</em>, <em>SOX9</em>, <em>TCF7L2</em>, <em>UBE2J2</em>) or an increase of lipids (175 genes in total, with six high-confidence genes: <em>ACACA</em>, <em>DGAT2</em>, <em>HILPDA</em>, <em>MBTPS1</em>, <em>SCAP</em>, <em>SRPR</em>) (Supplementary File 1).
Four gene modules were significantly enriched for these two gene-sets (FDR&lt;0.05) (Supplementary Table <a href="#tbl:sup:lipids_crispr:modules_enriched">1</a>).
<!--  -->
Next, we used these two lipids gene-sets to assess whether single gene-trait associations in PhenomeXcan recapitulated lipids-related traits.
We show that our gene module-based approach can 1) robustly identify genetic associations even when known single-gene associations are not significant and 2) contextualize these results by identifying tissue and cell type-specific associations.</p>
<p>First, we assessed the genes’ effects on all phenotypes by adding their <span class="math inline">\(p\)</span>-values (as <span class="math inline">\(z\)</span>-scores) and obtaining a ranked list of traits.
The top associated traits for genes in the decreasing-lipids gene-set were highly relevant to lipid levels, such as hypertension, diastolic and systolic blood pressure, and vascular diseases, including also asthma and lung function (Supplementary Table <a href="#tbl:sup:single_genes:lipids_decreasing:top_traits">2</a>).
We performed the same operation for our gene module-based approach by considering 24 modules nominally enriched (unadjusted <span class="math inline">\(p\)</span>-value &lt; 0.05) with the decreasing-lipids gene-set (Gene-set enrichment analysis).
In this case, we also found highly lipids-relevant traits among the top 25, including hypertension, blood pressure, cardiometabolic diseases like atherosclerosis, and celiac disease (Supplementary Table <a href="#tbl:sup:modules:lipids_decreasing:top_traits">3</a>).
This is particularly relevant because each of the 24 modules aggregated a specific weighted combination of almost 3,000 genes’ <span class="math inline">\(z\)</span>-scores across all traits.
Thus, aggregating the effects of this number of genes and obtaining top-ranked lipids-relevant traits is unlikely to happen by chance (<span class="math inline">\(p\)</span>-value &lt; 0.01), suggesting that gene co-expression patterns are concordant with gene-trait associations.</p>
<div id="fig:lv246" class="fignos">
<figure>
<img src="images/lvs_analysis/lv246/lv246.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 2:</span> <strong>Tissues and traits associated with a gene module related to lipids metabolism (LV246).</strong>
<!--  -->
<strong>a)</strong> Top cell types/tissues where genes in LV246 are expressed on.
Values in the <span class="math inline">\(y\)</span>-axis come from matrix <span class="math inline">\(\mathbf{B}\)</span> in the MultiPLIER models (Figure <a href="#fig:entire_process">1</a> a).
In the <span class="math inline">\(x\)</span>-axis, cell types/tissues are sorted by the maximum value.
<!--  -->
<strong>b)</strong> Gene-trait associations (S-MultiXcan; thresholded at -log(<span class="math inline">\(p\)</span>)=10) and colocalization probability (fastENLOC) for the top traits in LV246.
The top 40 genes in LV246 are shown, sorted by their module weight, from largest (top gene <em>SCD</em>) to smallest (gene <em>FAR2</em>);
<em>DGAT2</em> and <em>ACACA</em>, in bold, are two of the six high-confidence genes in the increasing-lipids gene set from our CRISPR analyses.
<!--  -->
SGBS: Simpson Golabi Behmel Syndrome;
CH2DB: CH<sub>2</sub> groups to double bonds ratio;
NMR: nuclear magnetic resonance;
HDL: high-density lipoprotein;
RCP: locus regional colocalization probability.
<!--  --></figcaption>
</figure>
</div>
<p>When we considered the increasing-lipids set, genes and modules were associated with a more diverse set of traits, such as blood count tests, impedance measures, and bone-densitometry (Supplementary Tables <a href="#tbl:sup:single_genes:lipids_increasing:top_traits">4</a> and <a href="#tbl:sup:modules:lipids_increasing:top_traits">5</a>).
Additionally, gene modules were also associated with lung function, arterial stiffness, intraocular pressure, handgrip strength, rheumatoid arthritis, and celiac disease.
One gene module (LV246) was also associated with lipids metabolism and triglyceride biosynthesis pathways in MultiPLIER (Supplementary Table <a href="#tbl:sup:multiplier_pathways:lv246">6</a>).
Using our approach, we found that its genes were mainly co-expressed in adipose tissue (Figure <a href="#fig:lv246">2</a> a), which plays a key role in coordinating and regulating lipids metabolism.
Gene-trait associations and colocalization for the top genes in this LV are shown in Figure <a href="#fig:lv246">2</a> b).
Additionally, gene weights for this LV significantly predicted gene associations for blood lipids and hypercholesterolemia (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv246">7</a>).
Two high-confidence genes from our CRISPR screening, <em>DGAT2</em> and <em>ACACA</em>, are responsible for encoding enzymes for triglycerides and fatty acid synthesis, and were among the top genes of LV246.
However, as it can be seen in Figure <a href="#fig:lv246">2</a> b, these two genes are not strongly associated with any of the top traits for this LV;
other members of this module, such as <em>SCD</em>, <em>LPL</em>, <em>FADS2</em>, <em>HMGCR</em>, and <em>LDLR</em> were instead significantly associated and colocalized with lipid-relevant traits.
This suggests that a module-based perspective can contextualize and reprioritize TWAS hits using modules of functionally related genes.</p>
<h3 id="our-gene-module-based-approach-more-accurately-predicts-known-disease-therapeutics">Our gene module-based approach more accurately predicts known disease therapeutics</h3>
<p>We systematically evaluated our gene module-based approach from a translational perspective by assessing whether it could more accurately predict known treatments for disease.
For this, we used the transcriptional responses to small molecule perturbations profiled in LINCS L1000 <span class="citation" data-cites="F7lIlh2N">[<a href="#ref-F7lIlh2N" role="doc-biblioref">31</a>]</span>, which were further processed and mapped to DrubBank IDs <span class="citation" data-cites="6PR8LEXK O21tn8vf 1DJZvtwP1">[<a href="#ref-6PR8LEXK" role="doc-biblioref">32</a>,<a href="#ref-O21tn8vf" role="doc-biblioref">33</a>,<a href="#ref-1DJZvtwP1" role="doc-biblioref">34</a>]</span>.
Based on the established drug repurposing strategy that looks for reversed transcriptome patterns between genes and drug-induced perturbations <span class="citation" data-cites="1ClBKizD7 mZjkE1xU">[<a href="#ref-1ClBKizD7" role="doc-biblioref">35</a>,<a href="#ref-mZjkE1xU" role="doc-biblioref">36</a>]</span>, we used a framework for prioritizing drug candidates that uses imputed transcriptomes from GWAS <span class="citation" data-cites="17oeJ0CXy">[<a href="#ref-17oeJ0CXy" role="doc-biblioref">37</a>]</span>.
For this, we computed a drug-disease score by anti-correlating the <span class="math inline">\(z\)</span>-scores for a disease (from TWAS) and the <span class="math inline">\(z\)</span>-scores for a drug (from LINCS) across sets of genes of different size (see Methods).
Therefore, a large score for a drug-disease pair indicates that a higher (lower) predicted expression of disease-associated genes are down (up)-regulated by the drug, thus predicting a potential treatment.
Similarly for the gene module approach, we estimated how pharmacological perturbations affected the gene module activity by projecting expression profiles of drugs into our latent representation (Methods).
We used a manually-curated gold standard of drug-disease medical indications <span class="citation" data-cites="O21tn8vf 10KA5jTBQ">[<a href="#ref-O21tn8vf" role="doc-biblioref">33</a>,<a href="#ref-10KA5jTBQ" role="doc-biblioref">38</a>]</span> across 53 diseases and 322 compounds to evaluate and compare the prediction performance.</p>
<div id="fig:drug_disease:roc_pr" class="fignos">
<figure>
<img src="images/drug_disease_prediction/roc_pr_curves.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 3:</span> <strong>Drug-disease prediction performance for gene-based and module-based approaches.</strong>
<!--  -->
The receiver operating characteristic (ROC) (left) and the precision-recall curves (right) for a gene-based and our module-based approach.
<!--  -->
AUC: area under the curve; AP: average precision.
<!--  --></figcaption>
</figure>
</div>
<p>The ROC and precision-recall (PR) curves comparing both approaches are shown in Figure <a href="#fig:drug_disease:roc_pr">3</a>.
Our proposed gene module-based method outperformed the gene-based one with an area under the curve of 0.632 and an average precision of 0.858.
<!--  -->
It is important to note that the gene-trait associations and drug-induced expression profiles projected into the latent space represent a compressed version of the entire set of results.
The prediction results show that this low-dimensional space captures biologically meaningful patterns that can better represent and link pathophysiological processes with the mechanisms of action of drugs.
In the following, with the aim to understand these results, we examined specific a drug-disease pair where both methods disagreed.</p>
<p>Nicotinic acid (niacin) is a B vitamin widely used clinically to treat lipid disorders.
Niacin exerts its effects on a number tissues, although not all its mechanisms have been documented <span class="citation" data-cites="LVihFr3g 7OgaYjeL">[<a href="#ref-LVihFr3g" role="doc-biblioref">39</a>,<a href="#ref-7OgaYjeL" role="doc-biblioref">40</a>]</span>.
This compound can increase high-density lipoprotein (HDL) by inhibiting an HDL catabolism receptor in liver.
Niacin also inhibits diacylglycerol acyltransferase–2 (DGAT2), which decrease production of low-density lipoproteins (LDL) by modulating triglyceride synthesis in hepatocytes, or by inhibiting adipocyte triglyceride lipolysis <span class="citation" data-cites="LVihFr3g">[<a href="#ref-LVihFr3g" role="doc-biblioref">39</a>]</span>.
Niacin was categorized in our gold standard as a disease-modifying indication for atherosclerosis (AT) and coronary artery disease (CAD), and not for pancreatitis.
For pancreatitis, both the gene-based and module-based methods assigned a negative score (below their averages), which agrees with the gold standard in that niacin does not therapeutically change the biology of this disease.
For AT, the module-based approach predicted niacin as a therapeutic drug by scoring them with 0.52 (above the mean), whereas the gene-based method assigned negative scores of -0.01 (below the mean), respectively.
To understand why the predictions by the module-based method were different, we obtained the LVs that positively contributed to the score by looking at large positive (negative) LV values for the disease and large negative (positive) LV values for the drug of interest (Supplementary Table ).
Notably, LV246 (analyzed previously) was among the top 20 modules contributing to the prediction of niacin as a therapeutic drug for AT.
As shown in Figure <a href="#fig:lv246">2</a>, this module is mainly expressed in adipose cells and liver cells, and its top genes encode important enzymes involved in lipid biosynthesis.
The module as a whole (considering the gene weights) is significantly associated with cardiovascular traits (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv246">7</a>), and several of its top genes are significantly associated and colocalized with cardiovascular-related traits:
<em>SCD</em> (10q24.31) is associated with hypercholesterolemia (P=1.9e-5) and its GWAS and eQTL signals are fully colocalized (RCP=1.0);
<em>LPL</em> (8p21.3), which is known to be linked to different disorders of lipoprotein metabolism, is strongly associated with hypercholesterolemia (P=7.5e-17, RCP=0.26), and family history of heart disease (P=1.7e-5, RCP=0.22);
other genes associated with hypercholesterolemia in this module are
<em>FADS2</em> (11q12.2) (P=9.42e-5, RCP=0.623),
<em>HMGCR</em> (5q13.3) (P=1.3e-42, RCP=0.23),
and <em>LDLR</em> (19p13.2) (P=9.9e-136, RCP=0.41).</p>
<p>The analysis of other niacin-AT-contributing modules revealed additional known mechanisms of action of niacin.
<!--  -->
For example, GPR109A/HCAR2 is a G protein-coupled high-affinity niacin receptor in adipocytes and immune cells, including monocytes, macrophages, neutrophils and dendritic cells <span class="citation" data-cites="izbPw2kc wI0IjT3i">[<a href="#ref-izbPw2kc" role="doc-biblioref">41</a>,<a href="#ref-wI0IjT3i" role="doc-biblioref">42</a>]</span>.
It was initially thought that the antiatherogenic effects of niacin were solely due to inhibition of lipolysis in adipose tissue.
However, it has been shown that nicotinic acid can reduce atherosclerosis progression independently of its antidyslipidemic activity through the activation of GPR109A in immune cells <span class="citation" data-cites="1Bz0jRHYo">[<a href="#ref-1Bz0jRHYo" role="doc-biblioref">43</a>]</span>, thus boosting anti-inflammatory processes and reversing cholesterol transport <span class="citation" data-cites="1DblG8swn">[<a href="#ref-1DblG8swn" role="doc-biblioref">44</a>]</span>.
This alternative mechanism for niacin could have been hypothesized by examining the cell types where top modules positively contributing to the niacin-AT prediction are expressed: LV116 and LV931 (Supplementary Figures <a href="#fig:sup:lv116">9</a> and <a href="#fig:sup:lv931">10</a>).
Among these, we also found LV678 positively contributing to this prediction, which was significantly enriched with the lipids-decreasing genes from our CRISPR screening (Supplementary Table <a href="#tbl:sup:lipids_crispr:modules_enriched">1</a>).
This module was expressed in the heart and muscle cells (Supplementary Figure <a href="#fig:sup:lv678">8</a>).
<!-- Among these, we also found other potentially interesting modules that could represent mechanisms to explore, such as LV536 expressed in the bladder (Supplementary Figure @fig:sup:lv536) and LV885/LV840 expressed in kidneys (Supplementary Figures @fig:sup:lv885 and @fig:sup:lv840). --></p>
<p>The projection of these two types of data into a common latent gene module-based representation could provide a more powerful framework for drug repositioning using data from genetic studies.
Additionally, our approach could also be helpful to understand better the mechanism of pharmacological effect of known or experimental drugs.
For example, one of the top modules affected by niacin (LV66, Supplementary Figure <a href="#fig:sup:lv66">11</a>) is mainly expressed in ovarian granulosa cells.
This compound has been very recently considered as a potential therapeutic for ovarian diseases <span class="citation" data-cites="kLRErKXz TovvsrDr">[<a href="#ref-kLRErKXz" role="doc-biblioref">45</a>,<a href="#ref-TovvsrDr" role="doc-biblioref">46</a>]</span>, as it was found to promote follicle growth and inhibit granulosa cell apoptosis in animal models.
Our proposed approach could be helpful to generate novel hypotheses to evaluate potential mechanisms of action of different drugs.</p>
<h3 id="clusters-of-traits-in-the-gene-module-space-are-associated-with-relevant-transcriptional-processes">Clusters of traits in the gene module space are associated with relevant transcriptional processes</h3>
<div id="fig:clustering:design" class="fignos">
<figure>
<img src="images/clustering/clustering_design.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 4:</span> <strong>Cluster analysis on traits using the latent gene expression representation.</strong>
<!--  -->
<strong>a)</strong> The projection of TWAS results on <span class="math inline">\(n\)</span>=3,752 traits into the latent gene expression representation is the input data to the clustering process.
A linear (PCA) and non-linear (UMAP) dimensionality reduction techniques are applied to the input data, and the three data versions are processed by five different clustering algorithms.
These algorithms derive partitions from the data using different sets of parameters (such as the number of clusters), leading to an ensemble of 4,428 partitions.
Then, a distance matrix is derived by counting how many times a pair of traits were grouped in different clusters across the ensemble.
Finally, a consensus function is applied to the distance matrix to generate consolidated partitions with different number of clusters (from 2 to <span class="math inline">\(\sqrt{n}\approx\)</span> 60).
These final solutions are represented in the clustering tree (Figure <a href="#fig:clustering:tree">5</a>).
<!--  -->
<strong>b)</strong> The clusters found by the consensus function are used as labels to train a decision tree classifier on the original input data, which detects the LVs that better differentiate groups of traits.
<!--  --></figcaption>
</figure>
</div>
<p>The previous results suggest that <span class="math inline">\(\hat{\mathbf{M}}\)</span> represents a less noisy low-dimentional version of the data.
Thus, we conducted cluster analysis on <span class="math inline">\(\hat{\mathbf{M}}\)</span> to find groups of traits that are similarly affected by the same transcriptional processes.
To avoid using a single clustering algorithm (which implies using a single assumption about the structure of the data), we employed a consensus clustering approach where different methods with varying sets parameters were applied on the data, and later combined into a consolidated solution <span class="citation" data-cites="17CbjLzCP uw3AnEgA 8js8Q3pF">[<a href="#ref-17CbjLzCP" role="doc-biblioref">47</a>,<a href="#ref-uw3AnEgA" role="doc-biblioref">48</a>,<a href="#ref-8js8Q3pF" role="doc-biblioref">49</a>]</span> (Figure <a href="#fig:clustering:design">4</a>).
Our clustering pipeline generated 15 final consensus clustering solutions with 5 to 29 clusters (Supplementary Figure <a href="#fig:sup:consensus_agreement">28</a>).
Instead of selecting a specific number of clusters, we used a clustering tree <span class="citation" data-cites="xhtEAzx6">[<a href="#ref-xhtEAzx6" role="doc-biblioref">50</a>]</span> (Figure <a href="#fig:clustering:tree">5</a>) to examine stable groups of traits across multiple resolutions.
<!--  -->
Finally, for the interpretation of the clusters, we trained a decision tree classifier (a highly interpretable machine learning model) on the input data <span class="math inline">\(\hat{\mathbf{M}}\)</span> using the clusters found as labels.
This allowed us to quickly identify the latent variables/gene modules that better differentiated the groups of traits found (see Methods).</p>
<div id="fig:clustering:tree" class="fignos">
<figure>
<img src="images/clustering/clustering_tree.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 5:</span> <strong>Clustering tree using multiple resolutions for clusters of traits.</strong>
<!--  -->
Each row represents a partition/grouping of the traits, and each circle is a cluster from that partition, and the number of clusters go from 5 to 29.
Arrows indicate how traits in one cluster move across clusters from different partitions.
Most of the clusters are preserved across different resolutions, showing highly stable solutions even with independent runs of the clustering algorithm.
<!--  -->
MCV: mean corpuscular volume;
MCH: mean corpuscular hemoglobin;
MRV: mean reticulocyte volume;
MSCV: mean sphered cell volume;
RDW: red cell (erythrocyte) distribution width;
BMI: body mass index;
WC: waist circumference;
HC: hip circumference;
BMR: basal metabolic rate;
RA: rheumatoid arthritis;
SLE: systemic lupus erythematosus;
IBD: inflammatory bowel disease;
<!--  -->
<em>Descriptions of traits by cluster IDs (from left to right):</em>
12: also includes lymphocyte count and allergies such as allergic rhinitis or eczema;
4: includes reticulocyte count and percentage, immature reticulocyte fraction, and high light scatter reticulocytes count and percentage;
5: includes erythrocyte count, hemoglobin concentration, and hematocrit percentage;
18: also includes ankle spacing width;
1: includes platelet count, crit, mean volume, and distribution width;
13: diabetes refers to age when the diabetes was first diagnosed;
25: includes vascular problems such as angina, deep vein thrombosis (DVT), intraocular pressure, eye and mouth problems, pulse rate, hand-grip strength, several measurements of physical activity, jobs involving heavy physical work, types of transport used, intake of vitamin/mineral supplements, and various types of body pain and medications for pain relief;
21: also includes attention deficit hyperactivity disorder (ADHD), number of years of schooling completed, bone density, and intracranial volume measurement;
28: includes diabetes, gout, arthrosis, and respiratory diseases (and related medications such as ramipril, allopurinol, and lisinopril), urine assays, female-specific factors (age at menarche, menopause, first/last live birth), and several environmental/behavioral factors such as intake of a range of food/drink items including alcohol, time spent outdoors and watching TV, smoking and sleeping habits, early-life factors (breastfed as a baby, maternal smoking around birth), education attainment, psychological and mental health, and health satisfaction;
11: also includes fasting blood glucose and insulin measurement;
16: lipids include high and low-density lipoprotein cholesterol (HDL and LDL), triglycerides, and average number of methylene groups per a double bond;
14: includes myocardial infarction, coronary atherosclerosis, ischaemic heart disease (wide definition).
9: includes neutrophil count, neutrophil+basophil count, neutrophil+eosinophil count, granulocyte count, leukocyte count, and myeloid cell count.
<!--  --></figcaption>
</figure>
</div>
<p>The clustering tree in Figure <a href="#fig:clustering:tree">5</a> shows five clear branches that are shown at the top with different numerical labels (from left to right):
0) a “large” branch that includes most of the traits that start to be subdivided only at <span class="math inline">\(k\)</span>=16 (with asthma, subjective well-being traits, and nutrient intake clusters),
1) heel bone-densitometry measurements,
2) hematological assays on red blood cells,
3) physical measures, including spirometry and body impedance, and anthropometric traits with fat-free and fat mass measures in separate sub-branches, and
4) a “complex” branch including keratometry measurements, assays on white blood cells and platelets, skin and hair color traits, autoimmune disorders (type 1 diabetes, psoriasis, hyper/hypothyroidism, rheumatoid arthritis, systemic lupus erythematosus, celiac disease), and cardiovascular diseases (hypertension, coronary artery disease, myocardial infraction, hypercholesterolemia, and other cardiovascular-related traits such hand-grip strength <span class="citation" data-cites="aBVh8zt1">[<a href="#ref-aBVh8zt1" role="doc-biblioref">51</a>]</span>, and environmental/behavioral factors such as physical activity and diet) (See Supplementary Files 1-5 for clustering results).
<!--  -->
All branches show relatively highly stable clusters, where the same traits are clustered together across different resolutions even with the consensus algorithm using random seeds at each level.
<!--  -->
The arrows between different clusters show how traits move from one group to another across different resolutions.
This mainly happens between clusters within the “complex” branch, and between clusters from the “large” branch to the “complex” branch.
This is expected, since the “complex” branch contains traits related to a wide range of different factors and thus are hard to categorize into a single cluster.</p>
<div id="fig:clustering:heatmap" class="fignos">
<figure>
<img src="images/clustering/global_clustermap-plain.svg" style="width:100.0%" alt="" /><figcaption><span>Figure 6:</span> <strong>Cluster-specific and general transcriptional processes.</strong>
<!--  -->
The plot shows a submatrix of <span class="math inline">\(\hat{\mathbf{M}}\)</span> for the main trait clusters at <span class="math inline">\(k\)</span>=29, considering only gene modules (rows) that align well with at least one known pathway.
<!-- Labels on the right show LV names and their main tissues/cell types where gene modules are expressed in. -->
Values are standardized from -5 (lighter color) to 16 (darker color).
<!--  --></figcaption>
</figure>
</div>
<p>Next, we analyzed which gene modules are driving these clusters of traits.
For that, we trained decision tree classifiers on the input data (Figure <a href="#fig:clustering:design">4</a>) using each cluster at <span class="math inline">\(k\)</span>=29 (bottom of Figure <a href="#fig:clustering:tree">5</a>) as labels (see Methods).
This yielded for each cluster the top gene modules, where several of them were well-aligned to existing pathways, and others were “novel” and expressed in relevant tissues.
We summarized this in Figure <a href="#fig:clustering:heatmap">6</a>, where it can be seen that some modules are highly specific to certain types of traits, and others seem to be associated with a wide range of different traits and diseases, thus potentially involved in more general biological functions.
For example, modules such as LV928 and LV30 (Supplementary Figures <a href="#fig:sup:lv928">12</a> and <a href="#fig:sup:lv30">13</a>), which are known to be related to early progenitors of the erythrocytes lineage <span class="citation" data-cites="Zk82GvJV">[<a href="#ref-Zk82GvJV" role="doc-biblioref">52</a>]</span>, are predominantly expressed in early differentiation stages of erythropoiesis, and strongly associated with different assays on red blood cells (erythrocytes and reticulocytes).
On the other side, others are highly specific, such as LV730, expressed in thrombocytes from different cancer samples (Supplementary Figures <a href="#fig:sup:lv730">14</a>), and strongly associated with hemathological assays on platelets;
or LV598, whose genes are expressed in corneal endothelial cells (Supplementary Figures <a href="#fig:sup:lv598">15</a>) and associated to keratometry measurements (FDR &lt; 0.05; Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv598">12</a>).
<!-- 
LV154 is also expressed in corneal endothelial cells and associated with keratometry, but not exclusively.
 --></p>
<p>The autoimmune diseases sub-branch also has significant gene modules associations expressed in relevant cell types.
<!--  -->
LV155 was strongly expressed in thyroid (Supplementary Figures <a href="#fig:sup:lv155">16</a>), and significantly associated with hypothyroidism both in PhenomeXcan and eMERGE (FDR &lt; 0.05; Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv155">13</a> and <a href="#tbl:sup:emerge_assocs:lv155">14</a>).
<!--  -->
LV844 was the most strongly associated gene module with autoimmune disorders (FDR &lt; 1e-15; Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv844">15</a> and <a href="#tbl:sup:emerge_assocs:lv844">16</a>), and was expressed in a wide range of cell types, including blood, breast organoids, myeloma cells, lung fibroblasts, and different cell types from the brain (Supplementary Figures <a href="#fig:sup:lv844">17</a>).
<!--  -->
Other important gene modules associated with autoimmunity in both PhenomeXcan and eMERGE are LV57 expressed in T cells (Supplementary Figure <a href="#fig:sup:lv57">18</a>, and Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv57">17</a> and <a href="#tbl:sup:emerge_assocs:lv57">18</a>), and LV54 expressed in different soft tissue tumors, breast, lung, pterygia and epithelial cells (Supplementary Figure <a href="#fig:sup:lv54">19</a>, and Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv54">19</a> and <a href="#tbl:sup:emerge_assocs:lv54">20</a>).</p>
<p>The cardiovascular sub-branch also exhibited significant associations, such as LV847 (Supplementary Figure <a href="#fig:sup:lv847">20</a>) with blood pressure traits and hypertension (Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv847">21</a> and <a href="#tbl:sup:emerge_assocs:lv847">22</a>), which was expressed in CD19 (B cells) (which are related to preeclampsia <span class="citation" data-cites="YyXvsbU9">[<a href="#ref-YyXvsbU9" role="doc-biblioref">53</a>]</span>), Jurkcat cells (T lymphocyte cells), and cervical carcinoma cell lines (the uterus was previously reported to be linked to blood pressure through a potential hormonal pathway <span class="citation" data-cites="Om8ZhS06 meFYGVLK">[<a href="#ref-Om8ZhS06" role="doc-biblioref">54</a>,<a href="#ref-meFYGVLK" role="doc-biblioref">55</a>]</span>).
<!--  -->
LV136 was aligned with known collagen formation and muscle contraction pathways, and it was associated to coronary artery disease, myocardial infarction and keratometry measurements (Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv136">23</a> and <a href="#tbl:sup:emerge_assocs:lv136">24</a>), and expressed in a wide range of cell types, including fibroblasts, mesenchymal stem cells, osteoblasts, pancreatic stellate cells, cardiomyocytes, and adipocytes (Supplementary Figure <a href="#fig:sup:lv136">21</a>).
<!--  -->
Lipids were clustered with chronotype and Alzheimer’s disease, and were significantly associated with several modules expressed mainly in brain cell types, including LV93 (Supplementary Figure <a href="#fig:sup:lv93">22</a>, and Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv93">25</a> and <a href="#tbl:sup:emerge_assocs:lv93">26</a>), LV206 (Supplementary Figure <a href="#fig:sup:lv206">23</a>, and Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv206">27</a> and <a href="#tbl:sup:emerge_assocs:lv206">28</a>), and LV260 (Supplementary Figure <a href="#fig:sup:lv260">24</a>, and Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv260">29</a> and <a href="#tbl:sup:emerge_assocs:lv260">30</a>).
These modules were associated mainly with cardiovascular traits in eMERGE.</p>
<p>Within the cardiovascular sub-branch, we also found mental and neurodevelopmental disorders such as Alzheimer’s disease, schizophrenia, and attention deficit hyperactivity disorder (ADHD).
These disorders were previously linked to the cardiovascular system <span class="citation" data-cites="10qjLoufR j2Sl4DAE 17LYMnG9n 13t4TuFeJ">[<a href="#ref-10qjLoufR" role="doc-biblioref">56</a>,<a href="#ref-j2Sl4DAE" role="doc-biblioref">57</a>,<a href="#ref-17LYMnG9n" role="doc-biblioref">58</a>,<a href="#ref-13t4TuFeJ" role="doc-biblioref">59</a>]</span>, and share several risk factors, including hypertension, high cholesterol, obesity, smoking, among others <span class="citation" data-cites="D83Aqhga 9BGyO071">[<a href="#ref-D83Aqhga" role="doc-biblioref">60</a>,<a href="#ref-9BGyO071" role="doc-biblioref">61</a>]</span>.
<!--  -->
In our results, however, these diseases were grouped by potentially shared transcriptional processes expressed in specific tissues/cell types.
<!--  -->
Alzheimer’s disease, for example, was significantly associated with LV21 (FDR &lt; 1e-18) and with LV5 (FDR &lt; 0.01) (Supplementary Tables <a href="#tbl:sup:phenomexcan_assocs:lv21">31</a> and <a href="#tbl:sup:phenomexcan_assocs:lv5">33</a>).
LV21 was strongly expressed in a variety of soft tissue sarcomas, monocytes/macrophages (including microglia from cortex samples), and aortic valves (Supplementary Figure <a href="#fig:sup:lv21">25</a>);
as discussed previously, macrophages play a key role in the reverse cholesterol transport and thus atherogenesis <span class="citation" data-cites="idlBgtFz">[<a href="#ref-idlBgtFz" role="doc-biblioref">62</a>]</span>.
LV5 was expressed in breast cancer and brain glioma samples, microglia (cortex), liver, and kidney, among other cell types (Supplementary Figure <a href="#fig:sup:lv5">26</a>).
LV21 and LV5 were also strongly associated with lipids: LDL cholesterol (FDR &lt; 0.001) and triglycerides (FDR &lt; 0.05 and FDR &lt; 0.001, respectively).
Additionally, LV5 was associated with depression traits from the UK Biobank.
<!--  -->
ADHD was the only significantly associated trait for LV434 (FDR &lt; 0.01) (Supplementary Table <a href="#tbl:sup:phenomexcan_assocs:lv434">35</a>), which was expressed in breast cancer and glioma cells, cerebral organoids, and several different cell populations from the brain: fetal neurons (replicating and quiescence), microglia, and astrocytes (Supplementary Figure <a href="#fig:sup:lv434">27</a>).
<!-- 
https://link.springer.com/article/10.1007/s11060-019-03165-4
Clinical diagnosis of attention-deficit/hyperactivity disorder in survivors of pediatric brain tumors
    - very interesting
 -->
<!--  -->
Schizophrenia was not significantly associated with any gene module tested in our analysis.
<!--  -->
None of these LVs were aligned to prior pathways, which might represent potentially novel transcriptional processes affecting the cardiovascular and central nervous system.</p>
<h2 id="discussion">Discussion</h2>
<p>We have introduced a novel computational approach that can map existing TWAS results into a latent gene expression representation to infer cell type-specific effects for complex diseases.
Leveraging one of the largest gene expression compendia, we integrated gene co-expression patterns on tens of thousands of RNA-seq samples covering different tissues and cell types under a variety of stimuli and developmental stages.
We showed that this latent representation is robust to capture relevant associations even when known single gene-trait effects are not detected.
Moreover, projecting gene-trait and gene-drug associations into this common representation can better link pathophysiological processes with the mechanisms of action of drugs, which led to a more accurate prediction of known disease treatments.
Finally, we showed how the approach could be used to group diseases and traits affected by the same transcriptional processes, which showed disease-specific modules expressed in highly relevant tissues.</p>
<p>The key difference of our approach with other alternatives is that we can incorporate patterns learned in RNA-seq datasets with tissues and cell types measured in a large variety of conditions.
<!--  -->
As we showed, gene modules are expressed in very specific cell types, and this has the advantage of identifying potentially causal cell types with more precision.
We provide examples of modules clearly expressed in one type of cell (such as adipose in LV246, thyroid in LV155, or ovary in LV66), whereas others seem to be activated across different contexts and thus capture a more realistic multi-tissue scenario for complex diseases (for example, LV136 is associated with coronary artery disease and expressed in fibroblasts, osteoblasts, pancreas, liver, and cardiomyocytes).
To our knowledge, this is a novel approach for the identification of tissue and cell type-specific effects that is computationally simple to implement.</p>
<p>Our approach assumes that gene modules with coordinated expression controlled by a perturbed transcriptional process will also manifest coordinated pathological effects on relevant diseases.
This leads to several limitations related to
1) the unsupervised learning process that detects modules of co-expressed genes (MultiPLIER),
and 2) the statistical methods that link SNP-trait to gene-trait associations through eQTL (TWAS/PhenomeXcan).
<!--  -->
First, MultiPLIER applies a matrix factorization algorithm to extract latent variables/modules across a large expression dataset.
These modules could represent real transcriptional processes or technical factors (“batch effects”).
Although the deconvolution process successfully uses prior knowledge (pathways) to segregate noise from authentic biological patterns, several detected modules could be either novel, in the best scenario, or just noise.
On the other hand, the underlying factorization method is restricted to linear combinations, which could miss important and more complex co-expression patterns.
<!--  -->
Second, TWAS approaches have several limitations that can lead to false positives <span class="citation" data-cites="l6ogswV3 ndd3tW4g">[<a href="#ref-l6ogswV3" role="doc-biblioref">63</a>,<a href="#ref-ndd3tW4g" role="doc-biblioref">64</a>]</span>.
Like GWAS, which generally detects groups of associated variants in LD (linkage disequilibrium), TWAS usually identifies several genes within the same locus <span class="citation" data-cites="AxVJwanp Z8bvDdVq">[<a href="#ref-Z8bvDdVq" role="doc-biblioref">14</a>,<a href="#ref-AxVJwanp" role="doc-biblioref">65</a>]</span>.
This is due to sharing of GWAS variants in gene expression models, to correlated expression of nearby genes, or even correlation of their predicted expression due to eQTLs in LD, among others <span class="citation" data-cites="l6ogswV3">[<a href="#ref-l6ogswV3" role="doc-biblioref">63</a>]</span>.
<!--  -->
These two factors can combine to produce spurious module-trait associations, but our results suggest that the first aspect (module detection) could be the most problematic and probably more challenging to address.</p>
<p>Our approach is also in line with previous studies showing that drugs with genetic support are more likely to succeed through the drug development pipeline <span class="citation" data-cites="REXpV7nA 17oeJ0CXy">[<a href="#ref-17oeJ0CXy" role="doc-biblioref">37</a>,<a href="#ref-REXpV7nA" role="doc-biblioref">66</a>]</span>.
We show that a gene module perspective can more accurately predict known therapeutics and potentially serve as a more effective drug repositioning framework than considering single-gene effects.
This outcome is explained by the biologically meaningful patterns captured by the latent gene expression space.
The projection of single gene-trait associations and gene-compound profiles into this common representation translated into a stronger link between pathophysiological processes with the mechanisms of action of drugs.</p>
<p>Our consensus clustering pipeline combined different algorithms and assumptions on data structure to deliver highly stable groups of traits differentiated by relevant transcriptional processes.
Considering groups of related diseases was previously shown to be more powerful to detect shared genetic etiology <span class="citation" data-cites="LyJmyoQr veADXImD">[<a href="#ref-LyJmyoQr" role="doc-biblioref">67</a>,<a href="#ref-veADXImD" role="doc-biblioref">68</a>]</span>;
Our low-dimensional expression representation was used to detect groups of diseases previously suspected to share common pathophysiological mechanisms, such as cardiovascular, autoimmune, and mental diseases.
The approach, however, also has several limitations related to 1) the data we used for grouping traits and 2) the fit of the clustering algorithms used for highly multi-factorial traits.
First, we used the S-MultiXcan associations, which only provide the association strength between a gene and a trait, but with no direction of effect.
This makes groups of traits harder to interpret since we can only infer that transcriptional processes are associated with traits in the same cluster, but they could actually have opposite effects for some traits than others.
Second, we employed hard-partitioning algorithms (one trait belongs exclusively to one cluster) where the distance between two traits takes into account all gene modules.
However, it is likely for two complex diseases to share just a few biological processes instead of being similar across all of them.
These more complex scenarios cannot be captured by our clustering pipeline, as it can be seen for traits in the “complex” branch, which are hard to categorize into a single cluster or branch.
This limitation was circumvented to some extent by using the hierarchical structure provided by clustering trees.</p>
<p>The proposed approach will benefit from larger gene expression datasets on different tissues and cell types measured on highly heterogeneous conditions.
High-quality attributes for RNA-seq samples are critical for a successful application of our method, and we welcome current efforts to improve this situation <span class="citation" data-cites="fnDaLjFy">[<a href="#ref-fnDaLjFy" role="doc-biblioref">69</a>]</span>.
This will enable the identification of more precise tissue- and cell lineage-specific effects on complex diseases.</p>
<!--
- talk about the omnigenic model!
-->
<h2 id="methods">Methods</h2>
<h3 id="phenomexcan-gene-based-associations-on-4091-traits">PhenomeXcan: gene-based associations on 4,091 traits</h3>
<p>We used TWAS results from PhenomeXcan <span class="citation" data-cites="lY5ln3dB">[<a href="#ref-lY5ln3dB" role="doc-biblioref">24</a>]</span> on 4091 traits for 22515 genes.
PhenomeXcan was built using publicly available GWAS summary statistics to compute
1) gene-based associations with the PrediXcan family of methods <span class="citation" data-cites="Z8bvDdVq vLyTudUB 1FFzCXo1s">[<a href="#ref-1FFzCXo1s" role="doc-biblioref">13</a>,<a href="#ref-Z8bvDdVq" role="doc-biblioref">14</a>,<a href="#ref-vLyTudUB" role="doc-biblioref">70</a>]</span>, and
2) a posterior probability of colocalization between GWAS loci and <em>cis</em>-eQTL with fastENLOC <span class="citation" data-cites="lY5ln3dB wyKoQS8N">[<a href="#ref-lY5ln3dB" role="doc-biblioref">24</a>,<a href="#ref-wyKoQS8N" role="doc-biblioref">71</a>]</span>.
The PrediXcan family of methods first build prediction models using data from the Genotype-Tissue Expression project (GTEx v8) <span class="citation" data-cites="9Pr9idng">[<a href="#ref-9Pr9idng" role="doc-biblioref">4</a>]</span> for gene expression imputation and then correlate this predicted expression with the phenotype of interest.
This family is comprised of
S-PrediXcan <span class="citation" data-cites="vLyTudUB">[<a href="#ref-vLyTudUB" role="doc-biblioref">70</a>]</span> (which computes a gene-tissue-trait association using GWAS as input)
and S-MultiXcan <span class="citation" data-cites="1FFzCXo1s">[<a href="#ref-1FFzCXo1s" role="doc-biblioref">13</a>]</span> (which computes a gene-trait association by aggregating evidence of associations across all tissues).</p>
<p>We refer to the standardized effect sizes (<span class="math inline">\(z\)</span>-scores) of S-PrediXcan across <span class="math inline">\(n\)</span> traits and <span class="math inline">\(m\)</span> genes in tissue <span class="math inline">\(t\)</span> as <span class="math inline">\(\mathbf{M}^{t} \in \mathbb{R}^{n \times m}\)</span>.
For S-MultiXcan we do not have the direction of effect, and we used the <span class="math inline">\(p\)</span>-values converted to <span class="math inline">\(z\)</span>-scores <span class="math inline">\(\mathbf{M}=\Phi^{-1}(1 - p/2)\)</span>, where <span class="math inline">\(\Phi^{-1}\)</span> is the probit function.
Higher <span class="math inline">\(z\)</span>-scores correspond to stronger associations.</p>
<h3 id="multiplier-and-pathway-level-information-extractor-plier">MultiPLIER and Pathway-level information extractor (PLIER)</h3>
<p>MultiPLIER <span class="citation" data-cites="14rnBunuZ">[<a href="#ref-14rnBunuZ" role="doc-biblioref">25</a>]</span> extracts patterns of co-expressed genes from recount2 <span class="citation" data-cites="6SPTvFXq">[<a href="#ref-6SPTvFXq" role="doc-biblioref">8</a>]</span>, a large gene expression dataset.
The approach applies the pathway-level information extractor method (PLIER) <span class="citation" data-cites="Ki2ij7zE">[<a href="#ref-Ki2ij7zE" role="doc-biblioref">23</a>]</span>, which performs unsupervised learning using prior knowledge (canonical pathways) to reduce technical noise.
Via a matrix factorization approach, PLIER deconvolutes the gene expression data into a set of latent variables (LV), where each represents a gene module.
This reduced the data dimensionality into 987 latent variables or gene modules.</p>
<p>Given a gene expression dataset <span class="math inline">\(\mathbf{Y}^{m \times c}\)</span> with <span class="math inline">\(m\)</span> genes and <span class="math inline">\(c\)</span> experimental conditions and a prior knowledge matrix <span class="math inline">\(\mathbf{C} \in \{0,1\}^{m \times p}\)</span> for <span class="math inline">\(p\)</span> MSigDB pathways <span class="citation" data-cites="1CbVoEpNJ">[<a href="#ref-1CbVoEpNJ" role="doc-biblioref">72</a>]</span> (so that <span class="math inline">\(\mathbf{C}_{ij} = 1\)</span> if gene <span class="math inline">\(i\)</span> belongs to pathway <span class="math inline">\(j\)</span>), PLIER finds <span class="math inline">\(\mathbf{U}\)</span>, <span class="math inline">\(\mathbf{Z}\)</span>, and <span class="math inline">\(\mathbf{B}\)</span> minimizing</p>
<p><span id="eq:met:plier_func" class="eqnos"><span class="math display">\[
||\mathbf{Y} - \mathbf{Z}\mathbf{B}||^{2}_{F} + \lambda_1 ||\mathbf{Z} - \mathbf{C}\mathbf{U}||^{2}_{F} + \lambda_2 ||\mathbf{B}||^{2}_{F} + \lambda_3 ||\mathbf{U}||_{L^1}
\]</span><span class="eqnos-number">(1)</span></span> </p>
<p>subject to <span class="math inline">\(\mathbf{U}&gt;0, \mathbf{Z}&gt;0\)</span>;
<span class="math inline">\(\mathbf{Z}^{m \times l}\)</span> are the gene loadings with <span class="math inline">\(l\)</span> latent variables,
<span class="math inline">\(\mathbf{B}^{l \times c}\)</span> is the latent space for <span class="math inline">\(c\)</span> conditions,
<span class="math inline">\(\mathbf{U}^{p \times l}\)</span> specifies which of the <span class="math inline">\(p\)</span> prior-information pathways in <span class="math inline">\(\mathbf{C}\)</span> are represented for each LV,
and <span class="math inline">\(\lambda_i\)</span> are different regularization parameters used in the training step.
<!--  -->
<span class="math inline">\(\mathbf{Z}\)</span> is a low-dimensional representation of the gene space where each LV aligns as much as possible to prior knowledge, and it might represent either a known or novel gene module (i.e., a meaningful biological pattern) or noise.</p>
<p>We projected <span class="math inline">\(\mathbf{M}\)</span> (either from S-PrediXcan across each tissue, or S-MultiXcan) into the low-dimensional gene module space learned by MultiPLIER using:</p>
<p><span id="eq:proj" class="eqnos"><span class="math display">\[
\hat{\mathbf{M}} = (\mathbf{Z}^{\top} \mathbf{Z} + \lambda_{2} \mathbf{I})^{-1} \mathbf{Z}^{\top} \mathbf{M}.
\]</span><span class="eqnos-number">(2)</span></span> </p>
<p>where in <span class="math inline">\(\hat{\mathbf{M}}^{l \times n}\)</span> all traits in PhenomeXcan are now described by gene modules.</p>
<h3 id="crispr-cas9-screening">CRISPR-Cas9 screening</h3>
<p><code class="red">Add details</code></p>
<h3 id="gene-module-trait-associations">Gene module-trait associations</h3>
<p>To compute an association between a gene module and a trait, we used an approach similar to the gene-property analysis in MAGMA <span class="citation" data-cites="19XiXgYmd">[<a href="#ref-19XiXgYmd" role="doc-biblioref">30</a>]</span>,
which is essentially a competitive test using gene weights from <span class="math inline">\(\mathbf{Z}\)</span> to predict gene <span class="math inline">\(z\)</span>-scores from <span class="math inline">\(\mathbf{M}\)</span>.
Thus, the regression model uses genes as data points by fitting <span class="math inline">\(\mathbf{m}=\beta_0 + \mathbf{z} \beta_z + \epsilon\)</span>, where <span class="math inline">\(\epsilon \sim \mathrm{MVN}(0, \hat{\Sigma})\)</span>, <span class="math inline">\(\mathbf{m}\)</span> are gene <span class="math inline">\(p\)</span>-values (for a trait) from S-MultiXcan that we transformed to <span class="math inline">\(z\)</span>-scores as mentioned before.
Since we are only interested in whether genes with a stronger membership to a module (highest weights) are more associated with the phenotype, we performed a one-sided test on the coefficient <span class="math inline">\(\beta_z\)</span> with the null hypothesis of <span class="math inline">\(\beta_z = 0\)</span> against the alternative <span class="math inline">\(\beta_z&gt;0\)</span>.
Since the error terms <span class="math inline">\(\epsilon\)</span> could be correlated due to correlation between predicted expression, we used a generalized least squares approach instead of standard linear regression.
To calculate <span class="math inline">\(\hat\Sigma\)</span>, we first estimated the correlation of predicted expression for each gene pair <span class="math inline">\((\mathbf{t}_i, \mathbf{t}_j)\)</span> in tissue <span class="math inline">\(t\)</span> using equations from <span class="citation" data-cites="1FFzCXo1s vLyTudUB">[<a href="#ref-1FFzCXo1s" role="doc-biblioref">13</a>,<a href="#ref-vLyTudUB" role="doc-biblioref">70</a>]</span>:</p>
<p><span id="eq:gene_corr" class="eqnos"><span class="math display">\[
\begin{split}
\hat{\Sigma}_{ij}^{t} &amp; = Cor(\mathbf{t}_i, \mathbf{t}_j) \\
 &amp; = \frac{ Cov(\mathbf{t}_i, \mathbf{t}_j) } { \sqrt{\widehat{\mathrm{var}}(\mathbf{t}_i) \widehat{\mathrm{var}}(\mathbf{t}_j)} } \\
 &amp; = \frac{ Cov(\sum_{a \in \mathrm{model}_i} w_a^i X_a, \sum_{b \in \mathrm{model}_j} w_b^j X_b) }  {\sqrt{\widehat{\mathrm{var}}(\mathbf{t}_i) \widehat{\mathrm{var}}(\mathbf{t}_j)} } \\
 &amp; = \frac{ \sum_{a \in \mathrm{model}_i \\ b \in \mathrm{model}_j} w_a^i w_b^j Cov(X_a, X_b)} {\sqrt{\widehat{\mathrm{var}}(\mathbf{t}_i) \widehat{\mathrm{var}}(\mathbf{t}_j)} } \\
 &amp; = \frac{ \sum_{a \in \mathrm{model}_i \\ b \in \mathrm{model}_j} w_a^i w_b^j \Gamma_{ab}} {\sqrt{\widehat{\mathrm{var}}(\mathbf{t}_i) \widehat{\mathrm{var}}(\mathbf{t}_j)} },
\end{split}
\]</span><span class="eqnos-number">(3)</span></span> </p>
<p>where <span class="math inline">\(\Gamma = \widehat{\mathrm{var}}(\mathbf{X}) = (\mathbf{X} - \mathbf{\bar{X}})^{\top} (\mathbf{X} - \mathbf{\bar{X}}) / (m-1)\)</span> is the genotype covariance matrix using 1000 Genomes Project data <span class="citation" data-cites="11PJIeJyi 1BWx1HXgP">[<a href="#ref-11PJIeJyi" role="doc-biblioref">73</a>,<a href="#ref-1BWx1HXgP" role="doc-biblioref">74</a>]</span>.
The variances for predicted gene expression of gene <span class="math inline">\(i\)</span> is estimated as:</p>
<p><span id="eq:gene_var" class="eqnos"><span class="math display">\[
\begin{split}
\widehat{\mathrm{var}}(\mathbf{t}_i) &amp; = (\mathbf{W}^i)^\top \Gamma^i \mathbf{W}^i \\
 &amp; = \sum_{a \in \mathrm{model}_i \\ b \in \mathrm{model}_i} w_a^i w_b^i \Gamma_{ab}^i.
\end{split}
\]</span><span class="eqnos-number">(4)</span></span> </p>
<p>Finally, <span class="math inline">\(\hat{\Sigma} = \sum_t \hat{\Sigma}^t / |t|\)</span> where <span class="math inline">\(|t|\)</span>=49 is the number of tissues.</p>
<h3 id="drug-disease-prediction">Drug-disease prediction</h3>
<p>For the drug-disease prediction, we used a method based on a drug repositioning framework previously used for psychiatry traits <span class="citation" data-cites="17oeJ0CXy">[<a href="#ref-17oeJ0CXy" role="doc-biblioref">37</a>]</span> where gene-trait associations are anticorrelated with expression profiles for drugs.
For the single-gene approach, we computed a drug-disease score by multiplying each S-PrediXcan set of results in tissue <span class="math inline">\(t\)</span>, <span class="math inline">\(\mathbf{M}^t\)</span>, with the transcriptional responses profiled in LINCS L1000 <span class="citation" data-cites="F7lIlh2N">[<a href="#ref-F7lIlh2N" role="doc-biblioref">31</a>]</span>, <span class="math inline">\(\mathbf{L}^{c \times m}\)</span> (for <span class="math inline">\(c\)</span> compounds): <span class="math inline">\(\mathbf{D}^{t,k}=-1 \cdot \mathbf{M}^{t,k} \mathbf{L}^\top\)</span>, where <span class="math inline">\(k\)</span> refers to the number of most significant gene associations in <span class="math inline">\(\mathbf{M}^t\)</span> for each trait.
As suggested in <span class="citation" data-cites="17oeJ0CXy">[<a href="#ref-17oeJ0CXy" role="doc-biblioref">37</a>]</span>, <span class="math inline">\(k\)</span> could be either all genes or the top 50, 100, 250, and 500; then we average score ranks across all <span class="math inline">\(k\)</span> and obtain <span class="math inline">\(\mathbf{D}^t\)</span>.
Finally, for each drug-disease pair, we took the maximum prediction score across all tissues: <span class="math inline">\(\mathbf{D}_{ij} = \max \{ \mathbf{D}_{ij}^t \mid \forall t \}\)</span>.</p>
<p>The same procedure was used for the gene module-based approach, where we projected S-PrediXcan results into our latent representation, leading to <span class="math inline">\(\hat{\mathbf{M}}^t\)</span>;
and also <span class="math inline">\(\mathbf{L}\)</span>, leading to <span class="math inline">\(\hat{\mathbf{L}}^{l \times c}\)</span>.
Finally, <span class="math inline">\(\mathbf{D}^{t,k}=-1 \cdot \hat{\mathbf{M}}^{t,k} \hat{\mathbf{L}}^\top\)</span>, where in this case <span class="math inline">\(k\)</span> could be all LVs or the top 5, 10, 25 and 50 (since have an order of magnitude less LVs than genes).</p>
<p>Since the gold standard of drug-disease medical indications used contained Disease Ontology IDs (DOID) <span class="citation" data-cites="1FsruosUW">[<a href="#ref-1FsruosUW" role="doc-biblioref">75</a>]</span>, we mapped PhenomeXcan traits to the Experimental Factor Ontology <span class="citation" data-cites="9okjVu3s">[<a href="#ref-9okjVu3s" role="doc-biblioref">76</a>]</span> using <span class="citation" data-cites="16RTdMKxI">[<a href="#ref-16RTdMKxI" role="doc-biblioref">77</a>]</span>, and then to DOID.</p>
<h3 id="consensus-clustering-of-traits">Consensus clustering of traits</h3>
<p>We performed two preprocessing steps on the S-MultiXcan results before the cluster analysis procedure.
First, we combined results in <span class="math inline">\(\mathbf{M}\)</span> (S-MultiXcan) for traits that mapped to the same Experimental Factor Ontology (EFO) <span class="citation" data-cites="9okjVu3s">[<a href="#ref-9okjVu3s" role="doc-biblioref">76</a>]</span> term using the Stouffer’s method: <span class="math inline">\(\sum w_i M_{ij} / \sqrt{\sum w_i^2}\)</span>, where <span class="math inline">\(w_i\)</span> is a weight based on the GWAS sample size for trait <span class="math inline">\(i\)</span>, and <span class="math inline">\(M_{ij}\)</span> is the <span class="math inline">\(z\)</span>-score for gene <span class="math inline">\(j\)</span>.
Second, we standardized all <span class="math inline">\(z\)</span>-scores for each trait <span class="math inline">\(i\)</span> by their sum to reduce the effect of highly polygenic traits: <span class="math inline">\(M_{ij} / \sum M_{ij}\)</span>.
Finally, we projected this data matrix using Equation <a href="#eq:proj">2</a>, obtaining <span class="math inline">\(\hat{\mathbf{M}}\)</span> with <span class="math inline">\(n\)</span>=3752 traits and <span class="math inline">\(l\)</span>=987 LVs as the input of our clustering pipeline.</p>
<p>A partitioning of <span class="math inline">\(\hat{\mathbf{M}}\)</span> with <span class="math inline">\(n\)</span> traits into <span class="math inline">\(k\)</span> clusters is represented as a label vector <span class="math inline">\(\pi \in \mathbb{N}^n\)</span>.
<!--  -->
Consensus clustering approaches consist of two steps:
1) the generation of an ensemble <span class="math inline">\(\Pi\)</span> with <span class="math inline">\(r\)</span> partitions of the dataset: <span class="math inline">\(\Pi=\{\pi_1, \pi_2, \ldots, \pi_r\}\)</span>,
and 2) the combination of the ensemble into a consolidated solution defined as:</p>
<p><span id="eq:consensus:obj_func" class="eqnos"><span class="math display">\[
\pi^* = \mathrm{arg}\,\underset{\hat{\pi}}{\max} Q(\{ \lvert \mathcal{L}^i \lvert \phi(\hat{\pi}_{\mathcal{L}^i}, \pi_{i \mathcal{L}^i}) \mid i \in \{1,\ldots,r\} \}),
\]</span><span class="eqnos-number">(5)</span></span> </p>
<p>where <span class="math inline">\(\mathcal{L}^i\)</span> is a set of data indices with known cluster labels for partition <span class="math inline">\(i\)</span>,
<span class="math inline">\(\phi\colon \mathbb{N}^n \times \mathbb{N}^n \to \mathbb{R}\)</span> is a function that measures the similarity between two partitions,
and <span class="math inline">\(Q\)</span> is a measure of central tendency, such as the mean or median.
We used the adjusted Rand index (ARI) <span class="citation" data-cites="e4AuoW8N">[<a href="#ref-e4AuoW8N" role="doc-biblioref">78</a>]</span> for <span class="math inline">\(\phi\)</span>, and the median for <span class="math inline">\(Q\)</span>.
<!--  -->
To obtain <span class="math inline">\(\pi^*\)</span>, we define a consensus function <span class="math inline">\(\Gamma\colon \mathbb{N}^{n \times r} \to \mathbb{N}^n\)</span> with <span class="math inline">\(\Pi\)</span> as the input.
We used consensus functions based on the evidence accumulation clustering (EAC) paradigm <span class="citation" data-cites="cuROQDFa">[<a href="#ref-cuROQDFa" role="doc-biblioref">79</a>]</span>, where <span class="math inline">\(\Pi\)</span> is first transformed into a distance matrix
<!--  -->
<!-- $\mathbf{D}_{ij} = \frac{d_{ij}}{r}$, -->
<span class="math inline">\(\mathbf{D}_{ij} = d_{ij} / r\)</span>,
<!--  -->
where <span class="math inline">\(d_{ij}\)</span> is the number of times traits <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> were grouped in different clusters across all <span class="math inline">\(r\)</span> partitions in <span class="math inline">\(\Pi\)</span>.
Then, <span class="math inline">\(\Gamma\)</span> can be any similarity-based clustering algorithm, which is applied on <span class="math inline">\(\mathbf{D}\)</span> to derive the final partition <span class="math inline">\(\pi^*\)</span>.</p>
<p>For the ensemble generation step, we used different algorithms to create a highly diverse set of partitions (see Figure <a href="#fig:clustering:design">4</a>), since diversity is an important property for ensembles <span class="citation" data-cites="8js8Q3pF rcTMvL18 t5p3UpxZ">[<a href="#ref-8js8Q3pF" role="doc-biblioref">49</a>,<a href="#ref-rcTMvL18" role="doc-biblioref">80</a>,<a href="#ref-t5p3UpxZ" role="doc-biblioref">81</a>]</span>.
We used three data representations: the raw dataset, its projection into the top 50 principal components, and the embedding learned by UMAP <span class="citation" data-cites="157h5hA34">[<a href="#ref-157h5hA34" role="doc-biblioref">82</a>]</span> using 50 components.
<!--  -->
For each of these, we applied five clustering algorithms, covering a wide range of different assumptions on the data structure: <span class="math inline">\(k\)</span>-means <span class="citation" data-cites="MxGpAiPu">[<a href="#ref-MxGpAiPu" role="doc-biblioref">83</a>]</span>, spectral clustering <span class="citation" data-cites="x3CT24TB">[<a href="#ref-x3CT24TB" role="doc-biblioref">84</a>]</span>, a Gaussian mixture model (GMM), hierarchical clustering, and DBSCAN <span class="citation" data-cites="JrL3iQea">[<a href="#ref-JrL3iQea" role="doc-biblioref">85</a>]</span>.
<!--  -->
For <span class="math inline">\(k\)</span>-means, spectral clustering and GMM, we specified a range of <span class="math inline">\(k\)</span> between 2 and <span class="math inline">\(\sqrt{n} \approx 60\)</span>, and for each <span class="math inline">\(k\)</span> we generated five partitions using random seeds.
<!--  -->
For hierarchical clustering, for each <span class="math inline">\(k\)</span> we generated four partitions using four common linkage criteria: ward, complete, average and single.
<!--  -->
For DBSCAN, we combined different ranges for parameters <span class="math inline">\(\epsilon\)</span> (the maximum distance between two data points to be considered part of the same neighborhood) and <em>minPts</em> (the minimum number of data points in a neighborhood for a data point to be considered a core point).
Specifically, we used <em>minPts</em> values from 2 to 125, and for each data version, we determined a plausible range of <span class="math inline">\(\epsilon\)</span> values by observing the distribution of the mean distance of the <em>minPts</em>-nearest neighbors across all data points.
Since some combinations of <em>minPts</em> and <span class="math inline">\(\epsilon\)</span> might not produce a meaningful partition (for instance, when all points are detected as noisy or only one cluster is found), we resampled partitions generated by DBSCAN to ensure an equal representation in the ensemble.
<!--  -->
This procedure generated a final ensemble of 4428 partitions.</p>
<p>Finally, we used spectral clustering on <span class="math inline">\(\mathbf{D}\)</span> to derive the final consensus partitions.
<span class="math inline">\(\mathbf{D}\)</span> was first transformed into a similarity matrix by applying an RBF kernel <span class="math inline">\(\mathrm{exp}(-\gamma \mathbf{D}^2)\)</span> using four different values for <span class="math inline">\(\gamma\)</span> that we empirically determined to work best.
Thus for each <span class="math inline">\(k\)</span> between 2 and 60, we derived four consensus partitions and selected the one that maximized Equation <a href="#eq:consensus:obj_func">5</a>.
<!--  -->
We further filtered this set of 59 solutions to keep only those with an ensemble agreement larger than the 75th percentile, leaving a total of 15 final consensus partitions shown in Figure <a href="#fig:clustering:tree">5</a>.</p>
<h3 id="clusters-interpretation">Clusters interpretation</h3>
<p>We used a supervised learning approach to interpret clustering results by detecting which gene modules are the most important for clusters of traits.
For this, we used the highest resolution partition (<span class="math inline">\(k\)</span>=29, although any could be used) to train a decision tree model using each of the clusters as labels and the projected data <span class="math inline">\(\hat{\mathbf{M}}\)</span> as the training samples.
For each <span class="math inline">\(k\)</span>, we built a set of binary labels with the current cluster’s traits as the positive class and the rest of the traits as the negative class.
Then, we selected the LV in the root node of the trained model only if its threshold was positive and larger than one standard deviation.
Next, we removed this LV from <span class="math inline">\(\hat{\mathbf{M}}\)</span> (regardless of being previously selected or not) and trained the model again.
We repeated this procedure 20 times to extract the top 20 LVs that better discriminate traits in a cluster from the rest.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-O0e3EhY6">
<p>1. <strong>Relaxed purifying selection and possibly high rate of adaptation in primate lineage-specific genes.</strong> <br />
James J Cai, Dmitri A Petrov<br />
<em>Genome biology and evolution</em> (2010-07-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20624743">https://www.ncbi.nlm.nih.gov/pubmed/20624743</a> <br />
DOI: <a href="https://doi.org/10.1093/gbe/evq019">10.1093/gbe/evq019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20624743">20624743</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2997544">PMC2997544</a></p>
</div>
<div id="ref-18jYvPauB">
<p>2. <strong>Elevated rates of protein secretion, evolution, and disease among tissue-specific genes.</strong> <br />
Eitan E Winter, Leo Goodstadt, Chris P Ponting<br />
<em>Genome research</em> (2004-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14707169">https://www.ncbi.nlm.nih.gov/pubmed/14707169</a> <br />
DOI: <a href="https://doi.org/10.1101/gr.1924004">10.1101/gr.1924004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14707169">14707169</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC314278">PMC314278</a></p>
</div>
<div id="ref-wNE0EQlN">
<p>3. <strong>A large-scale analysis of tissue-specific pathology and gene expression of human disease genes and complexes</strong> <br />
K. Lage, N. T. Hansen, E. O. Karlberg, A. C. Eklund, F. S. Roque, P. K. Donahoe, Z. Szallasi, T. S. Jensen, S. Brunak<br />
<em>Proceedings of the National Academy of Sciences</em> (2008-12-22) <a href="https://doi.org/d5qcv9">https://doi.org/d5qcv9</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.0810772105">10.1073/pnas.0810772105</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19104045">19104045</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2606902">PMC2606902</a></p>
</div>
<div id="ref-9Pr9idng">
<p>4. <strong>The GTEx Consortium atlas of genetic regulatory effects across human tissues</strong> <br />
The GTEx Consortium<br />
<em>Science</em> (2020-09-11) <a href="https://doi.org/ghbnhr">https://doi.org/ghbnhr</a> <br />
DOI: <a href="https://doi.org/10.1126/science.aaz1776">10.1126/science.aaz1776</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32913098">32913098</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7737656">PMC7737656</a></p>
</div>
<div id="ref-1DoyZS7y0">
<p>5. <strong>Index and biological spectrum of human DNase I hypersensitive sites</strong> <br />
Wouter Meuleman, Alexander Muratov, Eric Rynes, Jessica Halow, Kristen Lee, Daniel Bates, Morgan Diegel, Douglas Dunn, Fidencio Neri, Athanasios Teodosiadis, … John Stamatoyannopoulos<br />
<em>Nature</em> (2020-07-29) <a href="https://doi.org/gg6dhp">https://doi.org/gg6dhp</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2559-3">10.1038/s41586-020-2559-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32728217">32728217</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7422677">PMC7422677</a></p>
</div>
<div id="ref-nhaocxmR">
<p>6. <strong>Mechanisms of tissue and cell-type specificity in heritable traits and diseases</strong> <br />
Idan Hekselman, Esti Yeger-Lotem<br />
<em>Nature Reviews Genetics</em> (2020-01-08) <a href="https://doi.org/ggkx9v">https://doi.org/ggkx9v</a> <br />
DOI: <a href="https://doi.org/10.1038/s41576-019-0200-9">10.1038/s41576-019-0200-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31913361">31913361</a></p>
</div>
<div id="ref-xRGqPsT2">
<p>7. <strong>Regulatory genomic circuitry of human disease loci by integrative epigenomics</strong> <br />
Carles A. Boix, Benjamin T. James, Yongjin P. Park, Wouter Meuleman, Manolis Kellis<br />
<em>Nature</em> (2021-02-03) <a href="https://doi.org/ghzkhr">https://doi.org/ghzkhr</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-020-03145-z">10.1038/s41586-020-03145-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33536621">33536621</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7875769">PMC7875769</a></p>
</div>
<div id="ref-6SPTvFXq">
<p>8. <strong>Reproducible RNA-seq analysis using recount2</strong> <br />
Leonardo Collado-Torres, Abhinav Nellore, Kai Kammers, Shannon E Ellis, Margaret A Taub, Kasper D Hansen, Andrew E Jaffe, Ben Langmead, Jeffrey T Leek<br />
<em>Nature Biotechnology</em> (2017-04-11) <a href="https://doi.org/gf75hp">https://doi.org/gf75hp</a> <br />
DOI: <a href="https://doi.org/10.1038/nbt.3838">10.1038/nbt.3838</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28398307">28398307</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6742427">PMC6742427</a></p>
</div>
<div id="ref-Nz3IMEzd">
<p>9. <strong>Massive mining of publicly available RNA-seq data from human and mouse</strong> <br />
Alexander Lachmann, Denis Torre, Alexandra B. Keenan, Kathleen M. Jagodnik, Hoyjin J. Lee, Lily Wang, Moshe C. Silverstein, Avi Ma’ayan<br />
<em>Nature Communications</em> (2018-04-10) <a href="https://doi.org/gc92dr">https://doi.org/gc92dr</a> <br />
DOI: <a href="https://doi.org/10.1038/s41467-018-03751-6">10.1038/s41467-018-03751-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29636450">29636450</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5893633">PMC5893633</a></p>
</div>
<div id="ref-57TjOMEA">
<p>10. <strong>Identification of therapeutic targets from genetic association studies using hierarchical component analysis</strong> <br />
Hao-Chih Lee, Osamu Ichikawa, Benjamin S. Glicksberg, Aparna A. Divaraniya, Christine E. Becker, Pankaj Agarwal, Joel T. Dudley<br />
<em>BioData Mining</em> (2020-06-17) <a href="https://doi.org/gjp5pf">https://doi.org/gjp5pf</a> <br />
DOI: <a href="https://doi.org/10.1186/s13040-020-00216-9">10.1186/s13040-020-00216-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32565911">32565911</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7301559">PMC7301559</a></p>
</div>
<div id="ref-yEdRP9Xx">
<p>11. <strong>Novel Variance-Component TWAS method for studying complex human diseases with applications to Alzheimer’s dementia</strong> <br />
Shizhen Tang, Aron S. Buchman, Philip L. De Jager, David A. Bennett, Michael P. Epstein, Jingjing Yang<br />
<em>PLOS Genetics</em> (2021-04-02) <a href="https://doi.org/gjpr3j">https://doi.org/gjpr3j</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pgen.1009482">10.1371/journal.pgen.1009482</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33798195">33798195</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8046351">PMC8046351</a></p>
</div>
<div id="ref-1D63fEEPb">
<p>12. <strong>Integrative approaches for large-scale transcriptome-wide association studies</strong> <br />
Alexander Gusev, Arthur Ko, Huwenbo Shi, Gaurav Bhatia, Wonil Chung, Brenda WJH Penninx, Rick Jansen, Eco JC de Geus, Dorret I Boomsma, Fred A Wright, … Bogdan Pasaniuc<br />
<em>Nature Genetics</em> (2016-02-08) <a href="https://doi.org/f3vf4p">https://doi.org/f3vf4p</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.3506">10.1038/ng.3506</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26854917">26854917</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4767558">PMC4767558</a></p>
</div>
<div id="ref-1FFzCXo1s">
<p>13. <strong>Integrating predicted transcriptome from multiple tissues improves association detection</strong> <br />
Alvaro N. Barbeira, Milton Pividori, Jiamao Zheng, Heather E. Wheeler, Dan L. Nicolae, Hae Kyung Im<br />
<em>PLOS Genetics</em> (2019-01-22) <a href="https://doi.org/ghs8vx">https://doi.org/ghs8vx</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pgen.1007889">10.1371/journal.pgen.1007889</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30668570">30668570</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6358100">PMC6358100</a></p>
</div>
<div id="ref-Z8bvDdVq">
<p>14. <strong>A gene-based association method for mapping traits using reference transcriptome data</strong> <br />
Eric R Gamazon, Heather E Wheeler, Kaanan P Shah, Sahar V Mozaffari, Keston Aquino-Michaels, Robert J Carroll, Anne E Eyler, Joshua C Denny, Dan L Nicolae, Nancy J Cox, … GTEx Consortium<br />
<em>Nature Genetics</em> (2015-08-10) <a href="https://doi.org/f7p9zv">https://doi.org/f7p9zv</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.3367">10.1038/ng.3367</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26258848">26258848</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4552594">PMC4552594</a></p>
</div>
<div id="ref-4iMVo07x">
<p>15. <strong>Integrating Gene Expression with Summary Association Statistics to Identify Genes Associated with 30 Complex Traits</strong> <br />
Nicholas Mancuso, Huwenbo Shi, Pagé Goddard, Gleb Kichaev, Alexander Gusev, Bogdan Pasaniuc<br />
<em>The American Journal of Human Genetics</em> (2017-03) <a href="https://doi.org/f9wvsg">https://doi.org/f9wvsg</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ajhg.2017.01.031">10.1016/j.ajhg.2017.01.031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28238358">28238358</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339290">PMC5339290</a></p>
</div>
<div id="ref-WFslDIWl">
<p>16. <strong>Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types</strong> <br />
Hilary K. Finucane, Yakir A. Reshef, Verneri Anttila, Kamil Slowikowski, Alexander Gusev, Andrea Byrnes, Steven Gazal, Po-Ru Loh, Caleb Lareau, Noam Shoresh, … The Brainstorm Consortium<br />
<em>Nature Genetics</em> (2018-04-09) <a href="https://doi.org/gdfjqt">https://doi.org/gdfjqt</a> <br />
DOI: <a href="https://doi.org/10.1038/s41588-018-0081-4">10.1038/s41588-018-0081-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29632380">29632380</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5896795">PMC5896795</a></p>
</div>
<div id="ref-nga58tSh">
<p>17. <strong>Integrating Autoimmune Risk Loci with Gene-Expression Data Identifies Specific Pathogenic Immune Cell Subsets</strong> <br />
Xinli Hu, Hyun Kim, Eli Stahl, Robert Plenge, Mark Daly, Soumya Raychaudhuri<br />
<em>The American Journal of Human Genetics</em> (2011-10) <a href="https://doi.org/fpghp4">https://doi.org/fpghp4</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ajhg.2011.09.002">10.1016/j.ajhg.2011.09.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21963258">21963258</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3188838">PMC3188838</a></p>
</div>
<div id="ref-kXUuzq6I">
<p>18. <strong>SNPsea: an algorithm to identify cell types, tissues and pathways affected by risk loci</strong> <br />
Kamil Slowikowski, Xinli Hu, Soumya Raychaudhuri<br />
<em>Bioinformatics</em> (2014-09-01) <a href="https://doi.org/f6j6v3">https://doi.org/f6j6v3</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btu326">10.1093/bioinformatics/btu326</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24813542">24813542</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147889">PMC4147889</a></p>
</div>
<div id="ref-kUDav9OX">
<p>19. <strong>Meta-analysis of 375,000 individuals identifies 38 susceptibility loci for migraine</strong> <br />
Padhraig Gormley, Verneri Anttila, Bendik S Winsvold, Priit Palta, Tonu Esko, Tune H Pers, Kai-How Farh, Ester Cuenca-Leon, Mikko Muona, Nicholas A Furlotte, … International Headache Genetics Consortium<br />
<em>Nature Genetics</em> (2016-06-20) <a href="https://doi.org/bmzx">https://doi.org/bmzx</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.3598">10.1038/ng.3598</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27322543">27322543</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5331903">PMC5331903</a></p>
</div>
<div id="ref-z8MQTAnJ">
<p>20. <strong>Biological interpretation of genome-wide association studies using predicted gene functions</strong> <br />
Tune H. Pers, Juha M. Karjalainen, Yingleong Chan, Harm-Jan Westra, Andrew R. Wood, Jian Yang, Julian C. Lui, Sailaja Vedantam, Stefan Gustafsson, Tonu Esko, … Genetic Investigation of ANthropometric Traits (GIANT) Consortium<br />
<em>Nature Communications</em> (2015-01-19) <a href="https://doi.org/f3mwhd">https://doi.org/f3mwhd</a> <br />
DOI: <a href="https://doi.org/10.1038/ncomms6890">10.1038/ncomms6890</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25597830">25597830</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4420238">PMC4420238</a></p>
</div>
<div id="ref-12w2U7X9m">
<p>21. <strong>Estimating the causal tissues for complex traits and diseases</strong> <br />
Halit Ongen, Andrew A Brown, Olivier Delaneau, Nikolaos I Panousis, Alexandra C Nica, Emmanouil T Dermitzakis, GTEx Consortium<br />
<em>Nature Genetics</em> (2017-10-23) <a href="https://doi.org/ggrr72">https://doi.org/ggrr72</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.3981">10.1038/ng.3981</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29058715">29058715</a></p>
</div>
<div id="ref-mPnIAH38">
<p>22. <strong>Tackling the widespread and critical impact of batch effects in high-throughput data</strong> <br />
Jeffrey T. Leek, Robert B. Scharpf, Héctor Corrada Bravo, David Simcha, Benjamin Langmead, W. Evan Johnson, Donald Geman, Keith Baggerly, Rafael A. Irizarry<br />
<em>Nature Reviews Genetics</em> (2010-09-14) <a href="https://doi.org/cfr324">https://doi.org/cfr324</a> <br />
DOI: <a href="https://doi.org/10.1038/nrg2825">10.1038/nrg2825</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20838408">20838408</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3880143">PMC3880143</a></p>
</div>
<div id="ref-Ki2ij7zE">
<p>23. <strong>Pathway-level information extractor (PLIER) for gene expression data</strong> <br />
Weiguang Mao, Elena Zaslavsky, Boris M. Hartmann, Stuart C. Sealfon, Maria Chikina<br />
<em>Nature Methods</em> (2019-06-27) <a href="https://doi.org/gf75g6">https://doi.org/gf75g6</a> <br />
DOI: <a href="https://doi.org/10.1038/s41592-019-0456-1">10.1038/s41592-019-0456-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31249421">31249421</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7262669">PMC7262669</a></p>
</div>
<div id="ref-lY5ln3dB">
<p>24. <strong>PhenomeXcan: Mapping the genome to the phenome through the transcriptome</strong> <br />
Milton Pividori, Padma S. Rajagopal, Alvaro Barbeira, Yanyu Liang, Owen Melia, Lisa Bastarache, YoSon Park, GTEx Consortium, Xiaoquan Wen, Hae K. Im<br />
<em>Science Advances</em> (2020-09) <a href="https://doi.org/ghbvbf">https://doi.org/ghbvbf</a> <br />
DOI: <a href="https://doi.org/10.1126/sciadv.aba2083">10.1126/sciadv.aba2083</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32917697">32917697</a></p>
</div>
<div id="ref-14rnBunuZ">
<p>25. <strong>MultiPLIER: A Transfer Learning Framework for Transcriptomics Reveals Systemic Features of Rare Disease</strong> <br />
Jaclyn N. Taroni, Peter C. Grayson, Qiwen Hu, Sean Eddy, Matthias Kretzler, Peter A. Merkel, Casey S. Greene<br />
<em>Cell Systems</em> (2019-05) <a href="https://doi.org/gf75g5">https://doi.org/gf75g5</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cels.2019.04.003">10.1016/j.cels.2019.04.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31121115">31121115</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6538307">PMC6538307</a></p>
</div>
<div id="ref-wfqjCerX">
<p>26. <strong>The Electronic Medical Records and Genomics (eMERGE) Network: past, present, and future</strong> <br />
Omri Gottesman, Helena Kuivaniemi, Gerard Tromp, W. Andrew Faucett, Rongling Li, Teri A. Manolio, Saskia C. Sanderson, Joseph Kannry, Randi Zinberg, Melissa A. Basford, … and The eMERGE Network<br />
<em>Genetics in Medicine</em> (2013-06-06) <a href="https://doi.org/f5dwbt">https://doi.org/f5dwbt</a> <br />
DOI: <a href="https://doi.org/10.1038/gim.2013.72">10.1038/gim.2013.72</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23743551">23743551</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3795928">PMC3795928</a></p>
</div>
<div id="ref-nmJxPpE5">
<p>27. <strong>The UK Biobank resource with deep phenotyping and genomic data</strong> <br />
Clare Bycroft, Colin Freeman, Desislava Petkova, Gavin Band, Lloyd T. Elliott, Kevin Sharp, Allan Motyer, Damjan Vukcevic, Olivier Delaneau, Jared O’Connell, … Jonathan Marchini<br />
<em>Nature</em> (2018-10-10) <a href="https://doi.org/gfb7h2">https://doi.org/gfb7h2</a> <br />
DOI: <a href="https://doi.org/10.1038/s41586-018-0579-z">10.1038/s41586-018-0579-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30305743">30305743</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786975">PMC6786975</a></p>
</div>
<div id="ref-1FVd2WW6G">
<p>28. <strong>Finding function: evaluation methods for functional genomic data</strong> <br />
Chad L Myers, Daniel R Barrett, Matthew A Hibbs, Curtis Huttenhower, Olga G Troyanskaya<br />
<em>BMC Genomics</em> (2006-07-25) <a href="https://doi.org/fg6wnk">https://doi.org/fg6wnk</a> <br />
DOI: <a href="https://doi.org/10.1186/1471-2164-7-187">10.1186/1471-2164-7-187</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16869964">16869964</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1560386">PMC1560386</a></p>
</div>
<div id="ref-DN7TyZzb">
<p>29. <strong>The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens</strong> <br />
Naihui Zhou, Yuxiang Jiang, Timothy R. Bergquist, Alexandra J. Lee, Balint Z. Kacsoh, Alex W. Crocker, Kimberley A. Lewis, George Georghiou, Huy N. Nguyen, Md Nafiz Hamid, … Iddo Friedberg<br />
<em>Genome Biology</em> (2019-11-19) <a href="https://doi.org/ggnxpz">https://doi.org/ggnxpz</a> <br />
DOI: <a href="https://doi.org/10.1186/s13059-019-1835-8">10.1186/s13059-019-1835-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31744546">31744546</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6864930">PMC6864930</a></p>
</div>
<div id="ref-19XiXgYmd">
<p>30. <strong>MAGMA: Generalized Gene-Set Analysis of GWAS Data</strong> <br />
Christiaan A. de Leeuw, Joris M. Mooij, Tom Heskes, Danielle Posthuma<br />
<em>PLOS Computational Biology</em> (2015-04-17) <a href="https://doi.org/gf92gp">https://doi.org/gf92gp</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1004219">10.1371/journal.pcbi.1004219</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25885710">25885710</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4401657">PMC4401657</a></p>
</div>
<div id="ref-F7lIlh2N">
<p>31. <strong>A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles</strong> <br />
Aravind Subramanian, Rajiv Narayan, Steven M. Corsello, David D. Peck, Ted E. Natoli, Xiaodong Lu, Joshua Gould, John F. Davis, Andrew A. Tubelli, Jacob K. Asiedu, … Todd R. Golub<br />
<em>Cell</em> (2017-11) <a href="https://doi.org/cgwt">https://doi.org/cgwt</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2017.10.049">10.1016/j.cell.2017.10.049</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29195078">29195078</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5990023">PMC5990023</a></p>
</div>
<div id="ref-6PR8LEXK">
<p>32. <strong>DrugBank 4.0: shedding new light on drug metabolism</strong> <br />
Vivian Law, Craig Knox, Yannick Djoumbou, Tim Jewison, An Chi Guo, Yifeng Liu, Adam Maciejewski, David Arndt, Michael Wilson, Vanessa Neveu, … David S. Wishart<br />
<em>Nucleic Acids Research</em> (2014-01) <a href="https://doi.org/f3mn6d">https://doi.org/f3mn6d</a> <br />
DOI: <a href="https://doi.org/10.1093/nar/gkt1068">10.1093/nar/gkt1068</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24203711">24203711</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3965102">PMC3965102</a></p>
</div>
<div id="ref-O21tn8vf">
<p>33. <strong>Systematic integration of biomedical knowledge prioritizes drugs for repurposing</strong> <br />
Daniel Scott Himmelstein, Antoine Lizee, Christine Hessler, Leo Brueggeman, Sabrina L Chen, Dexter Hadley, Ari Green, Pouya Khankhanian, Sergio E Baranzini<br />
<em>eLife</em> (2017-09-22) <a href="https://doi.org/cdfk">https://doi.org/cdfk</a> <br />
DOI: <a href="https://doi.org/10.7554/elife.26726">10.7554/elife.26726</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28936969">28936969</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640425">PMC5640425</a></p>
</div>
<div id="ref-1DJZvtwP1">
<p>34. <strong>Dhimmel/Lincs V2.0: Refined Consensus Signatures From Lincs L1000</strong> <br />
Daniel Himmelstein, Leo Brueggeman, Sergio Baranzini<br />
<em>Zenodo</em> (2016-03-08) <a href="https://doi.org/f3mqvr">https://doi.org/f3mqvr</a> <br />
DOI: <a href="https://doi.org/10.5281/zenodo.47223">10.5281/zenodo.47223</a></p>
</div>
<div id="ref-1ClBKizD7">
<p>35. <strong>Computational Repositioning of the Anticonvulsant Topiramate for Inflammatory Bowel Disease</strong> <br />
J. T. Dudley, M. Sirota, M. Shenoy, R. K. Pai, S. Roedder, A. P. Chiang, A. A. Morgan, M. M. Sarwal, P. J. Pasricha, A. J. Butte<br />
<em>Science Translational Medicine</em> (2011-08-17) <a href="https://doi.org/bmh5ts">https://doi.org/bmh5ts</a> <br />
DOI: <a href="https://doi.org/10.1126/scitranslmed.3002648">10.1126/scitranslmed.3002648</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21849664">21849664</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3479650">PMC3479650</a></p>
</div>
<div id="ref-mZjkE1xU">
<p>36. <strong>Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data</strong> <br />
M. Sirota, J. T. Dudley, J. Kim, A. P. Chiang, A. A. Morgan, A. Sweet-Cordero, J. Sage, A. J. Butte<br />
<em>Science Translational Medicine</em> (2011-08-17) <a href="https://doi.org/c3fwxv">https://doi.org/c3fwxv</a> <br />
DOI: <a href="https://doi.org/10.1126/scitranslmed.3001318">10.1126/scitranslmed.3001318</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21849665">21849665</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3502016">PMC3502016</a></p>
</div>
<div id="ref-17oeJ0CXy">
<p>37. <strong>Analysis of genome-wide association data highlights candidates for drug repositioning in psychiatry</strong> <br />
Hon-Cheong So, Carlos Kwan-Long Chau, Wan-To Chiu, Kin-Sang Ho, Cho-Pong Lo, Stephanie Ho-Yue Yim, Pak-Chung Sham<br />
<em>Nature Neuroscience</em> (2017-08-14) <a href="https://doi.org/gbrssh">https://doi.org/gbrssh</a> <br />
DOI: <a href="https://doi.org/10.1038/nn.4618">10.1038/nn.4618</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28805813">28805813</a></p>
</div>
<div id="ref-10KA5jTBQ">
<p>38. <strong>Dhimmel/Indications V1.0. Pharmacotherapydb: The Open Catalog Of Drug Therapies For Disease</strong> <br />
Daniel S. Himmelstein, Pouya Khankhanian, Christine S. Hessler, Ari J. Green, Sergio E. Baranzini<br />
<em>Zenodo</em> (2016-03-15) <a href="https://doi.org/f3mqwb">https://doi.org/f3mqwb</a> <br />
DOI: <a href="https://doi.org/10.5281/zenodo.47664">10.5281/zenodo.47664</a></p>
</div>
<div id="ref-LVihFr3g">
<p>39. <strong>Mechanism of Action of Niacin</strong> <br />
Vaijinath S. Kamanna, Moti L. Kashyap<br />
<em>The American Journal of Cardiology</em> (2008-04) <a href="https://doi.org/c8zwdt">https://doi.org/c8zwdt</a> <br />
DOI: <a href="https://doi.org/10.1016/j.amjcard.2008.02.029">10.1016/j.amjcard.2008.02.029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18375237">18375237</a></p>
</div>
<div id="ref-7OgaYjeL">
<p>40. <strong>Niacin: an old lipid drug in a new NAD+ dress</strong> <br />
Mario Romani, Dina Carina Hofer, Elena Katsyuba, Johan Auwerx<br />
<em>Journal of Lipid Research</em> (2019-04) <a href="https://doi.org/gjpjft">https://doi.org/gjpjft</a> <br />
DOI: <a href="https://doi.org/10.1194/jlr.s092007">10.1194/jlr.s092007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30782960">30782960</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6446705">PMC6446705</a></p>
</div>
<div id="ref-izbPw2kc">
<p>41. <strong>The nicotinic acid receptor GPR109A (HM74A or PUMA-G) as a new therapeutic target</strong> <br />
S OFFERMANNS<br />
<em>Trends in Pharmacological Sciences</em> (2006-07) <a href="https://doi.org/fgb4tr">https://doi.org/fgb4tr</a> <br />
DOI: <a href="https://doi.org/10.1016/j.tips.2006.05.008">10.1016/j.tips.2006.05.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16766048">16766048</a></p>
</div>
<div id="ref-wI0IjT3i">
<p>42. <strong>Langerhans Cells Release Prostaglandin D2 in Response to Nicotinic Acid</strong> <br />
Dominique Maciejewski-Lenoir, Jeremy G. Richman, Yaron Hakak, Ibragim Gaidarov, Dominic P. Behan, Daniel T. Connolly<br />
<em>Journal of Investigative Dermatology</em> (2006-12) <a href="https://doi.org/dgxg75">https://doi.org/dgxg75</a> <br />
DOI: <a href="https://doi.org/10.1038/sj.jid.5700586">10.1038/sj.jid.5700586</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17008871">17008871</a></p>
</div>
<div id="ref-1Bz0jRHYo">
<p>43. <strong>Nicotinic acid inhibits progression of atherosclerosis in mice through its receptor GPR109A expressed by immune cells</strong> <br />
Martina Lukasova, Camille Malaval, Andreas Gille, Jukka Kero, Stefan Offermanns<br />
<em>Journal of Clinical Investigation</em> (2011-03-01) <a href="https://doi.org/cqftcq">https://doi.org/cqftcq</a> <br />
DOI: <a href="https://doi.org/10.1172/jci41651">10.1172/jci41651</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21317532">21317532</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3048854">PMC3048854</a></p>
</div>
<div id="ref-1DblG8swn">
<p>44. <strong>Role of HDL, ABCA1, and ABCG1 Transporters in Cholesterol Efflux and Immune Responses</strong> <br />
Laurent Yvan-Charvet, Nan Wang, Alan R. Tall<br />
<em>Arteriosclerosis, Thrombosis, and Vascular Biology</em> (2010-02) <a href="https://doi.org/ds23w6">https://doi.org/ds23w6</a> <br />
DOI: <a href="https://doi.org/10.1161/atvbaha.108.179283">10.1161/atvbaha.108.179283</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19797709">19797709</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812788">PMC2812788</a></p>
</div>
<div id="ref-kLRErKXz">
<p>45. <strong>Niacin Inhibits Apoptosis and Rescues Premature Ovarian Failure</strong> <br />
Shufang Wang, Min Sun, Ling Yu, Yixuan Wang, Yuanqing Yao, Deqing Wang<br />
<em>Cellular Physiology and Biochemistry</em> (2018) <a href="https://doi.org/gfqvcq">https://doi.org/gfqvcq</a> <br />
DOI: <a href="https://doi.org/10.1159/000495051">10.1159/000495051</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30415247">30415247</a></p>
</div>
<div id="ref-TovvsrDr">
<p>46. <strong>Chronic niacin administration ameliorates ovulation, histological changes in the ovary and adiponectin concentrations in a rat model of polycystic ovary syndrome</strong> <br />
Negin Asadi, Mahin Izadi, Ali Aflatounian, Mansour Esmaeili-Dehaj, Mohammad Ebrahim Rezvani, Zeinab Hafizi<br />
<em>Reproduction, Fertility and Development</em> (2021) <a href="https://doi.org/gjpjkt">https://doi.org/gjpjkt</a> <br />
DOI: <a href="https://doi.org/10.1071/rd20306">10.1071/rd20306</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33751926">33751926</a></p>
</div>
<div id="ref-17CbjLzCP">
<p>47. <strong>Clustering ensembles: models of consensus and weak partitions</strong> <br />
A. Topchy, A. K. Jain, W. Punch<br />
<em>IEEE Transactions on Pattern Analysis and Machine Intelligence</em> (2005-12) <a href="https://doi.org/c8z32x">https://doi.org/c8z32x</a> <br />
DOI: <a href="https://doi.org/10.1109/tpami.2005.237">10.1109/tpami.2005.237</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16355656">16355656</a></p>
</div>
<div id="ref-uw3AnEgA">
<p>48. <strong>Cluster Ensembles – A Knowledge Reuse Framework for Combining Multiple Partitions</strong> <br />
Alexander Strehl, Ghosh Joydeep<br />
<em>Journal of Machine Learning Research</em> (2002) <a href="https://www.jmlr.org/papers/v3/strehl02a.html">https://www.jmlr.org/papers/v3/strehl02a.html</a></p>
</div>
<div id="ref-8js8Q3pF">
<p>49. <strong>Diversity control for improving the analysis of consensus clustering</strong> <br />
Milton Pividori, Georgina Stegmayer, Diego H. Milone<br />
<em>Information Sciences</em> (2016-09) <a href="https://doi.org/ghtqbk">https://doi.org/ghtqbk</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ins.2016.04.027">10.1016/j.ins.2016.04.027</a></p>
</div>
<div id="ref-xhtEAzx6">
<p>50. <strong>Clustering trees: a visualization for evaluating clusterings at multiple resolutions</strong> <br />
Luke Zappia, Alicia Oshlack<br />
<em>GigaScience</em> (2018-07) <a href="https://doi.org/gfzqf5">https://doi.org/gfzqf5</a> <br />
DOI: <a href="https://doi.org/10.1093/gigascience/giy083">10.1093/gigascience/giy083</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30010766">30010766</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6057528">PMC6057528</a></p>
</div>
<div id="ref-aBVh8zt1">
<p>51. <strong>Prognostic value of grip strength: findings from the Prospective Urban Rural Epidemiology (PURE) study.</strong> <br />
Darryl P Leong, Koon K Teo, Sumathy Rangarajan, Patricio Lopez-Jaramillo, Alvaro Avezum, Andres Orlandini, Pamela Seron, Suad H Ahmed, Annika Rosengren, Roya Kelishadi, … <br />
<em>Lancet (London, England)</em> (2015-05-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25982160">https://www.ncbi.nlm.nih.gov/pubmed/25982160</a> <br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(14)62000-6">10.1016/s0140-6736(14)62000-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25982160">25982160</a></p>
</div>
<div id="ref-Zk82GvJV">
<p>52. <strong>Densely Interconnected Transcriptional Circuits Control Cell States in Human Hematopoiesis</strong> <br />
Noa Novershtern, Aravind Subramanian, Lee N. Lawton, Raymond H. Mak, W. Nicholas Haining, Marie E. McConkey, Naomi Habib, Nir Yosef, Cindy Y. Chang, Tal Shay, … Benjamin L. Ebert<br />
<em>Cell</em> (2011-01) <a href="https://doi.org/cf5k92">https://doi.org/cf5k92</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2011.01.004">10.1016/j.cell.2011.01.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21241896">21241896</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049864">PMC3049864</a></p>
</div>
<div id="ref-YyXvsbU9">
<p>53. <strong>CD19 <sup>+</sup> CD5 <sup>+</sup> Cells as Indicators of Preeclampsia</strong> <br />
Federico Jensen, Gerd Wallukat, Florian Herse, Oliver Budner, Tarek El-Mousleh, Serban-Dan Costa, Ralf Dechend, Ana Claudia Zenclussen<br />
<em>Hypertension</em> (2012-04) <a href="https://doi.org/gj36rs">https://doi.org/gj36rs</a> <br />
DOI: <a href="https://doi.org/10.1161/hypertensionaha.111.188276">10.1161/hypertensionaha.111.188276</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22353610">22353610</a></p>
</div>
<div id="ref-Om8ZhS06">
<p>54. <strong>Conditional and interaction gene-set analysis reveals novel functional pathways for blood pressure</strong> <br />
Christiaan A. de Leeuw, Sven Stringer, Ilona A. Dekkers, Tom Heskes, Danielle Posthuma<br />
<em>Nature Communications</em> (2018-09-14) <a href="https://doi.org/gd6d85">https://doi.org/gd6d85</a> <br />
DOI: <a href="https://doi.org/10.1038/s41467-018-06022-6">10.1038/s41467-018-06022-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30218068">30218068</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6138636">PMC6138636</a></p>
</div>
<div id="ref-meFYGVLK">
<p>55. <strong>Estrogen and hypertension</strong> <br />
Muhammad S. Ashraf, Wanpen Vongpatanasin<br />
<em>Current Hypertension Reports</em> (2006-09) <a href="https://doi.org/d638rf">https://doi.org/d638rf</a> <br />
DOI: <a href="https://doi.org/10.1007/s11906-006-0080-1">10.1007/s11906-006-0080-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16965722">16965722</a></p>
</div>
<div id="ref-10qjLoufR">
<p>56. <strong>Depression as a predictor for coronary heart disease. a review and meta-analysis.</strong> <br />
Reiner Rugulies<br />
<em>American journal of preventive medicine</em> (2002-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12093424">https://www.ncbi.nlm.nih.gov/pubmed/12093424</a> <br />
DOI: <a href="https://doi.org/10.1016/s0749-3797(02)00439-7">10.1016/s0749-3797(02)00439-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12093424">12093424</a></p>
</div>
<div id="ref-j2Sl4DAE">
<p>57. <strong>Mental Disorders Across the Adult Life Course and Future Coronary Heart Disease</strong> <br />
Catharine R. Gale, G. David Batty, David P. J. Osborn, Per Tynelius, Finn Rasmussen<br />
<em>Circulation</em> (2014-01-14) <a href="https://doi.org/qm4">https://doi.org/qm4</a> <br />
DOI: <a href="https://doi.org/10.1161/circulationaha.113.002065">10.1161/circulationaha.113.002065</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24190959">24190959</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4107269">PMC4107269</a></p>
</div>
<div id="ref-17LYMnG9n">
<p>58. <strong>Mortality gap for people with bipolar disorder and schizophrenia: UK-based cohort study 2000–2014</strong> <br />
Joseph F. Hayes, Louise Marston, Kate Walters, Michael B. King, David P. J. Osborn<br />
<em>British Journal of Psychiatry</em> (2018-01-02) <a href="https://doi.org/gbwcjx">https://doi.org/gbwcjx</a> <br />
DOI: <a href="https://doi.org/10.1192/bjp.bp.117.202606">10.1192/bjp.bp.117.202606</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28684403">28684403</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5579328">PMC5579328</a></p>
</div>
<div id="ref-13t4TuFeJ">
<p>59. <strong>Getting to the Heart of Alzheimer Disease</strong> <br />
Joshua M. Tublin, Jeremy M. Adelstein, Federica del Monte, Colin K. Combs, Loren E. Wold<br />
<em>Circulation Research</em> (2019-01-04) <a href="https://doi.org/gjzjgq">https://doi.org/gjzjgq</a> <br />
DOI: <a href="https://doi.org/10.1161/circresaha.118.313563">10.1161/circresaha.118.313563</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30605407">30605407</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6319653">PMC6319653</a></p>
</div>
<div id="ref-D83Aqhga">
<p>60. <strong>The overlap between vascular disease and Alzheimer’s disease - lessons from pathology</strong> <br />
Johannes Attems, Kurt A Jellinger<br />
<em>BMC Medicine</em> (2014-11-11) <a href="https://doi.org/f6pjd4">https://doi.org/f6pjd4</a> <br />
DOI: <a href="https://doi.org/10.1186/s12916-014-0206-2">10.1186/s12916-014-0206-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25385447">25385447</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4226890">PMC4226890</a></p>
</div>
<div id="ref-9BGyO071">
<p>61. <strong>Cardiovascular Risk Factors for Alzheimer’s Disease</strong> <br />
Clive Rosendorff, Michal S. Beeri, Jeremy M. Silverman<br />
<em>The American Journal of Geriatric Cardiology</em> (2007-03) <a href="https://doi.org/bpfw5d">https://doi.org/bpfw5d</a> <br />
DOI: <a href="https://doi.org/10.1111/j.1076-7460.2007.06696.x">10.1111/j.1076-7460.2007.06696.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17483665">17483665</a></p>
</div>
<div id="ref-idlBgtFz">
<p>62. <strong>Reverse cholesterol transport and cholesterol efflux in atherosclerosis</strong> <br />
R. Ohashi, H. Mu, X. Wang, Q. Yao, C. Chen<br />
<em>QJM: An International Journal of Medicine</em> (2005-12) <a href="https://doi.org/dn2fgt">https://doi.org/dn2fgt</a> <br />
DOI: <a href="https://doi.org/10.1093/qjmed/hci136">10.1093/qjmed/hci136</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16258026">16258026</a></p>
</div>
<div id="ref-l6ogswV3">
<p>63. <strong>Opportunities and challenges for transcriptome-wide association studies</strong> <br />
Michael Wainberg, Nasa Sinnott-Armstrong, Nicholas Mancuso, Alvaro N. Barbeira, David A. Knowles, David Golan, Raili Ermel, Arno Ruusalepp, Thomas Quertermous, Ke Hao, … Anshul Kundaje<br />
<em>Nature Genetics</em> (2019-03-29) <a href="https://doi.org/gf3hmr">https://doi.org/gf3hmr</a> <br />
DOI: <a href="https://doi.org/10.1038/s41588-019-0385-z">10.1038/s41588-019-0385-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30926968">30926968</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6777347">PMC6777347</a></p>
</div>
<div id="ref-ndd3tW4g">
<p>64. <strong>Probabilistic colocalization of genetic variants from complex and molecular traits: promise and limitations</strong> <br />
Abhay Hukku, Milton Pividori, Francesca Luca, Roger Pique-Regi, Hae Kyung Im, Xiaoquan Wen<br />
<em>The American Journal of Human Genetics</em> (2021-01) <a href="https://doi.org/gj58gg">https://doi.org/gj58gg</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ajhg.2020.11.012">10.1016/j.ajhg.2020.11.012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33308443">33308443</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7820626">PMC7820626</a></p>
</div>
<div id="ref-AxVJwanp">
<p>65. <strong>Transcriptome-wide association study of schizophrenia and chromatin activity yields mechanistic disease insights</strong> <br />
Alexander Gusev, Nicholas Mancuso, Hyejung Won, Maria Kousi, Hilary K. Finucane, Yakir Reshef, Lingyun Song, Alexias Safi, Steven McCarroll, Benjamin M. Neale, … Schizophrenia Working Group of the Psychiatric Genomics Consortium<br />
<em>Nature Genetics</em> (2018-04-09) <a href="https://doi.org/gdfdf2">https://doi.org/gdfdf2</a> <br />
DOI: <a href="https://doi.org/10.1038/s41588-018-0092-1">10.1038/s41588-018-0092-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29632383">29632383</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5942893">PMC5942893</a></p>
</div>
<div id="ref-REXpV7nA">
<p>66. <strong>The support of human genetic evidence for approved drug indications</strong> <br />
Matthew R Nelson, Hannah Tipney, Jeffery L Painter, Judong Shen, Paola Nicoletti, Yufeng Shen, Aris Floratos, Pak Chung Sham, Mulin Jun Li, Junwen Wang, … Philippe Sanseau<br />
<em>Nature Genetics</em> (2015-06-29) <a href="https://doi.org/f3mn52">https://doi.org/f3mn52</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.3314">10.1038/ng.3314</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26121088">26121088</a></p>
</div>
<div id="ref-LyJmyoQr">
<p>67. <strong>Shared genetic origin of asthma, hay fever and eczema elucidates allergic disease biology</strong> <br />
Manuel A Ferreira, Judith M Vonk, Hansjörg Baurecht, Ingo Marenholz, Chao Tian, Joshua D Hoffman, Quinta Helmer, Annika Tillander, Vilhelmina Ullemar, Jenny van Dongen, … LifeLines Cohort Study<br />
<em>Nature Genetics</em> (2017-10-30) <a href="https://doi.org/gchg62">https://doi.org/gchg62</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.3985">10.1038/ng.3985</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29083406">29083406</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5989923">PMC5989923</a></p>
</div>
<div id="ref-veADXImD">
<p>68. <strong>A genome-wide cross-trait analysis from UK Biobank highlights the shared genetic architecture of asthma and allergic diseases</strong> <br />
Zhaozhong Zhu, Phil H. Lee, Mark D. Chaffin, Wonil Chung, Po-Ru Loh, Quan Lu, David C. Christiani, Liming Liang<br />
<em>Nature Genetics</em> (2018-05-21) <a href="https://doi.org/gdpmtn">https://doi.org/gdpmtn</a> <br />
DOI: <a href="https://doi.org/10.1038/s41588-018-0121-0">10.1038/s41588-018-0121-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29785011">29785011</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5980765">PMC5980765</a></p>
</div>
<div id="ref-fnDaLjFy">
<p>69. <strong>Systematic tissue annotations of –omics samples by modeling unstructured metadata</strong> <br />
Nathaniel T. Hawkins, Marc Maldaver, Anna Yannakopoulos, Lindsay A. Guare, Arjun Krishnan<br />
<em>Cold Spring Harbor Laboratory</em> (2021-05-20) <a href="https://doi.org/gj2pkc">https://doi.org/gj2pkc</a> <br />
DOI: <a href="https://doi.org/10.1101/2021.05.10.443525">10.1101/2021.05.10.443525</a></p>
</div>
<div id="ref-vLyTudUB">
<p>70. <strong>Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics</strong> <br />
Alvaro N. Barbeira, Scott P. Dickinson, Rodrigo Bonazzola, Jiamao Zheng, Heather E. Wheeler, Jason M. Torres, Eric S. Torstenson, Kaanan P. Shah, Tzintzuni Garcia, Todd L. Edwards, … GTEx Consortium<br />
<em>Nature Communications</em> (2018-05-08) <a href="https://doi.org/gdjvp5">https://doi.org/gdjvp5</a> <br />
DOI: <a href="https://doi.org/10.1038/s41467-018-03621-1">10.1038/s41467-018-03621-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29739930">29739930</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940825">PMC5940825</a></p>
</div>
<div id="ref-wyKoQS8N">
<p>71. <strong>Probabilistic Colocalization of Genetic Variants from Complex and Molecular Traits: Promise and Limitations</strong> <br />
Abhay Hukku, Milton Pividori, Francesca Luca, Roger Pique-Regi, Hae Kyung Im, Xiaoquan Wen<br />
<em>Cold Spring Harbor Laboratory</em> (2020-07-01) <a href="https://doi.org/gjwh3p">https://doi.org/gjwh3p</a> <br />
DOI: <a href="https://doi.org/10.1101/2020.07.01.182097">10.1101/2020.07.01.182097</a></p>
</div>
<div id="ref-1CbVoEpNJ">
<p>72. <strong>The Molecular Signatures Database Hallmark Gene Set Collection</strong> <br />
Arthur Liberzon, Chet Birger, Helga Thorvaldsdóttir, Mahmoud Ghandi, Jill P. Mesirov, Pablo Tamayo<br />
<em>Cell Systems</em> (2015-12) <a href="https://doi.org/gf78hq">https://doi.org/gf78hq</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cels.2015.12.004">10.1016/j.cels.2015.12.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26771021">26771021</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4707969">PMC4707969</a></p>
</div>
<div id="ref-11PJIeJyi">
<p>73. <strong>A global reference for human genetic variation</strong> <br />
The 1000 Genomes Project Consortium<br />
<em>Nature</em> (2015-09-30) <a href="https://doi.org/73d">https://doi.org/73d</a> <br />
DOI: <a href="https://doi.org/10.1038/nature15393">10.1038/nature15393</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26432245">26432245</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4750478">PMC4750478</a></p>
</div>
<div id="ref-1BWx1HXgP">
<p>74. <strong>GWAS summary statistics imputation support data and integration with PrediXcan MASHR</strong> <br />
Alvaro Numa Barbeira, Hae Kyung Im<br />
<em>Zenodo</em> (2019-12-10) <a href="https://doi.org/gj6jwq">https://doi.org/gj6jwq</a> <br />
DOI: <a href="https://doi.org/10.5281/zenodo.3657902">10.5281/zenodo.3657902</a></p>
</div>
<div id="ref-1FsruosUW">
<p>75. <strong>Human Disease Ontology 2018 update: classification, content and workflow expansion</strong> <br />
Lynn M Schriml, Elvira Mitraka, James Munro, Becky Tauber, Mike Schor, Lance Nickle, Victor Felix, Linda Jeng, Cynthia Bearer, Richard Lichenstein, … Carol Greene<br />
<em>Nucleic Acids Research</em> (2019-01-08) <a href="https://doi.org/ggx9wp">https://doi.org/ggx9wp</a> <br />
DOI: <a href="https://doi.org/10.1093/nar/gky1032">10.1093/nar/gky1032</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30407550">30407550</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323977">PMC6323977</a></p>
</div>
<div id="ref-9okjVu3s">
<p>76. <strong>Modeling sample variables with an Experimental Factor Ontology</strong> <br />
James Malone, Ele Holloway, Tomasz Adamusiak, Misha Kapushesky, Jie Zheng, Nikolay Kolesnikov, Anna Zhukova, Alvis Brazma, Helen Parkinson<br />
<em>Bioinformatics</em> (2010-04-15) <a href="https://doi.org/dsb6vt">https://doi.org/dsb6vt</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btq099">10.1093/bioinformatics/btq099</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20200009">20200009</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2853691">PMC2853691</a></p>
</div>
<div id="ref-16RTdMKxI">
<p>77. <strong>EBISPOT/EFO-UKB-mappings</strong> <br />
EBISPOT<br />
(2021-03-15) <a href="https://github.com/EBISPOT/EFO-UKB-mappings">https://github.com/EBISPOT/EFO-UKB-mappings</a></p>
</div>
<div id="ref-e4AuoW8N">
<p>78. <strong>Comparing partitions</strong> <br />
Lawrence Hubert, Phipps Arabie<br />
<em>Journal of Classification</em> (1985-12) <a href="https://doi.org/bphmzh">https://doi.org/bphmzh</a> <br />
DOI: <a href="https://doi.org/10.1007/bf01908075">10.1007/bf01908075</a></p>
</div>
<div id="ref-cuROQDFa">
<p>79. <strong>Combining multiple clusterings using evidence accumulation</strong> <br />
Ana L. N. Fred, Anil K. Jain<br />
<em>IEEE Transactions on Pattern Analysis and Machine Intelligence</em> (2005-06) <a href="https://doi.org/bsknv6">https://doi.org/bsknv6</a> <br />
DOI: <a href="https://doi.org/10.1109/tpami.2005.113">10.1109/tpami.2005.113</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15943417">15943417</a></p>
</div>
<div id="ref-rcTMvL18">
<p>80. <strong>A Link-Based Approach to the Cluster Ensemble Problem</strong> <br />
Natthakan Iam-On, Tossapon Boongoen, Simon Garrett, Chris Price<br />
<em>IEEE Transactions on Pattern Analysis and Machine Intelligence</em> (2011-12) <a href="https://doi.org/cqgkh3">https://doi.org/cqgkh3</a> <br />
DOI: <a href="https://doi.org/10.1109/tpami.2011.84">10.1109/tpami.2011.84</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21576752">21576752</a></p>
</div>
<div id="ref-t5p3UpxZ">
<p>81. <strong>Hybrid clustering solution selection strategy</strong> <br />
Zhiwen Yu, Le Li, Yunjun Gao, Jane You, Jiming Liu, Hau-San Wong, Guoqiang Han<br />
<em>Pattern Recognition</em> (2014-10) <a href="https://doi.org/ghtzwt">https://doi.org/ghtzwt</a> <br />
DOI: <a href="https://doi.org/10.1016/j.patcog.2014.04.005">10.1016/j.patcog.2014.04.005</a></p>
</div>
<div id="ref-157h5hA34">
<p>82. <strong>UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction</strong> <br />
Leland McInnes, John Healy, James Melville<br />
<em>arXiv</em> (2020-09-21) <a href="https://arxiv.org/abs/1802.03426">https://arxiv.org/abs/1802.03426</a></p>
</div>
<div id="ref-MxGpAiPu">
<p>83. <strong>k-means++: the advantages of careful seeding</strong> <br />
David Arthur, Sergei Vassilvitskii<br />
<em>Proceedings of the Eighteenth Annual ACM-SIAM Symposium on Discrete Algorithms</em> (2007) <a href="http://ilpubs.stanford.edu:8090/778/1/2006-13.pdf">http://ilpubs.stanford.edu:8090/778/1/2006-13.pdf</a></p>
</div>
<div id="ref-x3CT24TB">
<p>84. <strong>On Spectral Clustering: Analysis and an algorithm</strong> <br />
Andrew Ng, Michael Jordan, Yair Weiss<br />
<em>Advances in Neural Information Processing Systems</em> (2001) <a href="https://ai.stanford.edu/~ang/papers/nips01-spectral.pdf">https://ai.stanford.edu/~ang/papers/nips01-spectral.pdf</a></p>
</div>
<div id="ref-JrL3iQea">
<p>85. <strong>A Density-Based Algorithm for Discovering Clusters in Large Spatial Databases with Noise</strong> <br />
Martin Ester, Hans-Peter Kriegel, Jörg Sander, Xiaowei Xu<br />
<em>Proceedings of the Second International Conference on Knowledge Discovery and Data Mining</em> (1996) <a href="https://www.aaai.org/Papers/KDD/1996/KDD96-037.pdf">https://www.aaai.org/Papers/KDD/1996/KDD96-037.pdf</a></p>
</div>
<div id="ref-3oFmSTW2">
<p>86. <strong>Human pluripotent stem cell-derived neural constructs for predicting neural toxicity</strong> <br />
Michael P. Schwartz, Zhonggang Hou, Nicholas E. Propson, Jue Zhang, Collin J. Engstrom, Vitor Santos Costa, Peng Jiang, Bao Kim Nguyen, Jennifer M. Bolin, William Daly, … James A. Thomson<br />
<em>Proceedings of the National Academy of Sciences</em> (2015-10-06) <a href="https://doi.org/f7vpzd">https://doi.org/f7vpzd</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1516645112">10.1073/pnas.1516645112</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26392547">26392547</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4603492">PMC4603492</a></p>
</div>
<div id="ref-VKYS05n1">
<p>87. <strong>Homo sapiens (ID 232177) - BioProject - NCBI</strong> <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA232177">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA232177</a></p>
</div>
</div>
<h2 id="supplementary-material">Supplementary material</h2>
<h3 id="gene-modules-enriched-for-lipids-gene-sets">Gene modules enriched for lipids gene-sets</h3>
<!-- lipids_gene_sets:modules_enriched:start -->
<div id="tbl:sup:lipids_crispr:modules_enriched" class="tablenos">
<table>
<caption><span>Table 1:</span> Gene modules (LVs) enriched for lipids gene-sets found with CRISPR screening. 
<!-- lipids_gene_sets:modules_enriched:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Gene module</th>
<th style="text-align: left;">Lipids gene-set</th>
<th style="text-align: left;">p-value</th>
<th style="text-align: left;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">LV678</td>
<td style="text-align: left;">decrease</td>
<td style="text-align: left;">2.61e-07</td>
<td style="text-align: left;">2.57e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">LV707</td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;">1.74e-07</td>
<td style="text-align: left;">2.57e-04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">LV905</td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;">4.29e-05</td>
<td style="text-align: left;">2.82e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">LV915</td>
<td style="text-align: left;">increase</td>
<td style="text-align: left;">6.37e-05</td>
<td style="text-align: left;">3.14e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="top-traits-across-lipids-associated-genes-and-modules">Top traits across lipids-associated genes and modules</h3>
<!-- single_genes:lipids_decreasing:start -->
<div id="tbl:sup:single_genes:lipids_decreasing:top_traits" class="tablenos">
<table>
<caption><span>Table 2:</span> Top 25 traits associated with genes from the lipids-decreasing gene-set found with CRISPR screening. 
<!-- single_genes:lipids_decreasing:end --></caption>
<colgroup>
<col style="width: 6%" />
<col style="width: 75%" />
<col style="width: 18%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Order</th>
<th style="text-align: left;">Trait</th>
<th style="text-align: left;">Category</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1</td>
<td style="text-align: left;">Vascular/heart problems diagnosed by doctor: High blood pressure</td>
<td style="text-align: left;">Diseases (cardiovascular)</td>
</tr>
<tr class="even">
<td style="text-align: left;">2</td>
<td style="text-align: left;">Diastolic blood pressure, automated reading</td>
<td style="text-align: left;">Blood pressure</td>
</tr>
<tr class="odd">
<td style="text-align: left;">3</td>
<td style="text-align: left;">Non-cancer illness code, self-reported: hypertension</td>
<td style="text-align: left;">Diseases (cardiovascular)</td>
</tr>
<tr class="even">
<td style="text-align: left;">4</td>
<td style="text-align: left;">Suggestive for eosinophilic asthma</td>
<td style="text-align: left;">Diseases (FinnGen)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">5</td>
<td style="text-align: left;">Medication for cholesterol, blood pressure, diabetes, or take exogenous hormones: Blood pressure medication</td>
<td style="text-align: left;">Medication</td>
</tr>
<tr class="even">
<td style="text-align: left;">6</td>
<td style="text-align: left;">Forced expiratory volume in 1-second (FEV1), predicted</td>
<td style="text-align: left;">Spirometry</td>
</tr>
<tr class="odd">
<td style="text-align: left;">7</td>
<td style="text-align: left;">Vascular/heart problems diagnosed by doctor: None of the above</td>
<td style="text-align: left;">Diseases (cardiovascular)</td>
</tr>
<tr class="even">
<td style="text-align: left;">8</td>
<td style="text-align: left;">Treatment/medication code: levothyroxine sodium</td>
<td style="text-align: left;">Medications</td>
</tr>
<tr class="odd">
<td style="text-align: left;">9</td>
<td style="text-align: left;">Haematocrit percentage</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">10</td>
<td style="text-align: left;">Treatment/medication code: lisinopril</td>
<td style="text-align: left;">Medications</td>
</tr>
<tr class="odd">
<td style="text-align: left;">11</td>
<td style="text-align: left;">Haemoglobin concentration</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">12</td>
<td style="text-align: left;">Job coding: counter clerk, bank clerk, cashier, post office clerk</td>
<td style="text-align: left;">Employment history</td>
</tr>
<tr class="odd">
<td style="text-align: left;">13</td>
<td style="text-align: left;">Acute alcohol intoxication</td>
<td style="text-align: left;">Diseases (FinnGen)</td>
</tr>
<tr class="even">
<td style="text-align: left;">14</td>
<td style="text-align: left;">Systolic blood pressure, automated reading</td>
<td style="text-align: left;">Blood pressure</td>
</tr>
<tr class="odd">
<td style="text-align: left;">15</td>
<td style="text-align: left;">Platelet count</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">16</td>
<td style="text-align: left;">Red Blood Cell Count</td>
<td style="text-align: left;">Blood</td>
</tr>
<tr class="odd">
<td style="text-align: left;">17</td>
<td style="text-align: left;">Peak expiratory flow (PEF)</td>
<td style="text-align: left;">Spirometry</td>
</tr>
<tr class="even">
<td style="text-align: left;">18</td>
<td style="text-align: left;">Sitting height</td>
<td style="text-align: left;">Body size measures</td>
</tr>
<tr class="odd">
<td style="text-align: left;">19</td>
<td style="text-align: left;">Treatment/medication code: bendroflumethiazide</td>
<td style="text-align: left;">Medications</td>
</tr>
<tr class="even">
<td style="text-align: left;">20</td>
<td style="text-align: left;">Age started wearing glasses or contact lenses</td>
<td style="text-align: left;">Eyesight</td>
</tr>
<tr class="odd">
<td style="text-align: left;">21</td>
<td style="text-align: left;">Comparative height size at age 10</td>
<td style="text-align: left;">Early life factors</td>
</tr>
<tr class="even">
<td style="text-align: left;">22</td>
<td style="text-align: left;">Workplace very cold: Often</td>
<td style="text-align: left;">Employment history</td>
</tr>
<tr class="odd">
<td style="text-align: left;">23</td>
<td style="text-align: left;">Salt added to food</td>
<td style="text-align: left;">Diet</td>
</tr>
<tr class="even">
<td style="text-align: left;">24</td>
<td style="text-align: left;">Difficulty concentrating during worst period of anxiety</td>
<td style="text-align: left;">Anxiety</td>
</tr>
<tr class="odd">
<td style="text-align: left;">25</td>
<td style="text-align: left;">Treatment/medication code: metformin</td>
<td style="text-align: left;">Medications</td>
</tr>
</tbody>
</table>
</div>
<!-- modules:lipids_decreasing:start -->
<div id="tbl:sup:modules:lipids_decreasing:top_traits" class="tablenos">
<table>
<caption><span>Table 3:</span> Top 25 traits associated with gene modules (LVs) enriched for the lipids-decreasing gene-set found with CRISPR screening. 
<!-- modules:lipids_decreasing:end --></caption>
<colgroup>
<col style="width: 7%" />
<col style="width: 59%" />
<col style="width: 32%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Order</th>
<th style="text-align: left;">Trait</th>
<th style="text-align: left;">Category</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1</td>
<td style="text-align: left;">Non-cancer illness code, self-reported: malabsorption/coeliac disease</td>
<td style="text-align: left;">Diseases (gastrointestinal/abdominal)</td>
</tr>
<tr class="even">
<td style="text-align: left;">2</td>
<td style="text-align: left;">Diastolic blood pressure, automated reading</td>
<td style="text-align: left;">Blood pressure</td>
</tr>
<tr class="odd">
<td style="text-align: left;">3</td>
<td style="text-align: left;">Immature reticulocyte fraction</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">4</td>
<td style="text-align: left;">Treatment/medication code: ferrous salt product</td>
<td style="text-align: left;">Medications</td>
</tr>
<tr class="odd">
<td style="text-align: left;">5</td>
<td style="text-align: left;">Vascular/heart problems diagnosed by doctor: None of the above</td>
<td style="text-align: left;">Diseases (cardiovascular)</td>
</tr>
<tr class="even">
<td style="text-align: left;">6</td>
<td style="text-align: left;">Platelet distribution width</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="odd">
<td style="text-align: left;">7</td>
<td style="text-align: left;">Unstable angina pectoris</td>
<td style="text-align: left;">Diseases (FinnGen)</td>
</tr>
<tr class="even">
<td style="text-align: left;">8</td>
<td style="text-align: left;">Vascular/heart problems diagnosed by doctor: High blood pressure</td>
<td style="text-align: left;">Diseases (cardiovascular)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">9</td>
<td style="text-align: left;">Nucleated red blood cell count</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">10</td>
<td style="text-align: left;">Diagnoses - main ICD10: K90 Intestinal malabsorption</td>
<td style="text-align: left;">Diseases (ICD10 main)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">11</td>
<td style="text-align: left;">Coeliac disease</td>
<td style="text-align: left;">Diseases (FinnGen)</td>
</tr>
<tr class="even">
<td style="text-align: left;">12</td>
<td style="text-align: left;">Non-cancer illness code, self-reported: hypertension</td>
<td style="text-align: left;">Diseases (cardiovascular)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">13</td>
<td style="text-align: left;">Nucleated red blood cell percentage</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">14</td>
<td style="text-align: left;">Relative age of first facial hair</td>
<td style="text-align: left;">Male-specific factors</td>
</tr>
<tr class="odd">
<td style="text-align: left;">15</td>
<td style="text-align: left;">Treatment/medication code: thiamine preparation</td>
<td style="text-align: left;">Medications</td>
</tr>
<tr class="even">
<td style="text-align: left;">16</td>
<td style="text-align: left;">Diagnoses - main ICD10: I70 Atherosclerosis</td>
<td style="text-align: left;">Diseases (ICD10 main)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">17</td>
<td style="text-align: left;">White Blood Cell Count</td>
<td style="text-align: left;">Blood</td>
</tr>
<tr class="even">
<td style="text-align: left;">18</td>
<td style="text-align: left;">Treatment/medication code: gtn 400micrograms spray</td>
<td style="text-align: left;">Medications</td>
</tr>
<tr class="odd">
<td style="text-align: left;">19</td>
<td style="text-align: left;">Treatment/medication code: singulair 10mg tablet</td>
<td style="text-align: left;">Medications</td>
</tr>
<tr class="even">
<td style="text-align: left;">20</td>
<td style="text-align: left;">Difficulty not smoking for 1 day</td>
<td style="text-align: left;">Smoking</td>
</tr>
<tr class="odd">
<td style="text-align: left;">21</td>
<td style="text-align: left;">Mean reticulocyte volume</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">22</td>
<td style="text-align: left;">Other malignant neoplasms of skin</td>
<td style="text-align: left;">Diseases (FinnGen)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">23</td>
<td style="text-align: left;">Length of working week for main job</td>
<td style="text-align: left;">Employment</td>
</tr>
<tr class="even">
<td style="text-align: left;">24</td>
<td style="text-align: left;">Pulse rate, automated reading</td>
<td style="text-align: left;">Blood pressure</td>
</tr>
<tr class="odd">
<td style="text-align: left;">25</td>
<td style="text-align: left;">Milk type used: Skimmed</td>
<td style="text-align: left;">Diet</td>
</tr>
</tbody>
</table>
</div>
<!-- single_genes:lipids_increasing:start -->
<div id="tbl:sup:single_genes:lipids_increasing:top_traits" class="tablenos">
<table>
<caption><span>Table 4:</span> Top 25 traits associated with genes from the lipids-increasing gene-set found with CRISPR screening. 
<!-- single_genes:lipids_increasing:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Order</th>
<th style="text-align: left;">Trait</th>
<th style="text-align: left;">Category</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1</td>
<td style="text-align: left;">Lymphocyte percentage</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">2</td>
<td style="text-align: left;">Neutrophill percentage</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="odd">
<td style="text-align: left;">3</td>
<td style="text-align: left;">Neutrophill count</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">4</td>
<td style="text-align: left;">Red blood cell (erythrocyte) count</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="odd">
<td style="text-align: left;">5</td>
<td style="text-align: left;">Lymphocyte Count</td>
<td style="text-align: left;">Blood</td>
</tr>
<tr class="even">
<td style="text-align: left;">6</td>
<td style="text-align: left;">Trunk predicted mass</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="odd">
<td style="text-align: left;">7</td>
<td style="text-align: left;">Trunk fat-free mass</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="even">
<td style="text-align: left;">8</td>
<td style="text-align: left;">Mean corpuscular volume</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="odd">
<td style="text-align: left;">9</td>
<td style="text-align: left;">Mean sphered cell volume</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">10</td>
<td style="text-align: left;">White blood cell (leukocyte) count</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="odd">
<td style="text-align: left;">11</td>
<td style="text-align: left;">Skin colour</td>
<td style="text-align: left;">Sun exposure</td>
</tr>
<tr class="even">
<td style="text-align: left;">12</td>
<td style="text-align: left;">Arm fat-free mass (left)</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="odd">
<td style="text-align: left;">13</td>
<td style="text-align: left;">Impedance of arm (left)</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="even">
<td style="text-align: left;">14</td>
<td style="text-align: left;">Mean reticulocyte volume</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="odd">
<td style="text-align: left;">15</td>
<td style="text-align: left;">Whole body water mass</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="even">
<td style="text-align: left;">16</td>
<td style="text-align: left;">Impedance of arm (right)</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="odd">
<td style="text-align: left;">17</td>
<td style="text-align: left;">Mean corpuscular haemoglobin</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">18</td>
<td style="text-align: left;">Whole body fat-free mass</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="odd">
<td style="text-align: left;">19</td>
<td style="text-align: left;">Arm predicted mass (left)</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="even">
<td style="text-align: left;">20</td>
<td style="text-align: left;">Arm predicted mass (right)</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="odd">
<td style="text-align: left;">21</td>
<td style="text-align: left;">Arm fat-free mass (right)</td>
<td style="text-align: left;">Impedance measures</td>
</tr>
<tr class="even">
<td style="text-align: left;">22</td>
<td style="text-align: left;">Hair colour (natural, before greying): Red</td>
<td style="text-align: left;">Sun exposure</td>
</tr>
<tr class="odd">
<td style="text-align: left;">23</td>
<td style="text-align: left;">Ease of skin tanning</td>
<td style="text-align: left;">Sun exposure</td>
</tr>
<tr class="even">
<td style="text-align: left;">24</td>
<td style="text-align: left;">High light scatter reticulocyte count</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="odd">
<td style="text-align: left;">25</td>
<td style="text-align: left;">White Blood Cell Count</td>
<td style="text-align: left;">Blood</td>
</tr>
</tbody>
</table>
</div>
<!-- modules:lipids_increasing:start -->
<div id="tbl:sup:modules:lipids_increasing:top_traits" class="tablenos">
<table>
<caption><span>Table 5:</span> Top 25 traits associated with gene modules (LVs) enriched for the lipids-increasing gene-set found with CRISPR screening. 
<!-- modules:lipids_increasing:end --></caption>
<colgroup>
<col style="width: 5%" />
<col style="width: 69%" />
<col style="width: 24%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Order</th>
<th style="text-align: left;">Trait</th>
<th style="text-align: left;">Category</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1</td>
<td style="text-align: left;">Ankle spacing width</td>
<td style="text-align: left;">Bone-densitometry of heel</td>
</tr>
<tr class="even">
<td style="text-align: left;">2</td>
<td style="text-align: left;">Ankle spacing width (left)</td>
<td style="text-align: left;">Bone-densitometry of heel</td>
</tr>
<tr class="odd">
<td style="text-align: left;">3</td>
<td style="text-align: left;">Ankle spacing width (right)</td>
<td style="text-align: left;">Bone-densitometry of heel</td>
</tr>
<tr class="even">
<td style="text-align: left;">4</td>
<td style="text-align: left;">Job SOC coding: Advertising and public relations managers</td>
<td style="text-align: left;">Employment history</td>
</tr>
<tr class="odd">
<td style="text-align: left;">5</td>
<td style="text-align: left;">Hair colour (natural, before greying): Red</td>
<td style="text-align: left;">Sun exposure</td>
</tr>
<tr class="even">
<td style="text-align: left;">6</td>
<td style="text-align: left;">Sitting height</td>
<td style="text-align: left;">Body size measures</td>
</tr>
<tr class="odd">
<td style="text-align: left;">7</td>
<td style="text-align: left;">Platelet distribution width</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">8</td>
<td style="text-align: left;">Non-cancer illness code, self-reported: malabsorption/coeliac disease</td>
<td style="text-align: left;">Diseases (gastrointestinal/abdominal)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">9</td>
<td style="text-align: left;">Job coding: advertising or public relations manager, media/publicity manager, campaign/fundraising manager</td>
<td style="text-align: left;">Employment history</td>
</tr>
<tr class="even">
<td style="text-align: left;">10</td>
<td style="text-align: left;">Forced expiratory volume in 1-second (FEV1), predicted</td>
<td style="text-align: left;">Spirometry</td>
</tr>
<tr class="odd">
<td style="text-align: left;">11</td>
<td style="text-align: left;">Heel Broadband ultrasound attenuation, direct entry</td>
<td style="text-align: left;">Bone-densitometry of heel</td>
</tr>
<tr class="even">
<td style="text-align: left;">12</td>
<td style="text-align: left;">Intra-ocular pressure, Goldmann-correlated (right)</td>
<td style="text-align: left;">Intraocular pressure</td>
</tr>
<tr class="odd">
<td style="text-align: left;">13</td>
<td style="text-align: left;">Hearing test done: No, I am unable to do this</td>
<td style="text-align: left;">Hearing test</td>
</tr>
<tr class="even">
<td style="text-align: left;">14</td>
<td style="text-align: left;">Rheumatoid Arthritis</td>
<td style="text-align: left;">Diseases (ICD10 main)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">15</td>
<td style="text-align: left;">Red blood cell (erythrocyte) distribution width</td>
<td style="text-align: left;">Blood count</td>
</tr>
<tr class="even">
<td style="text-align: left;">16</td>
<td style="text-align: left;">Job coding: childminder, au pair, children’s nanny</td>
<td style="text-align: left;">Employment history</td>
</tr>
<tr class="odd">
<td style="text-align: left;">17</td>
<td style="text-align: left;">Heel bone mineral density (BMD)</td>
<td style="text-align: left;">Bone-densitometry of heel</td>
</tr>
<tr class="even">
<td style="text-align: left;">18</td>
<td style="text-align: left;">Heel quantitative ultrasound index (QUI), direct entry</td>
<td style="text-align: left;">Bone-densitometry of heel</td>
</tr>
<tr class="odd">
<td style="text-align: left;">19</td>
<td style="text-align: left;">Heel bone mineral density (BMD) T-score, automated</td>
<td style="text-align: left;">Bone-densitometry of heel</td>
</tr>
<tr class="even">
<td style="text-align: left;">20</td>
<td style="text-align: left;">Job SOC coding: Hand craft occupations n.e.c.</td>
<td style="text-align: left;">Employment history</td>
</tr>
<tr class="odd">
<td style="text-align: left;">21</td>
<td style="text-align: left;">Reason for glasses/contact lenses: For just reading/near work as you are getting older (called ‘presbyopia’)</td>
<td style="text-align: left;">Eyesight</td>
</tr>
<tr class="even">
<td style="text-align: left;">22</td>
<td style="text-align: left;">Intra-ocular pressure, Goldmann-correlated (left)</td>
<td style="text-align: left;">Intraocular pressure</td>
</tr>
<tr class="odd">
<td style="text-align: left;">23</td>
<td style="text-align: left;">Pulse wave peak to peak time</td>
<td style="text-align: left;">Arterial stiffness</td>
</tr>
<tr class="even">
<td style="text-align: left;">24</td>
<td style="text-align: left;">Hand grip strength (left)</td>
<td style="text-align: left;">Hand grip strength</td>
</tr>
<tr class="odd">
<td style="text-align: left;">25</td>
<td style="text-align: left;">Treatment/medication code: luteine</td>
<td style="text-align: left;">Medications</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv246">LV246</h3>
<div id="fig:sup:lv246" class="fignos">
<figure>
<img src="images/lvs_analysis/lv246/lv246-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 7:</span> <strong>Cell types for LV246.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV246:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv246" class="tablenos">
<table>
<caption><span>Table 6:</span> Pathways aligned to LV246. 
<!-- LV246:multiplier_pathways:end --></caption>
<colgroup>
<col style="width: 68%" />
<col style="width: 7%" />
<col style="width: 23%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">REACTOME_FATTY_ACID_TRIACYLGLYCEROL_AND_KETONE_BODY_METABOLISM</td>
<td style="text-align: left;">0.89</td>
<td style="text-align: left;">3.97e-16</td>
</tr>
<tr class="even">
<td style="text-align: left;">REACTOME_METABOLISM_OF_LIPIDS_AND_LIPOPROTEINS</td>
<td style="text-align: left;">0.67</td>
<td style="text-align: left;">1.14e-08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">REACTOME_TRIGLYCERIDE_BIOSYNTHESIS</td>
<td style="text-align: left;">0.86</td>
<td style="text-align: left;">6.52e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">KEGG_PYRUVATE_METABOLISM</td>
<td style="text-align: left;">0.82</td>
<td style="text-align: left;">2.66e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">KEGG_PROPANOATE_METABOLISM</td>
<td style="text-align: left;">0.83</td>
<td style="text-align: left;">4.27e-03</td>
</tr>
</tbody>
</table>
</div>
<!-- LV246:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv246" class="tablenos">
<table>
<caption><span>Table 7:</span> Significant trait associations of LV246 in PhenomeXcan. 
<!-- LV246:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 60%" />
<col style="width: 7%" />
<col style="width: 4%" />
<col style="width: 14%" />
<col style="width: 11%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">CH2DB NMR</td>
<td style="text-align: left;">24,154</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">9.36e-11</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-cancer illness code, self-reported: high cholesterol</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">43,957</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">5.24e-05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Medication for cholesterol, blood pressure, diabetes, or take exogenous hormones: Cholesterol lowering medication</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">24,247</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">9.34e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">HDL Cholesterol NMR</td>
<td style="text-align: left;">19,270</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">9.34e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Fasting Glucose</td>
<td style="text-align: left;">46,186</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 11</td>
<td style="text-align: left;">4.13e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV246:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv246" class="tablenos">
<table>
<caption><span>Table 8:</span> Significant trait associations of LV246 in eMERGE. 
<!-- LV246:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th>Phecode</th>
<th>Trait description</th>
<th>Sample size</th>
<th>Cases</th>
<th>p-value (adjusted)</th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<h3 id="lv678">LV678</h3>
<div id="fig:sup:lv678" class="fignos">
<figure>
<img src="images/lvs_analysis/lv678/lv678-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 8:</span> <strong>Cell types for LV678.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV678:multiplier_pathways:start -->
<div id="tbl:sup:multiplier_pathways:lv678" class="tablenos">
<table>
<caption><span>Table 9:</span> Pathways aligned to LV678. 
<!-- LV678:multiplier_pathways:end --></caption>
<colgroup>
<col style="width: 81%" />
<col style="width: 4%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Pathway</th>
<th style="text-align: left;">AUC</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">KEGG_OXIDATIVE_PHOSPHORYLATION</td>
<td style="text-align: left;">0.98</td>
<td style="text-align: left;">5.75e-14</td>
</tr>
<tr class="even">
<td style="text-align: left;">REACTOME_RESPIRATORY_ELECTRON_TRANSPORT_ATP_SYNTHESIS_BY_CHEMIOSMOTIC_COUPLING_AND_HEAT_PRODUCTION_BY_UNCOUPLING_PROTEINS_</td>
<td style="text-align: left;">0.99</td>
<td style="text-align: left;">5.94e-11</td>
</tr>
<tr class="odd">
<td style="text-align: left;">REACTOME_RESPIRATORY_ELECTRON_TRANSPORT</td>
<td style="text-align: left;">1.00</td>
<td style="text-align: left;">3.10e-09</td>
</tr>
<tr class="even">
<td style="text-align: left;">REACTOME_TCA_CYCLE_AND_RESPIRATORY_ELECTRON_TRANSPORT</td>
<td style="text-align: left;">0.86</td>
<td style="text-align: left;">9.66e-09</td>
</tr>
<tr class="odd">
<td style="text-align: left;">MIPS_55S_RIBOSOME_MITOCHONDRIAL</td>
<td style="text-align: left;">0.81</td>
<td style="text-align: left;">8.20e-05</td>
</tr>
<tr class="even">
<td style="text-align: left;">REACTOME_SRP_DEPENDENT_COTRANSLATIONAL_PROTEIN_TARGETING_TO_MEMBRANE</td>
<td style="text-align: left;">0.69</td>
<td style="text-align: left;">6.03e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">REACTOME_MITOCHONDRIAL_PROTEIN_IMPORT</td>
<td style="text-align: left;">0.74</td>
<td style="text-align: left;">1.99e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV678:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv678" class="tablenos">
<table>
<caption><span>Table 10:</span> Significant trait associations of LV678 in PhenomeXcan. 
<!-- LV678:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 50%" />
<col style="width: 10%" />
<col style="width: 6%" />
<col style="width: 18%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Vascular/heart problems diagnosed by doctor: Heart attack</td>
<td style="text-align: left;">360,420</td>
<td style="text-align: left;">8,288</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">1.08e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Inflammatory Bowel Disease</td>
<td style="text-align: left;">34,652</td>
<td style="text-align: left;">12,882</td>
<td style="text-align: left;">29 / 21</td>
<td style="text-align: left;">2.35e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: heart attack/myocardial infarction</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">8,239</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">2.35e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV678:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv678" class="tablenos">
<table>
<caption><span>Table 11:</span> Significant trait associations of LV678 in eMERGE. 
<!-- LV678:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th>Phecode</th>
<th>Trait description</th>
<th>Sample size</th>
<th>Cases</th>
<th>p-value (adjusted)</th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<h3 id="lv116">LV116</h3>
<div id="fig:sup:lv116" class="fignos">
<figure>
<img src="images/lvs_analysis/lv116/lv116-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 9:</span> <strong>Cell types for LV116.</strong>
<!--  --></figcaption>
</figure>
</div>
<h3 id="lv931">LV931</h3>
<div id="fig:sup:lv931" class="fignos">
<figure>
<img src="images/lvs_analysis/lv931/lv931-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 10:</span> <strong>Cell types for LV931.</strong>
<!--  --></figcaption>
</figure>
</div>
<h3 id="lv66">LV66</h3>
<div id="fig:sup:lv66" class="fignos">
<figure>
<img src="images/lvs_analysis/lv66/lv66-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 11:</span> <strong>Cell types for LV66.</strong>
<!--  --></figcaption>
</figure>
</div>
<h3 id="lv928">LV928</h3>
<div id="fig:sup:lv928" class="fignos">
<figure>
<img src="images/lvs_analysis/lv928/lv928-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 12:</span> <strong>Cell types for LV928.</strong>
<!--  --></figcaption>
</figure>
</div>
<h3 id="lv30">LV30</h3>
<div id="fig:sup:lv30" class="fignos">
<figure>
<img src="images/lvs_analysis/lv30/lv30-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 13:</span> <strong>Cell types for LV30.</strong>
<!--  --></figcaption>
</figure>
</div>
<h3 id="lv730">LV730</h3>
<div id="fig:sup:lv730" class="fignos">
<figure>
<img src="images/lvs_analysis/lv730/lv730-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 14:</span> <strong>Cell types for LV730.</strong>
<!--  --></figcaption>
</figure>
</div>
<h3 id="lv598">LV598</h3>
<div id="fig:sup:lv598" class="fignos">
<figure>
<img src="images/lvs_analysis/lv598/lv598-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 15:</span> <strong>Cell types for LV598.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV598:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv598" class="tablenos">
<table>
<caption><span>Table 12:</span> Significant trait associations of LV598 in PhenomeXcan. 
<!-- LV598:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 28%" />
<col style="width: 14%" />
<col style="width: 8%" />
<col style="width: 27%" />
<col style="width: 21%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">6mm strong meridian (right)</td>
<td style="text-align: left;">66,256</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">4.13e-07</td>
</tr>
<tr class="even">
<td style="text-align: left;">6mm weak meridian (right)</td>
<td style="text-align: left;">66,256</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">2.63e-06</td>
</tr>
<tr class="odd">
<td style="text-align: left;">6mm strong meridian (left)</td>
<td style="text-align: left;">65,551</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">3.13e-06</td>
</tr>
<tr class="even">
<td style="text-align: left;">3mm strong meridian (left)</td>
<td style="text-align: left;">75,398</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">3.24e-06</td>
</tr>
<tr class="odd">
<td style="text-align: left;">6mm weak meridian (left)</td>
<td style="text-align: left;">65,551</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">1.53e-05</td>
</tr>
<tr class="even">
<td style="text-align: left;">3mm weak meridian (left)</td>
<td style="text-align: left;">75,398</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">2.00e-05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">3mm strong meridian (right)</td>
<td style="text-align: left;">75,410</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">3.70e-05</td>
</tr>
<tr class="even">
<td style="text-align: left;">3mm weak meridian (right)</td>
<td style="text-align: left;">75,410</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">4.81e-05</td>
</tr>
</tbody>
</table>
</div>
<!-- DISABLED LV598:emerge_traits_assocs:start -->
<!-- DISABLED LV598:emerge_traits_assocs:end -->
<h3 id="lv155">LV155</h3>
<div id="fig:sup:lv155" class="fignos">
<figure>
<img src="images/lvs_analysis/lv155/lv155-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 16:</span> <strong>Cell types for LV155.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV155:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv155" class="tablenos">
<table>
<caption><span>Table 13:</span> Significant trait associations of LV155 in PhenomeXcan. 
<!-- LV155:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 49%" />
<col style="width: 10%" />
<col style="width: 6%" />
<col style="width: 19%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: hypothyroidism/myxoedema</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">17,574</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">2.01e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-cancer illness code, self-reported: hyperthyroidism/thyrotoxicosis</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">2,730</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">1.29e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Treatment/medication code: levothyroxine sodium</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">14,689</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">1.41e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV155:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv155" class="tablenos">
<table>
<caption><span>Table 14:</span> Significant trait associations of LV155 in eMERGE. 
<!-- LV155:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">244.2</td>
<td style="text-align: left;">Acquired hypothyroidism</td>
<td style="text-align: left;">45,839</td>
<td style="text-align: left;">1,155</td>
<td style="text-align: left;">2.19e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">427.9</td>
<td style="text-align: left;">Palpitations</td>
<td style="text-align: left;">35,214</td>
<td style="text-align: left;">6,092</td>
<td style="text-align: left;">4.43e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv844">LV844</h3>
<div id="fig:sup:lv844" class="fignos">
<figure>
<img src="images/lvs_analysis/lv844/lv844-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 17:</span> <strong>Cell types for LV844.</strong>
<!--  -->
<code class="red">Note</code> “Cell types from brain” come from <span class="citation" data-cites="3oFmSTW2">[<a href="#ref-3oFmSTW2" role="doc-biblioref">86</a>]</span>, treated with different chemichals; I don’t have enough information to separate cell types.
<!--  --></figcaption>
</figure>
</div>
<!-- LV844:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv844" class="tablenos">
<table>
<caption><span>Table 15:</span> Significant trait associations of LV844 in PhenomeXcan. 
<!-- LV844:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 55%" />
<col style="width: 9%" />
<col style="width: 5%" />
<col style="width: 16%" />
<col style="width: 13%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Rheumatoid Arthritis</td>
<td style="text-align: left;">80,799</td>
<td style="text-align: left;">19,234</td>
<td style="text-align: left;">29 / 26</td>
<td style="text-align: left;">4.27e-57</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-cancer illness code, self-reported: malabsorption/coeliac disease</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">1,587</td>
<td style="text-align: left;">29 / 8</td>
<td style="text-align: left;">4.83e-43</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Coeliac disease</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">842</td>
<td style="text-align: left;">29 / 8</td>
<td style="text-align: left;">4.76e-41</td>
</tr>
<tr class="even">
<td style="text-align: left;">Diagnoses - main ICD10: K90 Intestinal malabsorption</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">922</td>
<td style="text-align: left;">29 / 8</td>
<td style="text-align: left;">1.41e-39</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Started insulin within one year diagnosis of diabetes</td>
<td style="text-align: left;">16,415</td>
<td style="text-align: left;">1,999</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">1.78e-37</td>
</tr>
<tr class="even">
<td style="text-align: left;">Systemic Lupus Erythematosus</td>
<td style="text-align: left;">23,210</td>
<td style="text-align: left;">7,219</td>
<td style="text-align: left;">29 / 26</td>
<td style="text-align: left;">1.41e-34</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Age diabetes diagnosed</td>
<td style="text-align: left;">16,166</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">3.93e-34</td>
</tr>
<tr class="even">
<td style="text-align: left;">Never eat eggs, dairy, wheat, sugar: Wheat products</td>
<td style="text-align: left;">359,777</td>
<td style="text-align: left;">9,573</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">2.78e-31</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: hyperthyroidism/thyrotoxicosis</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">2,730</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">6.08e-30</td>
</tr>
<tr class="even">
<td style="text-align: left;">Treatment/medication code: insulin product</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">3,545</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">3.05e-25</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Medication for cholesterol, blood pressure, diabetes, or take exogenous hormones: Insulin</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">1,476</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">4.63e-23</td>
</tr>
<tr class="even">
<td style="text-align: left;">Medication for cholesterol, blood pressure or diabetes: Insulin</td>
<td style="text-align: left;">165,340</td>
<td style="text-align: left;">2,248</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">1.92e-20</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: hypothyroidism/myxoedema</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">17,574</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">4.96e-20</td>
</tr>
<tr class="even">
<td style="text-align: left;">Treatment/medication code: levothyroxine sodium</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">14,689</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">4.01e-19</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: psoriasis</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">4,192</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">9.28e-16</td>
</tr>
</tbody>
</table>
</div>
<!-- LV844:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv844" class="tablenos">
<table>
<caption><span>Table 16:</span> Significant trait associations of LV844 in eMERGE. 
<!-- LV844:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 9%" />
<col style="width: 52%" />
<col style="width: 12%" />
<col style="width: 7%" />
<col style="width: 18%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">714.1</td>
<td style="text-align: left;">Rheumatoid arthritis</td>
<td style="text-align: left;">49,453</td>
<td style="text-align: left;">2,541</td>
<td style="text-align: left;">8.22e-09</td>
</tr>
<tr class="even">
<td style="text-align: left;">250.1</td>
<td style="text-align: left;">Type 1 diabetes</td>
<td style="text-align: left;">42,723</td>
<td style="text-align: left;">2,450</td>
<td style="text-align: left;">2.54e-08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">714</td>
<td style="text-align: left;">Rheumatoid arthritis and other inflammatory polyarthropathies</td>
<td style="text-align: left;">50,215</td>
<td style="text-align: left;">3,303</td>
<td style="text-align: left;">5.06e-07</td>
</tr>
<tr class="even">
<td style="text-align: left;">440</td>
<td style="text-align: left;">Atherosclerosis</td>
<td style="text-align: left;">47,471</td>
<td style="text-align: left;">4,993</td>
<td style="text-align: left;">3.15e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">578.8</td>
<td style="text-align: left;">Hemorrhage of rectum and anus</td>
<td style="text-align: left;">47,545</td>
<td style="text-align: left;">1,991</td>
<td style="text-align: left;">3.15e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">585.32</td>
<td style="text-align: left;">End stage renal disease</td>
<td style="text-align: left;">43,309</td>
<td style="text-align: left;">1,842</td>
<td style="text-align: left;">4.38e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">440.2</td>
<td style="text-align: left;">Atherosclerosis of the extremities</td>
<td style="text-align: left;">45,524</td>
<td style="text-align: left;">3,046</td>
<td style="text-align: left;">5.00e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">514.2</td>
<td style="text-align: left;">Solitary pulmonary nodule</td>
<td style="text-align: left;">50,389</td>
<td style="text-align: left;">2,270</td>
<td style="text-align: left;">6.16e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">444</td>
<td style="text-align: left;">Arterial embolism and thrombosis</td>
<td style="text-align: left;">43,378</td>
<td style="text-align: left;">900</td>
<td style="text-align: left;">1.36e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">558</td>
<td style="text-align: left;">Noninfectious gastroenteritis</td>
<td style="text-align: left;">40,177</td>
<td style="text-align: left;">3,191</td>
<td style="text-align: left;">2.94e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">747.11</td>
<td style="text-align: left;">Cardiac shunt/ heart septal defect</td>
<td style="text-align: left;">58,364</td>
<td style="text-align: left;">1,037</td>
<td style="text-align: left;">3.60e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">585</td>
<td style="text-align: left;">Renal failure</td>
<td style="text-align: left;">51,437</td>
<td style="text-align: left;">9,970</td>
<td style="text-align: left;">3.87e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">443.9</td>
<td style="text-align: left;">Peripheral vascular disease, unspecified</td>
<td style="text-align: left;">46,926</td>
<td style="text-align: left;">4,448</td>
<td style="text-align: left;">4.43e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv57">LV57</h3>
<div id="fig:sup:lv57" class="fignos">
<figure>
<img src="images/lvs_analysis/lv57/lv57-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 18:</span> <strong>Cell types for LV57.</strong>
<!--  -->
<!--  --></figcaption>
</figure>
</div>
<!-- LV57:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv57" class="tablenos">
<table>
<caption><span>Table 17:</span> Significant trait associations of LV57 in PhenomeXcan. 
<!-- LV57:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 55%" />
<col style="width: 9%" />
<col style="width: 5%" />
<col style="width: 16%" />
<col style="width: 13%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: hypothyroidism/myxoedema</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">17,574</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">1.17e-24</td>
</tr>
<tr class="even">
<td style="text-align: left;">Treatment/medication code: levothyroxine sodium</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">14,689</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">6.07e-23</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: hyperthyroidism/thyrotoxicosis</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">2,730</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">1.16e-06</td>
</tr>
<tr class="even">
<td style="text-align: left;">Started insulin within one year diagnosis of diabetes</td>
<td style="text-align: left;">16,415</td>
<td style="text-align: left;">1,999</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">8.17e-05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Treatment/medication code: insulin product</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">3,545</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">6.33e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Medication for cholesterol, blood pressure, diabetes, or take exogenous hormones: Insulin</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">1,476</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">1.13e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Medication for cholesterol, blood pressure or diabetes: Insulin</td>
<td style="text-align: left;">165,340</td>
<td style="text-align: left;">2,248</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">4.50e-03</td>
</tr>
</tbody>
</table>
</div>
<!-- LV57:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv57" class="tablenos">
<table>
<caption><span>Table 18:</span> Significant trait associations of LV57 in eMERGE. 
<!-- LV57:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 9%" />
<col style="width: 52%" />
<col style="width: 12%" />
<col style="width: 7%" />
<col style="width: 18%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">244</td>
<td style="text-align: left;">Hypothyroidism</td>
<td style="text-align: left;">54,404</td>
<td style="text-align: left;">9,720</td>
<td style="text-align: left;">3.97e-09</td>
</tr>
<tr class="even">
<td style="text-align: left;">244.4</td>
<td style="text-align: left;">Hypothyroidism NOS</td>
<td style="text-align: left;">53,968</td>
<td style="text-align: left;">9,284</td>
<td style="text-align: left;">3.97e-09</td>
</tr>
<tr class="odd">
<td style="text-align: left;">279</td>
<td style="text-align: left;">Disorders involving the immune mechanism</td>
<td style="text-align: left;">56,771</td>
<td style="text-align: left;">3,309</td>
<td style="text-align: left;">4.93e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">514.2</td>
<td style="text-align: left;">Solitary pulmonary nodule</td>
<td style="text-align: left;">50,389</td>
<td style="text-align: left;">2,270</td>
<td style="text-align: left;">1.19e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">714</td>
<td style="text-align: left;">Rheumatoid arthritis and other inflammatory polyarthropathies</td>
<td style="text-align: left;">50,215</td>
<td style="text-align: left;">3,303</td>
<td style="text-align: left;">1.68e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">452.2</td>
<td style="text-align: left;">Deep vein thrombosis [DVT]</td>
<td style="text-align: left;">38,791</td>
<td style="text-align: left;">2,131</td>
<td style="text-align: left;">4.37e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv54">LV54</h3>
<div id="fig:sup:lv54" class="fignos">
<figure>
<img src="images/lvs_analysis/lv54/lv54-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 19:</span> <strong>Cell types for LV54.</strong>
<!--  -->
<!--  --></figcaption>
</figure>
</div>
<!-- LV54:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv54" class="tablenos">
<table>
<caption><span>Table 19:</span> Significant trait associations of LV54 in PhenomeXcan. 
<!-- LV54:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 55%" />
<col style="width: 9%" />
<col style="width: 5%" />
<col style="width: 16%" />
<col style="width: 13%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Diagnoses - main ICD10: K90 Intestinal malabsorption</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">922</td>
<td style="text-align: left;">29 / 8</td>
<td style="text-align: left;">5.09e-25</td>
</tr>
<tr class="even">
<td style="text-align: left;">Coeliac disease</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">842</td>
<td style="text-align: left;">29 / 8</td>
<td style="text-align: left;">7.77e-25</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Never eat eggs, dairy, wheat, sugar: Wheat products</td>
<td style="text-align: left;">359,777</td>
<td style="text-align: left;">9,573</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">6.33e-23</td>
</tr>
<tr class="even">
<td style="text-align: left;">Systemic Lupus Erythematosus</td>
<td style="text-align: left;">23,210</td>
<td style="text-align: left;">7,219</td>
<td style="text-align: left;">29 / 26</td>
<td style="text-align: left;">1.32e-22</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Started insulin within one year diagnosis of diabetes</td>
<td style="text-align: left;">16,415</td>
<td style="text-align: left;">1,999</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">3.84e-20</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-cancer illness code, self-reported: hyperthyroidism/thyrotoxicosis</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">2,730</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">9.59e-19</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Treatment/medication code: insulin product</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">3,545</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">5.07e-18</td>
</tr>
<tr class="even">
<td style="text-align: left;">Age diabetes diagnosed</td>
<td style="text-align: left;">16,166</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">1.28e-17</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: malabsorption/coeliac disease</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">1,587</td>
<td style="text-align: left;">29 / 8</td>
<td style="text-align: left;">1.36e-14</td>
</tr>
<tr class="even">
<td style="text-align: left;">Medication for cholesterol, blood pressure or diabetes: Insulin</td>
<td style="text-align: left;">165,340</td>
<td style="text-align: left;">2,248</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">8.67e-14</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: psoriasis</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">4,192</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">2.61e-13</td>
</tr>
<tr class="even">
<td style="text-align: left;">Rheumatoid Arthritis</td>
<td style="text-align: left;">80,799</td>
<td style="text-align: left;">19,234</td>
<td style="text-align: left;">29 / 26</td>
<td style="text-align: left;">3.11e-13</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Medication for cholesterol, blood pressure, diabetes, or take exogenous hormones: Insulin</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">1,476</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">3.89e-12</td>
</tr>
<tr class="even">
<td style="text-align: left;">Treatment/medication code: levothyroxine sodium</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">14,689</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">5.92e-10</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-cancer illness code, self-reported: hypothyroidism/myxoedema</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">17,574</td>
<td style="text-align: left;">29 / 13</td>
<td style="text-align: left;">3.31e-08</td>
</tr>
</tbody>
</table>
</div>
<!-- LV54:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv54" class="tablenos">
<table>
<caption><span>Table 20:</span> Significant trait associations of LV54 in eMERGE. 
<!-- LV54:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 9%" />
<col style="width: 52%" />
<col style="width: 12%" />
<col style="width: 7%" />
<col style="width: 18%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">250.1</td>
<td style="text-align: left;">Type 1 diabetes</td>
<td style="text-align: left;">42,723</td>
<td style="text-align: left;">2,450</td>
<td style="text-align: left;">2.04e-13</td>
</tr>
<tr class="even">
<td style="text-align: left;">244</td>
<td style="text-align: left;">Hypothyroidism</td>
<td style="text-align: left;">54,404</td>
<td style="text-align: left;">9,720</td>
<td style="text-align: left;">5.10e-06</td>
</tr>
<tr class="odd">
<td style="text-align: left;">244.4</td>
<td style="text-align: left;">Hypothyroidism NOS</td>
<td style="text-align: left;">53,968</td>
<td style="text-align: left;">9,284</td>
<td style="text-align: left;">5.37e-06</td>
</tr>
<tr class="even">
<td style="text-align: left;">695</td>
<td style="text-align: left;">Erythematous conditions</td>
<td style="text-align: left;">48,347</td>
<td style="text-align: left;">4,210</td>
<td style="text-align: left;">4.25e-05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">714</td>
<td style="text-align: left;">Rheumatoid arthritis and other inflammatory polyarthropathies</td>
<td style="text-align: left;">50,215</td>
<td style="text-align: left;">3,303</td>
<td style="text-align: left;">3.06e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">440</td>
<td style="text-align: left;">Atherosclerosis</td>
<td style="text-align: left;">47,471</td>
<td style="text-align: left;">4,993</td>
<td style="text-align: left;">8.88e-04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">585</td>
<td style="text-align: left;">Renal failure</td>
<td style="text-align: left;">51,437</td>
<td style="text-align: left;">9,970</td>
<td style="text-align: left;">3.40e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">585.32</td>
<td style="text-align: left;">End stage renal disease</td>
<td style="text-align: left;">43,309</td>
<td style="text-align: left;">1,842</td>
<td style="text-align: left;">3.64e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">585.33</td>
<td style="text-align: left;">Chronic Kidney Disease, Stage III</td>
<td style="text-align: left;">46,279</td>
<td style="text-align: left;">4,812</td>
<td style="text-align: left;">3.64e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">285.2</td>
<td style="text-align: left;">Anemia of chronic disease</td>
<td style="text-align: left;">39,673</td>
<td style="text-align: left;">2,606</td>
<td style="text-align: left;">7.62e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">415.1</td>
<td style="text-align: left;">Acute pulmonary heart disease</td>
<td style="text-align: left;">49,887</td>
<td style="text-align: left;">1,857</td>
<td style="text-align: left;">8.67e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">285.21</td>
<td style="text-align: left;">Anemia in chronic kidney disease</td>
<td style="text-align: left;">38,616</td>
<td style="text-align: left;">1,549</td>
<td style="text-align: left;">1.16e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">743</td>
<td style="text-align: left;">Osteoporosis, osteopenia and pathological fracture</td>
<td style="text-align: left;">55,165</td>
<td style="text-align: left;">11,990</td>
<td style="text-align: left;">1.31e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">415.11</td>
<td style="text-align: left;">Pulmonary embolism and infarction, acute</td>
<td style="text-align: left;">49,867</td>
<td style="text-align: left;">1,837</td>
<td style="text-align: left;">1.39e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">577</td>
<td style="text-align: left;">Diseases of pancreas</td>
<td style="text-align: left;">60,538</td>
<td style="text-align: left;">1,795</td>
<td style="text-align: left;">1.42e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">585.1</td>
<td style="text-align: left;">Acute renal failure</td>
<td style="text-align: left;">46,803</td>
<td style="text-align: left;">5,336</td>
<td style="text-align: left;">1.51e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">195</td>
<td style="text-align: left;">Cancer, suspected or other</td>
<td style="text-align: left;">50,040</td>
<td style="text-align: left;">2,250</td>
<td style="text-align: left;">1.52e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">440.2</td>
<td style="text-align: left;">Atherosclerosis of the extremities</td>
<td style="text-align: left;">45,524</td>
<td style="text-align: left;">3,046</td>
<td style="text-align: left;">1.89e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">714.1</td>
<td style="text-align: left;">Rheumatoid arthritis</td>
<td style="text-align: left;">49,453</td>
<td style="text-align: left;">2,541</td>
<td style="text-align: left;">3.18e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">458.9</td>
<td style="text-align: left;">Hypotension NOS</td>
<td style="text-align: left;">50,150</td>
<td style="text-align: left;">3,241</td>
<td style="text-align: left;">3.32e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv847">LV847</h3>
<div id="fig:sup:lv847" class="fignos">
<figure>
<img src="images/lvs_analysis/lv847/lv847-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 20:</span> <strong>Cell types for LV847.</strong>
<!--  -->
<!--  --></figcaption>
</figure>
</div>
<!-- LV847:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv847" class="tablenos">
<table style="width:100%;">
<caption><span>Table 21:</span> Significant trait associations of LV847 in PhenomeXcan. 
<!-- LV847:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 59%" />
<col style="width: 8%" />
<col style="width: 4%" />
<col style="width: 15%" />
<col style="width: 12%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Medication for cholesterol, blood pressure, diabetes, or take exogenous hormones: Blood pressure medication</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">33,519</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">1.95e-18</td>
</tr>
<tr class="even">
<td style="text-align: left;">Vascular/heart problems diagnosed by doctor: None of the above</td>
<td style="text-align: left;">360,420</td>
<td style="text-align: left;">253,565</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">4.07e-15</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Vascular/heart problems diagnosed by doctor: High blood pressure</td>
<td style="text-align: left;">360,420</td>
<td style="text-align: left;">97,139</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">6.99e-14</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-cancer illness code, self-reported: hypertension</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">93,560</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">1.48e-13</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Treatment/medication code: bendroflumethiazide</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">20,196</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">1.41e-08</td>
</tr>
<tr class="even">
<td style="text-align: left;">Medication for cholesterol, blood pressure or diabetes: Blood pressure medication</td>
<td style="text-align: left;">165,340</td>
<td style="text-align: left;">40,987</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">1.47e-07</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Medication for cholesterol, blood pressure, diabetes, or take exogenous hormones: None of the above</td>
<td style="text-align: left;">193,148</td>
<td style="text-align: left;">133,338</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">1.55e-06</td>
</tr>
<tr class="even">
<td style="text-align: left;">Diastolic blood pressure, automated reading</td>
<td style="text-align: left;">340,162</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">3.76e-06</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Medication for cholesterol, blood pressure or diabetes: None of the above</td>
<td style="text-align: left;">165,340</td>
<td style="text-align: left;">110,372</td>
<td style="text-align: left;">29 / 17</td>
<td style="text-align: left;">6.36e-06</td>
</tr>
</tbody>
</table>
</div>
<!-- LV847:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv847" class="tablenos">
<table>
<caption><span>Table 22:</span> Significant trait associations of LV847 in eMERGE. 
<!-- LV847:emerge_traits_assocs:end --></caption>
<colgroup>
<col style="width: 9%" />
<col style="width: 52%" />
<col style="width: 12%" />
<col style="width: 7%" />
<col style="width: 18%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">585.32</td>
<td style="text-align: left;">End stage renal disease</td>
<td style="text-align: left;">43,309</td>
<td style="text-align: left;">1,842</td>
<td style="text-align: left;">1.88e-08</td>
</tr>
<tr class="even">
<td style="text-align: left;">442.1</td>
<td style="text-align: left;">Aortic aneurysm</td>
<td style="text-align: left;">45,589</td>
<td style="text-align: left;">3,111</td>
<td style="text-align: left;">5.23e-06</td>
</tr>
<tr class="odd">
<td style="text-align: left;">411.3</td>
<td style="text-align: left;">Angina pectoris</td>
<td style="text-align: left;">43,503</td>
<td style="text-align: left;">4,382</td>
<td style="text-align: left;">2.14e-05</td>
</tr>
<tr class="even">
<td style="text-align: left;">415.11</td>
<td style="text-align: left;">Pulmonary embolism and infarction, acute</td>
<td style="text-align: left;">49,867</td>
<td style="text-align: left;">1,837</td>
<td style="text-align: left;">5.13e-05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">416</td>
<td style="text-align: left;">Cardiomegaly</td>
<td style="text-align: left;">53,289</td>
<td style="text-align: left;">5,259</td>
<td style="text-align: left;">6.50e-05</td>
</tr>
<tr class="even">
<td style="text-align: left;">415.1</td>
<td style="text-align: left;">Acute pulmonary heart disease</td>
<td style="text-align: left;">49,887</td>
<td style="text-align: left;">1,857</td>
<td style="text-align: left;">7.28e-05</td>
</tr>
<tr class="odd">
<td style="text-align: left;">411</td>
<td style="text-align: left;">Ischemic Heart Disease</td>
<td style="text-align: left;">54,275</td>
<td style="text-align: left;">15,154</td>
<td style="text-align: left;">5.49e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">401.2</td>
<td style="text-align: left;">Hypertensive heart and/or renal disease</td>
<td style="text-align: left;">30,405</td>
<td style="text-align: left;">6,253</td>
<td style="text-align: left;">1.28e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">519</td>
<td style="text-align: left;">Other diseases of respiratory system, not elsewhere classified</td>
<td style="text-align: left;">56,909</td>
<td style="text-align: left;">2,056</td>
<td style="text-align: left;">1.28e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">411.8</td>
<td style="text-align: left;">Other chronic ischemic heart disease, unspecified</td>
<td style="text-align: left;">44,123</td>
<td style="text-align: left;">5,002</td>
<td style="text-align: left;">1.42e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">427.6</td>
<td style="text-align: left;">Premature beats</td>
<td style="text-align: left;">31,575</td>
<td style="text-align: left;">2,453</td>
<td style="text-align: left;">5.65e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">687.1</td>
<td style="text-align: left;">Rash and other nonspecific skin eruption</td>
<td style="text-align: left;">47,039</td>
<td style="text-align: left;">4,964</td>
<td style="text-align: left;">9.88e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">185</td>
<td style="text-align: left;">Cancer of prostate</td>
<td style="text-align: left;">52,630</td>
<td style="text-align: left;">2,815</td>
<td style="text-align: left;">1.03e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">591</td>
<td style="text-align: left;">Urinary tract infection</td>
<td style="text-align: left;">49,727</td>
<td style="text-align: left;">10,016</td>
<td style="text-align: left;">1.34e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">442.11</td>
<td style="text-align: left;">Abdominal aortic aneurysm</td>
<td style="text-align: left;">44,531</td>
<td style="text-align: left;">2,053</td>
<td style="text-align: left;">2.08e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">427.21</td>
<td style="text-align: left;">Atrial fibrillation</td>
<td style="text-align: left;">37,743</td>
<td style="text-align: left;">8,621</td>
<td style="text-align: left;">2.26e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">389.1</td>
<td style="text-align: left;">Sensorineural hearing loss</td>
<td style="text-align: left;">53,672</td>
<td style="text-align: left;">4,318</td>
<td style="text-align: left;">2.73e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">427.2</td>
<td style="text-align: left;">Atrial fibrillation and flutter</td>
<td style="text-align: left;">37,934</td>
<td style="text-align: left;">8,812</td>
<td style="text-align: left;">4.50e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv136">LV136</h3>
<div id="fig:sup:lv136" class="fignos">
<figure>
<img src="images/lvs_analysis/lv136/lv136-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 21:</span> <strong>Cell types for LV136.</strong>
<!--  -->
Pulmonary microvascular endothelial cells were exposed to hypoxia for 24 hours or more <span class="citation" data-cites="VKYS05n1">[<a href="#ref-VKYS05n1" role="doc-biblioref">87</a>]</span>;</figcaption>
</figure>
</div>
<!-- LV136:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv136" class="tablenos">
<table>
<caption><span>Table 23:</span> Significant trait associations of LV136 in PhenomeXcan. 
<!-- LV136:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 50%" />
<col style="width: 10%" />
<col style="width: 6%" />
<col style="width: 18%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">3mm strong meridian (right)</td>
<td style="text-align: left;">75,410</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">9.19e-11</td>
</tr>
<tr class="even">
<td style="text-align: left;">6mm strong meridian (left)</td>
<td style="text-align: left;">65,551</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">2.06e-09</td>
</tr>
<tr class="odd">
<td style="text-align: left;">6mm strong meridian (right)</td>
<td style="text-align: left;">66,256</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">2.38e-09</td>
</tr>
<tr class="even">
<td style="text-align: left;">3mm strong meridian (left)</td>
<td style="text-align: left;">75,398</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">1.34e-08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">3mm weak meridian (right)</td>
<td style="text-align: left;">75,410</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">1.67e-08</td>
</tr>
<tr class="even">
<td style="text-align: left;">Coronary Artery Disease</td>
<td style="text-align: left;">184,305</td>
<td style="text-align: left;">60,801</td>
<td style="text-align: left;">29 / 11</td>
<td style="text-align: left;">1.67e-08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">6mm weak meridian (right)</td>
<td style="text-align: left;">66,256</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">3.21e-08</td>
</tr>
<tr class="even">
<td style="text-align: left;">3mm weak meridian (left)</td>
<td style="text-align: left;">75,398</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">5.20e-08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">6mm weak meridian (left)</td>
<td style="text-align: left;">65,551</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 10</td>
<td style="text-align: left;">1.21e-07</td>
</tr>
<tr class="even">
<td style="text-align: left;">Coronary atherosclerosis</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">14,334</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">3.90e-06</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Ischaemic heart disease, wide definition</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">20,857</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">7.22e-06</td>
</tr>
<tr class="even">
<td style="text-align: left;">Vascular/heart problems diagnosed by doctor: Heart attack</td>
<td style="text-align: left;">360,420</td>
<td style="text-align: left;">8,288</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">2.93e-04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Myocardial infarction</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">7,018</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">6.33e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Myocardial infarction, strict</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">7,018</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">6.33e-04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Diagnoses - main ICD10: I21 Acute myocardial infarction</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">5,948</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">9.92e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-cancer illness code, self-reported: heart attack/myocardial infarction</td>
<td style="text-align: left;">361,141</td>
<td style="text-align: left;">8,239</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">1.40e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Major coronary heart disease event excluding revascularizations</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">10,157</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">1.85e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Major coronary heart disease event</td>
<td style="text-align: left;">361,194</td>
<td style="text-align: left;">10,157</td>
<td style="text-align: left;">29 / 14</td>
<td style="text-align: left;">1.85e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Fasting Insulin</td>
<td style="text-align: left;">38,238</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 11</td>
<td style="text-align: left;">3.85e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV136:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv136" class="tablenos">
<table>
<caption><span>Table 24:</span> Significant trait associations of LV136 in eMERGE. 
<!-- LV136:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">747.1</td>
<td style="text-align: left;">Cardiac congenital anomalies</td>
<td style="text-align: left;">59,198</td>
<td style="text-align: left;">1,871</td>
<td style="text-align: left;">4.71e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">411.4</td>
<td style="text-align: left;">Coronary atherosclerosis</td>
<td style="text-align: left;">52,836</td>
<td style="text-align: left;">13,715</td>
<td style="text-align: left;">4.80e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv93">LV93</h3>
<div id="fig:sup:lv93" class="fignos">
<figure>
<img src="images/lvs_analysis/lv93/lv93-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 22:</span> <strong>Cell types for LV93.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV93:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv93" class="tablenos">
<table>
<caption><span>Table 25:</span> Significant trait associations of LV93 in PhenomeXcan. 
<!-- LV93:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 22%" />
<col style="width: 15%" />
<col style="width: 9%" />
<col style="width: 29%" />
<col style="width: 23%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">CH2DB NMR</td>
<td style="text-align: left;">24,154</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">9.61e-24</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chronotype</td>
<td style="text-align: left;">128,266</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">1.17e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">HDL Cholesterol NMR</td>
<td style="text-align: left;">19,270</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">2.99e-03</td>
</tr>
</tbody>
</table>
</div>
<!-- LV93:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv93" class="tablenos">
<table>
<caption><span>Table 26:</span> Significant trait associations of LV93 in eMERGE. 
<!-- LV93:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">208</td>
<td style="text-align: left;">Benign neoplasm of colon</td>
<td style="text-align: left;">55,694</td>
<td style="text-align: left;">8,597</td>
<td style="text-align: left;">6.21e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">440.2</td>
<td style="text-align: left;">Atherosclerosis of the extremities</td>
<td style="text-align: left;">45,524</td>
<td style="text-align: left;">3,046</td>
<td style="text-align: left;">1.31e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">444</td>
<td style="text-align: left;">Arterial embolism and thrombosis</td>
<td style="text-align: left;">43,378</td>
<td style="text-align: left;">900</td>
<td style="text-align: left;">4.06e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv206">LV206</h3>
<div id="fig:sup:lv206" class="fignos">
<figure>
<img src="images/lvs_analysis/lv206/lv206-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 23:</span> <strong>Cell types for LV206.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV206:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv206" class="tablenos">
<table>
<caption><span>Table 27:</span> Significant trait associations of LV206 in PhenomeXcan. 
<!-- LV206:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 22%" />
<col style="width: 15%" />
<col style="width: 9%" />
<col style="width: 29%" />
<col style="width: 23%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">CH2DB NMR</td>
<td style="text-align: left;">24,154</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">7.67e-21</td>
</tr>
<tr class="even">
<td style="text-align: left;">HDL Cholesterol NMR</td>
<td style="text-align: left;">19,270</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">6.46e-03</td>
</tr>
</tbody>
</table>
</div>
<!-- LV206:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv206" class="tablenos">
<table>
<caption><span>Table 28:</span> Significant trait associations of LV206 in eMERGE. 
<!-- LV206:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">458</td>
<td style="text-align: left;">Hypotension</td>
<td style="text-align: left;">51,341</td>
<td style="text-align: left;">4,432</td>
<td style="text-align: left;">1.41e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">286.9</td>
<td style="text-align: left;">Abnormal coagulation profile</td>
<td style="text-align: left;">48,006</td>
<td style="text-align: left;">800</td>
<td style="text-align: left;">1.54e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">458.9</td>
<td style="text-align: left;">Hypotension NOS</td>
<td style="text-align: left;">50,150</td>
<td style="text-align: left;">3,241</td>
<td style="text-align: left;">1.58e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">428.2</td>
<td style="text-align: left;">Heart failure NOS</td>
<td style="text-align: left;">48,178</td>
<td style="text-align: left;">3,584</td>
<td style="text-align: left;">1.65e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv260">LV260</h3>
<div id="fig:sup:lv260" class="fignos">
<figure>
<img src="images/lvs_analysis/lv260/lv260-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 24:</span> <strong>Cell types for LV260.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV260:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv260" class="tablenos">
<table>
<caption><span>Table 29:</span> Significant trait associations of LV260 in PhenomeXcan. 
<!-- LV260:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 22%" />
<col style="width: 15%" />
<col style="width: 9%" />
<col style="width: 29%" />
<col style="width: 23%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">CH2DB NMR</td>
<td style="text-align: left;">24,154</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">5.96e-17</td>
</tr>
<tr class="even">
<td style="text-align: left;">HDL Cholesterol NMR</td>
<td style="text-align: left;">19,270</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">2.37e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV260:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv260" class="tablenos">
<table>
<caption><span>Table 30:</span> Significant trait associations of LV260 in eMERGE. 
<!-- LV260:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">427.6</td>
<td style="text-align: left;">Premature beats</td>
<td style="text-align: left;">31,575</td>
<td style="text-align: left;">2,453</td>
<td style="text-align: left;">2.85e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">426.3</td>
<td style="text-align: left;">Bundle branch block</td>
<td style="text-align: left;">31,827</td>
<td style="text-align: left;">2,705</td>
<td style="text-align: left;">4.80e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv21">LV21</h3>
<div id="fig:sup:lv21" class="fignos">
<figure>
<img src="images/lvs_analysis/lv21/lv21-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 25:</span> <strong>Cell types for LV21.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV21:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv21" class="tablenos">
<table>
<caption><span>Table 31:</span> Significant trait associations of LV21 in PhenomeXcan. 
<!-- LV21:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 22%" />
<col style="width: 15%" />
<col style="width: 9%" />
<col style="width: 29%" />
<col style="width: 23%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Alzheimers Disease</td>
<td style="text-align: left;">54,162</td>
<td style="text-align: left;">17,008</td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">1.64e-19</td>
</tr>
<tr class="even">
<td style="text-align: left;">LDL Cholesterol NMR</td>
<td style="text-align: left;">13,527</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">1.18e-04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Triglycerides NMR</td>
<td style="text-align: left;">21,559</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">2.19e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV21:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv21" class="tablenos">
<table>
<caption><span>Table 32:</span> Significant trait associations of LV21 in eMERGE. 
<!-- LV21:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">573</td>
<td style="text-align: left;">Other disorders of liver</td>
<td style="text-align: left;">47,826</td>
<td style="text-align: left;">2,524</td>
<td style="text-align: left;">1.37e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">577</td>
<td style="text-align: left;">Diseases of pancreas</td>
<td style="text-align: left;">60,538</td>
<td style="text-align: left;">1,795</td>
<td style="text-align: left;">2.15e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv5">LV5</h3>
<div id="fig:sup:lv5" class="fignos">
<figure>
<img src="images/lvs_analysis/lv5/lv5-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 26:</span> <strong>Cell types for LV5.</strong>
<!--  --></figcaption>
</figure>
</div>
<!-- LV5:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv5" class="tablenos">
<table>
<caption><span>Table 33:</span> Significant trait associations of LV5 in PhenomeXcan. 
<!-- LV5:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 54%" />
<col style="width: 9%" />
<col style="width: 5%" />
<col style="width: 17%" />
<col style="width: 13%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">LDL Cholesterol NMR</td>
<td style="text-align: left;">13,527</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">1.78e-04</td>
</tr>
<tr class="even">
<td style="text-align: left;">Triglycerides NMR</td>
<td style="text-align: left;">21,559</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">5.00e-04</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Alzheimers Disease</td>
<td style="text-align: left;">54,162</td>
<td style="text-align: left;">17,008</td>
<td style="text-align: left;">29 / 16</td>
<td style="text-align: left;">3.06e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">Ever had prolonged feelings of sadness or depression</td>
<td style="text-align: left;">117,763</td>
<td style="text-align: left;">64,374</td>
<td style="text-align: left;">29 / 27</td>
<td style="text-align: left;">8.69e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Substances taken for depression: Medication prescribed to you (for at least two weeks)</td>
<td style="text-align: left;">117,763</td>
<td style="text-align: left;">28,351</td>
<td style="text-align: left;">29 / 27</td>
<td style="text-align: left;">1.03e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Recent feelings of depression</td>
<td style="text-align: left;">117,656</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 27</td>
<td style="text-align: left;">1.32e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Ever contemplated self-harm</td>
<td style="text-align: left;">117,610</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 27</td>
<td style="text-align: left;">1.89e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Recent lack of interest or pleasure in doing things</td>
<td style="text-align: left;">117,757</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 27</td>
<td style="text-align: left;">2.08e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Amount of alcohol drunk on a typical drinking day</td>
<td style="text-align: left;">108,256</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 27</td>
<td style="text-align: left;">3.50e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">Ever sought or received professional help for mental distress</td>
<td style="text-align: left;">117,677</td>
<td style="text-align: left;">46,020</td>
<td style="text-align: left;">29 / 27</td>
<td style="text-align: left;">3.92e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">General happiness</td>
<td style="text-align: left;">117,442</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">29 / 27</td>
<td style="text-align: left;">4.74e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- LV5:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv5" class="tablenos">
<table>
<caption><span>Table 34:</span> Significant trait associations of LV5 in eMERGE. 
<!-- LV5:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">241</td>
<td style="text-align: left;">Nontoxic nodular goiter</td>
<td style="text-align: left;">47,842</td>
<td style="text-align: left;">3,158</td>
<td style="text-align: left;">8.98e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">241.1</td>
<td style="text-align: left;">Nontoxic uninodular goiter</td>
<td style="text-align: left;">47,125</td>
<td style="text-align: left;">2,441</td>
<td style="text-align: left;">2.57e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">241.2</td>
<td style="text-align: left;">Nontoxic multinodular goiter</td>
<td style="text-align: left;">46,465</td>
<td style="text-align: left;">1,781</td>
<td style="text-align: left;">4.43e-02</td>
</tr>
</tbody>
</table>
</div>
<h3 id="lv434">LV434</h3>
<div id="fig:sup:lv434" class="fignos">
<figure>
<img src="images/lvs_analysis/lv434/lv434-cell_types.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 27:</span> <strong>Cell types for LV434.</strong>
<!--  -->
<code class="red">FIXME:</code> create a section for all LVs and clarify how figures were generated, like:
The top samples in human breast cancer cells have larger LV values that were thresholded at 3.0 for visualization purposes.
<!--  -->
HEK293 is a cell line derived from human embryonic kidney cells;
3T3 is a cell line derived from mouse embryonic fibroblasts.</figcaption>
</figure>
</div>
<!-- LV434:phenomexcan_traits_assocs:start -->
<div id="tbl:sup:phenomexcan_assocs:lv434" class="tablenos">
<table>
<caption><span>Table 35:</span> Significant trait associations of LV434 in PhenomeXcan. 
<!-- LV434:phenomexcan_traits_assocs:end --></caption>
<colgroup>
<col style="width: 36%" />
<col style="width: 12%" />
<col style="width: 7%" />
<col style="width: 24%" />
<col style="width: 18%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">Partition/cluster number</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Attention Deficit Hyperactivity Disorder</td>
<td style="text-align: left;">53,293</td>
<td style="text-align: left;">19,099</td>
<td style="text-align: left;">29 / 21</td>
<td style="text-align: left;">7.01e-03</td>
</tr>
</tbody>
</table>
</div>
<!-- LV434:emerge_traits_assocs:start -->
<div id="tbl:sup:emerge_assocs:lv434" class="tablenos">
<table>
<caption><span>Table 36:</span> Significant trait associations of LV434 in eMERGE. 
<!-- LV434:emerge_traits_assocs:end --></caption>
<thead>
<tr class="header">
<th style="text-align: left;">Phecode</th>
<th style="text-align: left;">Trait description</th>
<th style="text-align: left;">Sample size</th>
<th style="text-align: left;">Cases</th>
<th style="text-align: left;">p-value (adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">722</td>
<td style="text-align: left;">Intervertebral disc disorders</td>
<td style="text-align: left;">47,659</td>
<td style="text-align: left;">7,458</td>
<td style="text-align: left;">6.65e-03</td>
</tr>
<tr class="even">
<td style="text-align: left;">721</td>
<td style="text-align: left;">Spondylosis and allied disorders</td>
<td style="text-align: left;">47,517</td>
<td style="text-align: left;">7,316</td>
<td style="text-align: left;">7.62e-03</td>
</tr>
<tr class="odd">
<td style="text-align: left;">250.4</td>
<td style="text-align: left;">Abnormal glucose</td>
<td style="text-align: left;">45,220</td>
<td style="text-align: left;">4,947</td>
<td style="text-align: left;">1.02e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">721.1</td>
<td style="text-align: left;">Spondylosis without myelopathy</td>
<td style="text-align: left;">47,315</td>
<td style="text-align: left;">7,114</td>
<td style="text-align: left;">1.22e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">720</td>
<td style="text-align: left;">Spinal stenosis</td>
<td style="text-align: left;">44,807</td>
<td style="text-align: left;">4,606</td>
<td style="text-align: left;">1.74e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">288</td>
<td style="text-align: left;">Diseases of white blood cells</td>
<td style="text-align: left;">47,288</td>
<td style="text-align: left;">2,802</td>
<td style="text-align: left;">2.10e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">796</td>
<td style="text-align: left;">Elevated prostate specific antigen [PSA]</td>
<td style="text-align: left;">51,990</td>
<td style="text-align: left;">2,175</td>
<td style="text-align: left;">3.09e-02</td>
</tr>
<tr class="even">
<td style="text-align: left;">288.2</td>
<td style="text-align: left;">Elevated white blood cell count</td>
<td style="text-align: left;">46,595</td>
<td style="text-align: left;">2,109</td>
<td style="text-align: left;">3.54e-02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">079</td>
<td style="text-align: left;">Viral infection</td>
<td style="text-align: left;">46,991</td>
<td style="text-align: left;">1,934</td>
<td style="text-align: left;">4.19e-02</td>
</tr>
</tbody>
</table>
</div>
<!-- ### Top gene modules in the prediction of Niacin as a therapeutic drug

| LV        | Value for Niacin      | Value for atherosclerosis   | Difference   |
|:----------|:----------------------|:----------------------------|:-------------|

Table: Top gene modules/LVs responsible for predicting Niacin as a therapeutic drug for atherosclerosis. {#tbl:sup:emerge_assocs:lv434} -->
<h3 id="agreement-of-consensus-clustering-partitions-with-the-ensemble-by-number-of-clusters">Agreement of consensus clustering partitions with the ensemble by number of clusters</h3>
<div id="fig:sup:consensus_agreement" class="fignos">
<figure>
<img src="images/clustering/selected_best_partitions_by_k.svg" style="width:80.0%" alt="" /><figcaption><span>Figure 28:</span> <strong>Final selected partitions for follow-up analysis.</strong>
<!--  -->
From all consensus clustering partitions generated with <span class="math inline">\(k\)</span> from 2 to 60, we selected those with a median adjusted Rand index (ARI) with the ensemble members greater the 75th percentile.</figcaption>
</figure>
</div>
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        font-family: "Open Sans", "Helvetica", sans-serif;

        /* prevent text inflation on some mobile browsers */
        -webkit-text-size-adjust: none !important;
        -moz-text-size-adjust: none !important;
        -o-text-size-adjust: none !important;
        text-size-adjust: none !important;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            font-size: 12pt;
            line-height: 1.5;
            max-width: 8.5in;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin: 40px 0;
        text-align: center;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* heading font sizes */
    h1 {
        font-size: 2em;
    }
    h2 {
        font-size: 1.5em;
    }
    h3{
        font-size: 1.35em;
    }
    h4 {
        font-size: 1.25em;
    }
    h5 {
        font-size: 1.15em;
    }
    h6 {
        font-size: 1em;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        text-align: center;
        margin-top: 10px;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
        overflow-wrap: break-word;
    }

    /* normal links (not empty, not button link, not syntax highlighting link) */
    a:not(:empty):not(.button):not(.sourceLine) {
        padding-left: 1px;
        padding-right: 1px;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure > img,
    figure > svg {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type,
    svg + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        overflow-wrap: break-word;
        word-break: break-word;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #fafafa;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.5in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* "page" element */
        body {
            font-size: 11pt !important;
            line-height: 1.35;
        }

        /* all headings */
        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin: 15px 0;
        }

        /* figures and tables */
        figure, table {
            font-size: 0.85em;
        }

        /* table cells */
        th,
        td {
            padding: 5px;
        }

        /* shrink font awesome icons */
        i.fas,
        i.fab,
        i.far,
        i.fal {
            transform: scale(0.85);
        }

        /* decrease banner margins */
        .banner {
            margin-top: 15px;
            margin-bottom: 15px;
            padding: 15px;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* tablenos */
    /* -------------------------------------------------- */

    /* tablenos wrapper */
    .tablenos {
        /* show scrollbar on tables if necessary to prevent overflow */
        width: 100%;
        margin: 20px 0;
    }

    .tablenos > table {
        /* move margins from table to table_wrapper to allow margin collapsing */
        margin: 0;
    }

    @media only screen {
        /* tablenos wrapper */
        .tablenos {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow-x: auto !important;
        }

        .tablenos th,
        .tablenos td {
            overflow-wrap: unset !important;
            word-break: unset !important;
        }

        /* table in wrapper */
        .tablenos table,
        .tablenos table * {
            /* don't break table words */
            overflow-wrap: normal !important;
        }
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number and anchor */
        margin-right: 60px !important;
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            max-width: 75%;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin-bottom: 5px;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            overflow-wrap: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
            max-height: 320px;
            overflow-y: auto;
        }

        /* tooltip copy of <img> */
        #tooltip_content > figure > img,
        #tooltip_content > figure > svg {
            max-height: 260px;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            background: #ffffff;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-right: solid 1px #bdbdbd;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when open */
        #toc_panel[data-open="true"] {
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            border-right: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_header {
            box-sizing: border-box;
            display: flex;
            flex-direction: row;
            align-items: center;
            height: 60px;
            margin: 0;
            padding: 20px;
        }

        /* toc panel header when hovered */
        #toc_header:hover {
            cursor: pointer;
        }

        /* toc panel header when panel open */
        #toc_panel[data-open="true"] > #toc_header {
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        img.lightbox_document_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* hypothesis activation button */
    #hypothesis_button {
        box-sizing: border-box;
        position: fixed;
        top: 0;
        right: 0;
        width: 60px;
        height: 60px;
        background: #ffffff;
        border-radius: 0;
        border-left: solid 1px #bdbdbd;
        border-bottom: solid 1px #bdbdbd;
        box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
        z-index: 2;
    }

    /* hypothesis button svg */
    #hypothesis_button > svg {
        position: relative;
        top: -4px;
    }

    /* hypothesis annotation count */
    #hypothesis_count {
        position: absolute;
        left: 0;
        right: 0;
        bottom: 5px;
    }

    /* side panel */
    .annotator-frame {
        width: 280px !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide button, toolbar, and tooltip on print */
    @media only print {
        #hypothesis_button {
            display: none;
        }

        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
<!-- anchors plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds an anchor next to each of a certain type
        // of element that provides a human-readable url to that specific
        // item/position in the document (eg "manuscript.html#abstract"). It
        // also makes it such that scrolling out of view of a target removes
        // its identifier from the url.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'anchors';

        // default plugin options
        const options = {
            // which types of elements to add anchors next to, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3, [id^="fig:"], [id^="tbl:"], [id^="eq:"]',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // add anchor to each element of specified types
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements)
                addAnchor(element);

            // attach scroll listener to window
            window.addEventListener('scroll', onScroll);
        }

        // when window is scrolled
        function onScroll() {
            // if url has hash and user has scrolled out of view of hash
            // target, remove hash from url
            const tolerance = 100;
            const target = getHashTarget();
            if (target) {
                if (
                    target.getBoundingClientRect().top >
                        window.innerHeight + tolerance ||
                    target.getBoundingClientRect().bottom < 0 - tolerance
                )
                    history.pushState(null, null, ' ');
            }
        }

        // add anchor to element
        function addAnchor(element) {
            let addTo; // element to add anchor button to

            // if figure or table, modify withId and addTo to get expected
            // elements
            if (element.id.indexOf('fig:') === 0) {
                addTo = element.querySelector('figcaption');
            } else if (element.id.indexOf('tbl:') === 0) {
                addTo = element.querySelector('caption');
            } else if (element.id.indexOf('eq:') === 0) {
                addTo = element.querySelector('.eqnos-number');
            }

            addTo = addTo || element;
            const id = element.id || null;

            // do not add anchor if element doesn't have assigned id.
            // id is generated by pandoc and is assumed to be unique and
            // human-readable
            if (!id)
                return;

            // create anchor button
            const anchor = document.createElement('a');
            anchor.innerHTML = document.querySelector('.icon_link').innerHTML;
            anchor.title = 'Link to this part of the document';
            anchor.classList.add('icon_button', 'anchor');
            anchor.dataset.ignore = 'true';
            anchor.href = '#' + id;
            addTo.appendChild(anchor);
        }

        // get element that is target of link or url hash
        function getHashTarget() {
            const hash = window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- link icon -->

<template class="icon_link">
    <!-- modified from: https://fontawesome.com/icons/link -->
    <svg width="16" height="16" viewBox="0 0 512 512">
        <path
            fill="currentColor"
            d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
        ></path>
    </svg>
</template>
<!-- accordion plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows sections of content under <h2> headings
        // to be collapsible.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'accordion';

        // default plugin options
        const options = {
            // whether to always start expanded ('false'), always start
            // collapsed ('true'), or start collapsed when screen small ('auto')
            startCollapsed: 'auto',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <h2> heading
            const headings = document.querySelectorAll('h2');
            for (const heading of headings) {
                addArrow(heading);

                // start expanded/collapsed based on option
                if (
                    options.startCollapsed === 'true' ||
                    (options.startCollapsed === 'auto' && isSmallScreen())
                )
                    collapseHeading(heading);
                else
                    expandHeading(heading);
            }

            // attach hash change listener to window
            window.addEventListener('hashchange', onHashChange);
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                goToElement(target);
        }

        // add arrow to heading
        function addArrow(heading) {
            // add arrow button
            const arrow = document.createElement('button');
            arrow.innerHTML = document.querySelector(
                '.icon_angle_down'
            ).innerHTML;
            arrow.classList.add('icon_button', 'accordion_arrow');
            heading.insertBefore(arrow, heading.firstChild);

            // attach click listener to heading and button
            heading.addEventListener('click', onHeadingClick);
            arrow.addEventListener('click', onArrowClick);
        }

        // determine if on mobile-like device with small screen
        function isSmallScreen() {
            return Math.min(window.innerWidth, window.innerHeight) < 480;
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get element that is target of hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // when <h2> heading is clicked
        function onHeadingClick(event) {
            // only collapse if <h2> itself is target of click (eg, user did
            // not click on anchor within <h2>)
            if (event.target === this)
                toggleCollapse(this);
        }

        // when arrow button is clicked
        function onArrowClick() {
            toggleCollapse(this.parentNode);
        }

        // collapse section if expanded, expand if collapsed
        function toggleCollapse(heading) {
            if (heading.dataset.collapsed === 'false')
                collapseHeading(heading);
            else
                expandHeading(heading);
        }

        // elements to exclude from collapse, such as table of contents panel,
        // hypothesis panel, etc
        const exclude = '#toc_panel, div.annotator-frame, #lightbox_overlay';

        // collapse section
        function collapseHeading(heading) {
            heading.setAttribute('data-collapsed', 'true');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'true');
        }

        // expand section
        function expandHeading(heading) {
            heading.setAttribute('data-collapsed', 'false');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'false');
        }

        // get list of elements between this <h2> and next <h2> or <h1>
        // ("children" of the <h2> section)
        function getChildren(heading) {
            return nextUntil(heading, 'h2, h1', exclude);
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get list of elements after a start element up to element matching
        // query
        function nextUntil(element, query, exclude) {
            const elements = [];
            while (element = element.nextElementSibling, element) {
                if (element.matches(query))
                    break;
                if (!element.matches(exclude))
                    elements.push(element);
            }
            return elements;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
    <!-- modified from: https://fontawesome.com/icons/angle-down -->
    <svg width="16" height="16" viewBox="0 0 448 512">
        <path
            fill="currentColor"
            d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
        ></path>
    </svg>
</template>
<!-- tooltips plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when the user hovers or
        // focuses a link to a citation or figure, a tooltip appears with a
        // preview of the reference content, along with arrows to navigate
        // between instances of the same reference in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tooltips';

        // default plugin options
        const options = {
            // whether user must click off to close tooltip instead of just
            // un-hovering
            clickClose: 'false',
            // delay (in ms) between opening and closing tooltip
            delay: '100',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach hover and focus listeners to link
                link.addEventListener('mouseover', onLinkHover);
                link.addEventListener('mouseleave', onLinkUnhover);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('touchend', onLinkTouch);
            }

            // attach mouse, key, and resize listeners to window
            window.addEventListener('mousedown', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('keyup', onKeyUp);
            window.addEventListener('resize', onResize);
        }

        // when link is hovered
        function onLinkHover() {
            // function to open tooltip
            const delayOpenTooltip = function() {
                openTooltip(this);
            }.bind(this);

            // run open function after delay
            this.openTooltipTimer = window.setTimeout(
                delayOpenTooltip,
                options.delay
            );
        }

        // when mouse leaves link
        function onLinkUnhover() {
            // cancel opening tooltip
            window.clearTimeout(this.openTooltipTimer);

            // don't close on unhover if option specifies
            if (options.clickClose === 'true')
                return;

            // function to close tooltip
            const delayCloseTooltip = function() {
                // if tooltip open and if mouse isn't over tooltip, close
                const tooltip = document.getElementById('tooltip');
                if (tooltip && !tooltip.matches(':hover'))
                    closeTooltip();
            };

            // run close function after delay
            this.closeTooltipTimer = window.setTimeout(
                delayCloseTooltip,
                options.delay
            );
        }

        // when link is focused (tabbed to)
        function onLinkFocus(event) {
            openTooltip(this);
        }

        // when link is touched on touch screen
        function onLinkTouch(event) {
            // attempt to force hover state on first tap always, and trigger
            // regular link click (and navigation) on second tap
            if (event.target === document.activeElement)
                event.target.click();
            else {
                document.activeElement.blur();
                event.target.focus();
            }
            if (event.cancelable)
                event.preventDefault();
            event.stopPropagation();
            return false;
        }

        // when mouse is clicked anywhere in window
        function onClick(event) {
            closeTooltip();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'tooltip_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'tooltip_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeTooltip();
                    break;
            }
        }

        // when window is resized or zoomed
        function onResize() {
            closeTooltip();
        }

        // get all links of types we wish to handle
        function getLinks() {
            const queries = [];
            // exclude buttons, anchor links, toc links, etc
            const exclude =
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            queries.push('a[href^="#ref-"]' + exclude); // citation links
            queries.push('a[href^="#fig:"]' + exclude); // figure links
            const query = queries.join(', ');
            return document.querySelectorAll(query);
        }

        // get links with same target, get index of link in set, get total
        // same links
        function getSameLinks(link) {
            const sameLinks = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    sameLinks.push(otherLink);
            }

            return {
                elements: sameLinks,
                index: sameLinks.indexOf(link),
                total: sameLinks.length
            };
        }

        // open tooltip
        function openTooltip(link) {
            // delete tooltip if it exists, start fresh
            closeTooltip();

            // make tooltip element
            const tooltip = makeTooltip(link);

            // if source couldn't be found and tooltip not made, exit
            if (!tooltip)
                return;

            // make navbar elements
            const navBar = makeNavBar(link);
            if (navBar)
                tooltip.firstElementChild.appendChild(navBar);

            // attach tooltip to page
            document.body.appendChild(tooltip);

            // position tooltip
            const position = function() {
                positionTooltip(link);
            };
            position();

            // if tooltip contains images, position again after they've loaded
            const imgs = tooltip.querySelectorAll('img');
            for (const img of imgs)
                img.addEventListener('load', position);
        }

        // close (delete) tooltip
        function closeTooltip() {
            const tooltip = document.getElementById('tooltip');
            if (tooltip)
                tooltip.remove();
        }

        // make tooltip
        function makeTooltip(link) {
            // get target element that link points to
            const source = getSource(link);

            // if source can't be found, exit
            if (!source)
                return;

            // create new tooltip
            const tooltip = document.createElement('div');
            tooltip.id = 'tooltip';
            const tooltipContent = document.createElement('div');
            tooltipContent.id = 'tooltip_content';
            tooltip.appendChild(tooltipContent);

            // make copy of source node and put in tooltip
            const sourceCopy = makeCopy(source);
            tooltipContent.appendChild(sourceCopy);

            // attach mouse event listeners
            tooltip.addEventListener('click', onTooltipClick);
            tooltip.addEventListener('mousedown', onTooltipClick);
            tooltip.addEventListener('touchstart', onTooltipClick);
            tooltip.addEventListener('mouseleave', onTooltipUnhover);

            // (for interaction with lightbox plugin)
            // transfer click on tooltip copied img to original img
            const sourceImg = source.querySelector('img');
            const sourceCopyImg = sourceCopy.querySelector('img');
            if (sourceImg && sourceCopyImg) {
                const clickImg = function() {
                    sourceImg.click();
                    closeTooltip();
                };
                sourceCopyImg.addEventListener('click', clickImg);
            }

            return tooltip;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // when tooltip is clicked
        function onTooltipClick(event) {
            // when user clicks on tooltip, stop click from transferring
            // outside of tooltip (eg, click off to close tooltip, or eg click
            // off to unhighlight same refs)
            event.stopPropagation();
        }

        // when tooltip is unhovered
        function onTooltipUnhover(event) {
            if (options.clickClose === 'true')
                return;

            // make sure new mouse/touch/focus no longer over tooltip or any
            // element within it
            const tooltip = document.getElementById('tooltip');
            if (!tooltip)
                return;
            if (this.contains(event.relatedTarget))
                return;

            closeTooltip();
        }

        // make nav bar to go betwen prev/next instances of same reference
        function makeNavBar(link) {
            // find other links to the same source
            const sameLinks = getSameLinks(link);

            // don't show nav bar when singular reference
            if (sameLinks.total <= 1)
                return;

            // find prev/next links with same target
            const prevLink = getPrevLink(link, sameLinks);
            const nextLink = getNextLink(link, sameLinks);

            // create nav bar
            const navBar = document.createElement('div');
            navBar.id = 'tooltip_nav_bar';
            const text = sameLinks.index + 1 + ' of ' + sameLinks.total;

            // create nav bar prev/next buttons
            const prevButton = document.createElement('button');
            const nextButton = document.createElement('button');
            prevButton.id = 'tooltip_prev_button';
            nextButton.id = 'tooltip_next_button';
            prevButton.title =
                'Jump to the previous occurence of this item in the document [←]';
            nextButton.title =
                'Jump to the next occurence of this item in the document [→]';
            prevButton.classList.add('icon_button');
            nextButton.classList.add('icon_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;
            navBar.appendChild(prevButton);
            navBar.appendChild(document.createTextNode(text));
            navBar.appendChild(nextButton);

            // attach click listeners to buttons
            prevButton.addEventListener('click', function() {
                onPrevNextClick(link, prevLink);
            });
            nextButton.addEventListener('click', function() {
                onPrevNextClick(link, nextLink);
            });

            return navBar;
        }

        // get previous link with same target
        function getPrevLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if < 1
            let index;
            if (sameLinks.index - 1 >= 0)
                index = sameLinks.index - 1;
            else
                index = sameLinks.total - 1;
            return sameLinks.elements[index];
        }

        // get next link with same target
        function getNextLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if > total
            let index;
            if (sameLinks.index + 1 <= sameLinks.total - 1)
                index = sameLinks.index + 1;
            else
                index = 0;
            return sameLinks.elements[index];
        }

        // get element that is target of link or url hash
        function getSource(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if ref or figure, modify target to get expected element
            if (id.indexOf('ref-') === 0)
                target = target.querySelector('p');
            else if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');

            return target;
        }

        // when prev/next arrow button is clicked
        function onPrevNextClick(link, prevNextLink) {
            if (link && prevNextLink)
                goToElement(prevNextLink, window.innerHeight * 0.5);
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // determine position to place tooltip based on link position in
        // viewport and tooltip size
        function positionTooltip(link, left, top) {
            const tooltipElement = document.getElementById('tooltip');
            if (!tooltipElement)
                return;

            // get convenient vars for position/dimensions of
            // link/tooltip/page/view
            link = getRectInPage(link);
            const tooltip = getRectInPage(tooltipElement);
            const view = getRectInPage();

            // horizontal positioning
            if (left)
                // use explicit value
                left = left;
            else if (link.left + tooltip.width < view.right)
                // fit tooltip to right of link
                left = link.left;
            else if (link.right - tooltip.width > view.left)
                // fit tooltip to left of link
                left = link.right - tooltip.width;
            // center tooltip in view
            else
                left = (view.right - view.left) / 2 - tooltip.width / 2;

            // vertical positioning
            if (top)
                // use explicit value
                top = top;
            else if (link.top - tooltip.height > view.top)
                // fit tooltip above link
                top = link.top - tooltip.height;
            else if (link.bottom + tooltip.height < view.bottom)
                // fit tooltip below link
                top = link.bottom;
            else {
                // center tooltip in view
                top = view.top + view.height / 2 - tooltip.height / 2;
                // nudge off of link to left/right if possible
                if (link.right + tooltip.width < view.right)
                    left = link.right;
                else if (link.left - tooltip.width > view.left)
                    left = link.left - tooltip.width;
            }

            tooltipElement.style.left = left + 'px';
            tooltipElement.style.top = top + 'px';
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get position of element relative to page
        function getRectInPage(element) {
            const rect = getRectInView(element);
            const body = getRectInView(document.body);

            const newRect = {};
            newRect.left = rect.left - body.left;
            newRect.top = rect.top - body.top;
            newRect.right = rect.right - body.left;
            newRect.bottom = rect.bottom - body.top;
            newRect.width = rect.width;
            newRect.height = rect.height;

            return newRect;
        }

        // (for interaction with accordion plugin)
        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // (for interaction with accordion plugin)
        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- jump to first plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds a button next to each reference entry,
        // figure, and table that jumps the page to the first occurrence of a
        // link to that item in the manuscript.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'jumpToFirst';

        // default plugin options
        const options = {
            // whether to add buttons next to reference entries
            references: 'true',
            // whether to add buttons next to figures
            figures: 'true',
            // whether to add buttons next to tables
            tables: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            if (options.references !== 'false')
                makeReferenceButtons();
            if (options.figures !== 'false')
                makeFigureButtons();
            if (options.tables !== 'false')
                makeTableButtons();
        }

        // when jump button clicked
        function onButtonClick() {
            const first = getFirstOccurrence(this.dataset.id);
            if (!first)
                return;

            // update url hash so navigating "back" in history will return
            // user to jump button
            window.location.hash = this.dataset.id;
            // scroll to link
            window.setTimeout(function() {
                goToElement(first, window.innerHeight * 0.5);
            }, 0);
        }

        // get first occurence of link to item in document
        function getFirstOccurrence(id) {
            let query = 'a';
            query += '[href="#' + id + '"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelector(query);
        }

        // add button next to each reference entry
        function makeReferenceButtons() {
            const references = document.querySelectorAll('div[id^="ref-"]');
            for (const reference of references) {
                // get reference id and element to add button to
                const id = reference.id;
                const container = reference.firstElementChild;
                const first = getFirstOccurrence(id);

                // if can't find link to reference, ignore
                if (!first)
                    continue;

                // make jump button
                let button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this reference in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.innerHTML = button.outerHTML + container.innerHTML;
                button = container.firstElementChild;
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeFigureButtons() {
            const figures = document.querySelectorAll('[id^="fig:"]');
            for (const figure of figures) {
                // get figure id and element to add button to
                const id = figure.id;
                const container = figure.querySelector('figcaption') || figure;
                const first = getFirstOccurrence(id);

                // if can't find link to figure, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this figure in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeTableButtons() {
            const tables = document.querySelectorAll('[id^="tbl:"]');
            for (const table of tables) {
                // get ref id and element to add button to
                const id = table.id;
                const container = table.querySelector('caption') || table;
                const first = getFirstOccurrence(id);

                // if can't find link to table, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this table in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
    <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
    <svg width="16" height="16" viewBox="0 0 320 512">
        <path
            fill="currentColor"
            d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
        ></path>
    </svg>
</template>
<!-- link highlight plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user hovers or
        // focuses a link, other links that have the same target will be
        // highlighted. It also makes it such that when clicking a link, the
        // target of the link (eg reference, figure, table) is briefly
        // highlighted.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'linkHighlight';

        // default plugin options
        const options = {
            // whether to also highlight links that go to external urls
            externalLinks: 'false',
            // whether user must click off to unhighlight instead of just
            // un-hovering
            clickUnhighlight: 'false',
            // whether to also highlight links that are unique
            highlightUnique: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach mouse and focus listeners to link
                link.addEventListener('mouseenter', onLinkFocus);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('mouseleave', onLinkUnhover);
            }

            // attach click and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('hashchange', onHashChange);

            // run hash change on window load in case user has navigated
            // directly to hash
            onHashChange();
        }

        // when link is focused (tabbed to) or hovered
        function onLinkFocus() {
            highlight(this);
        }

        // when link is unhovered
        function onLinkUnhover() {
            if (options.clickUnhighlight !== 'true')
                unhighlightAll();
        }

        // when the mouse is clicked anywhere in window
        function onClick(event) {
            unhighlightAll();
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                glowElement(target);
        }

        // get element that is target of link or url hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            return target;
        }

        // start glow sequence on an element
        function glowElement(element) {
            const startGlow = function() {
                onGlowEnd();
                element.dataset.glow = 'true';
                element.addEventListener('animationend', onGlowEnd);
            };
            const onGlowEnd = function() {
                element.removeAttribute('data-glow');
                element.removeEventListener('animationend', onGlowEnd);
            };
            startGlow();
        }

        // highlight link and all others with same target
        function highlight(link) {
            // force unhighlight all to start fresh
            unhighlightAll();

            // get links with same target
            if (!link)
                return;
            const sameLinks = getSameLinks(link);

            // if link unique and option is off, exit and don't highlight
            if (sameLinks.length <= 1 && options.highlightUnique !== 'true')
                return;

            // highlight all same links, and "select" (special highlight) this
            // one
            for (const sameLink of sameLinks) {
                if (sameLink === link)
                    sameLink.setAttribute('data-selected', 'true');
                else
                    sameLink.setAttribute('data-highlighted', 'true');
            }
        }

        // unhighlight all links
        function unhighlightAll() {
            const links = getLinks();
            for (const link of links) {
                link.setAttribute('data-selected', 'false');
                link.setAttribute('data-highlighted', 'false');
            }
        }

        // get links with same target
        function getSameLinks(link) {
            const results = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    results.push(otherLink);
            }
            return results;
        }

        // get all links of types we wish to handle
        function getLinks() {
            let query = 'a';
            if (options.externalLinks !== 'true')
                query += '[href^="#"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelectorAll(query);
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin provides a "table of contents" (toc) panel on
        // the side of the document that allows the user to conveniently
        // navigate between sections of the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableOfContents';

        // default plugin options
        const options = {
            // which types of elements to add links for, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3',
            // whether toc starts open. use 'true' or 'false', or 'auto' to
            // use 'true' behavior when screen wide enough and 'false' when not
            startOpen: 'false',
            // whether toc closes when clicking on toc link. use 'true' or
            // 'false', or 'auto' to use 'false' behavior when screen wide
            // enough and 'true' when not
            clickClose: 'auto',
            // if list item is more than this many characters, text will be
            // truncated
            charLimit: '50',
            // whether or not to show bullets next to each toc item
            bullets: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // make toc panel and populate with entries (links to document
            // sections)
            const panel = makePanel();
            if (!panel)
                return;
            makeEntries(panel);
            // attach panel to document after making entries, so 'toc' heading
            // in panel isn't included in toc
            document.body.insertBefore(panel, document.body.firstChild);

            // initial panel state
            if (
                options.startOpen === 'true' ||
                (options.startOpen === 'auto' && !isSmallScreen())
            )
                openPanel();
            else
                closePanel();

            // attach click, scroll, and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('scroll', onScroll);
            window.addEventListener('hashchange', onScroll);
            window.addEventListener('keyup', onKeyUp);
            onScroll();

            // add class to push document body down out of way of toc button
            document.body.classList.add('toc_body_nudge');
        }

        // determine if screen wide enough to fit toc panel
        function isSmallScreen() {
            // in default theme:
            // 816px = 8.5in = width of "page" (<body>) element
            // 260px = min width of toc panel (*2 for both sides of <body>)
            return window.innerWidth < 816 + 260 * 2;
        }

        // when mouse is clicked anywhere in window
        function onClick() {
            if (isSmallScreen())
                closePanel();
        }

        // when window is scrolled or hash changed
        function onScroll() {
            highlightViewed();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            // close on esc
            if (event.key === 'Escape')
                closePanel();
        }

        // find entry of currently viewed document section in toc and highlight
        function highlightViewed() {
            const firstId = getFirstInView(options.typesQuery);

            // get toc entries (links), unhighlight all, then highlight viewed
            const list = document.getElementById('toc_list');
            if (!firstId || !list)
                return;
            const links = list.querySelectorAll('a');
            for (const link of links)
                link.dataset.viewing = 'false';
            const link = list.querySelector('a[href="#' + firstId + '"]');
            if (!link)
                return;
            link.dataset.viewing = 'true';
        }

        // get first or previous toc listed element in top half of view
        function getFirstInView(query) {
            // get all elements matching query and with id
            const elements = document.querySelectorAll(query);
            const elementsWithIds = [];
            for (const element of elements) {
                if (element.id)
                    elementsWithIds.push(element);
            }


            // get first or previous element in top half of view
            for (let i = 0; i < elementsWithIds.length; i++) {
                const element = elementsWithIds[i];
                const prevElement = elementsWithIds[Math.max(0, i - 1)];
                if (element.getBoundingClientRect().top >= 0) {
                    if (
                        element.getBoundingClientRect().top <
                        window.innerHeight / 2
                    )
                        return element.id;
                    else
                        return prevElement.id;
                }
            }
        }

        // make panel
        function makePanel() {
            // create panel
            const panel = document.createElement('div');
            panel.id = 'toc_panel';
            if (options.bullets === 'true')
                panel.dataset.bullets = 'true';

            // create header
            const header = document.createElement('div');
            header.id = 'toc_header';

            // create toc button
            const button = document.createElement('button');
            button.id = 'toc_button';
            button.innerHTML = document.querySelector('.icon_th_list').innerHTML;
            button.title = 'Table of Contents';
            button.classList.add('icon_button');

            // create header text
            const text = document.createElement('h4');
            text.innerHTML = 'Table of Contents';

            // create container for toc list
            const list = document.createElement('div');
            list.id = 'toc_list';

            // attach click listeners
            panel.addEventListener('click', onPanelClick);
            header.addEventListener('click', onHeaderClick);
            button.addEventListener('click', onButtonClick);

            // attach elements
            header.appendChild(button);
            header.appendChild(text);
            panel.appendChild(header);
            panel.appendChild(list);

            return panel;
        }

        // create toc entries (links) to each element of the specified types
        function makeEntries(panel) {
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements) {
                // do not add link if element doesn't have assigned id
                if (!element.id)
                    continue;

                // create link/list item
                const link = document.createElement('a');
                link.classList.add('toc_link');
                switch (element.tagName.toLowerCase()) {
                    case 'h1':
                        link.dataset.level = '1';
                        break;
                    case 'h2':
                        link.dataset.level = '2';
                        break;
                    case 'h3':
                        link.dataset.level = '3';
                        break;
                    case 'h4':
                        link.dataset.level = '4';
                        break;
                }
                link.title = element.innerText;
                let text = element.innerText;
                if (text.length > options.charLimit)
                    text = text.slice(0, options.charLimit) + '...';
                link.innerHTML = text;
                link.href = '#' + element.id;
                link.addEventListener('click', onLinkClick);

                // attach link
                panel.querySelector('#toc_list').appendChild(link);
            }
        }

        // when panel is clicked
        function onPanelClick(event) {
            // stop click from propagating to window/document and closing panel
            event.stopPropagation();
        }

        // when header itself is clicked
        function onHeaderClick(event) {
            togglePanel();
        }

        // when button is clicked
        function onButtonClick(event) {
            togglePanel();
            // stop header underneath button from also being clicked
            event.stopPropagation();
        }

        // when link is clicked
        function onLinkClick(event) {
            if (
                options.clickClose === 'true' ||
                (options.clickClose === 'auto' && isSmallScreen())
            )
                closePanel();
            else
                openPanel();
        }

        // open panel if closed, close if opened
        function togglePanel() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                closePanel();
            else
                openPanel();
        }

        // open panel
        function openPanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'true';
        }

        // close panel
        function closePanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'false';
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- th list icon -->

<template class="icon_th_list">
    <!-- modified from: https://fontawesome.com/icons/th-list -->
    <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
        <path
            fill="currentColor"
            d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- lightbox plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user clicks on an
        // image, the image fills the screen and the user can pan/drag/zoom
        // the image and navigate between other images in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'lightbox';

        // default plugin options
        const options = {
            // list of possible zoom/scale factors
            zoomSteps:
                '0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1,' +
                '1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8',
            // whether to fit image to view ('fit'), display at 100% and shrink
            // if necessary ('shrink'), or always display at 100% ('100')
            defaultZoom: 'fit',
            // whether to zoom in/out toward center of view ('true') or mouse
            // ('false')
            centerZoom: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <img> element
            const imgs = document.querySelectorAll('figure > img');
            let count = 1;
            for (const img of imgs) {
                img.classList.add('lightbox_document_img');
                img.dataset.number = count;
                img.dataset.total = imgs.length;
                img.addEventListener('click', openLightbox);
                count++;
            }

            // attach mouse and key listeners to window
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('keyup', onKeyUp);
        }

        // when mouse is moved anywhere in window
        function onWindowMouseMove(event) {
            window.mouseX = event.clientX;
            window.mouseY = event.clientY;
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'lightbox_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'lightbox_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeLightbox();
                    break;
            }
        }

        // open lightbox
        function openLightbox() {
            const lightbox = makeLightbox(this);
            if (!lightbox)
                return;

            blurBody(lightbox);
            document.body.appendChild(lightbox);
        }

        // make lightbox
        function makeLightbox(img) {
            // delete lightbox if it exists, start fresh
            closeLightbox();

            // create screen overlay containing lightbox
            const overlay = document.createElement('div');
            overlay.id = 'lightbox_overlay';

            // create image info boxes
            const numberInfo = document.createElement('div');
            const zoomInfo = document.createElement('div');
            numberInfo.id = 'lightbox_number_info';
            zoomInfo.id = 'lightbox_zoom_info';

            // create container for image
            const imageContainer = document.createElement('div');
            imageContainer.id = 'lightbox_image_container';
            const lightboxImg = makeLightboxImg(
                img,
                imageContainer,
                numberInfo,
                zoomInfo
            );
            imageContainer.appendChild(lightboxImg);

            // create bottom container for caption and navigation buttons
            const bottomContainer = document.createElement('div');
            bottomContainer.id = 'lightbox_bottom_container';
            const caption = makeCaption(img);
            const prevButton = makePrevButton(img);
            const nextButton = makeNextButton(img);
            bottomContainer.appendChild(prevButton);
            bottomContainer.appendChild(caption);
            bottomContainer.appendChild(nextButton);

            // attach top middle and bottom to overlay
            overlay.appendChild(numberInfo);
            overlay.appendChild(zoomInfo);
            overlay.appendChild(imageContainer);
            overlay.appendChild(bottomContainer);

            return overlay;
        }

        // make <img> object that is intuitively draggable and zoomable
        function makeLightboxImg(
            sourceImg,
            container,
            numberInfoBox,
            zoomInfoBox
        ) {
            // create copy of source <img>
            const img = sourceImg.cloneNode(true);
            img.classList.remove('lightbox_document_img');
            img.removeAttribute('id');
            img.removeAttribute('width');
            img.removeAttribute('height');
            img.style.position = 'unset';
            img.style.margin = '0';
            img.style.padding = '0';
            img.style.width = '';
            img.style.height = '';
            img.style.minWidth = '';
            img.style.minHeight = '';
            img.style.maxWidth = '';
            img.style.maxHeight = '';
            img.id = 'lightbox_img';

            // build sorted list of unique zoomSteps, always including a 100%
            let zoomSteps = [];
            const optionsZooms = options.zoomSteps.split(/[^0-9.]/);
            for (const optionZoom of optionsZooms) {
                const newZoom = parseFloat(optionZoom);
                if (newZoom && !zoomSteps.includes(newZoom))
                    zoomSteps.push(newZoom);
            }
            if (!zoomSteps.includes(1))
                zoomSteps.push(1);
            zoomSteps = zoomSteps.sort(function sortNumber(a, b) {
                return a - b;
            });

            // <img> object property variables
            let zoom = 1;
            let translateX = 0;
            let translateY = 0;
            let clickMouseX = undefined;
            let clickMouseY = undefined;
            let clickTranslateX = undefined;
            let clickTranslateY = undefined;

            updateNumberInfo();

            // update image numbers displayed in info box
            function updateNumberInfo() {
                numberInfoBox.innerHTML =
                    sourceImg.dataset.number + ' of ' + sourceImg.dataset.total;
            }

            // update zoom displayed in info box
            function updateZoomInfo() {
                let zoomInfo = zoom * 100;
                if (!Number.isInteger(zoomInfo))
                    zoomInfo = zoomInfo.toFixed(2);
                zoomInfoBox.innerHTML = zoomInfo + '%';
            }

            // move to closest zoom step above current zoom
            const zoomIn = function() {
                for (const zoomStep of zoomSteps) {
                    if (zoomStep > zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                updateTransform();
            };

            // move to closest zoom step above current zoom
            const zoomOut = function() {
                zoomSteps.reverse();
                for (const zoomStep of zoomSteps) {
                    if (zoomStep < zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                zoomSteps.reverse();

                updateTransform();
            };

            // update display of <img> based on scale/translate properties
            const updateTransform = function() {
                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                // get new width/height after scale
                const rect = img.getBoundingClientRect();
                // limit translate
                translateX = Math.max(translateX, -rect.width / 2);
                translateX = Math.min(translateX, rect.width / 2);
                translateY = Math.max(translateY, -rect.height / 2);
                translateY = Math.min(translateY, rect.height / 2);

                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                updateZoomInfo();
            };

            // fit <img> to container
            const fit = function() {
                // no x/y offset, 100% zoom by default
                translateX = 0;
                translateY = 0;
                zoom = 1;

                // widths of <img> and container
                const imgWidth = img.naturalWidth;
                const imgHeight = img.naturalHeight;
                const containerWidth = parseFloat(
                    window.getComputedStyle(container).width
                );
                const containerHeight = parseFloat(
                    window.getComputedStyle(container).height
                );

                // how much zooming is needed to fit <img> to container
                const xRatio = imgWidth / containerWidth;
                const yRatio = imgHeight / containerHeight;
                const maxRatio = Math.max(xRatio, yRatio);
                const newZoom = 1 / maxRatio;

                // fit <img> to container according to option
                if (options.defaultZoom === 'shrink') {
                    if (maxRatio > 1)
                        zoom = newZoom;
                } else if (options.defaultZoom === 'fit')
                    zoom = newZoom;

                updateTransform();
            };

            // when mouse wheel is rolled anywhere in container
            const onContainerWheel = function(event) {
                if (!event)
                    return;

                // let ctrl + mouse wheel to zoom behave as normal
                if (event.ctrlKey)
                    return;

                // prevent normal scroll behavior
                event.preventDefault();
                event.stopPropagation();

                // point around which to scale img
                const viewRect = container.getBoundingClientRect();
                const viewX = (viewRect.left + viewRect.right) / 2;
                const viewY = (viewRect.top + viewRect.bottom) / 2;
                const originX = options.centerZoom === 'true' ? viewX : mouseX;
                const originY = options.centerZoom === 'true' ? viewY : mouseY;

                // get point on image under origin
                const oldRect = img.getBoundingClientRect();
                const oldPercentX = (originX - oldRect.left) / oldRect.width;
                const oldPercentY = (originY - oldRect.top) / oldRect.height;

                // increment/decrement zoom
                if (event.deltaY < 0)
                    zoomIn();
                if (event.deltaY > 0)
                    zoomOut();

                // get offset between previous image point and origin
                const newRect = img.getBoundingClientRect();
                const offsetX =
                    originX - (newRect.left + newRect.width * oldPercentX);
                const offsetY =
                    originY - (newRect.top + newRect.height * oldPercentY);

                // translate image to keep image point under origin
                translateX += offsetX;
                translateY += offsetY;

                // perform translate
                updateTransform();
            };

            // when container is clicked
            function onContainerClick(event) {
                // if container itself is target of click, and not other
                // element above it
                if (event.target === this)
                    closeLightbox();
            }

            // when mouse button is pressed on image
            const onImageMouseDown = function(event) {
                // store original mouse position relative to image
                clickMouseX = window.mouseX;
                clickMouseY = window.mouseY;
                clickTranslateX = translateX;
                clickTranslateY = translateY;
                event.stopPropagation();
                event.preventDefault();
            };

            // when mouse button is released anywhere in window
            const onWindowMouseUp = function(event) {
                // reset original mouse position
                clickMouseX = undefined;
                clickMouseY = undefined;
                clickTranslateX = undefined;
                clickTranslateY = undefined;

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mouseup', onWindowMouseUp);
            };

            // when mouse is moved anywhere in window
            const onWindowMouseMove = function(event) {
                if (
                    clickMouseX === undefined ||
                    clickMouseY === undefined ||
                    clickTranslateX === undefined ||
                    clickTranslateY === undefined
                )
                    return;

                // offset image based on original and current mouse position
                translateX = clickTranslateX + window.mouseX - clickMouseX;
                translateY = clickTranslateY + window.mouseY - clickMouseY;
                updateTransform();
                event.preventDefault();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mousemove', onWindowMouseMove);
            };

            // when window is resized
            const onWindowResize = function(event) {
                fit();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('resize', onWindowResize);
            };

            // attach the necessary event listeners
            img.addEventListener('dblclick', fit);
            img.addEventListener('mousedown', onImageMouseDown);
            container.addEventListener('wheel', onContainerWheel);
            container.addEventListener('mousedown', onContainerClick);
            container.addEventListener('touchstart', onContainerClick);
            window.addEventListener('mouseup', onWindowMouseUp);
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('resize', onWindowResize);

            // run fit() after lightbox atttached to document and <img> Loaded
            // so needed container and img dimensions available
            img.addEventListener('load', fit);

            return img;
        }

        // make caption
        function makeCaption(img) {
            const caption = document.createElement('div');
            caption.id = 'lightbox_caption';
            const captionSource = img.nextElementSibling;
            if (captionSource.tagName.toLowerCase() === 'figcaption') {
                const captionCopy = makeCopy(captionSource);
                caption.innerHTML = captionCopy.innerHTML;
            }

            caption.addEventListener('touchstart', function(event) {
                event.stopPropagation();
            });

            return caption;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // make button to jump to previous image in document
        function makePrevButton(img) {
            const prevButton = document.createElement('button');
            prevButton.id = 'lightbox_prev_button';
            prevButton.title = 'Jump to the previous image in the document [←]';
            prevButton.classList.add('icon_button', 'lightbox_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;

            // attach click listeners to button
            prevButton.addEventListener('click', function() {
                getPrevImg(img).click();
            });

            return prevButton;
        }

        // make button to jump to next image in document
        function makeNextButton(img) {
            const nextButton = document.createElement('button');
            nextButton.id = 'lightbox_next_button';
            nextButton.title = 'Jump to the next image in the document [→]';
            nextButton.classList.add('icon_button', 'lightbox_button');
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;

            // attach click listeners to button
            nextButton.addEventListener('click', function() {
                getNextImg(img).click();
            });

            return nextButton;
        }

        // get previous image in document
        function getPrevImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if < 1
            if (index - 1 >= 0)
                index--;
            else
                index = imgs.length - 1;
            return imgs[index];
        }

        // get next image in document
        function getNextImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if > total
            if (index + 1 <= imgs.length - 1)
                index++;
            else
                index = 0;
            return imgs[index];
        }

        // close lightbox
        function closeLightbox() {
            focusBody();

            const lightbox = document.getElementById('lightbox_overlay');
            if (lightbox)
                lightbox.remove();
        }

        // make all elements behind lightbox non-focusable
        function blurBody(overlay) {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.tabIndex = -1;
            document.body.classList.add('body_no_scroll');
        }

        // make all elements focusable again
        function focusBody() {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.removeAttribute('tabIndex');
            document.body.classList.remove('body_no_scroll');
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- attributes plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows arbitrary HTML attributes to be attached
        // to (almost) any element. Place an HTML comment inside or next to the
        // desired element in the format <!-- $attribute="value" -->

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'attributes';

        // default plugin options
        const options = {
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // get list of comments in document
            const comments = findComments();

            for(const comment of comments)
                if (comment.parentElement)
                    addAttributes(
                        comment.parentElement,
                        comment.nodeValue.trim()
                    );
        }

        // add html attributes to specified element based on string of 
        // html attributes and values
        function addAttributes(element, text) {
            // regex's for finding attribute/value pairs in the format of
            // attribute="value" or attribute='value
            const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
            const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

            // loop through attribute/value pairs
            let match;
            while(match = text.match(regex2) || text.match(regex1)) {
                // get attribute and value from regex capture groups
                let attribute = match[1];
                let value = match[2];

                // remove from string
                text = text.substring(match.index + match[0].length);

                if (!attribute || !value)
                    break;

                // set attribute of parent element
                try {
                    element.setAttribute(attribute, value);
                } catch(error) {
                    console.log(error);
                }

                // special case for colspan
                if (attribute === 'colspan')
                    removeTableCells(element, value);
            }
        }

        // get list of comment elements in document
        function findComments() {
            const comments = [];

            // iterate over comment nodes in document
            function acceptNode(node) {
                return NodeFilter.FILTER_ACCEPT;
            }
            const iterator = document.createNodeIterator(
                document.body,
                NodeFilter.SHOW_COMMENT,
                acceptNode
            );
            let node;
            while(node = iterator.nextNode())
                comments.push(node);

            return comments;
        }

        // remove certain number of cells after specified cell
        function removeTableCells(cell, number) {
            number = parseInt(number);
            if (!number)
                return;

            // remove elements
            for(; number > 1; number--) {
                if (cell.nextElementSibling)
                    cell.nextElementSibling.remove();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- mathjax plugin configuration -->

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        "CommonHTML": { linebreaks: { automatic: true } },
        "HTML-CSS": { linebreaks: { automatic: true } },
        "SVG": { linebreaks: { automatic: true } },
        "fast-preview": { disabled: true }
  });
</script>

<!-- mathjax plugin -->

<script
    src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
    integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
    crossorigin="anonymous"
>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'MathJax' allows the proper rendering of
    // math/equations written in LaTeX.

    // https://www.mathjax.org/
</script>
<!-- annotations plugin -->

<script>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'Hypothesis' allows public annotation of the
    // manuscript.

    // https://web.hypothes.is/

    // plugin configuration
    window.hypothesisConfig = function() {
        return {
            branding: {
                accentColor: '#2196f3',
                appBackgroundColor: '#f8f8f8',
                ctaBackgroundColor: '#f8f8f8',
                ctaTextColor: '#000000',
                selectionFontFamily: 'Open Sans, Helvetica, sans serif',
                annotationFontFamily: 'Open Sans, Helvetica, sans serif'
            }
        };
    };

    // hypothesis client script
    const embed = 'https://hypothes.is/embed.js';
    // hypothesis annotation count query url
    const query = 'https://api.hypothes.is/api/search?limit=0&url='

    
    // start script
    function start() {
        const button = makeButton();
        document.body.insertBefore(button, document.body.firstChild);
        insertCount(button);
    }

    // make button
    function makeButton() {
        // create button
        const button = document.createElement('button');
        button.id = 'hypothesis_button';
        button.innerHTML = document.querySelector('.icon_hypothesis').innerHTML;
        button.title = 'Hypothesis annotations';
        button.classList.add('icon_button');

        function onClick(event) {
            onButtonClick(event, button);
        }

        // attach click listeners
        button.addEventListener('click', onClick);

        return button;
    }

    // insert annotations count
    async function insertCount(button) {
        // get annotation count from Hypothesis based on url
        let count = '-';
        try {
            const canonical = document.querySelector('link[rel="canonical"]');
            const location = window.location;
            const url = encodeURIComponent((canonical || location).href);
            const response = await fetch(query + url);
            const json = await response.json();
            count = json.total || '-';
        } catch(error) {
            console.log(error);
        }
        
        // put count into button
        const counter = document.createElement('span');
        counter.id = 'hypothesis_count';
        counter.innerHTML = count;
        button.title = 'View ' + count + ' Hypothesis annotations';
        button.append(counter);
    }

    // when button is clicked
    function onButtonClick(event, button) {
        const script = document.createElement('script');
        script.src = embed;
        document.body.append(script);
        button.remove();
    }

    window.addEventListener('load', start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
    <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
    <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
        <path
            fill="currentColor"
            d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- analytics plugin -->

<!-- copy and paste code from Google Analytics or similar service here -->
</body>
</html>
